{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# 系统发育推断"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 准备工作,给每个基因文件的标识符信息添加对应的物种名信息"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/home/liuji/project/evolution-tutorials\n"
     ]
    }
   ],
   "source": [
    "! pwd\n",
    "#调用脚本，内有注释，完成标识信息的添加\n",
    "! bash ./src/modified_ID.sh data/ modified_data/ fna"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 1、选取合适的分子标记进行多重比对、修剪、构树，本教程以直系同源单拷贝基因为例"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/home/liuji/project/evolution-tutorials\n"
     ]
    }
   ],
   "source": [
    "#根据基因名合并物种序列,准备构建基因树\n",
    "! pwd\n",
    "! python3 src/gene_split.py ./modified_data ./result/gene_fna fna\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "nthread = 8\n",
      "nadd = 0\n",
      "inputfile = infile\n",
      "thresholdtorev = 0.000000\n",
      "reflim = 5000\n",
      "mode = a\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "\n",
      "Step 1/2\n",
      "1   \n",
      "\n",
      "Step 2/2\n",
      " 1 / 4 (8 threads)   \n",
      "makedirectionlist (nuc) Version 7.526\n",
      "alg=m, model=DNA200 (2), 1.53 (4.59), 0.37 (1.11), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "directionfile = _direction\n",
      "inputfile = infile\n",
      "subalignment = 0\n",
      "subalignmentoffset = 0\n",
      "4 x 915 - 897 d\n",
      "outputhat23=16\n",
      "treein = 0\n",
      "compacttree = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "All-to-all alignment.\n",
      "tbfast-pair (nuc) Version 7.526\n",
      "alg=L, model=DNA200 (2), 2.00 (6.00), -0.10 (-0.30), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "outputhat23=16\n",
      "Loading 'hat3.seed' ... \n",
      "done.\n",
      "Writing hat3 for iterative refinement\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "tbutree = 1, compacttree = 0\n",
      "Constructing a UPGMA tree ... \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment ... \n",
      "STEP     3 /3 (thread    1) \n",
      "done.\n",
      "tbfast (nuc) Version 7.526\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "minimumweight = 0.000010\n",
      "autosubalignment = 0.000000\n",
      "nthread = 8\n",
      "randomseed = 0\n",
      "blosum 62 / kimura 200\n",
      "poffset = 0\n",
      "niter = 16\n",
      "sueff_global = 0.100000\n",
      "nadd = 16\n",
      "Loading 'hat3' ... done.\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "\n",
      "    0 / 4\n",
      "Segment   1/  1    1- 916\n",
      "001-0004-1 (thread    8) identical     \n",
      "Converged.\n",
      "done\n",
      "dvtditr (nuc) Version 7.526\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " L-INS-i (Probably most accurate, very slow)\n",
      " Iterative refinement method (<16) with LOCAL pairwise alignment information\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 8\n",
      "nadd = 0\n",
      "inputfile = infile\n",
      "thresholdtorev = 0.000000\n",
      "reflim = 5000\n",
      "mode = a\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "\n",
      "Step 1/2\n",
      "1   \n",
      "\n",
      "Step 2/2\n",
      " 1 / 4 (8 threads)   \n",
      "makedirectionlist (nuc) Version 7.526\n",
      "alg=m, model=DNA200 (2), 1.53 (4.59), 0.37 (1.11), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "directionfile = _direction\n",
      "inputfile = infile\n",
      "subalignment = 0\n",
      "subalignmentoffset = 0\n",
      "4 x 4140 - 3561 d\n",
      "nthread = 8\n",
      "nthreadpair = 8\n",
      "nthreadtb = 8\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "    1 / 4 (thread    0)\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "STEP     3 / 3 (thread    1)\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "    0 / 4 (thread    0)\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "STEP     3 / 3 (thread    2)\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.526\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "distout=h\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "    0 / 4 (thread    0)dndpre (nuc) Version 7.526\n",
      "alg=X, model=DNA200 (2), 1.53 (4.59), 0.37 (1.11), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "minimumweight = 0.000010\n",
      "autosubalignment = 0.000000\n",
      "nthread = 8\n",
      "randomseed = 0\n",
      "blosum 62 / kimura 200\n",
      "poffset = 0\n",
      "niter = 2\n",
      "sueff_global = 0.100000\n",
      "nadd = 2\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "\n",
      "    0 / 4\n",
      "Segment   1/ 40    1- 636\n",
      "001-0004-1 (thread    2) identical     \n",
      "Converged.\n",
      "Segment   2/ 40  636- 695\n",
      "001-0004-0 (thread    2) identical     \n",
      "Converged.\n",
      "Segment   3/ 40  695- 773\n",
      "001-0004-0 (thread    3) identical     \n",
      "Converged.\n",
      "Segment   4/ 40  773- 896\n",
      "001-0004-1 (thread    6) identical     \n",
      "Converged.\n",
      "Segment   5/ 40  896- 956\n",
      "001-0003-1 (thread    4) identical     \n",
      "Converged.\n",
      "Segment   6/ 40  956- 995\n",
      "001-0004-1 (thread    4) identical     \n",
      "Converged.\n",
      "Segment   7/ 40  995-1115\n",
      "002-0004-1 (thread    5) worse         001-0003-1 (thread    8) identical     \n",
      "Converged.\n",
      "\n",
      "Reached 2\n",
      "Segment   8/ 40 1115-1261\n",
      "001-0004-1 (thread    2) identical     \n",
      "Converged.\n",
      "Segment   9/ 40 1261-1341\n",
      "001-0004-0 (thread    1) identical     \n",
      "Converged.\n",
      "Segment  10/ 40 1341-1492\n",
      "001-0004-0 (thread    4) identical     \n",
      "Converged.\n",
      "Segment  11/ 40 1492-1643\n",
      "001-0004-1 (thread    7) identical     \n",
      "Converged.\n",
      "Segment  12/ 40 1643-1728\n",
      "001-0004-1 (thread    4) identical     \n",
      "Converged.\n",
      "Segment  13/ 40 1728-1836\n",
      "001-0004-0 (thread    2) identical     001-0001-0 (thread    1) identical     \n",
      "Converged.\n",
      "Segment  14/ 40 1836-1942\n",
      "001-0004-1 (thread    1) identical     \n",
      "Converged.\n",
      "Segment  15/ 40 1942-1978\n",
      "001-0004-0 (thread    4) identical     \n",
      "Converged.\n",
      "Segment  16/ 40 1978-1992\n",
      "001-0002-1 (thread    8) identical     \n",
      "Converged.\n",
      "Segment  17/ 40 1992-2074\n",
      "001-0004-1 (thread    6) identical     \n",
      "Converged.\n",
      "Segment  18/ 40 2074-2225\n",
      "001-0004-1 (thread    5) identical     \n",
      "Converged.\n",
      "Segment  19/ 40 2225-2330\n",
      "001-0004-1 (thread    5) identical     \n",
      "Converged.\n",
      "Segment  20/ 40 2330-2404\n",
      "001-0004-0 (thread    4) identical     \n",
      "Converged.\n",
      "Segment  21/ 40 2404-2440\n",
      "001-0004-0 (thread    1) identical     \n",
      "Converged.\n",
      "Segment  22/ 40 2440-2446\n",
      "001-0004-0 (thread    1) identical     \n",
      "Converged.\n",
      "Segment  23/ 40 2446-2452\n",
      "001-0004-0 (thread    2) identical     \n",
      "Converged.\n",
      "Segment  24/ 40 2452-2482\n",
      "001-0004-1 (thread    5) identical     \n",
      "Converged.\n",
      "Segment  25/ 40 2482-2599\n",
      "001-0004-0 (thread    1) identical     \n",
      "Converged.\n",
      "Segment  26/ 40 2599-2742\n",
      "001-0004-1 (thread    5) identical     \n",
      "Converged.\n",
      "Segment  27/ 40 2742-2876\n",
      "001-0004-0 (thread    2) identical     \n",
      "Converged.\n",
      "Segment  28/ 40 2876-3030\n",
      "001-0003-1 (thread    8) identical     \n",
      "Converged.\n",
      "Segment  29/ 40 3030-3181\n",
      "001-0004-0 (thread    4) identical     \n",
      "Converged.\n",
      "Segment  30/ 40 3181-3332\n",
      "001-0004-1 (thread    3) identical     \n",
      "Converged.\n",
      "Segment  31/ 40 3332-3426\n",
      "001-0004-1 (thread    4) identical     \n",
      "Converged.\n",
      "Segment  32/ 40 3426-3450\n",
      "001-0004-0 (thread    1) identical     \n",
      "Converged.\n",
      "Segment  33/ 40 3450-3458\n",
      "001-0004-0 (thread    1) identical     \n",
      "Converged.\n",
      "Segment  34/ 40 3458-3537\n",
      "001-0004-1 (thread    2) identical     \n",
      "Converged.\n",
      "Segment  35/ 40 3537-3688\n",
      "001-0004-1 (thread    1) identical     \n",
      "Converged.\n",
      "Segment  36/ 40 3688-3839\n",
      "001-0004-1 (thread    2) identical     \n",
      "Converged.\n",
      "Segment  37/ 40 3839-3990\n",
      "001-0004-1 (thread    3) identical     \n",
      "Converged.\n",
      "Segment  38/ 40 3990-4141\n",
      "001-0004-0 (thread    1) identical     \n",
      "Converged.\n",
      "Segment  39/ 40 4141-4218\n",
      "001-0004-1 (thread    5) identical     \n",
      "Converged.\n",
      "Segment  40/ 40 4218-4229\n",
      "001-0004-1 (thread    6) identical     \n",
      "Converged.\n",
      "done\n",
      "dvtditr (nuc) Version 7.526\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-i (Standard)\n",
      " Iterative refinement method (max. 2 iterations)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 8\n",
      "nadd = 0\n",
      "inputfile = infile\n",
      "thresholdtorev = 0.000000\n",
      "reflim = 5000\n",
      "mode = a\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "\n",
      "Step 1/2\n",
      "1   \n",
      "\n",
      "Step 2/2\n",
      " 1 / 4 (8 threads)   \n",
      "makedirectionlist (nuc) Version 7.526\n",
      "alg=m, model=DNA200 (2), 1.53 (4.59), 0.37 (1.11), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "directionfile = _direction\n",
      "inputfile = infile\n",
      "subalignment = 0\n",
      "subalignmentoffset = 0\n",
      "4 x 864 - 771 d\n",
      "outputhat23=16\n",
      "treein = 0\n",
      "compacttree = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "All-to-all alignment.\n",
      "tbfast-pair (nuc) Version 7.526\n",
      "alg=L, model=DNA200 (2), 2.00 (6.00), -0.10 (-0.30), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "outputhat23=16\n",
      "Loading 'hat3.seed' ... \n",
      "done.\n",
      "Writing hat3 for iterative refinement\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "tbutree = 1, compacttree = 0\n",
      "Constructing a UPGMA tree ... \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment ... \n",
      "STEP     3 /3 (thread    2) \n",
      "done.\n",
      "tbfast (nuc) Version 7.526\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "minimumweight = 0.000010\n",
      "autosubalignment = 0.000000\n",
      "nthread = 8\n",
      "randomseed = 0\n",
      "blosum 62 / kimura 200\n",
      "poffset = 0\n",
      "niter = 16\n",
      "sueff_global = 0.100000\n",
      "nadd = 16\n",
      "Loading 'hat3' ... done.\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "\n",
      "    0 / 4\n",
      "Segment   1/  1    1- 901\n",
      "002-0004-1 (thread    3) identical     \n",
      "Converged.\n",
      "done\n",
      "dvtditr (nuc) Version 7.526\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " L-INS-i (Probably most accurate, very slow)\n",
      " Iterative refinement method (<16) with LOCAL pairwise alignment information\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 8\n",
      "nadd = 0\n",
      "inputfile = infile\n",
      "thresholdtorev = 0.000000\n",
      "reflim = 5000\n",
      "mode = a\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "\n",
      "Step 1/2\n",
      "1   \n",
      "\n",
      "Step 2/2\n",
      " 1 / 4 (8 threads)   \n",
      "makedirectionlist (nuc) Version 7.526\n",
      "alg=m, model=DNA200 (2), 1.53 (4.59), 0.37 (1.11), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "directionfile = _direction\n",
      "inputfile = infile\n",
      "subalignment = 0\n",
      "subalignmentoffset = 0\n",
      "4 x 1599 - 1230 d\n",
      "outputhat23=16\n",
      "treein = 0\n",
      "compacttree = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "All-to-all alignment.\n",
      "tbfast-pair (nuc) Version 7.526\n",
      "alg=L, model=DNA200 (2), 2.00 (6.00), -0.10 (-0.30), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "outputhat23=16\n",
      "Loading 'hat3.seed' ... \n",
      "done.\n",
      "Writing hat3 for iterative refinement\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "tbutree = 1, compacttree = 0\n",
      "Constructing a UPGMA tree ... \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment ... \n",
      "STEP     3 /3 (thread    1) \n",
      "done.\n",
      "tbfast (nuc) Version 7.526\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "minimumweight = 0.000010\n",
      "autosubalignment = 0.000000\n",
      "nthread = 8\n",
      "randomseed = 0\n",
      "blosum 62 / kimura 200\n",
      "poffset = 0\n",
      "niter = 16\n",
      "sueff_global = 0.100000\n",
      "nadd = 16\n",
      "Loading 'hat3' ... done.\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "\n",
      "    0 / 4\n",
      "Segment   1/  1    1-1651\n",
      "001-0004-1 (thread    2) identical     \n",
      "Converged.\n",
      "done\n",
      "dvtditr (nuc) Version 7.526\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " L-INS-i (Probably most accurate, very slow)\n",
      " Iterative refinement method (<16) with LOCAL pairwise alignment information\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 8\n",
      "nadd = 0\n",
      "inputfile = infile\n",
      "thresholdtorev = 0.000000\n",
      "reflim = 5000\n",
      "mode = a\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "\n",
      "Step 1/2\n",
      "1   \n",
      "\n",
      "Step 2/2\n",
      " 1 / 4 (8 threads)   \n",
      "makedirectionlist (nuc) Version 7.526\n",
      "alg=m, model=DNA200 (2), 1.53 (4.59), 0.37 (1.11), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "directionfile = _direction\n",
      "inputfile = infile\n",
      "subalignment = 0\n",
      "subalignmentoffset = 0\n",
      "4 x 984 - 978 d\n",
      "outputhat23=16\n",
      "treein = 0\n",
      "compacttree = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "All-to-all alignment.\n",
      "tbfast-pair (nuc) Version 7.526\n",
      "alg=L, model=DNA200 (2), 2.00 (6.00), -0.10 (-0.30), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "outputhat23=16\n",
      "Loading 'hat3.seed' ... \n",
      "done.\n",
      "Writing hat3 for iterative refinement\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "tbutree = 1, compacttree = 0\n",
      "Constructing a UPGMA tree ... \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment ... \n",
      "STEP     3 /3 (thread    2) \n",
      "done.\n",
      "tbfast (nuc) Version 7.526\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "minimumweight = 0.000010\n",
      "autosubalignment = 0.000000\n",
      "nthread = 8\n",
      "randomseed = 0\n",
      "blosum 62 / kimura 200\n",
      "poffset = 0\n",
      "niter = 16\n",
      "sueff_global = 0.100000\n",
      "nadd = 16\n",
      "Loading 'hat3' ... done.\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "\n",
      "    0 / 4\n",
      "Segment   1/  1    1- 994\n",
      "001-0004-1 (thread    7) identical     \n",
      "Converged.\n",
      "done\n",
      "dvtditr (nuc) Version 7.526\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " L-INS-i (Probably most accurate, very slow)\n",
      " Iterative refinement method (<16) with LOCAL pairwise alignment information\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 8\n",
      "nadd = 0\n",
      "inputfile = infile\n",
      "thresholdtorev = 0.000000\n",
      "reflim = 5000\n",
      "mode = a\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "\n",
      "Step 1/2\n",
      "1   \n",
      "\n",
      "Step 2/2\n",
      " 1 / 4 (8 threads)   \n",
      "makedirectionlist (nuc) Version 7.526\n",
      "alg=m, model=DNA200 (2), 1.53 (4.59), 0.37 (1.11), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "directionfile = _direction\n",
      "inputfile = infile\n",
      "subalignment = 0\n",
      "subalignmentoffset = 0\n",
      "4 x 1008 - 999 d\n",
      "outputhat23=16\n",
      "treein = 0\n",
      "compacttree = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "All-to-all alignment.\n",
      "tbfast-pair (nuc) Version 7.526\n",
      "alg=L, model=DNA200 (2), 2.00 (6.00), -0.10 (-0.30), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "outputhat23=16\n",
      "Loading 'hat3.seed' ... \n",
      "done.\n",
      "Writing hat3 for iterative refinement\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "tbutree = 1, compacttree = 0\n",
      "Constructing a UPGMA tree ... \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment ... \n",
      "STEP     3 /3 (thread    3) \n",
      "done.\n",
      "tbfast (nuc) Version 7.526\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "minimumweight = 0.000010\n",
      "autosubalignment = 0.000000\n",
      "nthread = 8\n",
      "randomseed = 0\n",
      "blosum 62 / kimura 200\n",
      "poffset = 0\n",
      "niter = 16\n",
      "sueff_global = 0.100000\n",
      "nadd = 16\n",
      "Loading 'hat3' ... done.\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "\n",
      "    0 / 4\n",
      "Segment   1/  1    1-1015\n",
      "002-0004-0 (thread    7) identical     002-0002-1 (thread    8) identical     \n",
      "Converged.\n",
      "done\n",
      "dvtditr (nuc) Version 7.526\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " L-INS-i (Probably most accurate, very slow)\n",
      " Iterative refinement method (<16) with LOCAL pairwise alignment information\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 8\n",
      "nadd = 0\n",
      "inputfile = infile\n",
      "thresholdtorev = 0.000000\n",
      "reflim = 5000\n",
      "mode = a\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "\n",
      "Step 1/2\n",
      "1   \n",
      "\n",
      "Step 2/2\n",
      " 1 / 4 (8 threads)   \n",
      "makedirectionlist (nuc) Version 7.526\n",
      "alg=m, model=DNA200 (2), 1.53 (4.59), 0.37 (1.11), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "directionfile = _direction\n",
      "inputfile = infile\n",
      "subalignment = 0\n",
      "subalignmentoffset = 0\n",
      "4 x 795 - 795 d\n",
      "outputhat23=16\n",
      "treein = 0\n",
      "compacttree = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "All-to-all alignment.\n",
      "tbfast-pair (nuc) Version 7.526\n",
      "alg=L, model=DNA200 (2), 2.00 (6.00), -0.10 (-0.30), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "outputhat23=16\n",
      "Loading 'hat3.seed' ... \n",
      "done.\n",
      "Writing hat3 for iterative refinement\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "tbutree = 1, compacttree = 0\n",
      "Constructing a UPGMA tree ... \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment ... \n",
      "STEP     3 /3 (thread    2) \n",
      "done.\n",
      "tbfast (nuc) Version 7.526\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "minimumweight = 0.000010\n",
      "autosubalignment = 0.000000\n",
      "nthread = 8\n",
      "randomseed = 0\n",
      "blosum 62 / kimura 200\n",
      "poffset = 0\n",
      "niter = 16\n",
      "sueff_global = 0.100000\n",
      "nadd = 16\n",
      "Loading 'hat3' ... done.\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "\n",
      "    0 / 4\n",
      "Segment   1/  1    1- 796\n",
      "001-0004-0 (thread    1) identical     \n",
      "Converged.\n",
      "done\n",
      "dvtditr (nuc) Version 7.526\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " L-INS-i (Probably most accurate, very slow)\n",
      " Iterative refinement method (<16) with LOCAL pairwise alignment information\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 8\n",
      "nadd = 0\n",
      "inputfile = infile\n",
      "thresholdtorev = 0.000000\n",
      "reflim = 5000\n",
      "mode = a\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "\n",
      "Step 1/2\n",
      "1   \n",
      "\n",
      "Step 2/2\n",
      " 1 / 4 (8 threads)   \n",
      "makedirectionlist (nuc) Version 7.526\n",
      "alg=m, model=DNA200 (2), 1.53 (4.59), 0.37 (1.11), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "directionfile = _direction\n",
      "inputfile = infile\n",
      "subalignment = 0\n",
      "subalignmentoffset = 0\n",
      "4 x 810 - 810 d\n",
      "outputhat23=16\n",
      "treein = 0\n",
      "compacttree = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "All-to-all alignment.\n",
      "tbfast-pair (nuc) Version 7.526\n",
      "alg=L, model=DNA200 (2), 2.00 (6.00), -0.10 (-0.30), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "outputhat23=16\n",
      "Loading 'hat3.seed' ... \n",
      "done.\n",
      "Writing hat3 for iterative refinement\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "tbutree = 1, compacttree = 0\n",
      "Constructing a UPGMA tree ... \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment ... \n",
      "STEP     3 /3 (thread    2) \n",
      "done.\n",
      "tbfast (nuc) Version 7.526\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "minimumweight = 0.000010\n",
      "autosubalignment = 0.000000\n",
      "nthread = 8\n",
      "randomseed = 0\n",
      "blosum 62 / kimura 200\n",
      "poffset = 0\n",
      "niter = 16\n",
      "sueff_global = 0.100000\n",
      "nadd = 16\n",
      "Loading 'hat3' ... done.\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "\n",
      "    0 / 4\n",
      "Segment   1/  1    1- 811\n",
      "001-0004-1 (thread    8) identical     \n",
      "Converged.\n",
      "done\n",
      "dvtditr (nuc) Version 7.526\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " L-INS-i (Probably most accurate, very slow)\n",
      " Iterative refinement method (<16) with LOCAL pairwise alignment information\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 8\n",
      "nadd = 0\n",
      "inputfile = infile\n",
      "thresholdtorev = 0.000000\n",
      "reflim = 5000\n",
      "mode = a\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "\n",
      "Step 1/2\n",
      "1   \n",
      "\n",
      "Step 2/2\n",
      " 1 / 4 (8 threads)   \n",
      "makedirectionlist (nuc) Version 7.526\n",
      "alg=m, model=DNA200 (2), 1.53 (4.59), 0.37 (1.11), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "directionfile = _direction\n",
      "inputfile = infile\n",
      "subalignment = 0\n",
      "subalignmentoffset = 0\n",
      "4 x 1209 - 1206 d\n",
      "outputhat23=16\n",
      "treein = 0\n",
      "compacttree = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "All-to-all alignment.\n",
      "tbfast-pair (nuc) Version 7.526\n",
      "alg=L, model=DNA200 (2), 2.00 (6.00), -0.10 (-0.30), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "outputhat23=16\n",
      "Loading 'hat3.seed' ... \n",
      "done.\n",
      "Writing hat3 for iterative refinement\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "tbutree = 1, compacttree = 0\n",
      "Constructing a UPGMA tree ... \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment ... \n",
      "STEP     3 /3 (thread    2) \n",
      "done.\n",
      "tbfast (nuc) Version 7.526\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "minimumweight = 0.000010\n",
      "autosubalignment = 0.000000\n",
      "nthread = 8\n",
      "randomseed = 0\n",
      "blosum 62 / kimura 200\n",
      "poffset = 0\n",
      "niter = 16\n",
      "sueff_global = 0.100000\n",
      "nadd = 16\n",
      "Loading 'hat3' ... done.\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "\n",
      "    0 / 4\n",
      "Segment   1/  1    1-1210\n",
      "002-0004-1 (thread    6) identical     002-0000-0 (thread    4) identical     \n",
      "Converged.\n",
      "done\n",
      "dvtditr (nuc) Version 7.526\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " L-INS-i (Probably most accurate, very slow)\n",
      " Iterative refinement method (<16) with LOCAL pairwise alignment information\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    }
   ],
   "source": [
    "# 使用mafft完成比对\n",
    "! mkdir -p result/phylo_fna_gene\n",
    "! ls result/gene_fna/ | while read file;do mafft --thread 8 --adjustdirection --auto result/gene_fna/${file} > result/phylo_fna_gene/${file%.fna}_aln.fna;done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "metadata": {},
   "outputs": [],
   "source": [
    "#使用trimal完成修剪，去除gap\n",
    "! ls result/phylo_fna_gene/*_aln.fna | while read file; do trimal -in ${file} -out ${file%_aln.fna}_AlnTrimed.fna -gappyout;done\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "IQ-TREE multicore version 2.3.4 COVID-edition for Linux x86 64-bit built Jun 18 2024\n",
      "Developed by Bui Quang Minh, James Barbetti, Nguyen Lam Tung, Olga Chernomor,\n",
      "Heiko Schmidt, Dominik Schrempf, Michael Woodhams, Ly Trong Nhan, Thomas Wong\n",
      "\n",
      "Host:    RCNSZOR36402A (AVX512, FMA3, 7 GB RAM)\n",
      "Command: iqtree -s result/phylo_fna_gene/100045at50557_AlnTrimed.fna -nt 8 -m MFP -bb 1000 -alrt 1000 -pre result/phylo_fna_gene/100045at50557_iqtreeout/.tree\n",
      "Seed:    88676 (Using SPRNG - Scalable Parallel Random Number Generator)\n",
      "Time:    Sat Aug 31 21:08:42 2024\n",
      "Kernel:  AVX+FMA - 8 threads (8 CPU cores detected)\n",
      "\n",
      "Reading alignment file result/phylo_fna_gene/100045at50557_AlnTrimed.fna ... Fasta format detected\n",
      "Reading fasta file: done in 0.00011664 secs using 98.59% CPU\n",
      "Alignment most likely contains DNA/RNA sequences\n",
      "Alignment has 4 sequences with 897 columns, 58 distinct patterns\n",
      "15 parsimony-informative, 286 singleton sites, 596 constant sites\n",
      "                        Gap/Ambiguity  Composition  p-value\n",
      "Analyzing sequences: done in 0.00638073 secs using 229.6% CPU\n",
      "   1  Culex_pipiens             0.00%    passed     77.58%\n",
      "   2  Armigeres_subalbatus      0.00%    failed      0.04%\n",
      "   3  Culex_pipiens_molestus    0.00%    passed     23.83%\n",
      "   4  Culex_pipiens_pallens     0.00%    passed     67.86%\n",
      "****  TOTAL                     0.00%  1 sequences failed composition chi2 test (p-value<5%; df=3)\n",
      "\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "\n",
      "Create initial parsimony tree by phylogenetic likelihood library (PLL)... 0.000 seconds\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "Perform fast likelihood tree search using GTR+I+G model...\n",
      "Estimate model parameters (epsilon = 5.000)\n",
      "Perform nearest neighbor interchange...\n",
      "Estimate model parameters (epsilon = 1.000)\n",
      "1. Initial log-likelihood: -2394.120\n",
      "Optimal log-likelihood: -2394.072\n",
      "Rate parameters:  A-C: 1.73069  A-G: 4.29870  A-T: 1.77889  C-G: 0.92976  C-T: 9.41889  G-T: 1.00000\n",
      "Base frequencies:  A: 0.305  C: 0.231  G: 0.320  T: 0.144\n",
      "Proportion of invariable sites: 0.332\n",
      "Gamma shape alpha: 1.392\n",
      "Parameters optimization took 1 rounds (0.008 sec)\n",
      "Time for fast ML tree search: 0.055 seconds\n",
      "\n",
      "NOTE: ModelFinder requires 0 MB RAM!\n",
      "ModelFinder will test up to 484 DNA models (sample size: 897) ...\n",
      " No. Model         -LnL         df  AIC          AICc         BIC\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  1  GTR+F         2402.092     13  4830.185     4830.597     4892.573\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  2  GTR+F+I       2392.957     14  4813.913     4814.390     4881.100\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  3  GTR+F+G4      2394.582     14  4817.164     4817.641     4884.351\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  4  GTR+F+I+G4    2394.057     15  4818.113     4818.658     4890.099\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  5  GTR+F+R2      2394.257     15  4818.514     4819.059     4890.500\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  6  GTR+F+R3      2393.908     17  4821.815     4822.512     4903.399\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 14  GTR+F+I+R2    2393.880     16  4819.761     4820.379     4896.545\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 15  GTR+F+I+R3    2393.757     18  4823.515     4824.294     4909.898\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 24  SYM+I         2435.487     11  4892.975     4893.273     4945.765\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 25  SYM+G4        2435.266     11  4892.533     4892.831     4945.323\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 26  SYM+I+G4      2435.258     12  4894.517     4894.870     4952.106\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 27  SYM+R2        2435.167     12  4894.335     4894.688     4951.923\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 46  TVM+F+I       2396.169     13  4818.338     4818.750     4880.725\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 47  TVM+F+G4      2399.159     13  4824.317     4824.729     4886.705\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 48  TVM+F+I+G4    2398.620     14  4825.241     4825.717     4892.428\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 49  TVM+F+R2      2397.025     14  4822.050     4822.526     4889.237\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 68  TVMe+I        2435.475     10  4890.951     4891.199     4938.942\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 69  TVMe+G4       2435.233     10  4890.467     4890.715     4938.457\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 70  TVMe+I+G4     2435.155     11  4892.309     4892.607     4945.099\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 71  TVMe+R2       2435.191     11  4892.382     4892.680     4945.171\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 90  TIM3+F+I      2395.167     12  4814.334     4814.687     4871.922\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 91  TIM3+F+G4     2395.656     12  4815.311     4815.664     4872.900\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 92  TIM3+F+I+G4   2395.640     13  4817.280     4817.692     4879.667\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 93  TIM3+F+R2     2395.909     13  4817.818     4818.230     4880.205\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "112  TIM3e+I       2435.960     9   4889.919     4890.122     4933.111\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "113  TIM3e+G4      2435.798     9   4889.596     4889.799     4932.788\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "114  TIM3e+I+G4    2435.934     10  4891.868     4892.116     4939.858\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "115  TIM3e+R2      2435.823     10  4891.647     4891.895     4939.637\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "134  TIM2+F+I      2393.226     12  4810.452     4810.805     4868.041\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "135  TIM2+F+G4     2394.576     12  4813.152     4813.505     4870.740\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "136  TIM2+F+I+G4   2394.065     13  4814.129     4814.541     4876.517\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "137  TIM2+F+R2     2394.077     13  4814.153     4814.566     4876.541\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "156  TIM2e+I       2437.263     9   4892.525     4892.728     4935.717\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "157  TIM2e+G4      2438.240     9   4894.480     4894.683     4937.671\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "158  TIM2e+I+G4    2438.114     10  4896.228     4896.476     4944.218\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "159  TIM2e+R2      2438.241     10  4896.483     4896.731     4944.473\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "178  TIM+F+I       2395.088     12  4814.176     4814.529     4871.765\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "179  TIM+F+G4      2395.767     12  4815.535     4815.888     4873.124\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "180  TIM+F+I+G4    2395.517     13  4817.034     4817.446     4879.422\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "181  TIM+F+R2      2395.661     13  4817.322     4817.734     4879.710\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "200  TIMe+I        2439.061     9   4896.122     4896.325     4939.314\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "201  TIMe+G4       2439.249     9   4896.497     4896.700     4939.689\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "202  TIMe+I+G4     2439.273     10  4898.546     4898.794     4946.536\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "203  TIMe+R2       2439.407     10  4898.813     4899.061     4946.804\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "222  TPM3u+F+I     2400.024     11  4822.049     4822.347     4874.838\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "223  TPM3u+F+G4    2400.657     11  4823.314     4823.612     4876.103\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "224  TPM3u+F+I+G4  2400.665     12  4825.329     4825.682     4882.918\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "225  TPM3u+F+R2    2400.651     12  4825.302     4825.654     4882.890\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "244  TPM3+I        2435.954     8   4887.908     4888.071     4926.301\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "245  TPM3+G4       2435.816     8   4887.631     4887.793     4926.024\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "246  TPM3+I+G4     2435.928     9   4889.855     4890.058     4933.047\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "247  TPM3+R2       2435.862     9   4889.724     4889.927     4932.915\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "266  TPM2u+F+I     2396.462     11  4814.924     4815.222     4867.713\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "267  TPM2u+F+G4    2399.107     11  4820.214     4820.512     4873.003\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "268  TPM2u+F+I+G4  2398.624     12  4821.248     4821.601     4878.837\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "269  TPM2u+F+R2    2397.207     12  4818.413     4818.766     4876.002\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "288  TPM2+I        2437.301     8   4890.601     4890.764     4928.994\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "289  TPM2+G4       2438.234     8   4892.468     4892.630     4930.861\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "290  TPM2+I+G4     2438.110     9   4894.220     4894.423     4937.411\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "291  TPM2+R2       2438.128     9   4894.257     4894.460     4937.448\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "310  K3Pu+F+I      2399.917     11  4821.834     4822.132     4874.623\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "311  K3Pu+F+G4     2400.880     11  4823.759     4824.058     4876.549\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "312  K3Pu+F+I+G4   2400.671     12  4825.341     4825.694     4882.930\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "313  K3Pu+F+R2     2400.538     12  4825.076     4825.429     4882.664\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "332  K3P+I         2439.085     8   4894.170     4894.333     4932.563\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "333  K3P+G4        2439.250     8   4894.500     4894.663     4932.893\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "334  K3P+I+G4      2439.242     9   4896.483     4896.686     4939.675\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "335  K3P+R2        2439.396     9   4896.793     4896.996     4939.984\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "354  TN+F+I        2395.190     11  4812.380     4812.678     4865.170\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "355  TN+F+G4       2395.808     11  4813.616     4813.914     4866.405\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "356  TN+F+I+G4     2395.584     12  4815.168     4815.521     4872.756\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "357  TN+F+R2       2395.713     12  4815.426     4815.779     4873.014\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "376  TNe+I         2439.125     8   4894.250     4894.413     4932.643\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "377  TNe+G4        2439.291     8   4894.582     4894.744     4932.974\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "378  TNe+I+G4      2439.310     9   4896.621     4896.824     4939.812\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "379  TNe+R2        2439.429     9   4896.858     4897.061     4940.050\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "398  HKY+F+I       2400.026     10  4820.051     4820.300     4868.042\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "399  HKY+F+G4      2400.952     10  4821.905     4822.153     4869.895\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "400  HKY+F+I+G4    2400.776     11  4823.552     4823.851     4876.342\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "401  HKY+F+R2      2400.610     11  4823.221     4823.519     4876.010\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "420  K2P+I         2439.143     7   4892.286     4892.412     4925.879\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "421  K2P+G4        2439.294     7   4892.588     4892.714     4926.181\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "422  K2P+I+G4      2439.317     8   4894.633     4894.795     4933.026\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "423  K2P+R2        2439.506     8   4895.013     4895.175     4933.405\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "442  F81+F+I       2434.457     9   4886.914     4887.117     4930.106\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "443  F81+F+G4      2434.218     9   4886.436     4886.639     4929.627\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "444  F81+F+I+G4    2434.388     10  4888.776     4889.024     4936.766\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "445  F81+F+R2      2434.571     10  4889.142     4889.390     4937.133\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "464  JC+I          2480.341     6   4972.682     4972.777     5001.477\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "465  JC+G4         2480.144     6   4972.288     4972.382     5001.082\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "466  JC+I+G4       2480.331     7   4974.663     4974.788     5008.256\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "467  JC+R2         2480.386     7   4974.771     4974.897     5008.365\n",
      "Akaike Information Criterion:           TIM2+F+I\n",
      "Corrected Akaike Information Criterion: TIM2+F+I\n",
      "Bayesian Information Criterion:         TN+F+I\n",
      "Best-fit model: TN+F+I chosen according to BIC\n",
      "\n",
      "All model information printed to result/phylo_fna_gene/100045at50557_iqtreeout/.tree.model.gz\n",
      "CPU time for ModelFinder: 18.701 seconds (0h:0m:18s)\n",
      "Wall-clock time for ModelFinder: 2.687 seconds (0h:0m:2s)\n",
      "Generating 1000 samples for ultrafast bootstrap (seed: 88676)...\n",
      "WARNING: 8 threads for alignment length 58 will slow down analysis\n",
      "\n",
      "NOTE: 0 MB RAM (0 GB) is required!\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "Estimate model parameters (epsilon = 0.100)\n",
      "1. Initial log-likelihood: -2395.190\n",
      "Optimal log-likelihood: -2395.177\n",
      "Rate parameters:  A-C: 1.00000  A-G: 3.17857  A-T: 1.00000  C-G: 1.00000  C-T: 7.53636  G-T: 1.00000\n",
      "Base frequencies:  A: 0.305  C: 0.231  G: 0.320  T: 0.144\n",
      "Proportion of invariable sites: 0.465\n",
      "Parameters optimization took 1 rounds (0.012 sec)\n",
      "Wrote distance file to... \n",
      "Computing ML distances based on estimated model parameters...\n",
      "Calculating distance matrix: done in 0.00102165 secs using 171.8% CPU\n",
      "Computing ML distances took 0.011971 sec (of wall-clock time) 0.135171 sec (of CPU time)\n",
      "Setting up auxiliary I and S matrices: done in 0.00139712 secs\n",
      "Computing RapidNJ tree took 0.002621 sec (of wall-clock time) 0.000010 sec (of CPU time)\n",
      "Log-likelihood of RapidNJ tree: -2395.177\n",
      "--------------------------------------------------------------------\n",
      "|             INITIALIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "Generating 99 parsimony trees... 0.011 second\n",
      "Computing log-likelihood of 2 initial trees ... 0.002 seconds\n",
      "Current best score: -2395.177\n",
      "\n",
      "Do NNI search on 3 best initial trees\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2395.177\n",
      "Finish initializing candidate tree set (3)\n",
      "Current best tree score: -2395.177 / CPU time: 0.243\n",
      "Number of iterations: 3\n",
      "--------------------------------------------------------------------\n",
      "|               OPTIMIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2395.177\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2395.177\n",
      "Iteration 10 / LogL: -2395.177 / Time: 0h:0m:4s\n",
      "Iteration 20 / LogL: -2395.177 / Time: 0h:0m:5s\n",
      "Iteration 30 / LogL: -2395.177 / Time: 0h:0m:5s (0h:0m:7s left)\n",
      "Iteration 40 / LogL: -2395.177 / Time: 0h:0m:7s (0h:0m:6s left)\n",
      "Iteration 50 / LogL: -2395.177 / Time: 0h:0m:7s (0h:0m:5s left)\n",
      "Log-likelihood cutoff on original alignment: -2397.201\n",
      "Iteration 60 / LogL: -2395.177 / Time: 0h:0m:8s (0h:0m:3s left)\n",
      "Iteration 70 / LogL: -2395.177 / Time: 0h:0m:8s (0h:0m:2s left)\n",
      "Iteration 80 / LogL: -2395.177 / Time: 0h:0m:9s (0h:0m:1s left)\n",
      "Iteration 90 / LogL: -2395.177 / Time: 0h:0m:10s (0h:0m:0s left)\n",
      "Iteration 100 / LogL: -2395.177 / Time: 0h:0m:10s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -2397.201\n",
      "NOTE: Bootstrap correlation coefficient of split occurrence frequencies: 0.943\n",
      "NOTE: UFBoot does not converge, continue at least 100 more iterations\n",
      "Iteration 110 / LogL: -2395.177 / Time: 0h:0m:11s (0h:0m:7s left)\n",
      "Iteration 120 / LogL: -2395.177 / Time: 0h:0m:12s (0h:0m:6s left)\n",
      "Iteration 130 / LogL: -2395.177 / Time: 0h:0m:13s (0h:0m:5s left)\n",
      "Iteration 140 / LogL: -2395.177 / Time: 0h:0m:15s (0h:0m:5s left)\n",
      "Iteration 150 / LogL: -2395.177 / Time: 0h:0m:15s (0h:0m:4s left)\n",
      "Log-likelihood cutoff on original alignment: -2397.201\n",
      "Iteration 160 / LogL: -2395.177 / Time: 0h:0m:16s (0h:0m:3s left)\n",
      "Iteration 170 / LogL: -2395.177 / Time: 0h:0m:16s (0h:0m:2s left)\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2395.177\n",
      "Iteration 180 / LogL: -2395.177 / Time: 0h:0m:17s (0h:0m:1s left)\n",
      "Iteration 190 / LogL: -2395.177 / Time: 0h:0m:17s (0h:0m:0s left)\n",
      "Iteration 200 / LogL: -2395.177 / Time: 0h:0m:18s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -2397.201\n",
      "NOTE: Bootstrap correlation coefficient of split occurrence frequencies: 0.993\n",
      "TREE SEARCH COMPLETED AFTER 200 ITERATIONS / Time: 0h:0m:18s\n",
      "\n",
      "--------------------------------------------------------------------\n",
      "|                    FINALIZING TREE SEARCH                        |\n",
      "--------------------------------------------------------------------\n",
      "Performs final model parameters optimization\n",
      "Estimate model parameters (epsilon = 0.010)\n",
      "1. Initial log-likelihood: -2395.177\n",
      "Optimal log-likelihood: -2395.171\n",
      "Rate parameters:  A-C: 1.00000  A-G: 3.15121  A-T: 1.00000  C-G: 1.00000  C-T: 7.45229  G-T: 1.00000\n",
      "Base frequencies:  A: 0.305  C: 0.231  G: 0.320  T: 0.144\n",
      "Proportion of invariable sites: 0.463\n",
      "Parameters optimization took 1 rounds (0.135 sec)\n",
      "BEST SCORE FOUND : -2395.171\n",
      "\n",
      "Testing tree branches by SH-like aLRT with 1000 replicates...\n",
      "0.154 sec.\n",
      "Creating bootstrap support values...\n",
      "Split supports printed to NEXUS file result/phylo_fna_gene/100045at50557_iqtreeout/.tree.splits.nex\n",
      "Total tree length: 0.745\n",
      "\n",
      "Total number of iterations: 200\n",
      "CPU time used for tree search: 114.131 sec (0h:1m:54s)\n",
      "Wall-clock time used for tree search: 15.505 sec (0h:0m:15s)\n",
      "Total CPU time used: 135.468 sec (0h:2m:15s)\n",
      "Total wall-clock time used: 18.569 sec (0h:0m:18s)\n",
      "\n",
      "Computing bootstrap consensus tree...\n",
      "Reading input file result/phylo_fna_gene/100045at50557_iqtreeout/.tree.splits.nex...\n",
      "4 taxa and 7 splits.\n",
      "Consensus tree written to result/phylo_fna_gene/100045at50557_iqtreeout/.tree.contree\n",
      "Reading input trees file result/phylo_fna_gene/100045at50557_iqtreeout/.tree.contree\n",
      "Log-likelihood of consensus tree: -2395.171\n",
      "\n",
      "Analysis results written to: \n",
      "  IQ-TREE report:                result/phylo_fna_gene/100045at50557_iqtreeout/.tree.iqtree\n",
      "  Maximum-likelihood tree:       result/phylo_fna_gene/100045at50557_iqtreeout/.tree.treefile\n",
      "  Likelihood distances:          result/phylo_fna_gene/100045at50557_iqtreeout/.tree.mldist\n",
      "\n",
      "Ultrafast bootstrap approximation results written to:\n",
      "  Split support values:          result/phylo_fna_gene/100045at50557_iqtreeout/.tree.splits.nex\n",
      "  Consensus tree:                result/phylo_fna_gene/100045at50557_iqtreeout/.tree.contree\n",
      "  Screen log file:               result/phylo_fna_gene/100045at50557_iqtreeout/.tree.log\n",
      "\n",
      "Date and Time: Sat Aug 31 21:09:01 2024\n",
      "IQ-TREE multicore version 2.3.4 COVID-edition for Linux x86 64-bit built Jun 18 2024\n",
      "Developed by Bui Quang Minh, James Barbetti, Nguyen Lam Tung, Olga Chernomor,\n",
      "Heiko Schmidt, Dominik Schrempf, Michael Woodhams, Ly Trong Nhan, Thomas Wong\n",
      "\n",
      "Host:    RCNSZOR36402A (AVX512, FMA3, 7 GB RAM)\n",
      "Command: iqtree -s result/phylo_fna_gene/10008at50557_AlnTrimed.fna -nt 8 -m MFP -bb 1000 -alrt 1000 -pre result/phylo_fna_gene/10008at50557_iqtreeout/.tree\n",
      "Seed:    61684 (Using SPRNG - Scalable Parallel Random Number Generator)\n",
      "Time:    Sat Aug 31 21:09:01 2024\n",
      "Kernel:  AVX+FMA - 8 threads (8 CPU cores detected)\n",
      "\n",
      "Reading alignment file result/phylo_fna_gene/10008at50557_AlnTrimed.fna ... Fasta format detected\n",
      "Reading fasta file: done in 0.00042712 secs using 99.5% CPU\n",
      "Alignment most likely contains DNA/RNA sequences\n",
      "Alignment has 4 sequences with 3564 columns, 110 distinct patterns\n",
      "46 parsimony-informative, 838 singleton sites, 2680 constant sites\n",
      "                        Gap/Ambiguity  Composition  p-value\n",
      "Analyzing sequences: done in 0.0066851 secs using 1059% CPU\n",
      "   1  Culex_pipiens             0.17%    passed     52.50%\n",
      "   2  Armigeres_subalbatus      0.67%    failed      0.39%\n",
      "   3  Culex_pipiens_molestus    0.00%    passed     70.65%\n",
      "   4  Culex_pipiens_pallens     0.08%    passed     78.69%\n",
      "****  TOTAL                     0.23%  1 sequences failed composition chi2 test (p-value<5%; df=3)\n",
      "\n",
      "\n",
      "Create initial parsimony tree by phylogenetic likelihood library (PLL)... 0.000 seconds\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "Perform fast likelihood tree search using GTR+I+G model...\n",
      "Estimate model parameters (epsilon = 5.000)\n",
      "Perform nearest neighbor interchange...\n",
      "Estimate model parameters (epsilon = 1.000)\n",
      "1. Initial log-likelihood: -8597.386\n",
      "Optimal log-likelihood: -8597.382\n",
      "Rate parameters:  A-C: 0.86488  A-G: 3.10684  A-T: 0.82290  C-G: 1.41080  C-T: 5.01866  G-T: 1.00000\n",
      "Base frequencies:  A: 0.235  C: 0.274  G: 0.299  T: 0.192\n",
      "Proportion of invariable sites: 0.375\n",
      "Gamma shape alpha: 1.250\n",
      "Parameters optimization took 1 rounds (0.319 sec)\n",
      "Time for fast ML tree search: 1.528 seconds\n",
      "\n",
      "NOTE: ModelFinder requires 0 MB RAM!\n",
      "ModelFinder will test up to 484 DNA models (sample size: 3564) ...\n",
      " No. Model         -LnL         df  AIC          AICc         BIC\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  1  GTR+F         8627.406     13  17280.812    17280.914    17361.134\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  2  GTR+F+I       8597.775     14  17223.550    17223.668    17310.051\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  3  GTR+F+G4      8597.504     14  17223.008    17223.126    17309.509\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  4  GTR+F+I+G4    8597.382     15  17224.764    17224.900    17317.444\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  5  GTR+F+R2      8598.030     15  17226.059    17226.195    17318.739\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  6  GTR+F+R3      8597.586     17  17229.172    17229.345    17334.209\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 14  GTR+F+I+R2    8597.515     16  17227.029    17227.183    17325.888\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 15  GTR+F+I+R3    8597.417     18  17230.834    17231.027    17342.049\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 24  SYM+I         8647.628     11  17317.256    17317.330    17385.221\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 25  SYM+G4        8646.915     11  17315.831    17315.905    17383.796\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 26  SYM+I+G4      8646.996     12  17317.993    17318.081    17392.137\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 27  SYM+R2        8647.749     12  17319.499    17319.587    17393.642\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 46  TVM+F+I       8604.222     13  17234.445    17234.547    17314.767\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 47  TVM+F+G4      8603.896     13  17233.791    17233.894    17314.113\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 48  TVM+F+I+G4    8603.866     14  17235.733    17235.851    17322.234\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 49  TVM+F+R2      8604.499     14  17236.998    17237.117    17323.499\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 68  TVMe+I        8649.790     10  17319.581    17319.643    17381.367\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 69  TVMe+G4       8649.014     10  17318.028    17318.090    17379.815\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 70  TVMe+I+G4     8649.128     11  17320.255    17320.329    17388.220\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 71  TVMe+R2       8649.837     11  17321.674    17321.749    17389.639\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 90  TIM3+F+I      8600.466     12  17224.933    17225.021    17299.077\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 91  TIM3+F+G4     8600.325     12  17224.650    17224.738    17298.794\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 92  TIM3+F+I+G4   8600.299     13  17226.598    17226.700    17306.920\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 93  TIM3+F+R2     8600.616     13  17227.233    17227.335    17307.555\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "112  TIM3e+I       8653.801     9   17325.601    17325.652    17381.209\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "113  TIM3e+G4      8653.320     9   17324.640    17324.691    17380.248\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "114  TIM3e+I+G4    8653.371     10  17326.741    17326.803    17388.528\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "115  TIM3e+R2      8653.886     10  17327.772    17327.834    17389.559\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "134  TIM2+F+I      8598.556     12  17221.113    17221.201    17295.257\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "135  TIM2+F+G4     8598.505     12  17221.010    17221.098    17295.153\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "136  TIM2+F+I+G4   8598.505     13  17223.010    17223.113    17303.333\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "137  TIM2+F+R2     8598.840     13  17223.679    17223.782    17304.002\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "156  TIM2e+I       8653.258     9   17324.516    17324.566    17380.123\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "157  TIM2e+G4      8653.006     9   17324.013    17324.064    17379.621\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "158  TIM2e+I+G4    8653.000     10  17326.000    17326.062    17387.786\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "159  TIM2e+R2      8653.549     10  17327.097    17327.159    17388.884\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "178  TIM+F+I       8600.271     12  17224.543    17224.630    17298.686\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "179  TIM+F+G4      8600.190     12  17224.380    17224.468    17298.524\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "180  TIM+F+I+G4    8600.219     13  17226.437    17226.540    17306.759\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "181  TIM+F+R2      8600.564     13  17227.127    17227.230    17307.449\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "200  TIMe+I        8657.663     9   17333.326    17333.377    17388.934\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "201  TIMe+G4       8657.417     9   17332.833    17332.884    17388.441\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "202  TIMe+I+G4     8657.388     10  17334.777    17334.839    17396.563\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "203  TIMe+R2       8657.887     10  17335.774    17335.836    17397.560\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "222  TPM3u+F+I     8606.412     11  17234.824    17234.899    17302.789\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "223  TPM3u+F+G4    8606.383     11  17234.766    17234.840    17302.731\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "224  TPM3u+F+I+G4  8606.397     12  17236.794    17236.882    17310.938\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "225  TPM3u+F+R2    8606.695     12  17237.390    17237.478    17311.534\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "244  TPM3+I        8655.743     8   17327.487    17327.527    17376.916\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "245  TPM3+G4       8655.324     8   17326.649    17326.689    17376.078\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "246  TPM3+I+G4     8655.295     9   17328.589    17328.640    17384.197\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "247  TPM3+R2       8655.955     9   17329.909    17329.960    17385.517\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "266  TPM2u+F+I     8605.476     11  17232.952    17233.027    17300.917\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "267  TPM2u+F+G4    8605.291     11  17232.582    17232.656    17300.547\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "268  TPM2u+F+I+G4  8605.313     12  17234.627    17234.715    17308.771\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "269  TPM2u+F+R2    8605.712     12  17235.424    17235.512    17309.568\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "288  TPM2+I        8655.892     8   17327.784    17327.825    17377.213\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "289  TPM2+G4       8655.455     8   17326.910    17326.950    17376.339\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "290  TPM2+I+G4     8655.423     9   17328.846    17328.897    17384.454\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "291  TPM2+R2       8656.024     9   17330.048    17330.098    17385.655\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "310  K3Pu+F+I      8606.724     11  17235.447    17235.521    17303.412\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "311  K3Pu+F+G4     8606.655     11  17235.311    17235.385    17303.276\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "312  K3Pu+F+I+G4   8606.696     12  17237.392    17237.480    17311.536\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "313  K3Pu+F+R2     8607.074     12  17238.148    17238.236    17312.292\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "332  K3P+I         8660.020     8   17336.039    17336.080    17385.468\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "333  K3P+G4        8659.599     8   17335.199    17335.239    17384.628\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "334  K3P+I+G4      8659.591     9   17337.182    17337.233    17392.790\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "335  K3P+R2        8660.230     9   17338.460    17338.511    17394.068\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "354  TN+F+I        8600.922     11  17223.845    17223.919    17291.810\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "355  TN+F+G4       8600.955     11  17223.910    17223.985    17291.875\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "356  TN+F+I+G4     8601.027     12  17226.055    17226.143    17300.199\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "357  TN+F+R2       8601.141     12  17226.282    17226.370    17300.426\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "376  TNe+I         8658.559     8   17333.119    17333.159    17382.548\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "377  TNe+G4        8658.479     8   17332.957    17332.998    17382.386\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "378  TNe+I+G4      8658.487     9   17334.973    17335.024    17390.581\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "379  TNe+R2        8658.892     9   17335.783    17335.834    17391.391\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "398  HKY+F+I       8607.305     10  17234.611    17234.673    17296.397\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "399  HKY+F+G4      8607.395     10  17234.790    17234.852    17296.577\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "400  HKY+F+I+G4    8607.478     11  17236.956    17237.030    17304.921\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "401  HKY+F+R2      8607.596     11  17237.192    17237.267    17305.157\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "420  K2P+I         8660.950     7   17335.900    17335.931    17379.150\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "421  K2P+G4        8660.712     7   17335.424    17335.455    17378.674\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "422  K2P+I+G4      8660.732     8   17337.463    17337.504    17386.892\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "423  K2P+R2        8661.198     8   17338.395    17338.436    17387.824\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "442  F81+F+I       8710.567     9   17439.133    17439.184    17494.741\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "443  F81+F+G4      8710.712     9   17439.425    17439.475    17495.032\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "444  F81+F+I+G4    8710.718     10  17441.436    17441.497    17503.222\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "445  F81+F+R2      8711.024     10  17442.048    17442.109    17503.834\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "464  JC+I          8759.911     6   17531.823    17531.846    17568.895\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "465  JC+G4         8760.042     6   17532.083    17532.107    17569.155\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "466  JC+I+G4       8760.032     7   17534.064    17534.095    17577.314\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "467  JC+R2         8760.304     7   17534.609    17534.640    17577.859\n",
      "Akaike Information Criterion:           TIM2+F+G4\n",
      "Corrected Akaike Information Criterion: TIM2+F+G4\n",
      "Bayesian Information Criterion:         TN+F+I\n",
      "Best-fit model: TN+F+I chosen according to BIC\n",
      "\n",
      "All model information printed to result/phylo_fna_gene/10008at50557_iqtreeout/.tree.model.gz\n",
      "CPU time for ModelFinder: 449.258 seconds (0h:7m:29s)\n",
      "Wall-clock time for ModelFinder: 57.332 seconds (0h:0m:57s)\n",
      "Generating 1000 samples for ultrafast bootstrap (seed: 61684)...\n",
      "\n",
      "NOTE: 0 MB RAM (0 GB) is required!\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "Estimate model parameters (epsilon = 0.100)\n",
      "1. Initial log-likelihood: -8600.922\n",
      "Optimal log-likelihood: -8600.908\n",
      "Rate parameters:  A-C: 1.00000  A-G: 3.00347  A-T: 1.00000  C-G: 1.00000  C-T: 5.12428  G-T: 1.00000\n",
      "Base frequencies:  A: 0.235  C: 0.274  G: 0.299  T: 0.192\n",
      "Proportion of invariable sites: 0.565\n",
      "Parameters optimization took 1 rounds (0.590 sec)\n",
      "Wrote distance file to... \n",
      "Computing ML distances based on estimated model parameters...\n",
      "Calculating distance matrix: done in 0.0149806 secs using 773.7% CPU\n",
      "Computing ML distances took 0.069180 sec (of wall-clock time) 0.517292 sec (of CPU time)\n",
      "Setting up auxiliary I and S matrices: done in 0.00221717 secs using 185.5% CPU\n",
      "Computing RapidNJ tree took 0.004937 sec (of wall-clock time) 0.008069 sec (of CPU time)\n",
      "Log-likelihood of RapidNJ tree: -8600.909\n",
      "--------------------------------------------------------------------\n",
      "|             INITIALIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "Generating 99 parsimony trees... 0.030 second\n",
      "Computing log-likelihood of 2 initial trees ... 0.241 seconds\n",
      "Current best score: -8600.908\n",
      "\n",
      "Do NNI search on 3 best initial trees\n",
      "Finish initializing candidate tree set (3)\n",
      "Current best tree score: -8600.908 / CPU time: 0.388\n",
      "Number of iterations: 3\n",
      "--------------------------------------------------------------------\n",
      "|               OPTIMIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "UPDATE BEST LOG-LIKELIHOOD: -8600.908\n",
      "Iteration 10 / LogL: -8601.184 / Time: 0h:1m:5s\n",
      "UPDATE BEST LOG-LIKELIHOOD: -8600.908\n",
      "Iteration 20 / LogL: -8600.908 / Time: 0h:1m:6s\n",
      "Iteration 30 / LogL: -8601.177 / Time: 0h:1m:8s (0h:0m:25s left)\n",
      "Iteration 40 / LogL: -8601.182 / Time: 0h:1m:11s (0h:0m:21s left)\n",
      "Iteration 50 / LogL: -8601.173 / Time: 0h:1m:12s (0h:0m:15s left)\n",
      "Log-likelihood cutoff on original alignment: -8602.913\n",
      "Iteration 60 / LogL: -8601.173 / Time: 0h:1m:14s (0h:0m:11s left)\n",
      "Iteration 70 / LogL: -8601.184 / Time: 0h:1m:17s (0h:0m:8s left)\n",
      "Iteration 80 / LogL: -8601.184 / Time: 0h:1m:19s (0h:0m:5s left)\n",
      "Iteration 90 / LogL: -8601.173 / Time: 0h:1m:20s (0h:0m:2s left)\n",
      "Iteration 100 / LogL: -8601.173 / Time: 0h:1m:21s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -8602.913\n",
      "NOTE: Bootstrap correlation coefficient of split occurrence frequencies: 0.996\n",
      "TREE SEARCH COMPLETED AFTER 101 ITERATIONS / Time: 0h:1m:22s\n",
      "\n",
      "--------------------------------------------------------------------\n",
      "|                    FINALIZING TREE SEARCH                        |\n",
      "--------------------------------------------------------------------\n",
      "Performs final model parameters optimization\n",
      "Estimate model parameters (epsilon = 0.010)\n",
      "1. Initial log-likelihood: -8600.908\n",
      "Optimal log-likelihood: -8600.902\n",
      "Rate parameters:  A-C: 1.00000  A-G: 3.02612  A-T: 1.00000  C-G: 1.00000  C-T: 5.13811  G-T: 1.00000\n",
      "Base frequencies:  A: 0.235  C: 0.274  G: 0.299  T: 0.192\n",
      "Proportion of invariable sites: 0.566\n",
      "Parameters optimization took 1 rounds (0.005 sec)\n",
      "BEST SCORE FOUND : -8600.902\n",
      "\n",
      "Testing tree branches by SH-like aLRT with 1000 replicates...\n",
      "0.226 sec.\n",
      "Creating bootstrap support values...\n",
      "Split supports printed to NEXUS file result/phylo_fna_gene/10008at50557_iqtreeout/.tree.splits.nex\n",
      "Total tree length: 0.485\n",
      "\n",
      "Total number of iterations: 101\n",
      "CPU time used for tree search: 187.642 sec (0h:3m:7s)\n",
      "Wall-clock time used for tree search: 23.886 sec (0h:0m:23s)\n",
      "Total CPU time used: 644.785 sec (0h:10m:44s)\n",
      "Total wall-clock time used: 82.285 sec (0h:1m:22s)\n",
      "\n",
      "Computing bootstrap consensus tree...\n",
      "Reading input file result/phylo_fna_gene/10008at50557_iqtreeout/.tree.splits.nex...\n",
      "4 taxa and 7 splits.\n",
      "Consensus tree written to result/phylo_fna_gene/10008at50557_iqtreeout/.tree.contree\n",
      "Reading input trees file result/phylo_fna_gene/10008at50557_iqtreeout/.tree.contree\n",
      "Log-likelihood of consensus tree: -8600.903\n",
      "\n",
      "Analysis results written to: \n",
      "  IQ-TREE report:                result/phylo_fna_gene/10008at50557_iqtreeout/.tree.iqtree\n",
      "  Maximum-likelihood tree:       result/phylo_fna_gene/10008at50557_iqtreeout/.tree.treefile\n",
      "  Likelihood distances:          result/phylo_fna_gene/10008at50557_iqtreeout/.tree.mldist\n",
      "\n",
      "Ultrafast bootstrap approximation results written to:\n",
      "  Split support values:          result/phylo_fna_gene/10008at50557_iqtreeout/.tree.splits.nex\n",
      "  Consensus tree:                result/phylo_fna_gene/10008at50557_iqtreeout/.tree.contree\n",
      "  Screen log file:               result/phylo_fna_gene/10008at50557_iqtreeout/.tree.log\n",
      "\n",
      "Date and Time: Sat Aug 31 21:10:23 2024\n",
      "IQ-TREE multicore version 2.3.4 COVID-edition for Linux x86 64-bit built Jun 18 2024\n",
      "Developed by Bui Quang Minh, James Barbetti, Nguyen Lam Tung, Olga Chernomor,\n",
      "Heiko Schmidt, Dominik Schrempf, Michael Woodhams, Ly Trong Nhan, Thomas Wong\n",
      "\n",
      "Host:    RCNSZOR36402A (AVX512, FMA3, 7 GB RAM)\n",
      "Command: iqtree -s result/phylo_fna_gene/100138at50557_AlnTrimed.fna -nt 8 -m MFP -bb 1000 -alrt 1000 -pre result/phylo_fna_gene/100138at50557_iqtreeout/.tree\n",
      "Seed:    841031 (Using SPRNG - Scalable Parallel Random Number Generator)\n",
      "Time:    Sat Aug 31 21:10:23 2024\n",
      "Kernel:  AVX+FMA - 8 threads (8 CPU cores detected)\n",
      "\n",
      "Reading alignment file result/phylo_fna_gene/100138at50557_AlnTrimed.fna ... Fasta format detected\n",
      "Reading fasta file: done in 0.000173692 secs using 99.03% CPU\n",
      "Alignment most likely contains DNA/RNA sequences\n",
      "Alignment has 4 sequences with 801 columns, 41 distinct patterns\n",
      "8 parsimony-informative, 235 singleton sites, 558 constant sites\n",
      "                        Gap/Ambiguity  Composition  p-value\n",
      "Analyzing sequences: done in 0.00514076 secs using 227.2% CPU\n",
      "   1  Culex_pipiens             0.00%    passed     74.82%\n",
      "   2  Armigeres_subalbatus      5.24%    failed      0.46%\n",
      "   3  Culex_pipiens_molestus    0.00%    passed     75.75%\n",
      "   4  Culex_pipiens_pallens     0.00%    passed     62.61%\n",
      "****  TOTAL                     1.31%  1 sequences failed composition chi2 test (p-value<5%; df=3)\n",
      "\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "\n",
      "Create initial parsimony tree by phylogenetic likelihood library (PLL)... 0.000 seconds\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "Perform fast likelihood tree search using GTR+I+G model...\n",
      "Estimate model parameters (epsilon = 5.000)\n",
      "Perform nearest neighbor interchange...\n",
      "Estimate model parameters (epsilon = 1.000)\n",
      "1. Initial log-likelihood: -1974.986\n",
      "Optimal log-likelihood: -1974.877\n",
      "Rate parameters:  A-C: 3.29286  A-G: 4.73397  A-T: 0.97458  C-G: 1.47167  C-T: 6.15849  G-T: 1.00000\n",
      "Base frequencies:  A: 0.234  C: 0.272  G: 0.283  T: 0.211\n",
      "Proportion of invariable sites: 0.350\n",
      "Gamma shape alpha: 1.508\n",
      "Parameters optimization took 1 rounds (0.002 sec)\n",
      "Time for fast ML tree search: 0.064 seconds\n",
      "\n",
      "NOTE: ModelFinder requires 0 MB RAM!\n",
      "ModelFinder will test up to 484 DNA models (sample size: 801) ...\n",
      " No. Model         -LnL         df  AIC          AICc         BIC\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  1  GTR+F         1978.480     13  3982.960     3983.422     4043.876\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  2  GTR+F+I       1974.924     14  3977.847     3978.382     4043.449\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  3  GTR+F+G4      1974.654     14  3977.308     3977.842     4042.910\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  4  GTR+F+I+G4    1974.717     15  3979.435     3980.046     4049.723\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  5  GTR+F+R2      1975.164     15  3980.327     3980.939     4050.615\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  6  GTR+F+R3      1974.772     17  3983.544     3984.326     4063.204\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 14  GTR+F+I+R2    1974.809     16  3981.617     3982.311     4056.591\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 15  GTR+F+I+R3    1974.766     18  3985.531     3986.406     4069.877\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 23  SYM           1980.690     10  3981.379     3981.658     4028.238\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 24  SYM+I         1977.239     11  3976.478     3976.813     4028.023\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 25  SYM+G4        1976.889     11  3975.779     3976.114     4027.323\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 26  SYM+I+G4      1977.049     12  3978.099     3978.494     4034.329\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 27  SYM+R2        1977.543     12  3979.085     3979.481     4035.316\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 45  TVM+F         1978.813     12  3981.627     3982.023     4037.857\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 46  TVM+F+I       1975.277     13  3976.554     3977.017     4037.470\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 47  TVM+F+G4      1974.894     13  3975.789     3976.251     4036.705\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 48  TVM+F+I+G4    1975.081     14  3978.162     3978.696     4043.764\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 49  TVM+F+R2      1975.594     14  3979.189     3979.723     4044.791\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 67  TVMe          1980.785     9   3979.570     3979.797     4021.743\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 68  TVMe+I        1977.413     10  3974.826     3975.105     4021.685\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 69  TVMe+G4       1976.990     10  3973.980     3974.258     4020.838\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 70  TVMe+I+G4     1977.186     11  3976.372     3976.707     4027.917\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 71  TVMe+R2       1977.661     11  3977.322     3977.657     4028.866\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 89  TIM3+F        1980.518     11  3983.037     3983.372     4034.581\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 90  TIM3+F+I      1976.541     12  3977.082     3977.478     4033.313\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 91  TIM3+F+G4     1976.391     12  3976.781     3977.177     4033.011\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 92  TIM3+F+I+G4   1976.421     13  3978.843     3979.305     4039.759\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 93  TIM3+F+R2     1977.032     13  3980.064     3980.526     4040.980\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "111  TIM3e         1981.985     8   3979.970     3980.152     4017.457\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "112  TIM3e+I       1978.267     9   3974.534     3974.762     4016.707\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "113  TIM3e+G4      1978.029     9   3974.058     3974.285     4016.231\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "114  TIM3e+I+G4    1978.108     10  3976.216     3976.495     4023.075\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "115  TIM3e+R2      1978.622     10  3977.244     3977.522     4024.102\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "133  TIM2+F        1980.857     11  3983.715     3984.049     4035.259\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "134  TIM2+F+I      1976.838     12  3977.676     3978.072     4033.907\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "135  TIM2+F+G4     1976.874     12  3977.747     3978.143     4033.977\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "136  TIM2+F+I+G4   1976.878     13  3979.757     3980.219     4040.673\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "137  TIM2+F+R2     1977.310     13  3980.621     3981.083     4041.537\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "155  TIM2e         1984.454     8   3984.908     3985.089     4022.394\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "156  TIM2e+I       1980.617     9   3979.234     3979.461     4021.406\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "157  TIM2e+G4      1980.602     9   3979.204     3979.432     4021.377\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "158  TIM2e+I+G4    1980.629     10  3981.258     3981.537     4028.117\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "159  TIM2e+R2      1981.126     10  3982.251     3982.530     4029.110\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "177  TIM+F         1982.577     11  3987.154     3987.488     4038.698\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "178  TIM+F+I       1978.076     12  3980.151     3980.547     4036.381\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "179  TIM+F+G4      1977.946     12  3979.892     3980.288     4036.122\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "180  TIM+F+I+G4    1977.902     13  3981.804     3982.267     4042.720\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "181  TIM+F+R2      1978.405     13  3982.810     3983.273     4043.726\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "199  TIMe          1985.518     8   3987.036     3987.217     4024.523\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "200  TIMe+I        1981.318     9   3980.637     3980.864     4022.809\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "201  TIMe+G4       1981.150     9   3980.299     3980.527     4022.472\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "202  TIMe+I+G4     1981.152     10  3982.304     3982.583     4029.163\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "203  TIMe+R2       1981.683     10  3983.366     3983.645     4030.225\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "221  TPM3u+F       1980.835     10  3981.671     3981.949     4028.530\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "222  TPM3u+F+I     1976.896     11  3975.793     3976.127     4027.337\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "223  TPM3u+F+G4    1976.651     11  3975.301     3975.636     4026.846\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "224  TPM3u+F+I+G4  1976.711     12  3977.423     3977.819     4033.653\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "225  TPM3u+F+R2    1977.337     12  3978.675     3979.071     4034.905\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "243  TPM3          1982.081     7   3978.163     3978.304     4010.964\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "244  TPM3+I        1978.398     8   3972.796     3972.978     4010.283\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "245  TPM3+G4       1978.108     8   3972.216     3972.398     4009.703\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "246  TPM3+I+G4     1978.215     9   3974.430     3974.657     4016.602\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "247  TPM3+R2       1978.727     9   3975.453     3975.681     4017.626\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "265  TPM2u+F       1981.166     10  3982.331     3982.610     4029.190\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "266  TPM2u+F+I     1977.292     11  3976.583     3976.918     4028.128\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "267  TPM2u+F+G4    1977.183     11  3976.366     3976.701     4027.911\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "268  TPM2u+F+I+G4  1977.263     12  3978.527     3978.923     4034.757\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "269  TPM2u+F+R2    1977.746     12  3979.493     3979.889     4035.723\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "287  TPM2          1984.552     7   3983.103     3983.244     4015.904\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "288  TPM2+I        1980.813     8   3977.625     3977.807     4015.112\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "289  TPM2+G4       1980.707     8   3977.414     3977.596     4014.901\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "290  TPM2+I+G4     1980.782     9   3979.564     3979.792     4021.737\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "291  TPM2+R2       1981.247     9   3980.494     3980.722     4022.667\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "309  K3Pu+F        1982.875     10  3985.750     3986.028     4032.609\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "310  K3Pu+F+I      1978.503     11  3979.007     3979.341     4030.551\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "311  K3Pu+F+G4     1978.224     11  3978.447     3978.782     4029.992\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "312  K3Pu+F+I+G4   1978.240     12  3980.481     3980.877     4036.711\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "313  K3Pu+F+R2     1978.841     12  3981.682     3982.078     4037.913\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "331  K3P           1985.618     7   3985.236     3985.377     4018.037\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "332  K3P+I         1981.490     8   3978.979     3979.161     4016.466\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "333  K3P+G4        1981.228     8   3978.457     3978.639     4015.944\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "334  K3P+I+G4      1981.269     9   3980.539     3980.766     4022.712\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "335  K3P+R2        1981.801     9   3981.602     3981.829     4023.774\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "353  TN+F          1982.839     10  3985.678     3985.957     4032.537\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "354  TN+F+I        1978.321     11  3978.643     3978.977     4030.187\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "355  TN+F+G4       1978.350     11  3978.700     3979.035     4030.245\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "356  TN+F+I+G4     1978.271     12  3980.542     3980.938     4036.773\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "357  TN+F+R2       1978.741     12  3981.481     3981.877     4037.711\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "375  TNe           1985.707     7   3985.414     3985.555     4018.215\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "376  TNe+I         1981.463     8   3978.925     3979.107     4016.412\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "377  TNe+G4        1981.434     8   3978.867     3979.049     4016.354\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "378  TNe+I+G4      1981.398     9   3980.796     3981.023     4022.968\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "379  TNe+R2        1981.902     9   3981.805     3982.032     4023.977\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "397  HKY+F         1983.134     9   3984.268     3984.496     4026.441\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "398  HKY+F+I       1978.741     10  3977.483     3977.761     4024.341\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "399  HKY+F+G4      1978.646     10  3977.291     3977.570     4024.150\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "400  HKY+F+I+G4    1978.591     11  3979.181     3979.516     4030.726\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "401  HKY+F+R2      1979.165     11  3980.330     3980.664     4031.874\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "419  K2P           1985.808     6   3983.616     3983.721     4011.731\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "420  K2P+I         1981.641     7   3977.283     3977.424     4010.084\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "421  K2P+G4        1981.535     7   3977.071     3977.212     4009.872\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "422  K2P+I+G4      1981.519     8   3979.037     3979.219     4016.524\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "423  K2P+R2        1982.027     8   3980.054     3980.236     4017.541\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "441  F81+F         1998.566     8   4013.131     4013.313     4050.618\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "442  F81+F+I       1992.251     9   4002.502     4002.730     4044.675\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "443  F81+F+G4      1992.198     9   4002.397     4002.624     4044.569\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "444  F81+F+I+G4    1992.190     10  4004.381     4004.659     4051.239\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "445  F81+F+R2      1993.254     10  4006.508     4006.787     4053.367\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "463  JC            2000.994     5   4011.989     4012.064     4035.418\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "464  JC+I          1994.942     6   4001.885     4001.991     4030.000\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "465  JC+G4         1994.915     6   4001.829     4001.935     4029.945\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "466  JC+I+G4       1994.844     7   4003.688     4003.829     4036.489\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "467  JC+R2         1995.538     7   4005.076     4005.217     4037.877\n",
      "Akaike Information Criterion:           TPM3+G4\n",
      "Corrected Akaike Information Criterion: TPM3+G4\n",
      "Bayesian Information Criterion:         TPM3+G4\n",
      "Best-fit model: TPM3+G4 chosen according to BIC\n",
      "\n",
      "All model information printed to result/phylo_fna_gene/100138at50557_iqtreeout/.tree.model.gz\n",
      "CPU time for ModelFinder: 9.123 seconds (0h:0m:9s)\n",
      "Wall-clock time for ModelFinder: 1.557 seconds (0h:0m:1s)\n",
      "Generating 1000 samples for ultrafast bootstrap (seed: 841031)...\n",
      "WARNING: 8 threads for alignment length 41 will slow down analysis\n",
      "\n",
      "NOTE: 0 MB RAM (0 GB) is required!\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "Estimate model parameters (epsilon = 0.100)\n",
      "1. Initial log-likelihood: -1978.108\n",
      "Optimal log-likelihood: -1978.106\n",
      "Rate parameters:  A-C: 2.36010  A-G: 4.80800  A-T: 1.00000  C-G: 2.36010  C-T: 4.80800  G-T: 1.00000\n",
      "Base frequencies:  A: 0.250  C: 0.250  G: 0.250  T: 0.250\n",
      "Gamma shape alpha: 0.614\n",
      "Parameters optimization took 1 rounds (0.008 sec)\n",
      "Wrote distance file to... \n",
      "Computing ML distances based on estimated model parameters...\n",
      "Calculating distance matrix: done in 0.00377541 secs using 150.8% CPU\n",
      "Computing ML distances took 0.027784 sec (of wall-clock time) 0.201059 sec (of CPU time)\n",
      "Setting up auxiliary I and S matrices: done in 0.00229964 secs using 158% CPU\n",
      "Computing RapidNJ tree took 0.004478 sec (of wall-clock time) 0.003869 sec (of CPU time)\n",
      "Log-likelihood of RapidNJ tree: -1977.870\n",
      "--------------------------------------------------------------------\n",
      "|             INITIALIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "Generating 98 parsimony trees... 0.015 second\n",
      "Computing log-likelihood of 1 initial trees ... 0.015 seconds\n",
      "Current best score: -1977.870\n",
      "\n",
      "Do NNI search on 3 best initial trees\n",
      "Finish initializing candidate tree set (3)\n",
      "Current best tree score: -1977.870 / CPU time: 0.261\n",
      "Number of iterations: 3\n",
      "--------------------------------------------------------------------\n",
      "|               OPTIMIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "UPDATE BEST LOG-LIKELIHOOD: -1977.869\n",
      "WARNING: NNI search needs unusual large number of steps (4) to converge!\n",
      "WARNING: NNI search needs unusual large number of steps (4) to converge!\n",
      "WARNING: NNI search needs unusual large number of steps (4) to converge!\n",
      "Iteration 10 / LogL: -1977.956 / Time: 0h:0m:2s\n",
      "WARNING: NNI search needs unusual large number of steps (4) to converge!\n",
      "UPDATE BEST LOG-LIKELIHOOD: -1977.869\n",
      "UPDATE BEST LOG-LIKELIHOOD: -1977.869\n",
      "Iteration 20 / LogL: -1978.130 / Time: 0h:0m:3s\n",
      "UPDATE BEST LOG-LIKELIHOOD: -1977.868\n",
      "UPDATE BEST LOG-LIKELIHOOD: -1977.868\n",
      "WARNING: NNI search needs unusual large number of steps (4) to converge!\n",
      "Iteration 30 / LogL: -1978.102 / Time: 0h:0m:4s (0h:0m:5s left)\n",
      "UPDATE BEST LOG-LIKELIHOOD: -1977.868\n",
      "UPDATE BEST LOG-LIKELIHOOD: -1977.868\n",
      "WARNING: NNI search needs unusual large number of steps (4) to converge!\n",
      "WARNING: NNI search needs unusual large number of steps (4) to converge!\n",
      "WARNING: NNI search needs unusual large number of steps (4) to converge!\n",
      "Iteration 40 / LogL: -1977.993 / Time: 0h:0m:5s (0h:0m:5s left)\n",
      "WARNING: NNI search needs unusual large number of steps (4) to converge!\n",
      "Iteration 50 / LogL: -1978.130 / Time: 0h:0m:6s (0h:0m:4s left)\n",
      "Log-likelihood cutoff on original alignment: -1979.596\n",
      "Iteration 60 / LogL: -1978.130 / Time: 0h:0m:7s (0h:0m:3s left)\n",
      "UPDATE BEST LOG-LIKELIHOOD: -1977.868\n",
      "Iteration 70 / LogL: -1977.868 / Time: 0h:0m:7s (0h:0m:2s left)\n",
      "WARNING: NNI search needs unusual large number of steps (4) to converge!\n",
      "WARNING: NNI search needs unusual large number of steps (4) to converge!\n",
      "UPDATE BEST LOG-LIKELIHOOD: -1977.868\n",
      "Iteration 80 / LogL: -1978.102 / Time: 0h:0m:8s (0h:0m:1s left)\n",
      "UPDATE BEST LOG-LIKELIHOOD: -1977.868\n",
      "WARNING: NNI search needs unusual large number of steps (4) to converge!\n",
      "UPDATE BEST LOG-LIKELIHOOD: -1977.868\n",
      "Iteration 90 / LogL: -1978.031 / Time: 0h:0m:9s (0h:0m:0s left)\n",
      "UPDATE BEST LOG-LIKELIHOOD: -1977.868\n",
      "WARNING: NNI search needs unusual large number of steps (4) to converge!\n",
      "UPDATE BEST LOG-LIKELIHOOD: -1977.868\n",
      "Iteration 100 / LogL: -1978.031 / Time: 0h:0m:10s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -1979.597\n",
      "NOTE: Bootstrap correlation coefficient of split occurrence frequencies: 0.994\n",
      "TREE SEARCH COMPLETED AFTER 101 ITERATIONS / Time: 0h:0m:10s\n",
      "\n",
      "--------------------------------------------------------------------\n",
      "|                    FINALIZING TREE SEARCH                        |\n",
      "--------------------------------------------------------------------\n",
      "Performs final model parameters optimization\n",
      "Estimate model parameters (epsilon = 0.010)\n",
      "1. Initial log-likelihood: -1977.868\n",
      "2. Current log-likelihood: -1977.852\n",
      "Optimal log-likelihood: -1977.841\n",
      "Rate parameters:  A-C: 2.39883  A-G: 5.01445  A-T: 1.00000  C-G: 2.39883  C-T: 5.01445  G-T: 1.00000\n",
      "Base frequencies:  A: 0.250  C: 0.250  G: 0.250  T: 0.250\n",
      "Gamma shape alpha: 0.596\n",
      "Parameters optimization took 2 rounds (0.004 sec)\n",
      "BEST SCORE FOUND : -1977.841\n",
      "\n",
      "Testing tree branches by SH-like aLRT with 1000 replicates...\n",
      "0.013 sec.\n",
      "Creating bootstrap support values...\n",
      "Split supports printed to NEXUS file result/phylo_fna_gene/100138at50557_iqtreeout/.tree.splits.nex\n",
      "Total tree length: 0.634\n",
      "\n",
      "Total number of iterations: 101\n",
      "CPU time used for tree search: 59.372 sec (0h:0m:59s)\n",
      "Wall-clock time used for tree search: 8.963 sec (0h:0m:8s)\n",
      "Total CPU time used: 69.547 sec (0h:1m:9s)\n",
      "Total wall-clock time used: 10.682 sec (0h:0m:10s)\n",
      "\n",
      "Computing bootstrap consensus tree...\n",
      "Reading input file result/phylo_fna_gene/100138at50557_iqtreeout/.tree.splits.nex...\n",
      "4 taxa and 7 splits.\n",
      "Consensus tree written to result/phylo_fna_gene/100138at50557_iqtreeout/.tree.contree\n",
      "Reading input trees file result/phylo_fna_gene/100138at50557_iqtreeout/.tree.contree\n",
      "Log-likelihood of consensus tree: -1977.841\n",
      "\n",
      "Analysis results written to: \n",
      "  IQ-TREE report:                result/phylo_fna_gene/100138at50557_iqtreeout/.tree.iqtree\n",
      "  Maximum-likelihood tree:       result/phylo_fna_gene/100138at50557_iqtreeout/.tree.treefile\n",
      "  Likelihood distances:          result/phylo_fna_gene/100138at50557_iqtreeout/.tree.mldist\n",
      "\n",
      "Ultrafast bootstrap approximation results written to:\n",
      "  Split support values:          result/phylo_fna_gene/100138at50557_iqtreeout/.tree.splits.nex\n",
      "  Consensus tree:                result/phylo_fna_gene/100138at50557_iqtreeout/.tree.contree\n",
      "  Screen log file:               result/phylo_fna_gene/100138at50557_iqtreeout/.tree.log\n",
      "\n",
      "Date and Time: Sat Aug 31 21:10:34 2024\n",
      "IQ-TREE multicore version 2.3.4 COVID-edition for Linux x86 64-bit built Jun 18 2024\n",
      "Developed by Bui Quang Minh, James Barbetti, Nguyen Lam Tung, Olga Chernomor,\n",
      "Heiko Schmidt, Dominik Schrempf, Michael Woodhams, Ly Trong Nhan, Thomas Wong\n",
      "\n",
      "Host:    RCNSZOR36402A (AVX512, FMA3, 7 GB RAM)\n",
      "Command: iqtree -s result/phylo_fna_gene/100236at50557_AlnTrimed.fna -nt 8 -m MFP -bb 1000 -alrt 1000 -pre result/phylo_fna_gene/100236at50557_iqtreeout/.tree\n",
      "Seed:    575586 (Using SPRNG - Scalable Parallel Random Number Generator)\n",
      "Time:    Sat Aug 31 21:10:34 2024\n",
      "Kernel:  AVX+FMA - 8 threads (8 CPU cores detected)\n",
      "\n",
      "Reading alignment file result/phylo_fna_gene/100236at50557_AlnTrimed.fna ... Fasta format detected\n",
      "Reading fasta file: done in 0.00013628 secs\n",
      "Alignment most likely contains DNA/RNA sequences\n",
      "Alignment has 4 sequences with 1194 columns, 57 distinct patterns\n",
      "23 parsimony-informative, 261 singleton sites, 910 constant sites\n",
      "                        Gap/Ambiguity  Composition  p-value\n",
      "Analyzing sequences: done in 0.0157003 secs using 820.2% CPU\n",
      "   1  Culex_pipiens             0.00%    passed     71.90%\n",
      "   2  Armigeres_subalbatus      0.00%    failed      4.83%\n",
      "   3  Culex_pipiens_molestus    0.00%    passed     89.16%\n",
      "   4  Culex_pipiens_pallens     0.00%    passed     84.78%\n",
      "****  TOTAL                     0.00%  1 sequences failed composition chi2 test (p-value<5%; df=3)\n",
      "\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "\n",
      "Create initial parsimony tree by phylogenetic likelihood library (PLL)... 0.000 seconds\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "Perform fast likelihood tree search using GTR+I+G model...\n",
      "Estimate model parameters (epsilon = 5.000)\n",
      "Perform nearest neighbor interchange...\n",
      "Estimate model parameters (epsilon = 1.000)\n",
      "1. Initial log-likelihood: -2838.988\n",
      "Optimal log-likelihood: -2838.961\n",
      "Rate parameters:  A-C: 1.22901  A-G: 2.86155  A-T: 0.90578  C-G: 0.89691  C-T: 5.79447  G-T: 1.00000\n",
      "Base frequencies:  A: 0.238  C: 0.249  G: 0.286  T: 0.228\n",
      "Proportion of invariable sites: 0.379\n",
      "Gamma shape alpha: 1.157\n",
      "Parameters optimization took 1 rounds (0.225 sec)\n",
      "Time for fast ML tree search: 0.479 seconds\n",
      "\n",
      "NOTE: ModelFinder requires 0 MB RAM!\n",
      "ModelFinder will test up to 484 DNA models (sample size: 1194) ...\n",
      " No. Model         -LnL         df  AIC          AICc         BIC\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  1  GTR+F         2849.068     13  5724.137     5724.445     5790.242\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  2  GTR+F+I       2839.319     14  5706.638     5706.994     5777.828\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  3  GTR+F+G4      2839.077     14  5706.154     5706.510     5777.345\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  4  GTR+F+I+G4    2838.966     15  5707.932     5708.340     5784.208\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  5  GTR+F+R2      2839.659     15  5709.318     5709.726     5785.594\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  6  GTR+F+R3      2839.162     17  5712.324     5712.844     5798.770\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 14  GTR+F+I+R2    2839.136     16  5710.271     5710.733     5791.632\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 15  GTR+F+I+R3    2838.979     18  5713.958     5714.540     5805.489\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 24  SYM+I         2843.284     11  5708.569     5708.792     5764.504\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 25  SYM+G4        2843.059     11  5708.118     5708.341     5764.054\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 26  SYM+I+G4      2842.999     12  5709.997     5710.262     5771.018\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 27  SYM+R2        2843.646     12  5711.291     5711.556     5772.312\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 46  TVM+F+I       2843.397     13  5712.793     5713.102     5778.899\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 47  TVM+F+G4      2843.450     13  5712.901     5713.209     5779.006\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 48  TVM+F+I+G4    2843.457     14  5714.914     5715.271     5786.105\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 49  TVM+F+R2      2843.917     14  5715.834     5716.190     5787.025\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 68  TVMe+I        2846.441     10  5712.882     5713.068     5763.733\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 69  TVMe+G4       2846.323     10  5712.646     5712.832     5763.496\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 70  TVMe+I+G4     2846.306     11  5714.612     5714.835     5770.547\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 71  TVMe+R2       2846.919     11  5715.839     5716.062     5771.775\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 90  TIM3+F+I      2839.508     12  5703.015     5703.279     5764.036\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 91  TIM3+F+G4     2839.224     12  5702.447     5702.711     5763.468\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 92  TIM3+F+I+G4   2839.168     13  5704.336     5704.645     5770.442\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 93  TIM3+F+R2     2839.859     13  5705.717     5706.026     5771.823\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "112  TIM3e+I       2843.400     9   5704.800     5704.952     5750.565\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "113  TIM3e+G4      2843.136     9   5704.273     5704.425     5750.038\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "114  TIM3e+I+G4    2843.090     10  5706.179     5706.365     5757.030\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "115  TIM3e+R2      2843.756     10  5707.512     5707.698     5758.363\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "134  TIM2+F+I      2839.445     12  5702.889     5703.153     5763.910\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "135  TIM2+F+G4     2839.161     12  5702.321     5702.585     5763.342\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "136  TIM2+F+I+G4   2839.103     13  5704.206     5704.515     5770.312\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "137  TIM2+F+R2     2839.834     13  5705.669     5705.977     5771.774\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "156  TIM2e+I       2843.552     9   5705.103     5705.255     5750.869\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "157  TIM2e+G4      2843.333     9   5704.666     5704.818     5750.432\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "158  TIM2e+I+G4    2843.284     10  5706.568     5706.754     5757.419\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "159  TIM2e+R2      2843.864     10  5707.728     5707.914     5758.579\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "178  TIM+F+I       2839.426     12  5702.853     5703.117     5763.874\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "179  TIM+F+G4      2839.185     12  5702.371     5702.635     5763.391\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "180  TIM+F+I+G4    2839.126     13  5704.251     5704.560     5770.357\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "181  TIM+F+R2      2839.788     13  5705.576     5705.885     5771.682\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "200  TIMe+I        2843.434     9   5704.868     5705.020     5750.634\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "201  TIMe+G4       2843.232     9   5704.464     5704.617     5750.230\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "202  TIMe+I+G4     2843.179     10  5706.357     5706.543     5757.208\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "203  TIMe+R2       2843.815     10  5707.630     5707.816     5758.480\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "222  TPM3u+F+I     2843.861     11  5709.722     5709.946     5765.658\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "223  TPM3u+F+G4    2843.803     11  5709.606     5709.829     5765.541\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "224  TPM3u+F+I+G4  2843.786     12  5711.572     5711.836     5772.592\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "225  TPM3u+F+R2    2844.328     12  5712.656     5712.920     5773.677\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "244  TPM3+I        2846.527     8   5709.054     5709.176     5749.735\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "245  TPM3+G4       2846.420     8   5708.839     5708.961     5749.520\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "246  TPM3+I+G4     2846.408     9   5710.816     5710.968     5756.582\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "247  TPM3+R2       2846.937     9   5711.874     5712.026     5757.640\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "266  TPM2u+F+I     2843.480     11  5708.959     5709.183     5764.895\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "267  TPM2u+F+G4    2843.553     11  5709.106     5709.329     5765.041\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "268  TPM2u+F+I+G4  2843.599     12  5711.197     5711.462     5772.218\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "269  TPM2u+F+R2    2843.934     12  5711.869     5712.133     5772.890\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "288  TPM2+I        2846.655     8   5709.309     5709.431     5749.990\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "289  TPM2+G4       2846.564     8   5709.128     5709.249     5749.808\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "290  TPM2+I+G4     2846.546     9   5711.092     5711.244     5756.857\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "291  TPM2+R2       2847.110     9   5712.220     5712.372     5757.986\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "310  K3Pu+F+I      2843.799     11  5709.598     5709.821     5765.534\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "311  K3Pu+F+G4     2843.769     11  5709.539     5709.762     5765.474\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "312  K3Pu+F+I+G4   2843.772     12  5711.543     5711.807     5772.564\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "313  K3Pu+F+R2     2844.279     12  5712.559     5712.823     5773.580\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "332  K3P+I         2846.574     8   5709.148     5709.269     5749.828\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "333  K3P+G4        2846.490     8   5708.980     5709.101     5749.660\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "334  K3P+I+G4      2846.479     9   5710.957     5711.109     5756.723\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "335  K3P+R2        2846.999     9   5711.998     5712.150     5757.763\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "354  TN+F+I        2839.562     11  5701.124     5701.347     5757.060\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "355  TN+F+G4       2839.298     11  5700.597     5700.820     5756.532\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "356  TN+F+I+G4     2839.247     12  5702.494     5702.758     5763.514\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "357  TN+F+R2       2839.935     12  5703.871     5704.135     5764.892\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "376  TNe+I         2843.553     8   5703.106     5703.228     5743.787\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "377  TNe+G4        2843.337     8   5702.674     5702.796     5743.355\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "378  TNe+I+G4      2843.285     9   5704.570     5704.722     5750.336\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "379  TNe+R2        2843.935     9   5705.870     5706.022     5751.636\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "398  HKY+F+I       2843.909     10  5707.818     5708.004     5758.669\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "399  HKY+F+G4      2843.885     10  5707.771     5707.957     5758.621\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "400  HKY+F+I+G4    2843.855     11  5709.711     5709.934     5765.647\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "401  HKY+F+R2      2844.287     11  5710.574     5710.798     5766.510\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "420  K2P+I         2846.681     7   5707.362     5707.457     5742.958\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "421  K2P+G4        2846.597     7   5707.195     5707.289     5742.790\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "422  K2P+I+G4      2846.571     8   5709.142     5709.263     5749.822\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "423  K2P+R2        2847.101     8   5710.202     5710.323     5750.882\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "442  F81+F+I       2883.314     9   5784.629     5784.781     5830.394\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "443  F81+F+G4      2883.110     9   5784.221     5784.373     5829.986\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "444  F81+F+I+G4    2883.097     10  5786.194     5786.379     5837.044\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "445  F81+F+R2      2883.879     10  5787.759     5787.945     5838.609\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "464  JC+I          2885.708     6   5783.416     5783.487     5813.927\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "465  JC+G4         2885.527     6   5783.055     5783.126     5813.565\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "466  JC+I+G4       2885.516     7   5785.033     5785.127     5820.628\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "467  JC+R2         2886.287     7   5786.574     5786.669     5822.170\n",
      "Akaike Information Criterion:           TN+F+G4\n",
      "Corrected Akaike Information Criterion: TN+F+G4\n",
      "Bayesian Information Criterion:         K2P+G4\n",
      "Best-fit model: K2P+G4 chosen according to BIC\n",
      "\n",
      "All model information printed to result/phylo_fna_gene/100236at50557_iqtreeout/.tree.model.gz\n",
      "CPU time for ModelFinder: 26.538 seconds (0h:0m:26s)\n",
      "Wall-clock time for ModelFinder: 3.793 seconds (0h:0m:3s)\n",
      "Generating 1000 samples for ultrafast bootstrap (seed: 575586)...\n",
      "WARNING: 8 threads for alignment length 57 will slow down analysis\n",
      "\n",
      "NOTE: 0 MB RAM (0 GB) is required!\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "Estimate model parameters (epsilon = 0.100)\n",
      "1. Initial log-likelihood: -2846.597\n",
      "Optimal log-likelihood: -2846.581\n",
      "Rate parameters:  A-C: 1.00000  A-G: 4.27920  A-T: 1.00000  C-G: 1.00000  C-T: 4.27920  G-T: 1.00000\n",
      "Base frequencies:  A: 0.250  C: 0.250  G: 0.250  T: 0.250\n",
      "Gamma shape alpha: 0.318\n",
      "Parameters optimization took 1 rounds (0.006 sec)\n",
      "Wrote distance file to... \n",
      "Computing ML distances based on estimated model parameters...\n",
      "Calculating distance matrix: done in 0.00641152 secs using 296.4% CPU\n",
      "Computing ML distances took 0.012277 sec (of wall-clock time) 0.092122 sec (of CPU time)\n",
      "Setting up auxiliary I and S matrices: done in 0.00164695 secs using 175% CPU\n",
      "Computing RapidNJ tree took 0.003065 sec (of wall-clock time) 0.005131 sec (of CPU time)\n",
      "Log-likelihood of RapidNJ tree: -2846.581\n",
      "--------------------------------------------------------------------\n",
      "|             INITIALIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "Generating 99 parsimony trees... 0.015 second\n",
      "Computing log-likelihood of 2 initial trees ... 0.001 seconds\n",
      "Current best score: -2846.581\n",
      "\n",
      "Do NNI search on 3 best initial trees\n",
      "Finish initializing candidate tree set (3)\n",
      "Current best tree score: -2846.581 / CPU time: 0.166\n",
      "Number of iterations: 3\n",
      "--------------------------------------------------------------------\n",
      "|               OPTIMIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "Iteration 10 / LogL: -2846.855 / Time: 0h:0m:4s\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2846.581\n",
      "Iteration 20 / LogL: -2846.880 / Time: 0h:0m:5s\n",
      "Iteration 30 / LogL: -2846.880 / Time: 0h:0m:6s (0h:0m:7s left)\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2846.581\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2846.581\n",
      "Iteration 40 / LogL: -2846.880 / Time: 0h:0m:7s (0h:0m:4s left)\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2846.581\n",
      "Iteration 50 / LogL: -2846.855 / Time: 0h:0m:7s (0h:0m:3s left)\n",
      "Log-likelihood cutoff on original alignment: -2848.363\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2846.581\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2846.581\n",
      "Iteration 60 / LogL: -2846.833 / Time: 0h:0m:7s (0h:0m:2s left)\n",
      "Iteration 70 / LogL: -2846.917 / Time: 0h:0m:8s (0h:0m:2s left)\n",
      "Iteration 80 / LogL: -2846.581 / Time: 0h:0m:10s (0h:0m:1s left)\n",
      "Iteration 90 / LogL: -2846.581 / Time: 0h:0m:10s (0h:0m:0s left)\n",
      "Iteration 100 / LogL: -2846.607 / Time: 0h:0m:11s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -2848.361\n",
      "NOTE: Bootstrap correlation coefficient of split occurrence frequencies: 0.999\n",
      "TREE SEARCH COMPLETED AFTER 101 ITERATIONS / Time: 0h:0m:11s\n",
      "\n",
      "--------------------------------------------------------------------\n",
      "|                    FINALIZING TREE SEARCH                        |\n",
      "--------------------------------------------------------------------\n",
      "Performs final model parameters optimization\n",
      "Estimate model parameters (epsilon = 0.010)\n",
      "1. Initial log-likelihood: -2846.581\n",
      "Optimal log-likelihood: -2846.573\n",
      "Rate parameters:  A-C: 1.00000  A-G: 4.31230  A-T: 1.00000  C-G: 1.00000  C-T: 4.31230  G-T: 1.00000\n",
      "Base frequencies:  A: 0.250  C: 0.250  G: 0.250  T: 0.250\n",
      "Gamma shape alpha: 0.313\n",
      "Parameters optimization took 1 rounds (0.018 sec)\n",
      "BEST SCORE FOUND : -2846.573\n",
      "\n",
      "Testing tree branches by SH-like aLRT with 1000 replicates...\n",
      "0.003 sec.\n",
      "Creating bootstrap support values...\n",
      "Split supports printed to NEXUS file result/phylo_fna_gene/100236at50557_iqtreeout/.tree.splits.nex\n",
      "Total tree length: 0.476\n",
      "\n",
      "Total number of iterations: 101\n",
      "CPU time used for tree search: 57.207 sec (0h:0m:57s)\n",
      "Wall-clock time used for tree search: 7.893 sec (0h:0m:7s)\n",
      "Total CPU time used: 84.542 sec (0h:1m:24s)\n",
      "Total wall-clock time used: 11.799 sec (0h:0m:11s)\n",
      "\n",
      "Computing bootstrap consensus tree...\n",
      "Reading input file result/phylo_fna_gene/100236at50557_iqtreeout/.tree.splits.nex...\n",
      "4 taxa and 7 splits.\n",
      "Consensus tree written to result/phylo_fna_gene/100236at50557_iqtreeout/.tree.contree\n",
      "Reading input trees file result/phylo_fna_gene/100236at50557_iqtreeout/.tree.contree\n",
      "Log-likelihood of consensus tree: -2846.574\n",
      "\n",
      "Analysis results written to: \n",
      "  IQ-TREE report:                result/phylo_fna_gene/100236at50557_iqtreeout/.tree.iqtree\n",
      "  Maximum-likelihood tree:       result/phylo_fna_gene/100236at50557_iqtreeout/.tree.treefile\n",
      "  Likelihood distances:          result/phylo_fna_gene/100236at50557_iqtreeout/.tree.mldist\n",
      "\n",
      "Ultrafast bootstrap approximation results written to:\n",
      "  Split support values:          result/phylo_fna_gene/100236at50557_iqtreeout/.tree.splits.nex\n",
      "  Consensus tree:                result/phylo_fna_gene/100236at50557_iqtreeout/.tree.contree\n",
      "  Screen log file:               result/phylo_fna_gene/100236at50557_iqtreeout/.tree.log\n",
      "\n",
      "Date and Time: Sat Aug 31 21:10:46 2024\n",
      "IQ-TREE multicore version 2.3.4 COVID-edition for Linux x86 64-bit built Jun 18 2024\n",
      "Developed by Bui Quang Minh, James Barbetti, Nguyen Lam Tung, Olga Chernomor,\n",
      "Heiko Schmidt, Dominik Schrempf, Michael Woodhams, Ly Trong Nhan, Thomas Wong\n",
      "\n",
      "Host:    RCNSZOR36402A (AVX512, FMA3, 7 GB RAM)\n",
      "Command: iqtree -s result/phylo_fna_gene/100341at50557_AlnTrimed.fna -nt 8 -m MFP -bb 1000 -alrt 1000 -pre result/phylo_fna_gene/100341at50557_iqtreeout/.tree\n",
      "Seed:    518866 (Using SPRNG - Scalable Parallel Random Number Generator)\n",
      "Time:    Sat Aug 31 21:10:46 2024\n",
      "Kernel:  AVX+FMA - 8 threads (8 CPU cores detected)\n",
      "\n",
      "Reading alignment file result/phylo_fna_gene/100341at50557_AlnTrimed.fna ... Fasta format detected\n",
      "Reading fasta file: done in 0.000123117 secs using 99.09% CPU\n",
      "Alignment most likely contains DNA/RNA sequences\n",
      "Alignment has 4 sequences with 975 columns, 37 distinct patterns\n",
      "9 parsimony-informative, 259 singleton sites, 707 constant sites\n",
      "                        Gap/Ambiguity  Composition  p-value\n",
      "Analyzing sequences: done in 0.00157498 secs using 3350% CPU\n",
      "   1  Culex_pipiens             0.00%    passed     34.21%\n",
      "   2  Armigeres_subalbatus      0.00%    failed      0.00%\n",
      "   3  Culex_pipiens_molestus    0.00%    passed     28.09%\n",
      "   4  Culex_pipiens_pallens     0.00%    passed     44.39%\n",
      "****  TOTAL                     0.00%  1 sequences failed composition chi2 test (p-value<5%; df=3)\n",
      "\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "\n",
      "Create initial parsimony tree by phylogenetic likelihood library (PLL)... 0.000 seconds\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "Perform fast likelihood tree search using GTR+I+G model...\n",
      "Estimate model parameters (epsilon = 5.000)\n",
      "Perform nearest neighbor interchange...\n",
      "Estimate model parameters (epsilon = 1.000)\n",
      "1. Initial log-likelihood: -2321.617\n",
      "Optimal log-likelihood: -2321.552\n",
      "Rate parameters:  A-C: 2.68270  A-G: 3.99948  A-T: 0.62882  C-G: 2.27620  C-T: 11.68359  G-T: 1.00000\n",
      "Base frequencies:  A: 0.269  C: 0.253  G: 0.296  T: 0.182\n",
      "Proportion of invariable sites: 0.364\n",
      "Gamma shape alpha: 2.315\n",
      "Parameters optimization took 1 rounds (0.006 sec)\n",
      "Time for fast ML tree search: 0.043 seconds\n",
      "\n",
      "NOTE: ModelFinder requires 0 MB RAM!\n",
      "ModelFinder will test up to 484 DNA models (sample size: 975) ...\n",
      " No. Model         -LnL         df  AIC          AICc         BIC\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  1  GTR+F         2322.588     13  4671.176     4671.554     4734.647\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  2  GTR+F+I       2321.483     14  4670.965     4671.403     4739.319\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  3  GTR+F+G4      2321.166     14  4670.332     4670.770     4738.686\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  4  GTR+F+I+G4    2321.408     15  4672.817     4673.317     4746.053\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  5  GTR+F+R2      2321.381     15  4672.762     4673.262     4745.998\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  6  GTR+F+R3      2321.212     17  4676.424     4677.064     4759.426\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 14  GTR+F+I+R2    2321.624     16  4675.249     4675.817     4753.368\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 15  GTR+F+I+R3    2321.526     18  4679.051     4679.767     4766.935\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 23  SYM           2333.748     10  4687.495     4687.724     4736.320\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 24  SYM+I         2332.964     11  4687.928     4688.203     4741.635\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 25  SYM+G4        2332.636     11  4687.272     4687.546     4740.979\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 45  TVM+F         2331.971     12  4687.941     4688.266     4746.531\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 46  TVM+F+I       2329.862     13  4685.724     4686.103     4749.196\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 47  TVM+F+G4      2329.976     13  4685.953     4686.332     4749.425\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 67  TVMe          2337.726     9   4693.452     4693.639     4737.394\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 68  TVMe+I        2336.745     10  4693.490     4693.719     4742.315\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 69  TVMe+G4       2336.534     10  4693.068     4693.296     4741.892\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 89  TIM3+F        2322.758     11  4667.517     4667.791     4721.223\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 90  TIM3+F+I      2321.658     12  4667.315     4667.640     4725.905\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 91  TIM3+F+G4     2321.332     12  4666.664     4666.988     4725.253\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "111  TIM3e         2333.933     8   4683.867     4684.016     4722.926\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "112  TIM3e+I       2333.158     9   4684.316     4684.502     4728.258\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "113  TIM3e+G4      2332.797     9   4683.593     4683.780     4727.535\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "133  TIM2+F        2327.326     11  4676.652     4676.926     4730.359\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "134  TIM2+F+I      2325.422     12  4674.845     4675.169     4733.434\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "135  TIM2+F+G4     2325.279     12  4674.557     4674.882     4733.146\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "155  TIM2e         2341.172     8   4698.343     4698.493     4737.403\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "156  TIM2e+I       2339.599     9   4697.197     4697.384     4741.139\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "157  TIM2e+G4      2339.443     9   4696.885     4697.072     4740.827\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "177  TIM+F         2327.234     11  4676.467     4676.741     4730.174\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "178  TIM+F+I       2325.354     12  4674.709     4675.033     4733.298\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "179  TIM+F+G4      2325.215     12  4674.430     4674.754     4733.019\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "199  TIMe          2341.102     8   4698.204     4698.353     4737.263\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "200  TIMe+I        2339.554     9   4697.108     4697.295     4741.050\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "201  TIMe+G4       2339.389     9   4696.778     4696.965     4740.720\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "221  TPM3u+F       2332.145     10  4684.289     4684.517     4733.114\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "222  TPM3u+F+I     2330.026     11  4682.051     4682.326     4735.758\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "223  TPM3u+F+G4    2330.130     11  4682.261     4682.535     4735.967\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "243  TPM3          2337.915     7   4689.831     4689.947     4724.008\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "244  TPM3+I        2336.936     8   4689.871     4690.020     4728.931\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "245  TPM3+G4       2336.700     8   4689.401     4689.550     4728.460\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "265  TPM2u+F       2336.468     10  4692.936     4693.164     4741.760\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "266  TPM2u+F+I     2332.966     11  4687.931     4688.205     4741.638\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "267  TPM2u+F+G4    2333.731     11  4689.463     4689.737     4743.169\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "287  TPM2          2345.141     7   4704.283     4704.398     4738.460\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "288  TPM2+I        2343.359     8   4702.717     4702.866     4741.777\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "289  TPM2+G4       2343.313     8   4702.626     4702.775     4741.686\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "309  K3Pu+F        2336.377     10  4692.754     4692.982     4741.578\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "310  K3Pu+F+I      2333.041     11  4688.082     4688.356     4741.789\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "311  K3Pu+F+G4     2333.672     11  4689.345     4689.619     4743.052\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "331  K3P           2345.068     7   4704.135     4704.251     4738.312\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "332  K3P+I         2343.312     8   4702.624     4702.773     4741.683\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "333  K3P+G4        2343.260     8   4702.520     4702.669     4741.580\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "353  TN+F          2327.326     10  4674.652     4674.880     4723.476\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "354  TN+F+I        2325.366     11  4672.732     4673.006     4726.439\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "355  TN+F+G4       2325.279     11  4672.558     4672.832     4726.265\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "375  TNe           2341.190     7   4696.381     4696.497     4730.558\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "376  TNe+I         2339.616     8   4695.233     4695.382     4734.292\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "377  TNe+G4        2339.452     8   4694.904     4695.053     4733.963\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "397  HKY+F         2336.468     9   4690.936     4691.122     4734.878\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "398  HKY+F+I       2332.960     10  4685.919     4686.147     4734.744\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "399  HKY+F+G4      2333.728     10  4687.457     4687.685     4736.281\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "419  K2P           2345.158     6   4702.316     4702.403     4731.611\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "420  K2P+I         2343.377     7   4700.755     4700.870     4734.932\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "421  K2P+G4        2343.325     7   4700.650     4700.766     4734.827\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "441  F81+F         2361.681     8   4739.363     4739.512     4778.422\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "442  F81+F+I       2361.203     9   4740.405     4740.592     4784.347\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "443  F81+F+G4      2361.190     9   4740.380     4740.566     4784.322\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "463  JC            2372.553     5   4755.106     4755.168     4779.518\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "464  JC+I          2372.032     6   4756.064     4756.151     4785.359\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "465  JC+G4         2372.023     6   4756.046     4756.133     4785.341\n",
      "Akaike Information Criterion:           TIM3+F+G4\n",
      "Corrected Akaike Information Criterion: TIM3+F+G4\n",
      "Bayesian Information Criterion:         TIM3+F\n",
      "Best-fit model: TIM3+F chosen according to BIC\n",
      "\n",
      "All model information printed to result/phylo_fna_gene/100341at50557_iqtreeout/.tree.model.gz\n",
      "CPU time for ModelFinder: 3.877 seconds (0h:0m:3s)\n",
      "Wall-clock time for ModelFinder: 0.680 seconds (0h:0m:0s)\n",
      "Generating 1000 samples for ultrafast bootstrap (seed: 518866)...\n",
      "WARNING: 8 threads for alignment length 37 will slow down analysis\n",
      "\n",
      "NOTE: 0 MB RAM (0 GB) is required!\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "Estimate model parameters (epsilon = 0.100)\n",
      "1. Initial log-likelihood: -2322.758\n",
      "Optimal log-likelihood: -2322.758\n",
      "Rate parameters:  A-C: 2.11345  A-G: 3.07649  A-T: 1.00000  C-G: 2.11345  C-T: 6.84865  G-T: 1.00000\n",
      "Base frequencies:  A: 0.269  C: 0.253  G: 0.296  T: 0.182\n",
      "Parameters optimization took 1 rounds (0.005 sec)\n",
      "Wrote distance file to... \n",
      "Computing ML distances based on estimated model parameters...\n",
      "Calculating distance matrix: done in 0.00185613 secs using 568.4% CPU\n",
      "Computing ML distances took 0.021112 sec (of wall-clock time) 0.149855 sec (of CPU time)\n",
      "Setting up auxiliary I and S matrices: done in 0.00130606 secs using 180.4% CPU\n",
      "Computing RapidNJ tree took 0.003496 sec (of wall-clock time) 0.002431 sec (of CPU time)\n",
      "Log-likelihood of RapidNJ tree: -2322.758\n",
      "--------------------------------------------------------------------\n",
      "|             INITIALIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "Generating 99 parsimony trees... 0.009 second\n",
      "Computing log-likelihood of 2 initial trees ... 0.000 seconds\n",
      "Current best score: -2322.758\n",
      "\n",
      "Do NNI search on 3 best initial trees\n",
      "Finish initializing candidate tree set (3)\n",
      "Current best tree score: -2322.758 / CPU time: 0.022\n",
      "Number of iterations: 3\n",
      "--------------------------------------------------------------------\n",
      "|               OPTIMIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "Iteration 10 / LogL: -2322.769 / Time: 0h:0m:1s\n",
      "Iteration 20 / LogL: -2322.768 / Time: 0h:0m:1s\n",
      "Iteration 30 / LogL: -2322.768 / Time: 0h:0m:3s (0h:0m:5s left)\n",
      "Iteration 40 / LogL: -2322.773 / Time: 0h:0m:3s (0h:0m:4s left)\n",
      "Iteration 50 / LogL: -2322.769 / Time: 0h:0m:4s (0h:0m:3s left)\n",
      "Log-likelihood cutoff on original alignment: -2327.340\n",
      "Iteration 60 / LogL: -2322.758 / Time: 0h:0m:5s (0h:0m:2s left)\n",
      "Iteration 70 / LogL: -2322.768 / Time: 0h:0m:5s (0h:0m:2s left)\n",
      "Iteration 80 / LogL: -2322.781 / Time: 0h:0m:6s (0h:0m:1s left)\n",
      "Iteration 90 / LogL: -2322.773 / Time: 0h:0m:7s (0h:0m:0s left)\n",
      "Iteration 100 / LogL: -2322.758 / Time: 0h:0m:8s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -2327.340\n",
      "NOTE: Bootstrap correlation coefficient of split occurrence frequencies: 1.000\n",
      "TREE SEARCH COMPLETED AFTER 101 ITERATIONS / Time: 0h:0m:8s\n",
      "\n",
      "--------------------------------------------------------------------\n",
      "|                    FINALIZING TREE SEARCH                        |\n",
      "--------------------------------------------------------------------\n",
      "Performs final model parameters optimization\n",
      "Estimate model parameters (epsilon = 0.010)\n",
      "1. Initial log-likelihood: -2322.758\n",
      "Optimal log-likelihood: -2322.758\n",
      "Rate parameters:  A-C: 2.11203  A-G: 3.07664  A-T: 1.00000  C-G: 2.11203  C-T: 6.84997  G-T: 1.00000\n",
      "Base frequencies:  A: 0.269  C: 0.253  G: 0.296  T: 0.182\n",
      "Parameters optimization took 1 rounds (0.000 sec)\n",
      "BEST SCORE FOUND : -2322.758\n",
      "\n",
      "Testing tree branches by SH-like aLRT with 1000 replicates...\n",
      "0.027 sec.\n",
      "Creating bootstrap support values...\n",
      "Split supports printed to NEXUS file result/phylo_fna_gene/100341at50557_iqtreeout/.tree.splits.nex\n",
      "Total tree length: 0.354\n",
      "\n",
      "Total number of iterations: 101\n",
      "CPU time used for tree search: 49.262 sec (0h:0m:49s)\n",
      "Wall-clock time used for tree search: 7.673 sec (0h:0m:7s)\n",
      "Total CPU time used: 53.970 sec (0h:0m:53s)\n",
      "Total wall-clock time used: 8.475 sec (0h:0m:8s)\n",
      "\n",
      "Computing bootstrap consensus tree...\n",
      "Reading input file result/phylo_fna_gene/100341at50557_iqtreeout/.tree.splits.nex...\n",
      "4 taxa and 7 splits.\n",
      "Consensus tree written to result/phylo_fna_gene/100341at50557_iqtreeout/.tree.contree\n",
      "Reading input trees file result/phylo_fna_gene/100341at50557_iqtreeout/.tree.contree\n",
      "Log-likelihood of consensus tree: -2322.758\n",
      "\n",
      "Analysis results written to: \n",
      "  IQ-TREE report:                result/phylo_fna_gene/100341at50557_iqtreeout/.tree.iqtree\n",
      "  Maximum-likelihood tree:       result/phylo_fna_gene/100341at50557_iqtreeout/.tree.treefile\n",
      "  Likelihood distances:          result/phylo_fna_gene/100341at50557_iqtreeout/.tree.mldist\n",
      "\n",
      "Ultrafast bootstrap approximation results written to:\n",
      "  Split support values:          result/phylo_fna_gene/100341at50557_iqtreeout/.tree.splits.nex\n",
      "  Consensus tree:                result/phylo_fna_gene/100341at50557_iqtreeout/.tree.contree\n",
      "  Screen log file:               result/phylo_fna_gene/100341at50557_iqtreeout/.tree.log\n",
      "\n",
      "Date and Time: Sat Aug 31 21:10:55 2024\n",
      "IQ-TREE multicore version 2.3.4 COVID-edition for Linux x86 64-bit built Jun 18 2024\n",
      "Developed by Bui Quang Minh, James Barbetti, Nguyen Lam Tung, Olga Chernomor,\n",
      "Heiko Schmidt, Dominik Schrempf, Michael Woodhams, Ly Trong Nhan, Thomas Wong\n",
      "\n",
      "Host:    RCNSZOR36402A (AVX512, FMA3, 7 GB RAM)\n",
      "Command: iqtree -s result/phylo_fna_gene/100475at50557_AlnTrimed.fna -nt 8 -m MFP -bb 1000 -alrt 1000 -pre result/phylo_fna_gene/100475at50557_iqtreeout/.tree\n",
      "Seed:    56687 (Using SPRNG - Scalable Parallel Random Number Generator)\n",
      "Time:    Sat Aug 31 21:10:55 2024\n",
      "Kernel:  AVX+FMA - 8 threads (8 CPU cores detected)\n",
      "\n",
      "Reading alignment file result/phylo_fna_gene/100475at50557_AlnTrimed.fna ... Fasta format detected\n",
      "Reading fasta file: done in 8.8974e-05 secs\n",
      "Alignment most likely contains DNA/RNA sequences\n",
      "Alignment has 4 sequences with 993 columns, 39 distinct patterns\n",
      "8 parsimony-informative, 258 singleton sites, 727 constant sites\n",
      "                        Gap/Ambiguity  Composition  p-value\n",
      "Analyzing sequences: done in 6.34399e-06 secs\n",
      "   1  Culex_pipiens             0.00%    passed     75.16%\n",
      "   2  Armigeres_subalbatus      0.00%    failed      1.59%\n",
      "   3  Culex_pipiens_molestus    0.00%    passed     83.14%\n",
      "   4  Culex_pipiens_pallens     0.00%    passed     63.73%\n",
      "****  TOTAL                     0.00%  1 sequences failed composition chi2 test (p-value<5%; df=3)\n",
      "\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "\n",
      "Create initial parsimony tree by phylogenetic likelihood library (PLL)... 0.000 seconds\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "Perform fast likelihood tree search using GTR+I+G model...\n",
      "Estimate model parameters (epsilon = 5.000)\n",
      "Perform nearest neighbor interchange...\n",
      "Estimate model parameters (epsilon = 1.000)\n",
      "1. Initial log-likelihood: -2347.849\n",
      "Optimal log-likelihood: -2347.808\n",
      "Rate parameters:  A-C: 0.00010  A-G: 3.21021  A-T: 0.44318  C-G: 1.91714  C-T: 4.35706  G-T: 1.00000\n",
      "Warning! Some parameters hit the boundaries\n",
      "Base frequencies:  A: 0.261  C: 0.275  G: 0.296  T: 0.168\n",
      "Proportion of invariable sites: 0.365\n",
      "Gamma shape alpha: 1.155\n",
      "Parameters optimization took 1 rounds (0.002 sec)\n",
      "Time for fast ML tree search: 0.062 seconds\n",
      "\n",
      "NOTE: ModelFinder requires 0 MB RAM!\n",
      "ModelFinder will test up to 484 DNA models (sample size: 993) ...\n",
      " No. Model         -LnL         df  AIC          AICc         BIC\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  1  GTR+F         2357.481     13  4740.962     4741.334     4804.671\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  2  GTR+F+I       2346.102     14  4720.204     4720.634     4788.814\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  3  GTR+F+G4      2347.880     14  4723.761     4724.190     4792.371\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  4  GTR+F+I+G4    2347.848     15  4725.695     4726.187     4799.206\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  5  GTR+F+R2      2346.569     15  4723.138     4723.630     4796.649\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  6  GTR+F+R3      2346.698     17  4727.397     4728.024     4810.709\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 14  GTR+F+I+R2    2347.347     16  4726.695     4727.252     4805.106\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 15  GTR+F+I+R3    2346.748     18  4729.496     4730.198     4817.709\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 24  SYM+I         2367.451     11  4756.902     4757.171     4810.810\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 25  SYM+G4        2368.534     11  4759.068     4759.337     4812.976\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 27  SYM+R2        2367.644     12  4759.288     4759.606     4818.096\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 46  TVM+F+I       2347.366     13  4720.732     4721.104     4784.441\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 47  TVM+F+G4      2348.502     13  4723.005     4723.376     4786.714\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 49  TVM+F+R2      2347.664     14  4723.328     4723.757     4791.938\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 68  TVMe+I        2367.572     10  4755.144     4755.368     4804.151\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 69  TVMe+G4       2368.552     10  4757.104     4757.328     4806.112\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 71  TVMe+R2       2367.600     11  4757.200     4757.469     4811.108\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 90  TIM3+F+I      2357.381     12  4738.761     4739.079     4797.570\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 91  TIM3+F+G4     2358.771     12  4741.541     4741.860     4800.350\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 93  TIM3+F+R2     2359.524     13  4745.047     4745.419     4808.757\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "112  TIM3e+I       2379.659     9   4777.317     4777.500     4821.424\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "113  TIM3e+G4      2380.668     9   4779.336     4779.520     4823.443\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "115  TIM3e+R2      2381.580     10  4783.159     4783.383     4832.167\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "134  TIM2+F+I      2347.487     12  4718.974     4719.292     4777.782\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "135  TIM2+F+G4     2349.213     12  4722.426     4722.744     4781.234\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "137  TIM2+F+R2     2348.258     13  4722.515     4722.887     4786.225\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "156  TIM2e+I       2370.543     9   4759.086     4759.270     4803.193\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "157  TIM2e+G4      2371.635     9   4761.270     4761.453     4805.377\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "159  TIM2e+R2      2371.186     10  4762.372     4762.596     4811.379\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "178  TIM+F+I       2353.511     12  4731.022     4731.340     4789.831\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "179  TIM+F+G4      2355.812     12  4735.623     4735.942     4794.432\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "181  TIM+F+R2      2356.690     13  4739.379     4739.751     4803.089\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "200  TIMe+I        2378.927     9   4775.853     4776.036     4819.960\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "201  TIMe+G4       2380.144     9   4778.289     4778.472     4822.395\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "203  TIMe+R2       2381.051     10  4782.102     4782.326     4831.109\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "222  TPM3u+F+I     2358.540     11  4739.080     4739.349     4792.988\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "223  TPM3u+F+G4    2359.547     11  4741.093     4741.362     4795.001\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "225  TPM3u+F+R2    2360.341     12  4744.682     4745.000     4803.490\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "244  TPM3+I        2379.841     8   4775.682     4775.829     4814.888\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "245  TPM3+G4       2380.668     8   4777.335     4777.482     4816.541\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "247  TPM3+R2       2381.458     9   4780.916     4781.100     4825.023\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "266  TPM2u+F+I     2348.846     11  4719.691     4719.960     4773.599\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "267  TPM2u+F+G4    2349.814     11  4721.629     4721.898     4775.537\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "269  TPM2u+F+R2    2349.255     12  4722.509     4722.827     4781.318\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "288  TPM2+I        2370.583     8   4757.166     4757.312     4796.371\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "289  TPM2+G4       2371.726     8   4759.452     4759.598     4798.658\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "291  TPM2+R2       2371.149     9   4760.298     4760.481     4804.405\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "310  K3Pu+F+I      2355.209     11  4732.418     4732.688     4786.326\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "311  K3Pu+F+G4     2356.700     11  4735.400     4735.669     4789.308\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "313  K3Pu+F+R2     2357.932     12  4739.865     4740.183     4798.674\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "332  K3P+I         2378.941     8   4773.883     4774.029     4813.088\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "333  K3P+G4        2380.141     8   4776.281     4776.427     4815.487\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "335  K3P+R2        2381.486     9   4780.972     4781.155     4825.078\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "354  TN+F+I        2357.688     11  4737.377     4737.646     4791.285\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "355  TN+F+G4       2359.024     11  4740.048     4740.317     4793.956\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "357  TN+F+R2       2359.920     12  4743.839     4744.157     4802.648\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "376  TNe+I         2381.861     8   4779.721     4779.867     4818.927\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "377  TNe+G4        2382.780     8   4781.560     4781.706     4820.765\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "379  TNe+R2        2383.648     9   4785.296     4785.479     4829.402\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "398  HKY+F+I       2358.860     10  4737.720     4737.944     4786.728\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "399  HKY+F+G4      2359.822     10  4739.644     4739.869     4788.652\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "401  HKY+F+R2      2360.651     11  4743.302     4743.571     4797.210\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "420  K2P+I         2381.891     7   4777.781     4777.895     4812.086\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "421  K2P+G4        2382.797     7   4779.595     4779.708     4813.900\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "423  K2P+R2        2383.652     8   4783.304     4783.450     4822.510\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "442  F81+F+I       2387.857     9   4793.715     4793.898     4837.821\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "443  F81+F+G4      2388.013     9   4794.026     4794.210     4838.133\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "445  F81+F+R2      2389.580     10  4799.161     4799.385     4848.168\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "464  JC+I          2410.770     6   4833.539     4833.624     4862.944\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "465  JC+G4         2410.928     6   4833.856     4833.941     4863.260\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "467  JC+R2         2412.404     7   4838.807     4838.921     4873.112\n",
      "Akaike Information Criterion:           TIM2+F+I\n",
      "Corrected Akaike Information Criterion: TIM2+F+I\n",
      "Bayesian Information Criterion:         TPM2u+F+I\n",
      "Best-fit model: TPM2u+F+I chosen according to BIC\n",
      "\n",
      "All model information printed to result/phylo_fna_gene/100475at50557_iqtreeout/.tree.model.gz\n",
      "CPU time for ModelFinder: 5.775 seconds (0h:0m:5s)\n",
      "Wall-clock time for ModelFinder: 1.091 seconds (0h:0m:1s)\n",
      "Generating 1000 samples for ultrafast bootstrap (seed: 56687)...\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "\n",
      "NOTE: 0 MB RAM (0 GB) is required!\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "Estimate model parameters (epsilon = 0.100)\n",
      "1. Initial log-likelihood: -2348.846\n",
      "Optimal log-likelihood: -2348.837\n",
      "Rate parameters:  A-C: 0.06923  A-G: 2.40550  A-T: 0.06923  C-G: 1.00000  C-T: 2.40550  G-T: 1.00000\n",
      "Base frequencies:  A: 0.261  C: 0.275  G: 0.296  T: 0.168\n",
      "Proportion of invariable sites: 0.548\n",
      "Parameters optimization took 1 rounds (0.001 sec)\n",
      "Wrote distance file to... \n",
      "Computing ML distances based on estimated model parameters...\n",
      "Calculating distance matrix: done in 0.000116785 secs using 1.742e+04% CPU\n",
      "Computing ML distances took 0.031049 sec (of wall-clock time) 0.212359 sec (of CPU time)\n",
      "Setting up auxiliary I and S matrices: done in 0.00127859 secs using 170.2% CPU\n",
      "Computing RapidNJ tree took 0.002909 sec (of wall-clock time) 0.005141 sec (of CPU time)\n",
      "Log-likelihood of RapidNJ tree: -2348.835\n",
      "--------------------------------------------------------------------\n",
      "|             INITIALIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "Generating 99 parsimony trees... 0.008 second\n",
      "Computing log-likelihood of 2 initial trees ... 0.000 seconds\n",
      "Current best score: -2348.835\n",
      "\n",
      "Do NNI search on 3 best initial trees\n",
      "Finish initializing candidate tree set (3)\n",
      "Current best tree score: -2348.835 / CPU time: 0.218\n",
      "Number of iterations: 3\n",
      "--------------------------------------------------------------------\n",
      "|               OPTIMIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "Iteration 10 / LogL: -2348.868 / Time: 0h:0m:2s\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2348.834\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2348.834\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2348.834\n",
      "Iteration 20 / LogL: -2348.862 / Time: 0h:0m:3s\n",
      "Iteration 30 / LogL: -2348.862 / Time: 0h:0m:4s (0h:0m:7s left)\n",
      "Iteration 40 / LogL: -2348.862 / Time: 0h:0m:4s (0h:0m:5s left)\n",
      "Iteration 50 / LogL: -2348.868 / Time: 0h:0m:5s (0h:0m:4s left)\n",
      "Log-likelihood cutoff on original alignment: -2350.081\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2348.834\n",
      "Iteration 60 / LogL: -2348.862 / Time: 0h:0m:6s (0h:0m:3s left)\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2348.834\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2348.834\n",
      "Iteration 70 / LogL: -2348.863 / Time: 0h:0m:7s (0h:0m:2s left)\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2348.834\n",
      "Iteration 80 / LogL: -2348.867 / Time: 0h:0m:8s (0h:0m:1s left)\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2348.834\n",
      "Iteration 90 / LogL: -2348.834 / Time: 0h:0m:8s (0h:0m:0s left)\n",
      "Iteration 100 / LogL: -2348.863 / Time: 0h:0m:9s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -2350.081\n",
      "NOTE: Bootstrap correlation coefficient of split occurrence frequencies: 0.791\n",
      "NOTE: UFBoot does not converge, continue at least 100 more iterations\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2348.834\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2348.834\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2348.834\n",
      "Iteration 110 / LogL: -2348.863 / Time: 0h:0m:10s (0h:0m:7s left)\n",
      "Iteration 120 / LogL: -2348.868 / Time: 0h:0m:11s (0h:0m:6s left)\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2348.834\n",
      "Iteration 130 / LogL: -2348.868 / Time: 0h:0m:11s (0h:0m:5s left)\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2348.834\n",
      "Iteration 140 / LogL: -2348.862 / Time: 0h:0m:12s (0h:0m:4s left)\n",
      "Iteration 150 / LogL: -2348.863 / Time: 0h:0m:13s (0h:0m:4s left)\n",
      "Log-likelihood cutoff on original alignment: -2350.081\n",
      "Iteration 160 / LogL: -2348.863 / Time: 0h:0m:14s (0h:0m:3s left)\n",
      "Iteration 170 / LogL: -2348.863 / Time: 0h:0m:15s (0h:0m:2s left)\n",
      "Iteration 180 / LogL: -2348.868 / Time: 0h:0m:16s (0h:0m:1s left)\n",
      "Iteration 190 / LogL: -2348.862 / Time: 0h:0m:16s (0h:0m:0s left)\n",
      "Iteration 200 / LogL: -2348.867 / Time: 0h:0m:17s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -2350.081\n",
      "NOTE: Bootstrap correlation coefficient of split occurrence frequencies: 0.498\n",
      "NOTE: UFBoot does not converge, continue at least 100 more iterations\n",
      "Iteration 210 / LogL: -2348.863 / Time: 0h:0m:18s (0h:0m:7s left)\n",
      "Iteration 220 / LogL: -2348.862 / Time: 0h:0m:19s (0h:0m:6s left)\n",
      "Iteration 230 / LogL: -2348.867 / Time: 0h:0m:19s (0h:0m:5s left)\n",
      "Iteration 240 / LogL: -2348.863 / Time: 0h:0m:20s (0h:0m:4s left)\n",
      "Iteration 250 / LogL: -2348.868 / Time: 0h:0m:20s (0h:0m:3s left)\n",
      "Log-likelihood cutoff on original alignment: -2350.081\n",
      "Iteration 260 / LogL: -2348.867 / Time: 0h:0m:21s (0h:0m:3s left)\n",
      "Iteration 270 / LogL: -2348.834 / Time: 0h:0m:22s (0h:0m:2s left)\n",
      "Iteration 280 / LogL: -2348.862 / Time: 0h:0m:23s (0h:0m:1s left)\n",
      "Iteration 290 / LogL: -2348.867 / Time: 0h:0m:24s (0h:0m:0s left)\n",
      "Iteration 300 / LogL: -2348.862 / Time: 0h:0m:25s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -2350.081\n",
      "NOTE: Bootstrap correlation coefficient of split occurrence frequencies: 0.957\n",
      "NOTE: UFBoot does not converge, continue at least 100 more iterations\n",
      "Iteration 310 / LogL: -2348.867 / Time: 0h:0m:25s (0h:0m:7s left)\n",
      "Iteration 320 / LogL: -2348.868 / Time: 0h:0m:26s (0h:0m:6s left)\n",
      "Iteration 330 / LogL: -2348.868 / Time: 0h:0m:27s (0h:0m:5s left)\n",
      "Iteration 340 / LogL: -2348.862 / Time: 0h:0m:28s (0h:0m:4s left)\n",
      "Iteration 350 / LogL: -2348.862 / Time: 0h:0m:28s (0h:0m:3s left)\n",
      "Log-likelihood cutoff on original alignment: -2350.081\n",
      "Iteration 360 / LogL: -2348.862 / Time: 0h:0m:29s (0h:0m:3s left)\n",
      "Iteration 370 / LogL: -2348.862 / Time: 0h:0m:30s (0h:0m:2s left)\n",
      "Iteration 380 / LogL: -2348.868 / Time: 0h:0m:30s (0h:0m:1s left)\n",
      "Iteration 390 / LogL: -2348.868 / Time: 0h:0m:31s (0h:0m:0s left)\n",
      "Iteration 400 / LogL: -2348.834 / Time: 0h:0m:32s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -2350.081\n",
      "NOTE: Bootstrap correlation coefficient of split occurrence frequencies: 1.000\n",
      "TREE SEARCH COMPLETED AFTER 400 ITERATIONS / Time: 0h:0m:32s\n",
      "\n",
      "--------------------------------------------------------------------\n",
      "|                    FINALIZING TREE SEARCH                        |\n",
      "--------------------------------------------------------------------\n",
      "Performs final model parameters optimization\n",
      "Estimate model parameters (epsilon = 0.010)\n",
      "1. Initial log-likelihood: -2348.834\n",
      "Optimal log-likelihood: -2348.832\n",
      "Rate parameters:  A-C: 0.07192  A-G: 2.40512  A-T: 0.07192  C-G: 1.00000  C-T: 2.40512  G-T: 1.00000\n",
      "Base frequencies:  A: 0.261  C: 0.275  G: 0.296  T: 0.168\n",
      "Proportion of invariable sites: 0.547\n",
      "Parameters optimization took 1 rounds (0.002 sec)\n",
      "BEST SCORE FOUND : -2348.832\n",
      "\n",
      "Testing tree branches by SH-like aLRT with 1000 replicates...\n",
      "0.017 sec.\n",
      "Creating bootstrap support values...\n",
      "Split supports printed to NEXUS file result/phylo_fna_gene/100475at50557_iqtreeout/.tree.splits.nex\n",
      "Total tree length: 0.610\n",
      "\n",
      "Total number of iterations: 400\n",
      "CPU time used for tree search: 201.955 sec (0h:3m:21s)\n",
      "Wall-clock time used for tree search: 31.761 sec (0h:0m:31s)\n",
      "Total CPU time used: 208.388 sec (0h:3m:28s)\n",
      "Total wall-clock time used: 32.950 sec (0h:0m:32s)\n",
      "\n",
      "Computing bootstrap consensus tree...\n",
      "Reading input file result/phylo_fna_gene/100475at50557_iqtreeout/.tree.splits.nex...\n",
      "4 taxa and 7 splits.\n",
      "Consensus tree written to result/phylo_fna_gene/100475at50557_iqtreeout/.tree.contree\n",
      "Reading input trees file result/phylo_fna_gene/100475at50557_iqtreeout/.tree.contree\n",
      "Log-likelihood of consensus tree: -2348.833\n",
      "\n",
      "Analysis results written to: \n",
      "  IQ-TREE report:                result/phylo_fna_gene/100475at50557_iqtreeout/.tree.iqtree\n",
      "  Maximum-likelihood tree:       result/phylo_fna_gene/100475at50557_iqtreeout/.tree.treefile\n",
      "  Likelihood distances:          result/phylo_fna_gene/100475at50557_iqtreeout/.tree.mldist\n",
      "\n",
      "Ultrafast bootstrap approximation results written to:\n",
      "  Split support values:          result/phylo_fna_gene/100475at50557_iqtreeout/.tree.splits.nex\n",
      "  Consensus tree:                result/phylo_fna_gene/100475at50557_iqtreeout/.tree.contree\n",
      "  Screen log file:               result/phylo_fna_gene/100475at50557_iqtreeout/.tree.log\n",
      "\n",
      "Date and Time: Sat Aug 31 21:11:28 2024\n",
      "IQ-TREE multicore version 2.3.4 COVID-edition for Linux x86 64-bit built Jun 18 2024\n",
      "Developed by Bui Quang Minh, James Barbetti, Nguyen Lam Tung, Olga Chernomor,\n",
      "Heiko Schmidt, Dominik Schrempf, Michael Woodhams, Ly Trong Nhan, Thomas Wong\n",
      "\n",
      "Host:    RCNSZOR36402A (AVX512, FMA3, 7 GB RAM)\n",
      "Command: iqtree -s result/phylo_fna_gene/100579at50557_AlnTrimed.fna -nt 8 -m MFP -bb 1000 -alrt 1000 -pre result/phylo_fna_gene/100579at50557_iqtreeout/.tree\n",
      "Seed:    28836 (Using SPRNG - Scalable Parallel Random Number Generator)\n",
      "Time:    Sat Aug 31 21:11:28 2024\n",
      "Kernel:  AVX+FMA - 8 threads (8 CPU cores detected)\n",
      "\n",
      "Reading alignment file result/phylo_fna_gene/100579at50557_AlnTrimed.fna ... Fasta format detected\n",
      "Reading fasta file: done in 8.0773e-05 secs using 97.8% CPU\n",
      "Alignment most likely contains DNA/RNA sequences\n",
      "Alignment has 4 sequences with 795 columns, 25 distinct patterns\n",
      "6 parsimony-informative, 158 singleton sites, 631 constant sites\n",
      "                        Gap/Ambiguity  Composition  p-value\n",
      "Analyzing sequences: done in 8.829e-06 secs using 90.61% CPU\n",
      "   1  Culex_pipiens             0.00%    passed     97.19%\n",
      "   2  Armigeres_subalbatus      0.00%    passed     53.07%\n",
      "   3  Culex_pipiens_molestus    0.00%    passed     91.94%\n",
      "   4  Culex_pipiens_pallens     0.00%    passed     97.12%\n",
      "****  TOTAL                     0.00%  0 sequences failed composition chi2 test (p-value<5%; df=3)\n",
      "\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "\n",
      "Create initial parsimony tree by phylogenetic likelihood library (PLL)... 0.000 seconds\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "Perform fast likelihood tree search using GTR+I+G model...\n",
      "Estimate model parameters (epsilon = 5.000)\n",
      "Perform nearest neighbor interchange...\n",
      "Estimate model parameters (epsilon = 1.000)\n",
      "1. Initial log-likelihood: -1718.932\n",
      "Optimal log-likelihood: -1718.924\n",
      "Rate parameters:  A-C: 2.09516  A-G: 9.76604  A-T: 0.78342  C-G: 2.58801  C-T: 9.64784  G-T: 1.00000\n",
      "Base frequencies:  A: 0.266  C: 0.242  G: 0.252  T: 0.241\n",
      "Proportion of invariable sites: 0.396\n",
      "Gamma shape alpha: 1.408\n",
      "Parameters optimization took 1 rounds (0.063 sec)\n",
      "Time for fast ML tree search: 0.137 seconds\n",
      "\n",
      "NOTE: ModelFinder requires 0 MB RAM!\n",
      "ModelFinder will test up to 484 DNA models (sample size: 795) ...\n",
      " No. Model         -LnL         df  AIC          AICc         BIC\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  1  GTR+F         1720.652     13  3467.304     3467.770     3528.123\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  2  GTR+F+I       1719.049     14  3466.097     3466.636     3531.594\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  3  GTR+F+G4      1719.187     14  3466.373     3466.911     3531.870\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  4  GTR+F+I+G4    1718.962     15  3467.923     3468.539     3538.098\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  5  GTR+F+R2      1719.613     15  3469.227     3469.843     3539.402\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  6  GTR+F+R3      1719.404     17  3472.808     3473.595     3552.339\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 14  GTR+F+I+R2    1718.875     16  3469.749     3470.448     3544.603\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 15  GTR+F+I+R3    1718.909     18  3473.817     3474.699     3558.027\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 23  SYM           1721.659     10  3463.318     3463.599     3510.102\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 24  SYM+I         1719.967     11  3461.934     3462.271     3513.396\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 25  SYM+G4        1720.174     11  3462.348     3462.685     3513.809\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 26  SYM+I+G4      1719.950     12  3463.899     3464.298     3520.039\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 45  TVM+F         1720.701     12  3465.401     3465.800     3521.542\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 46  TVM+F+I       1719.031     13  3464.062     3464.528     3524.881\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 47  TVM+F+G4      1719.189     13  3464.378     3464.844     3525.197\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 48  TVM+F+I+G4    1718.969     14  3465.938     3466.477     3531.435\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 67  TVMe          1721.658     9   3461.316     3461.545     3503.421\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 68  TVMe+I        1720.100     10  3460.200     3460.480     3506.983\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 69  TVMe+G4       1720.209     10  3460.418     3460.698     3507.201\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 70  TVMe+I+G4     1719.998     11  3461.997     3462.334     3513.458\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 89  TIM3+F        1720.956     11  3463.912     3464.250     3515.374\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 90  TIM3+F+I      1719.107     12  3462.213     3462.612     3518.354\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 91  TIM3+F+G4     1719.276     12  3462.552     3462.951     3518.692\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 92  TIM3+F+I+G4   1719.101     13  3464.202     3464.668     3525.021\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "111  TIM3e         1721.912     8   3459.823     3460.006     3497.250\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "112  TIM3e+I       1720.037     9   3458.074     3458.303     3500.179\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "113  TIM3e+G4      1720.225     9   3458.450     3458.679     3500.555\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "114  TIM3e+I+G4    1720.047     10  3460.094     3460.375     3506.877\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "133  TIM2+F        1722.687     11  3467.374     3467.711     3518.836\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "134  TIM2+F+I      1720.917     12  3465.834     3466.233     3521.974\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "135  TIM2+F+G4     1721.178     12  3466.356     3466.755     3522.496\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "136  TIM2+F+I+G4   1720.950     13  3467.899     3468.365     3528.718\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "155  TIM2e         1723.695     8   3463.391     3463.574     3500.817\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "156  TIM2e+I       1721.904     9   3461.808     3462.037     3503.913\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "157  TIM2e+G4      1722.177     9   3462.353     3462.583     3504.458\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "158  TIM2e+I+G4    1721.937     10  3463.874     3464.155     3510.658\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "177  TIM+F         1722.829     11  3467.658     3467.995     3519.120\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "178  TIM+F+I       1721.051     12  3466.102     3466.501     3522.242\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "179  TIM+F+G4      1721.312     12  3466.625     3467.024     3522.765\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "180  TIM+F+I+G4    1721.074     13  3468.147     3468.613     3528.965\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "199  TIMe          1723.794     8   3463.588     3463.771     3501.015\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "200  TIMe+I        1721.993     9   3461.985     3462.215     3504.090\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "201  TIMe+G4       1722.269     9   3462.537     3462.767     3504.642\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "202  TIMe+I+G4     1722.025     10  3464.050     3464.331     3510.834\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "221  TPM3u+F       1721.005     10  3462.009     3462.290     3508.793\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "222  TPM3u+F+I     1719.110     11  3460.221     3460.558     3511.683\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "223  TPM3u+F+G4    1719.271     11  3460.542     3460.879     3512.004\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "224  TPM3u+F+I+G4  1719.112     12  3462.224     3462.623     3518.364\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "243  TPM3          1721.912     7   3457.823     3457.966     3490.572\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "244  TPM3+I        1720.124     8   3456.249     3456.432     3493.676\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "245  TPM3+G4       1720.318     8   3456.636     3456.820     3494.063\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "246  TPM3+I+G4     1720.106     9   3458.212     3458.442     3500.317\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "265  TPM2u+F       1722.733     10  3465.466     3465.746     3512.249\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "266  TPM2u+F+I     1720.914     11  3463.827     3464.164     3515.289\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "267  TPM2u+F+G4    1721.175     11  3464.350     3464.687     3515.812\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "268  TPM2u+F+I+G4  1720.950     12  3465.901     3466.300     3522.041\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "287  TPM2          1723.696     7   3461.391     3461.534     3494.140\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "288  TPM2+I        1722.032     8   3460.064     3460.247     3497.491\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "289  TPM2+G4       1722.197     8   3460.395     3460.578     3497.821\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "290  TPM2+I+G4     1721.987     9   3461.975     3462.204     3504.080\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "309  K3Pu+F        1722.875     10  3465.750     3466.031     3512.534\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "310  K3Pu+F+I      1721.038     11  3464.076     3464.413     3515.538\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "311  K3Pu+F+G4     1721.299     11  3464.597     3464.935     3516.059\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "312  K3Pu+F+I+G4   1721.071     12  3466.142     3466.541     3522.282\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "331  K3P           1723.794     7   3461.588     3461.730     3494.336\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "332  K3P+I         1722.121     8   3460.243     3460.426     3497.670\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "333  K3P+G4        1722.285     8   3460.570     3460.753     3497.996\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "334  K3P+I+G4      1722.074     9   3462.147     3462.377     3504.253\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "353  TN+F          1722.892     10  3465.783     3466.064     3512.567\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "354  TN+F+I        1721.045     11  3464.089     3464.426     3515.551\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "355  TN+F+G4       1721.256     11  3464.513     3464.850     3515.974\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "356  TN+F+I+G4     1721.087     12  3466.173     3466.572     3522.313\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "375  TNe           1723.851     7   3461.702     3461.844     3494.450\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "376  TNe+I         1721.976     8   3459.952     3460.136     3497.379\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "377  TNe+G4        1722.206     8   3460.411     3460.595     3497.838\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "378  TNe+I+G4      1722.032     9   3462.065     3462.294     3504.170\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "397  HKY+F         1722.937     9   3463.875     3464.104     3505.980\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "398  HKY+F+I       1721.037     10  3462.075     3462.355     3508.858\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "399  HKY+F+G4      1721.245     10  3462.491     3462.771     3509.274\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "400  HKY+F+I+G4    1721.087     11  3464.175     3464.512     3515.636\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "419  K2P           1723.851     6   3459.703     3459.809     3487.773\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "420  K2P+I         1722.053     7   3458.107     3458.249     3490.855\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "421  K2P+G4        1722.294     7   3458.588     3458.730     3491.336\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "422  K2P+I+G4      1722.084     8   3460.169     3460.352     3497.595\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "441  F81+F         1759.440     8   3534.881     3535.064     3572.307\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "442  F81+F+I       1758.288     9   3534.576     3534.805     3576.681\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "443  F81+F+G4      1758.312     9   3534.625     3534.854     3576.730\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "444  F81+F+I+G4    1758.224     10  3536.448     3536.729     3583.232\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "463  JC            1760.467     5   3530.934     3531.010     3554.326\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "464  JC+I          1759.349     6   3530.698     3530.805     3558.768\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "465  JC+G4         1759.372     6   3530.743     3530.850     3558.813\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "466  JC+I+G4       1759.298     7   3532.595     3532.738     3565.344\n",
      "Akaike Information Criterion:           TPM3+I\n",
      "Corrected Akaike Information Criterion: TPM3+I\n",
      "Bayesian Information Criterion:         K2P\n",
      "Best-fit model: K2P chosen according to BIC\n",
      "\n",
      "All model information printed to result/phylo_fna_gene/100579at50557_iqtreeout/.tree.model.gz\n",
      "CPU time for ModelFinder: 4.407 seconds (0h:0m:4s)\n",
      "Wall-clock time for ModelFinder: 0.819 seconds (0h:0m:0s)\n",
      "Generating 1000 samples for ultrafast bootstrap (seed: 28836)...\n",
      "WARNING: 8 threads for alignment length 25 will slow down analysis\n",
      "\n",
      "NOTE: 0 MB RAM (0 GB) is required!\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "Estimate model parameters (epsilon = 0.100)\n",
      "1. Initial log-likelihood: -1723.851\n",
      "Optimal log-likelihood: -1723.851\n",
      "Rate parameters:  A-C: 1.00000  A-G: 4.38052  A-T: 1.00000  C-G: 1.00000  C-T: 4.38052  G-T: 1.00000\n",
      "Base frequencies:  A: 0.250  C: 0.250  G: 0.250  T: 0.250\n",
      "Parameters optimization took 1 rounds (0.000 sec)\n",
      "Wrote distance file to... \n",
      "Computing ML distances based on estimated model parameters...\n",
      "Calculating distance matrix: done in 0.0127122 secs using 916% CPU\n",
      "Computing ML distances took 0.018513 sec (of wall-clock time) 0.132253 sec (of CPU time)\n",
      "Setting up auxiliary I and S matrices: done in 0.00160482 secs using 179.3% CPU\n",
      "Computing RapidNJ tree took 0.003428 sec (of wall-clock time) 0.006151 sec (of CPU time)\n",
      "Log-likelihood of RapidNJ tree: -1723.851\n",
      "--------------------------------------------------------------------\n",
      "|             INITIALIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "Generating 99 parsimony trees... 0.005 second\n",
      "Computing log-likelihood of 2 initial trees ... 0.000 seconds\n",
      "Current best score: -1723.851\n",
      "\n",
      "Do NNI search on 3 best initial trees\n",
      "UPDATE BEST LOG-LIKELIHOOD: -1723.851\n",
      "Finish initializing candidate tree set (3)\n",
      "Current best tree score: -1723.851 / CPU time: 0.265\n",
      "Number of iterations: 3\n",
      "--------------------------------------------------------------------\n",
      "|               OPTIMIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "Iteration 10 / LogL: -1723.860 / Time: 0h:0m:1s\n",
      "Iteration 20 / LogL: -1723.860 / Time: 0h:0m:2s\n",
      "Iteration 30 / LogL: -1723.860 / Time: 0h:0m:3s (0h:0m:5s left)\n",
      "Iteration 40 / LogL: -1723.860 / Time: 0h:0m:4s (0h:0m:5s left)\n",
      "Iteration 50 / LogL: -1723.860 / Time: 0h:0m:4s (0h:0m:4s left)\n",
      "Log-likelihood cutoff on original alignment: -1729.985\n",
      "Iteration 60 / LogL: -1723.851 / Time: 0h:0m:5s (0h:0m:3s left)\n",
      "Iteration 70 / LogL: -1723.860 / Time: 0h:0m:6s (0h:0m:2s left)\n",
      "Iteration 80 / LogL: -1723.860 / Time: 0h:0m:6s (0h:0m:1s left)\n",
      "Iteration 90 / LogL: -1723.860 / Time: 0h:0m:7s (0h:0m:0s left)\n",
      "Iteration 100 / LogL: -1723.851 / Time: 0h:0m:8s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -1729.985\n",
      "NOTE: Bootstrap correlation coefficient of split occurrence frequencies: 1.000\n",
      "TREE SEARCH COMPLETED AFTER 101 ITERATIONS / Time: 0h:0m:8s\n",
      "\n",
      "--------------------------------------------------------------------\n",
      "|                    FINALIZING TREE SEARCH                        |\n",
      "--------------------------------------------------------------------\n",
      "Performs final model parameters optimization\n",
      "Estimate model parameters (epsilon = 0.010)\n",
      "1. Initial log-likelihood: -1723.851\n",
      "Optimal log-likelihood: -1723.851\n",
      "Rate parameters:  A-C: 1.00000  A-G: 4.38052  A-T: 1.00000  C-G: 1.00000  C-T: 4.38052  G-T: 1.00000\n",
      "Base frequencies:  A: 0.250  C: 0.250  G: 0.250  T: 0.250\n",
      "Parameters optimization took 1 rounds (0.003 sec)\n",
      "BEST SCORE FOUND : -1723.851\n",
      "\n",
      "Testing tree branches by SH-like aLRT with 1000 replicates...\n",
      "0.025 sec.\n",
      "Creating bootstrap support values...\n",
      "Split supports printed to NEXUS file result/phylo_fna_gene/100579at50557_iqtreeout/.tree.splits.nex\n",
      "Total tree length: 0.249\n",
      "\n",
      "Total number of iterations: 101\n",
      "CPU time used for tree search: 49.142 sec (0h:0m:49s)\n",
      "Wall-clock time used for tree search: 8.053 sec (0h:0m:8s)\n",
      "Total CPU time used: 54.154 sec (0h:0m:54s)\n",
      "Total wall-clock time used: 8.972 sec (0h:0m:8s)\n",
      "\n",
      "Computing bootstrap consensus tree...\n",
      "Reading input file result/phylo_fna_gene/100579at50557_iqtreeout/.tree.splits.nex...\n",
      "4 taxa and 7 splits.\n",
      "Consensus tree written to result/phylo_fna_gene/100579at50557_iqtreeout/.tree.contree\n",
      "Reading input trees file result/phylo_fna_gene/100579at50557_iqtreeout/.tree.contree\n",
      "Log-likelihood of consensus tree: -1723.851\n",
      "\n",
      "Analysis results written to: \n",
      "  IQ-TREE report:                result/phylo_fna_gene/100579at50557_iqtreeout/.tree.iqtree\n",
      "  Maximum-likelihood tree:       result/phylo_fna_gene/100579at50557_iqtreeout/.tree.treefile\n",
      "  Likelihood distances:          result/phylo_fna_gene/100579at50557_iqtreeout/.tree.mldist\n",
      "\n",
      "Ultrafast bootstrap approximation results written to:\n",
      "  Split support values:          result/phylo_fna_gene/100579at50557_iqtreeout/.tree.splits.nex\n",
      "  Consensus tree:                result/phylo_fna_gene/100579at50557_iqtreeout/.tree.contree\n",
      "  Screen log file:               result/phylo_fna_gene/100579at50557_iqtreeout/.tree.log\n",
      "\n",
      "Date and Time: Sat Aug 31 21:11:37 2024\n",
      "IQ-TREE multicore version 2.3.4 COVID-edition for Linux x86 64-bit built Jun 18 2024\n",
      "Developed by Bui Quang Minh, James Barbetti, Nguyen Lam Tung, Olga Chernomor,\n",
      "Heiko Schmidt, Dominik Schrempf, Michael Woodhams, Ly Trong Nhan, Thomas Wong\n",
      "\n",
      "Host:    RCNSZOR36402A (AVX512, FMA3, 7 GB RAM)\n",
      "Command: iqtree -s result/phylo_fna_gene/100764at50557_AlnTrimed.fna -nt 8 -m MFP -bb 1000 -alrt 1000 -pre result/phylo_fna_gene/100764at50557_iqtreeout/.tree\n",
      "Seed:    23807 (Using SPRNG - Scalable Parallel Random Number Generator)\n",
      "Time:    Sat Aug 31 21:11:37 2024\n",
      "Kernel:  AVX+FMA - 8 threads (8 CPU cores detected)\n",
      "\n",
      "Reading alignment file result/phylo_fna_gene/100764at50557_AlnTrimed.fna ... Fasta format detected\n",
      "Reading fasta file: done in 0.000104209 secs\n",
      "Alignment most likely contains DNA/RNA sequences\n",
      "Alignment has 4 sequences with 810 columns, 35 distinct patterns\n",
      "10 parsimony-informative, 117 singleton sites, 683 constant sites\n",
      "                        Gap/Ambiguity  Composition  p-value\n",
      "Analyzing sequences: done in 0.0030321 secs using 83.84% CPU\n",
      "   1  Culex_pipiens             0.00%    passed     86.98%\n",
      "   2  Armigeres_subalbatus      0.00%    passed     11.10%\n",
      "   3  Culex_pipiens_molestus    0.00%    passed     84.87%\n",
      "   4  Culex_pipiens_pallens     0.00%    passed     91.04%\n",
      "****  TOTAL                     0.00%  0 sequences failed composition chi2 test (p-value<5%; df=3)\n",
      "\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "\n",
      "Create initial parsimony tree by phylogenetic likelihood library (PLL)... 0.000 seconds\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "Perform fast likelihood tree search using GTR+I+G model...\n",
      "Estimate model parameters (epsilon = 5.000)\n",
      "Perform nearest neighbor interchange...\n",
      "Estimate model parameters (epsilon = 1.000)\n",
      "1. Initial log-likelihood: -1685.699\n",
      "Optimal log-likelihood: -1685.680\n",
      "Rate parameters:  A-C: 0.43224  A-G: 2.49059  A-T: 0.00010  C-G: 1.78457  C-T: 2.38065  G-T: 1.00000\n",
      "Warning! Some parameters hit the boundaries\n",
      "Base frequencies:  A: 0.180  C: 0.285  G: 0.308  T: 0.227\n",
      "Proportion of invariable sites: 0.422\n",
      "Gamma shape alpha: 1.111\n",
      "Parameters optimization took 1 rounds (0.004 sec)\n",
      "Time for fast ML tree search: 0.099 seconds\n",
      "\n",
      "NOTE: ModelFinder requires 0 MB RAM!\n",
      "ModelFinder will test up to 484 DNA models (sample size: 810) ...\n",
      " No. Model         -LnL         df  AIC          AICc         BIC\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  1  GTR+F         1689.239     13  3404.477     3404.934     3465.539\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  2  GTR+F+I       1685.611     14  3399.221     3399.750     3464.980\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  3  GTR+F+G4      1685.650     14  3399.299     3399.828     3465.058\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  4  GTR+F+I+G4    1685.653     15  3401.307     3401.911     3471.762\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  5  GTR+F+R2      1686.832     15  3403.665     3404.269     3474.120\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  6  GTR+F+R3      1685.901     17  3405.803     3406.575     3485.652\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 14  GTR+F+I+R2    1685.688     16  3403.375     3404.061     3478.528\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 15  GTR+F+I+R3    1685.646     18  3407.292     3408.157     3491.839\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 23  SYM           1703.012     10  3426.024     3426.299     3472.994\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 24  SYM+I         1700.101     11  3422.203     3422.534     3473.870\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 25  SYM+G4        1700.195     11  3422.389     3422.720     3474.057\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 26  SYM+I+G4      1700.181     12  3424.362     3424.754     3480.727\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 27  SYM+R2        1701.181     12  3426.362     3426.753     3482.726\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 45  TVM+F         1689.286     12  3402.573     3402.964     3458.937\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 46  TVM+F+I       1685.650     13  3397.301     3397.758     3458.362\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 47  TVM+F+G4      1685.660     13  3397.319     3397.776     3458.381\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 48  TVM+F+I+G4    1685.681     14  3399.363     3399.891     3465.121\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 49  TVM+F+R2      1686.872     14  3401.743     3402.271     3467.501\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 67  TVMe          1703.019     9   3424.039     3424.264     3466.312\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 68  TVMe+I        1700.118     10  3420.236     3420.512     3467.207\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 69  TVMe+G4       1700.216     10  3420.431     3420.707     3467.402\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 70  TVMe+I+G4     1700.197     11  3422.395     3422.726     3474.062\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 71  TVMe+R2       1701.191     11  3424.381     3424.712     3476.049\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 89  TIM3+F        1697.671     11  3417.342     3417.673     3469.010\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 90  TIM3+F+I      1693.090     12  3410.179     3410.571     3466.544\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 91  TIM3+F+G4     1693.120     12  3410.240     3410.631     3466.604\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 92  TIM3+F+I+G4   1693.131     13  3412.263     3412.720     3473.324\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 93  TIM3+F+R2     1694.620     13  3415.240     3415.697     3476.301\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "111  TIM3e         1715.968     8   3447.935     3448.115     3485.512\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "112  TIM3e+I       1712.407     9   3442.813     3443.038     3485.086\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "113  TIM3e+G4      1712.450     9   3442.899     3443.124     3485.172\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "114  TIM3e+I+G4    1712.418     10  3444.836     3445.111     3491.806\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "115  TIM3e+R2      1713.730     10  3447.459     3447.735     3494.430\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "133  TIM2+F        1692.216     11  3406.432     3406.763     3458.099\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "134  TIM2+F+I      1688.515     12  3401.030     3401.421     3457.394\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "135  TIM2+F+G4     1688.537     12  3401.074     3401.466     3457.439\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "136  TIM2+F+I+G4   1688.536     13  3403.072     3403.529     3464.133\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "137  TIM2+F+R2     1689.786     13  3405.572     3406.029     3466.634\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "155  TIM2e         1707.094     8   3430.188     3430.368     3467.764\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "156  TIM2e+I       1704.214     9   3426.427     3426.652     3468.701\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "157  TIM2e+G4      1704.257     9   3426.513     3426.738     3468.787\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "158  TIM2e+I+G4    1704.242     10  3428.483     3428.759     3475.453\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "159  TIM2e+R2      1705.254     10  3430.508     3430.783     3477.478\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "177  TIM+F         1699.064     11  3420.127     3420.458     3471.794\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "178  TIM+F+I       1694.193     12  3412.385     3412.777     3468.750\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "179  TIM+F+G4      1694.213     12  3412.426     3412.818     3468.791\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "180  TIM+F+I+G4    1694.347     13  3414.693     3415.150     3475.754\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "181  TIM+F+R2      1695.805     13  3417.610     3418.067     3478.671\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "199  TIMe          1718.101     8   3452.201     3452.381     3489.777\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "200  TIMe+I        1714.303     9   3446.606     3446.831     3488.880\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "201  TIMe+G4       1714.402     9   3446.803     3447.028     3489.076\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "202  TIMe+I+G4     1714.409     10  3448.817     3449.092     3495.787\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "203  TIMe+R2       1715.693     10  3451.387     3451.662     3498.357\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "221  TPM3u+F       1697.703     10  3415.405     3415.681     3462.376\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "222  TPM3u+F+I     1693.067     11  3408.134     3408.465     3459.801\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "223  TPM3u+F+G4    1693.076     11  3408.153     3408.484     3459.820\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "224  TPM3u+F+I+G4  1693.139     12  3410.277     3410.669     3466.642\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "225  TPM3u+F+R2    1694.643     12  3413.285     3413.677     3469.650\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "243  TPM3          1715.977     7   3445.953     3446.093     3478.832\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "244  TPM3+I        1712.455     8   3440.910     3441.089     3478.486\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "245  TPM3+G4       1712.485     8   3440.970     3441.150     3478.546\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "246  TPM3+I+G4     1712.470     9   3442.941     3443.166     3485.214\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "247  TPM3+R2       1713.746     9   3445.491     3445.716     3487.765\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "265  TPM2u+F       1692.269     10  3404.537     3404.813     3451.508\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "266  TPM2u+F+I     1688.517     11  3399.034     3399.365     3450.702\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "267  TPM2u+F+G4    1688.541     11  3399.082     3399.413     3450.750\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "268  TPM2u+F+I+G4  1688.546     12  3401.092     3401.484     3457.457\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "269  TPM2u+F+R2    1689.734     12  3403.468     3403.859     3459.832\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "287  TPM2          1707.101     7   3428.201     3428.341     3461.080\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "288  TPM2+I        1704.230     8   3424.460     3424.640     3462.037\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "289  TPM2+G4       1704.268     8   3424.535     3424.715     3462.111\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "290  TPM2+I+G4     1704.256     9   3426.512     3426.737     3468.785\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "291  TPM2+R2       1705.260     9   3428.521     3428.746     3470.794\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "309  K3Pu+F        1699.101     10  3418.203     3418.478     3465.173\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "310  K3Pu+F+I      1694.172     11  3410.345     3410.676     3462.012\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "311  K3Pu+F+G4     1694.226     11  3410.453     3410.784     3462.120\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "312  K3Pu+F+I+G4   1694.350     12  3412.700     3413.092     3469.065\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "313  K3Pu+F+R2     1695.837     12  3415.674     3416.065     3472.038\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "331  K3P           1718.108     7   3450.216     3450.356     3483.095\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "332  K3P+I         1714.339     8   3444.679     3444.859     3482.255\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "333  K3P+G4        1714.430     8   3444.860     3445.040     3482.436\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "334  K3P+I+G4      1714.445     9   3446.889     3447.114     3489.163\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "335  K3P+R2        1715.704     9   3449.408     3449.633     3491.682\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "353  TN+F          1700.321     10  3420.642     3420.917     3467.612\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "354  TN+F+I        1695.440     11  3412.880     3413.211     3464.548\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "355  TN+F+G4       1695.411     11  3412.822     3413.152     3464.489\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "356  TN+F+I+G4     1695.414     12  3414.829     3415.220     3471.193\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "357  TN+F+R2       1697.062     12  3418.124     3418.516     3474.489\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "375  TNe           1719.644     7   3453.287     3453.427     3486.166\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "376  TNe+I         1715.917     8   3447.833     3448.013     3485.409\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "377  TNe+G4        1715.912     8   3447.823     3448.003     3485.400\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "378  TNe+I+G4      1715.876     9   3449.752     3449.977     3492.025\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "379  TNe+R2        1717.248     9   3452.495     3452.720     3494.769\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "397  HKY+F         1700.358     9   3418.716     3418.941     3460.990\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "398  HKY+F+I       1695.448     10  3410.896     3411.171     3457.866\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "399  HKY+F+G4      1695.414     10  3410.829     3411.104     3457.799\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "400  HKY+F+I+G4    1695.419     11  3412.838     3413.169     3464.505\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "401  HKY+F+R2      1697.092     11  3416.185     3416.516     3467.852\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "419  K2P           1719.651     6   3451.302     3451.407     3479.485\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "420  K2P+I         1715.958     7   3445.916     3446.056     3478.795\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "421  K2P+G4        1715.951     7   3445.902     3446.041     3478.781\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "422  K2P+I+G4      1715.918     8   3447.837     3448.017     3485.413\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "423  K2P+R2        1717.262     8   3450.525     3450.705     3488.101\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "441  F81+F         1709.185     8   3434.369     3434.549     3471.946\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "442  F81+F+I       1705.363     9   3428.725     3428.950     3470.999\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "443  F81+F+G4      1705.383     9   3428.766     3428.991     3471.039\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "444  F81+F+I+G4    1705.401     10  3430.801     3431.076     3477.771\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "445  F81+F+R2      1706.658     10  3433.317     3433.592     3480.287\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "463  JC            1727.750     5   3465.501     3465.575     3488.986\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "464  JC+I          1724.583     6   3461.166     3461.270     3489.348\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "465  JC+G4         1724.619     6   3461.239     3461.343     3489.421\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "466  JC+I+G4       1724.611     7   3463.221     3463.361     3496.100\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "467  JC+R2         1725.752     7   3465.504     3465.643     3498.383\n",
      "Akaike Information Criterion:           TVM+F+I\n",
      "Corrected Akaike Information Criterion: TVM+F+I\n",
      "Bayesian Information Criterion:         TPM2u+F+I\n",
      "Best-fit model: TPM2u+F+I chosen according to BIC\n",
      "\n",
      "All model information printed to result/phylo_fna_gene/100764at50557_iqtreeout/.tree.model.gz\n",
      "CPU time for ModelFinder: 6.360 seconds (0h:0m:6s)\n",
      "Wall-clock time for ModelFinder: 1.165 seconds (0h:0m:1s)\n",
      "Generating 1000 samples for ultrafast bootstrap (seed: 23807)...\n",
      "WARNING: 8 threads for alignment length 35 will slow down analysis\n",
      "\n",
      "NOTE: 0 MB RAM (0 GB) is required!\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "Estimate model parameters (epsilon = 0.100)\n",
      "1. Initial log-likelihood: -1688.517\n",
      "Optimal log-likelihood: -1688.510\n",
      "Rate parameters:  A-C: 0.15781  A-G: 1.77464  A-T: 0.15781  C-G: 1.00000  C-T: 1.77464  G-T: 1.00000\n",
      "Base frequencies:  A: 0.180  C: 0.285  G: 0.308  T: 0.227\n",
      "Proportion of invariable sites: 0.643\n",
      "Parameters optimization took 1 rounds (0.006 sec)\n",
      "Wrote distance file to... \n",
      "Computing ML distances based on estimated model parameters...\n",
      "Calculating distance matrix: done in 0.0158137 secs using 747.5% CPU\n",
      "Computing ML distances took 0.060611 sec (of wall-clock time) 0.452055 sec (of CPU time)\n",
      "Setting up auxiliary I and S matrices: done in 0.00153683 secs using 151.1% CPU\n",
      "Computing RapidNJ tree took 0.005210 sec (of wall-clock time) 0.007177 sec (of CPU time)\n",
      "Log-likelihood of RapidNJ tree: -1688.510\n",
      "--------------------------------------------------------------------\n",
      "|             INITIALIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "Generating 99 parsimony trees... 0.010 second\n",
      "Computing log-likelihood of 2 initial trees ... 0.001 seconds\n",
      "Current best score: -1688.510\n",
      "\n",
      "Do NNI search on 3 best initial trees\n",
      "Finish initializing candidate tree set (3)\n",
      "Current best tree score: -1688.510 / CPU time: 0.426\n",
      "Number of iterations: 3\n",
      "--------------------------------------------------------------------\n",
      "|               OPTIMIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "UPDATE BEST LOG-LIKELIHOOD: -1688.510\n",
      "UPDATE BEST LOG-LIKELIHOOD: -1688.510\n",
      "Iteration 10 / LogL: -1688.989 / Time: 0h:0m:2s\n",
      "UPDATE BEST LOG-LIKELIHOOD: -1688.510\n",
      "Iteration 20 / LogL: -1688.607 / Time: 0h:0m:3s\n",
      "Iteration 30 / LogL: -1688.989 / Time: 0h:0m:3s (0h:0m:5s left)\n",
      "Iteration 40 / LogL: -1688.989 / Time: 0h:0m:4s (0h:0m:5s left)\n",
      "Iteration 50 / LogL: -1688.989 / Time: 0h:0m:5s (0h:0m:4s left)\n",
      "Log-likelihood cutoff on original alignment: -1691.347\n",
      "Iteration 60 / LogL: -1688.510 / Time: 0h:0m:5s (0h:0m:3s left)\n",
      "Iteration 70 / LogL: -1688.989 / Time: 0h:0m:6s (0h:0m:2s left)\n",
      "Iteration 80 / LogL: -1688.989 / Time: 0h:0m:7s (0h:0m:1s left)\n",
      "Iteration 90 / LogL: -1688.989 / Time: 0h:0m:7s (0h:0m:0s left)\n",
      "Iteration 100 / LogL: -1688.989 / Time: 0h:0m:8s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -1691.347\n",
      "NOTE: Bootstrap correlation coefficient of split occurrence frequencies: 1.000\n",
      "TREE SEARCH COMPLETED AFTER 101 ITERATIONS / Time: 0h:0m:8s\n",
      "\n",
      "--------------------------------------------------------------------\n",
      "|                    FINALIZING TREE SEARCH                        |\n",
      "--------------------------------------------------------------------\n",
      "Performs final model parameters optimization\n",
      "Estimate model parameters (epsilon = 0.010)\n",
      "1. Initial log-likelihood: -1688.510\n",
      "Optimal log-likelihood: -1688.507\n",
      "Rate parameters:  A-C: 0.15431  A-G: 1.77534  A-T: 0.15431  C-G: 1.00000  C-T: 1.77534  G-T: 1.00000\n",
      "Base frequencies:  A: 0.180  C: 0.285  G: 0.308  T: 0.227\n",
      "Proportion of invariable sites: 0.646\n",
      "Parameters optimization took 1 rounds (0.001 sec)\n",
      "BEST SCORE FOUND : -1688.507\n",
      "\n",
      "Testing tree branches by SH-like aLRT with 1000 replicates...\n",
      "0.008 sec.\n",
      "Creating bootstrap support values...\n",
      "Split supports printed to NEXUS file result/phylo_fna_gene/100764at50557_iqtreeout/.tree.splits.nex\n",
      "Total tree length: 0.238\n",
      "\n",
      "Total number of iterations: 101\n",
      "CPU time used for tree search: 44.820 sec (0h:0m:44s)\n",
      "Wall-clock time used for tree search: 7.064 sec (0h:0m:7s)\n",
      "Total CPU time used: 52.446 sec (0h:0m:52s)\n",
      "Total wall-clock time used: 8.433 sec (0h:0m:8s)\n",
      "\n",
      "Computing bootstrap consensus tree...\n",
      "Reading input file result/phylo_fna_gene/100764at50557_iqtreeout/.tree.splits.nex...\n",
      "4 taxa and 7 splits.\n",
      "Consensus tree written to result/phylo_fna_gene/100764at50557_iqtreeout/.tree.contree\n",
      "Reading input trees file result/phylo_fna_gene/100764at50557_iqtreeout/.tree.contree\n",
      "Log-likelihood of consensus tree: -1688.507\n",
      "\n",
      "Analysis results written to: \n",
      "  IQ-TREE report:                result/phylo_fna_gene/100764at50557_iqtreeout/.tree.iqtree\n",
      "  Maximum-likelihood tree:       result/phylo_fna_gene/100764at50557_iqtreeout/.tree.treefile\n",
      "  Likelihood distances:          result/phylo_fna_gene/100764at50557_iqtreeout/.tree.mldist\n",
      "\n",
      "Ultrafast bootstrap approximation results written to:\n",
      "  Split support values:          result/phylo_fna_gene/100764at50557_iqtreeout/.tree.splits.nex\n",
      "  Consensus tree:                result/phylo_fna_gene/100764at50557_iqtreeout/.tree.contree\n",
      "  Screen log file:               result/phylo_fna_gene/100764at50557_iqtreeout/.tree.log\n",
      "\n",
      "Date and Time: Sat Aug 31 21:11:45 2024\n",
      "IQ-TREE multicore version 2.3.4 COVID-edition for Linux x86 64-bit built Jun 18 2024\n",
      "Developed by Bui Quang Minh, James Barbetti, Nguyen Lam Tung, Olga Chernomor,\n",
      "Heiko Schmidt, Dominik Schrempf, Michael Woodhams, Ly Trong Nhan, Thomas Wong\n",
      "\n",
      "Host:    RCNSZOR36402A (AVX512, FMA3, 7 GB RAM)\n",
      "Command: iqtree -s result/phylo_fna_gene/100854at50557_AlnTrimed.fna -nt 8 -m MFP -bb 1000 -alrt 1000 -pre result/phylo_fna_gene/100854at50557_iqtreeout/.tree\n",
      "Seed:    533994 (Using SPRNG - Scalable Parallel Random Number Generator)\n",
      "Time:    Sat Aug 31 21:11:45 2024\n",
      "Kernel:  AVX+FMA - 8 threads (8 CPU cores detected)\n",
      "\n",
      "Reading alignment file result/phylo_fna_gene/100854at50557_AlnTrimed.fna ... Fasta format detected\n",
      "Reading fasta file: done in 0.000145533 secs\n",
      "Alignment most likely contains DNA/RNA sequences\n",
      "Alignment has 4 sequences with 1206 columns, 39 distinct patterns\n",
      "14 parsimony-informative, 231 singleton sites, 961 constant sites\n",
      "                        Gap/Ambiguity  Composition  p-value\n",
      "Analyzing sequences: done in 8.02201e-06 secs\n",
      "   1  Culex_pipiens             0.00%    passed     72.10%\n",
      "   2  Armigeres_subalbatus      0.00%    failed      0.18%\n",
      "   3  Culex_pipiens_molestus    0.00%    passed     60.04%\n",
      "   4  Culex_pipiens_pallens     0.00%    passed     56.99%\n",
      "****  TOTAL                     0.00%  1 sequences failed composition chi2 test (p-value<5%; df=3)\n",
      "\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "\n",
      "Create initial parsimony tree by phylogenetic likelihood library (PLL)... 0.000 seconds\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "Perform fast likelihood tree search using GTR+I+G model...\n",
      "Estimate model parameters (epsilon = 5.000)\n",
      "Perform nearest neighbor interchange...\n",
      "Estimate model parameters (epsilon = 1.000)\n",
      "1. Initial log-likelihood: -2620.219\n",
      "Optimal log-likelihood: -2620.218\n",
      "Rate parameters:  A-C: 6.36881  A-G: 20.80038  A-T: 3.87428  C-G: 8.68364  C-T: 46.04987  G-T: 1.00000\n",
      "Base frequencies:  A: 0.308  C: 0.192  G: 0.293  T: 0.208\n",
      "Proportion of invariable sites: 0.397\n",
      "Gamma shape alpha: 1.738\n",
      "Parameters optimization took 1 rounds (0.008 sec)\n",
      "Time for fast ML tree search: 0.097 seconds\n",
      "\n",
      "NOTE: ModelFinder requires 0 MB RAM!\n",
      "ModelFinder will test up to 484 DNA models (sample size: 1206) ...\n",
      " No. Model         -LnL         df  AIC          AICc         BIC\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  1  GTR+F         2623.514     13  5273.029     5273.334     5339.265\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  2  GTR+F+I       2620.512     14  5269.023     5269.376     5340.354\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  3  GTR+F+G4      2620.276     14  5268.551     5268.904     5339.882\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  4  GTR+F+I+G4    2620.209     15  5270.419     5270.822     5346.845\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  5  GTR+F+R2      2621.150     15  5272.301     5272.704     5348.727\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  6  GTR+F+R3      2620.664     17  5275.329     5275.844     5361.945\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 14  GTR+F+I+R2    2620.350     16  5272.699     5273.157     5354.220\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 15  GTR+F+I+R3    2620.272     18  5276.544     5277.121     5368.256\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 23  SYM           2649.572     10  5319.144     5319.328     5370.095\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 24  SYM+I         2646.588     11  5315.176     5315.397     5371.222\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 25  SYM+G4        2646.310     11  5314.621     5314.842     5370.667\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 26  SYM+I+G4      2646.280     12  5316.561     5316.822     5377.701\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 27  SYM+R2        2647.154     12  5318.309     5318.570     5379.449\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 45  TVM+F         2630.645     12  5285.291     5285.552     5346.431\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 46  TVM+F+I       2626.718     13  5279.436     5279.741     5345.671\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 47  TVM+F+G4      2626.576     13  5279.152     5279.458     5345.388\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 48  TVM+F+I+G4    2626.489     14  5280.979     5281.331     5352.310\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 49  TVM+F+R2      2627.680     14  5283.361     5283.713     5354.692\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 67  TVMe          2650.559     9   5319.117     5319.268     5364.973\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 68  TVMe+I        2647.604     10  5315.208     5315.392     5366.159\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 69  TVMe+G4       2647.328     10  5314.656     5314.840     5365.607\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 70  TVMe+I+G4     2647.300     11  5316.601     5316.822     5372.646\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 71  TVMe+R2       2648.203     11  5318.406     5318.627     5374.452\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 89  TIM3+F        2623.559     11  5269.118     5269.339     5325.163\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 90  TIM3+F+I      2620.858     12  5265.715     5265.977     5326.856\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 91  TIM3+F+G4     2620.636     12  5265.273     5265.534     5326.414\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 92  TIM3+F+I+G4   2620.714     13  5267.427     5267.733     5333.663\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 93  TIM3+F+R2     2621.319     13  5268.638     5268.944     5334.874\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "111  TIM3e         2649.619     8   5315.238     5315.358     5355.999\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "112  TIM3e+I       2646.929     9   5311.858     5312.009     5357.714\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "113  TIM3e+G4      2646.707     9   5311.414     5311.564     5357.270\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "114  TIM3e+I+G4    2646.821     10  5313.643     5313.827     5364.594\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "115  TIM3e+R2      2647.405     10  5314.811     5314.995     5365.761\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "133  TIM2+F        2626.984     11  5275.968     5276.189     5332.014\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "134  TIM2+F+I      2623.963     12  5271.926     5272.188     5333.067\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "135  TIM2+F+G4     2623.899     12  5271.799     5272.060     5332.940\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "136  TIM2+F+I+G4   2623.862     13  5273.725     5274.030     5339.960\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "137  TIM2+F+R2     2624.675     13  5275.350     5275.656     5341.586\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "155  TIM2e         2652.545     8   5321.090     5321.210     5361.850\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "156  TIM2e+I       2649.553     9   5317.106     5317.257     5362.962\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "157  TIM2e+G4      2649.470     9   5316.940     5317.090     5362.795\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "158  TIM2e+I+G4    2649.477     10  5318.953     5319.138     5369.904\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "159  TIM2e+R2      2650.256     10  5320.512     5320.696     5371.462\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "177  TIM+F         2626.988     11  5275.977     5276.198     5332.022\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "178  TIM+F+I       2623.904     12  5271.808     5272.070     5332.949\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "179  TIM+F+G4      2623.833     12  5271.666     5271.927     5332.806\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "180  TIM+F+I+G4    2623.779     13  5273.559     5273.864     5339.795\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "181  TIM+F+R2      2624.644     13  5275.288     5275.594     5341.524\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "199  TIMe          2652.544     8   5321.087     5321.208     5361.848\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "200  TIMe+I        2649.483     9   5316.966     5317.117     5362.822\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "201  TIMe+G4       2649.401     9   5316.801     5316.952     5362.657\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "202  TIMe+I+G4     2649.386     10  5318.772     5318.956     5369.722\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "203  TIMe+R2       2650.141     10  5320.283     5320.467     5371.233\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "221  TPM3u+F       2630.682     10  5281.364     5281.548     5332.315\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "222  TPM3u+F+I     2627.113     11  5276.227     5276.448     5332.272\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "223  TPM3u+F+G4    2626.998     11  5275.997     5276.218     5332.042\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "224  TPM3u+F+I+G4  2626.960     12  5277.920     5278.181     5339.060\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "225  TPM3u+F+R2    2627.824     12  5279.647     5279.909     5340.788\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "243  TPM3          2650.556     7   5315.111     5315.205     5350.777\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "244  TPM3+I        2647.911     8   5311.821     5311.942     5352.582\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "245  TPM3+G4       2647.684     8   5311.368     5311.489     5352.129\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "246  TPM3+I+G4     2647.761     9   5313.523     5313.673     5359.378\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "247  TPM3+R2       2648.359     9   5314.717     5314.868     5360.573\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "265  TPM2u+F       2634.106     10  5288.213     5288.397     5339.164\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "266  TPM2u+F+I     2630.151     11  5282.302     5282.523     5338.347\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "267  TPM2u+F+G4    2630.147     11  5282.294     5282.515     5338.340\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "268  TPM2u+F+I+G4  2630.081     12  5284.162     5284.423     5345.302\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "269  TPM2u+F+R2    2630.944     12  5285.889     5286.150     5347.030\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "287  TPM2          2653.475     7   5320.951     5321.044     5356.616\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "288  TPM2+I        2650.566     8   5317.131     5317.252     5357.892\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "289  TPM2+G4       2650.455     8   5316.910     5317.030     5357.670\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "290  TPM2+I+G4     2650.458     9   5318.915     5319.066     5364.771\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "291  TPM2+R2       2651.220     9   5320.440     5320.591     5366.296\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "309  K3Pu+F        2634.110     10  5288.219     5288.404     5339.170\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "310  K3Pu+F+I      2630.109     11  5282.218     5282.439     5338.263\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "311  K3Pu+F+G4     2630.098     11  5282.195     5282.416     5338.241\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "312  K3Pu+F+I+G4   2630.035     12  5284.070     5284.331     5345.211\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "313  K3Pu+F+R2     2630.909     12  5285.818     5286.079     5346.958\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "331  K3P           2653.474     7   5320.948     5321.042     5356.614\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "332  K3P+I         2650.514     8   5317.028     5317.149     5357.789\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "333  K3P+G4        2650.399     8   5316.799     5316.919     5357.559\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "334  K3P+I+G4      2650.390     9   5318.780     5318.930     5364.635\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "335  K3P+R2        2651.180     9   5320.360     5320.510     5366.215\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "353  TN+F          2626.991     10  5273.982     5274.166     5324.933\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "354  TN+F+I        2623.971     11  5269.942     5270.163     5325.988\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "355  TN+F+G4       2623.903     11  5269.806     5270.028     5325.852\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "356  TN+F+I+G4     2623.879     12  5271.758     5272.020     5332.899\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "357  TN+F+R2       2624.673     12  5273.345     5273.607     5334.486\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "375  TNe           2652.548     7   5319.096     5319.189     5354.761\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "376  TNe+I         2649.572     8   5315.143     5315.264     5355.904\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "377  TNe+G4        2649.491     8   5314.983     5315.103     5355.743\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "378  TNe+I+G4      2649.517     9   5317.034     5317.184     5362.889\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "379  TNe+R2        2650.258     9   5318.516     5318.667     5364.372\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "397  HKY+F         2634.111     9   5286.223     5286.373     5332.078\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "398  HKY+F+I       2630.222     10  5280.443     5280.627     5331.394\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "399  HKY+F+G4      2630.185     10  5280.369     5280.553     5331.320\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "400  HKY+F+I+G4    2630.142     11  5282.285     5282.506     5338.330\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "401  HKY+F+R2      2631.035     11  5284.070     5284.291     5340.116\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "419  K2P           2653.479     6   5318.958     5319.028     5349.529\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "420  K2P+I         2650.590     7   5315.181     5315.274     5350.846\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "421  K2P+G4        2650.480     7   5314.960     5315.054     5350.626\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "422  K2P+I+G4      2650.497     8   5316.995     5317.115     5357.755\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "423  K2P+R2        2651.220     8   5318.439     5318.560     5359.200\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "441  F81+F         2691.277     8   5398.555     5398.675     5439.315\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "442  F81+F+I       2688.788     9   5395.576     5395.727     5441.432\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "443  F81+F+G4      2688.812     9   5395.623     5395.774     5441.479\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "444  F81+F+I+G4    2688.802     10  5397.605     5397.789     5448.556\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "445  F81+F+R2      2689.357     10  5398.713     5398.897     5449.664\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "463  JC            2717.204     5   5444.407     5444.457     5469.883\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "464  JC+I          2714.757     6   5441.514     5441.585     5472.085\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "465  JC+G4         2714.802     6   5441.605     5441.675     5472.175\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "466  JC+I+G4       2714.773     7   5443.546     5443.640     5479.212\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "467  JC+R2         2715.307     7   5444.615     5444.708     5480.280\n",
      "Akaike Information Criterion:           TIM3+F+G4\n",
      "Corrected Akaike Information Criterion: TIM3+F+G4\n",
      "Bayesian Information Criterion:         TN+F\n",
      "Best-fit model: TN+F chosen according to BIC\n",
      "\n",
      "All model information printed to result/phylo_fna_gene/100854at50557_iqtreeout/.tree.model.gz\n",
      "CPU time for ModelFinder: 8.757 seconds (0h:0m:8s)\n",
      "Wall-clock time for ModelFinder: 1.552 seconds (0h:0m:1s)\n",
      "Generating 1000 samples for ultrafast bootstrap (seed: 533994)...\n",
      "WARNING: 8 threads for alignment length 39 will slow down analysis\n",
      "\n",
      "NOTE: 0 MB RAM (0 GB) is required!\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "Estimate model parameters (epsilon = 0.100)\n",
      "1. Initial log-likelihood: -2626.991\n",
      "Optimal log-likelihood: -2626.991\n",
      "Rate parameters:  A-C: 1.00000  A-G: 3.50974  A-T: 1.00000  C-G: 1.00000  C-T: 6.62647  G-T: 1.00000\n",
      "Base frequencies:  A: 0.308  C: 0.192  G: 0.293  T: 0.208\n",
      "Parameters optimization took 1 rounds (0.006 sec)\n",
      "Wrote distance file to... \n",
      "Computing ML distances based on estimated model parameters...\n",
      "Calculating distance matrix: done in 0.0161366 secs using 786.9% CPU\n",
      "Computing ML distances took 0.071122 sec (of wall-clock time) 0.487869 sec (of CPU time)\n",
      "Setting up auxiliary I and S matrices: done in 0.00180922 secs using 169.5% CPU\n",
      "Computing RapidNJ tree took 0.003640 sec (of wall-clock time) 0.006095 sec (of CPU time)\n",
      "Log-likelihood of RapidNJ tree: -2626.991\n",
      "--------------------------------------------------------------------\n",
      "|             INITIALIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "Generating 99 parsimony trees... 0.011 second\n",
      "Computing log-likelihood of 2 initial trees ... 0.001 seconds\n",
      "Current best score: -2626.991\n",
      "\n",
      "Do NNI search on 3 best initial trees\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2626.991\n",
      "Finish initializing candidate tree set (3)\n",
      "Current best tree score: -2626.991 / CPU time: 0.400\n",
      "Number of iterations: 3\n",
      "--------------------------------------------------------------------\n",
      "|               OPTIMIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "Iteration 10 / LogL: -2626.997 / Time: 0h:0m:2s\n",
      "Iteration 20 / LogL: -2627.026 / Time: 0h:0m:2s\n",
      "Iteration 30 / LogL: -2626.997 / Time: 0h:0m:3s (0h:0m:4s left)\n",
      "Iteration 40 / LogL: -2635.983 / Time: 0h:0m:3s (0h:0m:3s left)\n",
      "Iteration 50 / LogL: -2627.047 / Time: 0h:0m:4s (0h:0m:2s left)\n",
      "Log-likelihood cutoff on original alignment: -2642.139\n",
      "Iteration 60 / LogL: -2626.997 / Time: 0h:0m:4s (0h:0m:2s left)\n",
      "Iteration 70 / LogL: -2635.983 / Time: 0h:0m:5s (0h:0m:1s left)\n",
      "Iteration 80 / LogL: -2635.983 / Time: 0h:0m:5s (0h:0m:1s left)\n",
      "Iteration 90 / LogL: -2635.983 / Time: 0h:0m:6s (0h:0m:0s left)\n",
      "Iteration 100 / LogL: -2627.001 / Time: 0h:0m:6s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -2642.774\n",
      "NOTE: Bootstrap correlation coefficient of split occurrence frequencies: 1.000\n",
      "TREE SEARCH COMPLETED AFTER 101 ITERATIONS / Time: 0h:0m:6s\n",
      "\n",
      "--------------------------------------------------------------------\n",
      "|                    FINALIZING TREE SEARCH                        |\n",
      "--------------------------------------------------------------------\n",
      "Performs final model parameters optimization\n",
      "Estimate model parameters (epsilon = 0.010)\n",
      "1. Initial log-likelihood: -2626.991\n",
      "Optimal log-likelihood: -2626.991\n",
      "Rate parameters:  A-C: 1.00000  A-G: 3.50955  A-T: 1.00000  C-G: 1.00000  C-T: 6.62660  G-T: 1.00000\n",
      "Base frequencies:  A: 0.308  C: 0.192  G: 0.293  T: 0.208\n",
      "Parameters optimization took 1 rounds (0.001 sec)\n",
      "BEST SCORE FOUND : -2626.991\n",
      "\n",
      "Testing tree branches by SH-like aLRT with 1000 replicates...\n",
      "0.017 sec.\n",
      "Creating bootstrap support values...\n",
      "Split supports printed to NEXUS file result/phylo_fna_gene/100854at50557_iqtreeout/.tree.splits.nex\n",
      "Total tree length: 0.247\n",
      "\n",
      "Total number of iterations: 101\n",
      "CPU time used for tree search: 30.296 sec (0h:0m:30s)\n",
      "Wall-clock time used for tree search: 4.919 sec (0h:0m:4s)\n",
      "Total CPU time used: 40.376 sec (0h:0m:40s)\n",
      "Total wall-clock time used: 6.676 sec (0h:0m:6s)\n",
      "\n",
      "Computing bootstrap consensus tree...\n",
      "Reading input file result/phylo_fna_gene/100854at50557_iqtreeout/.tree.splits.nex...\n",
      "4 taxa and 6 splits.\n",
      "Consensus tree written to result/phylo_fna_gene/100854at50557_iqtreeout/.tree.contree\n",
      "Reading input trees file result/phylo_fna_gene/100854at50557_iqtreeout/.tree.contree\n",
      "Log-likelihood of consensus tree: -2626.991\n",
      "\n",
      "Analysis results written to: \n",
      "  IQ-TREE report:                result/phylo_fna_gene/100854at50557_iqtreeout/.tree.iqtree\n",
      "  Maximum-likelihood tree:       result/phylo_fna_gene/100854at50557_iqtreeout/.tree.treefile\n",
      "  Likelihood distances:          result/phylo_fna_gene/100854at50557_iqtreeout/.tree.mldist\n",
      "\n",
      "Ultrafast bootstrap approximation results written to:\n",
      "  Split support values:          result/phylo_fna_gene/100854at50557_iqtreeout/.tree.splits.nex\n",
      "  Consensus tree:                result/phylo_fna_gene/100854at50557_iqtreeout/.tree.contree\n",
      "  Screen log file:               result/phylo_fna_gene/100854at50557_iqtreeout/.tree.log\n",
      "\n",
      "Date and Time: Sat Aug 31 21:11:52 2024\n"
     ]
    }
   ],
   "source": [
    "#使用iqtree完成，如果数据量过大，可以使用fasttree快速构建一个树文件，暂时代替\n",
    "! ls result/phylo_fna_gene/*_AlnTrimed.fna | while read file;do mkdir -p ${file%_AlnTrimed.fna}_iqtreeout;done\n",
    "! ls result/phylo_fna_gene/*_AlnTrimed.fna | while read file; do iqtree -s $file -nt 8 -m MFP -bb 1000 -alrt 1000 -pre ${file%_AlnTrimed.fna}_iqtreeout/$(basename ${i%_AlnTrimed.fna}.tree); done\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# 2、利用修剪后的基因比对序列建物种树"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "metadata": {},
   "outputs": [],
   "source": [
    "#先将Trimal后的序列提取出来\n",
    "! mkdir -p result/Aln_fna\n",
    "! cp -r result/phylo_fna_gene/*_AlnTrimed.fna result/Aln_fna/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "metadata": {},
   "outputs": [],
   "source": [
    "# 按照物种划分序列，得到对应物种的全序列，注意，缺失的基因用gap替换\n",
    "! python3 src/species.py result/Aln_fna/ result/species_fna/ fna"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 63,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "IQ-TREE multicore version 2.3.4 COVID-edition for Linux x86 64-bit built Jun 18 2024\n",
      "Developed by Bui Quang Minh, James Barbetti, Nguyen Lam Tung, Olga Chernomor,\n",
      "Heiko Schmidt, Dominik Schrempf, Michael Woodhams, Ly Trong Nhan, Thomas Wong\n",
      "\n",
      "Host:    RCNSZOR36402A (AVX512, FMA3, 7 GB RAM)\n",
      "Command: iqtree -s result/species.fna -nt 8 -m MFP -bb 1000 -alrt 1000 -pre result/species_iqtreeout/species.tree\n",
      "Seed:    17802 (Using SPRNG - Scalable Parallel Random Number Generator)\n",
      "Time:    Sun Sep  1 17:09:50 2024\n",
      "Kernel:  AVX+FMA - 8 threads (8 CPU cores detected)\n",
      "\n",
      "Reading alignment file result/species.fna ... Fasta format detected\n",
      "Reading fasta file: done in 0.000917375 secs using 37.5% CPU\n",
      "Alignment most likely contains DNA/RNA sequences\n",
      "Alignment has 4 sequences with 11235 columns, 153 distinct patterns\n",
      "139 parsimony-informative, 2643 singleton sites, 8453 constant sites\n",
      "                        Gap/Ambiguity  Composition  p-value\n",
      "Analyzing sequences: done in 0.00612641 secs using 99.98% CPU\n",
      "   1  Armigeres_subalbatus      0.59%    failed      0.00%\n",
      "   2  Culex_pipiens             0.05%    failed      1.94%\n",
      "   3  Culex_pipiens_molestus    0.00%    failed      2.11%\n",
      "   4  Culex_pipiens_pallens     0.03%    failed      4.29%\n",
      "****  TOTAL                     0.17%  4 sequences failed composition chi2 test (p-value<5%; df=3)\n",
      "\n",
      "\n",
      "Create initial parsimony tree by phylogenetic likelihood library (PLL)... 0.003 seconds\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "Perform fast likelihood tree search using GTR+I+G model...\n",
      "Estimate model parameters (epsilon = 5.000)\n",
      "Perform nearest neighbor interchange...\n",
      "Estimate model parameters (epsilon = 1.000)\n",
      "1. Initial log-likelihood: -26742.277\n",
      "Optimal log-likelihood: -26742.260\n",
      "Rate parameters:  A-C: 1.34058  A-G: 4.13292  A-T: 0.87265  C-G: 1.77323  C-T: 6.57432  G-T: 1.00000\n",
      "Base frequencies:  A: 0.252  C: 0.256  G: 0.294  T: 0.198\n",
      "Proportion of invariable sites: 0.377\n",
      "Gamma shape alpha: 1.323\n",
      "Parameters optimization took 1 rounds (0.053 sec)\n",
      "Time for fast ML tree search: 2.095 seconds\n",
      "\n",
      "NOTE: ModelFinder requires 0 MB RAM!\n",
      "ModelFinder will test up to 484 DNA models (sample size: 11235) ...\n",
      " No. Model         -LnL         df  AIC          AICc         BIC\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  1  GTR+F         26818.794    13  53663.589    53663.621    53758.837\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  2  GTR+F+I       26742.237    14  53512.474    53512.512    53615.049\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  3  GTR+F+G4      26742.679    14  53513.359    53513.396    53615.934\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  4  GTR+F+I+G4    26742.334    15  53514.667    53514.710    53624.569\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  5  GTR+F+R2      26742.706    15  53515.412    53515.454    53625.314\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  6  GTR+F+R3      26742.411    17  53518.822    53518.876    53643.377\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 14  GTR+F+I+R2    26742.260    16  53516.520    53516.568    53633.748\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 15  GTR+F+I+R3    26742.244    18  53520.487    53520.548    53652.369\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 24  SYM+I         26839.547    11  53701.094    53701.117    53781.688\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 25  SYM+G4        26839.152    11  53700.305    53700.328    53780.899\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 26  SYM+I+G4      26839.147    12  53702.294    53702.322    53790.216\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 46  TVM+F+I       26760.119    13  53546.237    53546.270    53641.486\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 47  TVM+F+G4      26761.415    13  53548.829    53548.861    53644.077\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 48  TVM+F+I+G4    26761.004    14  53550.008    53550.045    53652.583\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 68  TVMe+I        26843.616    10  53707.233    53707.252    53780.501\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 69  TVMe+G4       26843.419    10  53706.837    53706.857    53780.105\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 70  TVMe+I+G4     26843.231    11  53708.462    53708.486    53789.057\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 90  TIM3+F+I      26745.367    12  53514.735    53514.763    53602.656\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 91  TIM3+F+G4     26745.586    12  53515.172    53515.199    53603.093\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 92  TIM3+F+I+G4   26745.342    13  53516.684    53516.717    53611.932\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "112  TIM3e+I       26846.270    9   53710.540    53710.556    53776.481\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "113  TIM3e+G4      26845.942    9   53709.884    53709.900    53775.825\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "114  TIM3e+I+G4    26845.872    10  53711.743    53711.763    53785.011\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "134  TIM2+F+I      26752.310    12  53528.620    53528.647    53616.541\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "135  TIM2+F+G4     26753.875    12  53531.750    53531.778    53619.672\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "136  TIM2+F+I+G4   26753.733    13  53533.465    53533.497    53628.713\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "156  TIM2e+I       26864.222    9   53746.444    53746.460    53812.385\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "157  TIM2e+G4      26865.503    9   53749.007    53749.023    53814.948\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "158  TIM2e+I+G4    26865.223    10  53750.446    53750.465    53823.714\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "178  TIM+F+I       26754.101    12  53532.203    53532.230    53620.124\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "179  TIM+F+G4      26755.669    12  53535.337    53535.365    53623.259\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "180  TIM+F+I+G4    26755.572    13  53537.145    53537.177    53632.393\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "200  TIMe+I        26869.266    9   53756.532    53756.548    53822.473\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "201  TIMe+G4       26870.565    9   53759.130    53759.146    53825.071\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "202  TIMe+I+G4     26870.277    10  53760.554    53760.574    53833.822\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "222  TPM3u+F+I     26762.381    11  53546.761    53546.785    53627.356\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "223  TPM3u+F+G4    26763.730    11  53549.460    53549.483    53630.054\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "224  TPM3u+F+I+G4  26763.552    12  53551.105    53551.133    53639.026\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "244  TPM3+I        26849.960    8   53715.920    53715.933    53774.534\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "245  TPM3+G4       26849.878    8   53715.757    53715.770    53774.371\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "246  TPM3+I+G4     26849.615    9   53717.231    53717.247    53783.172\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "266  TPM2u+F+I     26771.549    11  53565.097    53565.121    53645.692\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "267  TPM2u+F+G4    26773.408    11  53568.817    53568.840    53649.412\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "268  TPM2u+F+I+G4  26773.452    12  53570.904    53570.932    53658.826\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "288  TPM2+I        26869.219    8   53754.438    53754.451    53813.053\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "289  TPM2+G4       26869.944    8   53755.889    53755.902    53814.503\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "290  TPM2+I+G4     26869.788    9   53757.576    53757.592    53823.517\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "310  K3Pu+F+I      26772.431    11  53566.861    53566.885    53647.456\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "311  K3Pu+F+G4     26774.570    11  53571.141    53571.165    53651.736\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "312  K3Pu+F+I+G4   26774.659    12  53573.319    53573.347    53661.240\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "332  K3P+I         26873.891    8   53763.782    53763.795    53822.396\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "333  K3P+G4        26874.658    8   53765.316    53765.329    53823.930\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "334  K3P+I+G4      26874.628    9   53767.256    53767.272    53833.197\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "354  TN+F+I        26754.903    11  53531.805    53531.829    53612.400\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "355  TN+F+G4       26756.434    11  53534.868    53534.892    53615.463\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "356  TN+F+I+G4     26756.380    12  53536.760    53536.788    53624.682\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "376  TNe+I         26870.061    8   53756.122    53756.135    53814.736\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "377  TNe+G4        26871.309    8   53758.619    53758.631    53817.233\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "378  TNe+I+G4      26871.164    9   53760.327    53760.343    53826.268\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "398  HKY+F+I       26773.254    10  53566.508    53566.527    53639.775\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "399  HKY+F+G4      26775.371    10  53570.742    53570.762    53644.010\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "400  HKY+F+I+G4    26775.515    11  53573.030    53573.054    53653.625\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "420  K2P+I         26874.695    7   53763.389    53763.399    53814.677\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "421  K2P+G4        26875.582    7   53765.163    53765.173    53816.451\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "422  K2P+I+G4      26875.521    8   53767.042    53767.055    53825.656\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "442  F81+F+I       27125.652    9   54269.305    54269.321    54335.246\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "443  F81+F+G4      27125.711    9   54269.423    54269.439    54335.364\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "444  F81+F+I+G4    27125.732    10  54271.463    54271.483    54344.731\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "464  JC+I          27227.663    6   54467.326    54467.334    54511.287\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "465  JC+G4         27227.722    6   54467.444    54467.452    54511.405\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "466  JC+I+G4       27227.702    7   54469.403    54469.413    54520.691\n",
      "Akaike Information Criterion:           GTR+F+I\n",
      "Corrected Akaike Information Criterion: GTR+F+I\n",
      "Bayesian Information Criterion:         TIM3+F+I\n",
      "Best-fit model: TIM3+F+I chosen according to BIC\n",
      "\n",
      "All model information printed to result/species_iqtreeout/species.tree.model.gz\n",
      "CPU time for ModelFinder: 286.239 seconds (0h:4m:46s)\n",
      "Wall-clock time for ModelFinder: 36.458 seconds (0h:0m:36s)\n",
      "Generating 1000 samples for ultrafast bootstrap (seed: 17802)...\n",
      "\n",
      "NOTE: 0 MB RAM (0 GB) is required!\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "Estimate model parameters (epsilon = 0.100)\n",
      "1. Initial log-likelihood: -26745.367\n",
      "Optimal log-likelihood: -26745.330\n",
      "Rate parameters:  A-C: 1.59476  A-G: 4.22471  A-T: 1.00000  C-G: 1.59476  C-T: 6.72996  G-T: 1.00000\n",
      "Base frequencies:  A: 0.252  C: 0.256  G: 0.294  T: 0.198\n",
      "Proportion of invariable sites: 0.537\n",
      "Parameters optimization took 1 rounds (0.001 sec)\n",
      "Wrote distance file to... \n",
      "Computing ML distances based on estimated model parameters...\n",
      "Calculating distance matrix: done in 0.00195477 secs using 3429% CPU\n",
      "Computing ML distances took 0.017963 sec (of wall-clock time) 0.158522 sec (of CPU time)\n",
      "Setting up auxiliary I and S matrices: done in 0.00171806 secs using 167.3% CPU\n",
      "Computing RapidNJ tree took 0.004076 sec (of wall-clock time) 0.006637 sec (of CPU time)\n",
      "Log-likelihood of RapidNJ tree: -26745.330\n",
      "--------------------------------------------------------------------\n",
      "|             INITIALIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "Generating 99 parsimony trees... 0.028 second\n",
      "Computing log-likelihood of 2 initial trees ... 0.042 seconds\n",
      "Current best score: -26745.330\n",
      "\n",
      "Do NNI search on 3 best initial trees\n",
      "Finish initializing candidate tree set (3)\n",
      "Current best tree score: -26745.330 / CPU time: 0.091\n",
      "Number of iterations: 3\n",
      "--------------------------------------------------------------------\n",
      "|               OPTIMIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "Iteration 10 / LogL: -26745.911 / Time: 0h:0m:39s\n",
      "UPDATE BEST LOG-LIKELIHOOD: -26745.330\n",
      "Iteration 20 / LogL: -26745.600 / Time: 0h:0m:41s\n",
      "UPDATE BEST LOG-LIKELIHOOD: -26745.330\n",
      "UPDATE BEST LOG-LIKELIHOOD: -26745.330\n",
      "Iteration 30 / LogL: -26745.549 / Time: 0h:0m:41s (0h:0m:12s left)\n",
      "Iteration 40 / LogL: -26745.549 / Time: 0h:0m:41s (0h:0m:8s left)\n",
      "Iteration 50 / LogL: -26745.913 / Time: 0h:0m:43s (0h:0m:7s left)\n",
      "Log-likelihood cutoff on original alignment: -26752.537\n",
      "Iteration 60 / LogL: -26745.911 / Time: 0h:0m:44s (0h:0m:5s left)\n",
      "Iteration 70 / LogL: -26745.911 / Time: 0h:0m:46s (0h:0m:4s left)\n",
      "Iteration 80 / LogL: -26745.549 / Time: 0h:0m:51s (0h:0m:3s left)\n",
      "Iteration 90 / LogL: -26745.913 / Time: 0h:0m:51s (0h:0m:1s left)\n",
      "Iteration 100 / LogL: -26745.929 / Time: 0h:0m:53s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -26752.537\n",
      "NOTE: Bootstrap correlation coefficient of split occurrence frequencies: 1.000\n",
      "TREE SEARCH COMPLETED AFTER 101 ITERATIONS / Time: 0h:0m:53s\n",
      "\n",
      "--------------------------------------------------------------------\n",
      "|                    FINALIZING TREE SEARCH                        |\n",
      "--------------------------------------------------------------------\n",
      "Performs final model parameters optimization\n",
      "Estimate model parameters (epsilon = 0.010)\n",
      "1. Initial log-likelihood: -26745.330\n",
      "Optimal log-likelihood: -26745.315\n",
      "Rate parameters:  A-C: 1.58860  A-G: 4.22653  A-T: 1.00000  C-G: 1.58860  C-T: 6.73919  G-T: 1.00000\n",
      "Base frequencies:  A: 0.252  C: 0.256  G: 0.294  T: 0.198\n",
      "Proportion of invariable sites: 0.539\n",
      "Parameters optimization took 1 rounds (0.010 sec)\n",
      "BEST SCORE FOUND : -26745.315\n",
      "\n",
      "Testing tree branches by SH-like aLRT with 1000 replicates...\n",
      "0.009 sec.\n",
      "Creating bootstrap support values...\n",
      "Split supports printed to NEXUS file result/species_iqtreeout/species.tree.splits.nex\n",
      "Total tree length: 0.454\n",
      "\n",
      "Total number of iterations: 101\n",
      "CPU time used for tree search: 131.068 sec (0h:2m:11s)\n",
      "Wall-clock time used for tree search: 16.656 sec (0h:0m:16s)\n",
      "Total CPU time used: 423.358 sec (0h:7m:3s)\n",
      "Total wall-clock time used: 53.903 sec (0h:0m:53s)\n",
      "\n",
      "Computing bootstrap consensus tree...\n",
      "Reading input file result/species_iqtreeout/species.tree.splits.nex...\n",
      "4 taxa and 7 splits.\n",
      "Consensus tree written to result/species_iqtreeout/species.tree.contree\n",
      "Reading input trees file result/species_iqtreeout/species.tree.contree\n",
      "Log-likelihood of consensus tree: -26745.315\n",
      "\n",
      "Analysis results written to: \n",
      "  IQ-TREE report:                result/species_iqtreeout/species.tree.iqtree\n",
      "  Maximum-likelihood tree:       result/species_iqtreeout/species.tree.treefile\n",
      "  Likelihood distances:          result/species_iqtreeout/species.tree.mldist\n",
      "\n",
      "Ultrafast bootstrap approximation results written to:\n",
      "  Split support values:          result/species_iqtreeout/species.tree.splits.nex\n",
      "  Consensus tree:                result/species_iqtreeout/species.tree.contree\n",
      "  Screen log file:               result/species_iqtreeout/species.tree.log\n",
      "\n",
      "Date and Time: Sun Sep  1 17:10:44 2024\n"
     ]
    }
   ],
   "source": [
    "#  由于本身序列就是比对修剪过后的，可以直接用来进行建树\n",
    "! cat result/species_fna/* > result/species.fna\n",
    "! mkdir -p result/species_iqtreeout\n",
    "! iqtree -s result/species.fna -nt 8 -m MFP -bb 1000 -alrt 1000 -pre result/species_iqtreeout/species.tree"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "base",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.12.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
