{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# 系统发育推断"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 准备工作,给每个基因文件的标识符信息添加对应的物种名信息"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "#准备miniconda软件\n",
    "# wget https://mirrors.tuna.tsinghua.edu.cn/anaconda/miniconda/Miniconda3-latest-Linux-x86_64.sh\n",
    "# bash Miniconda3-latest-Linux-x86_64.sh"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#安装建树一整套软件\n",
    "! source ~/.bashrc\n",
    "! conda install -c bioconda mafft trimal iqtree"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/workspaces/evolution\n"
     ]
    }
   ],
   "source": [
    "! pwd\n",
    "#调用脚本，内有注释，完成标识信息的添加\n",
    "! bash ./src/modified_ID.sh data/ modified_data/ fna"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 1、选取合适的分子标记进行多重比对、修剪、构树，本教程以直系同源单拷贝基因为例"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/workspaces/evolution\n"
     ]
    }
   ],
   "source": [
    "#根据基因名合并物种序列,准备构建基因树\n",
    "! pwd\n",
    "! python3 src/gene_split.py ./modified_data ./result/gene_fna fna\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "nthread = 8\n",
      "nadd = 0\n",
      "inputfile = infile\n",
      "thresholdtorev = 0.000000\n",
      "reflim = 5000\n",
      "mode = a\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "\n",
      "Step 1/2\n",
      "1   \n",
      "\n",
      "Step 2/2\n",
      " 1 / 4 (8 threads)   \n",
      "makedirectionlist (nuc) Version 7.526\n",
      "alg=m, model=DNA200 (2), 1.53 (4.59), 0.37 (1.11), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "directionfile = _direction\n",
      "inputfile = infile\n",
      "subalignment = 0\n",
      "subalignmentoffset = 0\n",
      "4 x 915 - 897 d\n",
      "outputhat23=16\n",
      "treein = 0\n",
      "compacttree = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "All-to-all alignment.\n",
      "tbfast-pair (nuc) Version 7.526\n",
      "alg=L, model=DNA200 (2), 2.00 (6.00), -0.10 (-0.30), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "outputhat23=16\n",
      "Loading 'hat3.seed' ... \n",
      "done.\n",
      "Writing hat3 for iterative refinement\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "tbutree = 1, compacttree = 0\n",
      "Constructing a UPGMA tree ... \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment ... \n",
      "STEP     3 /3 (thread    6) \n",
      "done.\n",
      "tbfast (nuc) Version 7.526\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "minimumweight = 0.000010\n",
      "autosubalignment = 0.000000\n",
      "nthread = 8\n",
      "randomseed = 0\n",
      "blosum 62 / kimura 200\n",
      "poffset = 0\n",
      "niter = 16\n",
      "sueff_global = 0.100000\n",
      "nadd = 16\n",
      "Loading 'hat3' ... done.\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "\n",
      "    0 / 4\n",
      "Segment   1/  1    1- 916\n",
      "001-0004-0 (thread    1) identical     \n",
      "Converged.\n",
      "done\n",
      "dvtditr (nuc) Version 7.526\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " L-INS-i (Probably most accurate, very slow)\n",
      " Iterative refinement method (<16) with LOCAL pairwise alignment information\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 8\n",
      "nadd = 0\n",
      "inputfile = infile\n",
      "thresholdtorev = 0.000000\n",
      "reflim = 5000\n",
      "mode = a\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "\n",
      "Step 1/2\n",
      "1   \n",
      "\n",
      "Step 2/2\n",
      " 1 / 4 (8 threads)   \n",
      "makedirectionlist (nuc) Version 7.526\n",
      "alg=m, model=DNA200 (2), 1.53 (4.59), 0.37 (1.11), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "directionfile = _direction\n",
      "inputfile = infile\n",
      "subalignment = 0\n",
      "subalignmentoffset = 0\n",
      "4 x 4140 - 3561 d\n",
      "nthread = 8\n",
      "nthreadpair = 8\n",
      "nthreadtb = 8\n",
      "ppenalty_ex = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "\n",
      "\n",
      "\n",
      "Making a distance matrix ..\n",
      "    1 / 4 (thread    0)\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=0) ... \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 1/2... \n",
      "STEP     3 / 3 (thread    5)\n",
      "done.\n",
      "\n",
      "Making a distance matrix from msa.. \n",
      "    0 / 4 (thread    0)\n",
      "done.\n",
      "\n",
      "Constructing a UPGMA tree (efffree=1) ... \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment 2/2... \n",
      "STEP     3 / 3 (thread    4)\n",
      "done.\n",
      "\n",
      "disttbfast (nuc) Version 7.526\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "distout=h\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "    0 / 4 (thread    0)dndpre (nuc) Version 7.526\n",
      "alg=X, model=DNA200 (2), 1.53 (4.59), 0.37 (1.11), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "minimumweight = 0.000010\n",
      "autosubalignment = 0.000000\n",
      "nthread = 8\n",
      "randomseed = 0\n",
      "blosum 62 / kimura 200\n",
      "poffset = 0\n",
      "niter = 2\n",
      "sueff_global = 0.100000\n",
      "nadd = 2\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "\n",
      "    0 / 4\n",
      "Segment   1/ 40    1- 636\n",
      "001-0004-1 (thread    3) identical     \n",
      "Converged.\n",
      "Segment   2/ 40  636- 695\n",
      "001-0004-1 (thread    5) identical     \n",
      "Converged.\n",
      "Segment   3/ 40  695- 773\n",
      "001-0004-0 (thread    4) identical     \n",
      "Converged.\n",
      "Segment   4/ 40  773- 896\n",
      "001-0004-0 (thread    5) identical     \n",
      "Converged.\n",
      "Segment   5/ 40  896- 956\n",
      "001-0004-0 (thread    7) identical     \n",
      "Converged.\n",
      "Segment   6/ 40  956- 995\n",
      "001-0004-1 (thread    6) identical     \n",
      "Converged.\n",
      "Segment   7/ 40  995-1115\n",
      "002-0004-0 (thread    5) identical     \n",
      "Converged.\n",
      "\n",
      "Reached 2\n",
      "Segment   8/ 40 1115-1261\n",
      "001-0000-1 (thread    8) identical     \n",
      "Converged.\n",
      "Segment   9/ 40 1261-1341\n",
      "001-0004-0 (thread    7) identical     \n",
      "Converged.\n",
      "Segment  10/ 40 1341-1492\n",
      "001-0004-1 (thread    5) identical     \n",
      "Converged.\n",
      "Segment  11/ 40 1492-1643\n",
      "001-0004-0 (thread    7) identical     \n",
      "Converged.\n",
      "Segment  12/ 40 1643-1728\n",
      "001-0004-1 (thread    4) identical     \n",
      "Converged.\n",
      "Segment  13/ 40 1728-1836\n",
      "001-0004-0 (thread    3) identical     \n",
      "Converged.\n",
      "Segment  14/ 40 1836-1942\n",
      "001-0004-1 (thread    6) identical     \n",
      "Converged.\n",
      "Segment  15/ 40 1942-1978\n",
      "001-0004-1 (thread    1) identical     \n",
      "Converged.\n",
      "Segment  16/ 40 1978-1992\n",
      "001-0004-1 (thread    4) identical     \n",
      "Converged.\n",
      "Segment  17/ 40 1992-2074\n",
      "001-0004-1 (thread    8) identical     \n",
      "Converged.\n",
      "Segment  18/ 40 2074-2225\n",
      "001-0004-1 (thread    6) identical     \n",
      "Converged.\n",
      "Segment  19/ 40 2225-2330\n",
      "001-0004-1 (thread    6) identical     \n",
      "Converged.\n",
      "Segment  20/ 40 2330-2404\n",
      "001-0004-1 (thread    6) identical     \n",
      "Converged.\n",
      "Segment  21/ 40 2404-2440\n",
      "001-0004-1 (thread    1) identical     \n",
      "Converged.\n",
      "Segment  22/ 40 2440-2446\n",
      "001-0004-1 (thread    6) identical     \n",
      "Converged.\n",
      "Segment  23/ 40 2446-2452\n",
      "001-0004-1 (thread    4) identical     001-0003-0 (thread    7) identical     \n",
      "Converged.\n",
      "Segment  24/ 40 2452-2482\n",
      "001-0004-1 (thread    2) identical     \n",
      "Converged.\n",
      "Segment  25/ 40 2482-2599\n",
      "001-0004-1 (thread    8) identical     \n",
      "Converged.\n",
      "Segment  26/ 40 2599-2742\n",
      "001-0004-1 (thread    4) identical     \n",
      "Converged.\n",
      "Segment  27/ 40 2742-2876\n",
      "001-0004-1 (thread    8) identical     \n",
      "Converged.\n",
      "Segment  28/ 40 2876-3030\n",
      "001-0003-1 (thread    8) identical     001-0004-1 (thread    4) identical     \n",
      "Converged.\n",
      "Segment  29/ 40 3030-3181\n",
      "001-0004-1 (thread    4) identical     \n",
      "Converged.\n",
      "Segment  30/ 40 3181-3332\n",
      "001-0004-0 (thread    8) identical     \n",
      "Converged.\n",
      "Segment  31/ 40 3332-3426\n",
      "001-0004-0 (thread    4) identical     \n",
      "Converged.\n",
      "Segment  32/ 40 3426-3450\n",
      "001-0004-1 (thread    5) identical     \n",
      "Converged.\n",
      "Segment  33/ 40 3450-3458\n",
      "001-0004-0 (thread    4) identical     \n",
      "Converged.\n",
      "Segment  34/ 40 3458-3537\n",
      "001-0004-1 (thread    4) identical     \n",
      "Converged.\n",
      "Segment  35/ 40 3537-3688\n",
      "001-0004-1 (thread    3) identical     \n",
      "Converged.\n",
      "Segment  36/ 40 3688-3839\n",
      "001-0004-1 (thread    4) identical     \n",
      "Converged.\n",
      "Segment  37/ 40 3839-3990\n",
      "001-0004-0 (thread    7) identical     \n",
      "Converged.\n",
      "Segment  38/ 40 3990-4141\n",
      "001-0004-0 (thread    3) identical     \n",
      "Converged.\n",
      "Segment  39/ 40 4141-4218\n",
      "001-0004-1 (thread    6) identical     \n",
      "Converged.\n",
      "Segment  40/ 40 4218-4229\n",
      "001-0004-0 (thread    6) identical     \n",
      "Converged.\n",
      "done\n",
      "dvtditr (nuc) Version 7.526\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " FFT-NS-i (Standard)\n",
      " Iterative refinement method (max. 2 iterations)\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 8\n",
      "nadd = 0\n",
      "inputfile = infile\n",
      "thresholdtorev = 0.000000\n",
      "reflim = 5000\n",
      "mode = a\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "\n",
      "Step 1/2\n",
      "1   \n",
      "\n",
      "Step 2/2\n",
      " 1 / 4 (8 threads)   \n",
      "makedirectionlist (nuc) Version 7.526\n",
      "alg=m, model=DNA200 (2), 1.53 (4.59), 0.37 (1.11), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "directionfile = _direction\n",
      "inputfile = infile\n",
      "subalignment = 0\n",
      "subalignmentoffset = 0\n",
      "4 x 864 - 771 d\n",
      "outputhat23=16\n",
      "treein = 0\n",
      "compacttree = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "All-to-all alignment.\n",
      "tbfast-pair (nuc) Version 7.526\n",
      "alg=L, model=DNA200 (2), 2.00 (6.00), -0.10 (-0.30), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "outputhat23=16\n",
      "Loading 'hat3.seed' ... \n",
      "done.\n",
      "Writing hat3 for iterative refinement\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "tbutree = 1, compacttree = 0\n",
      "Constructing a UPGMA tree ... \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment ... \n",
      "STEP     3 /3 (thread    3) \n",
      "done.\n",
      "tbfast (nuc) Version 7.526\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "minimumweight = 0.000010\n",
      "autosubalignment = 0.000000\n",
      "nthread = 8\n",
      "randomseed = 0\n",
      "blosum 62 / kimura 200\n",
      "poffset = 0\n",
      "niter = 16\n",
      "sueff_global = 0.100000\n",
      "nadd = 16\n",
      "Loading 'hat3' ... done.\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "\n",
      "    0 / 4\n",
      "Segment   1/  1    1- 901\n",
      "002-0004-1 (thread    6) identical     001-0003-1 (thread    3) identical     002-0001-0 (thread    7) identical     \n",
      "Converged.\n",
      "done\n",
      "dvtditr (nuc) Version 7.526\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " L-INS-i (Probably most accurate, very slow)\n",
      " Iterative refinement method (<16) with LOCAL pairwise alignment information\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 8\n",
      "nadd = 0\n",
      "inputfile = infile\n",
      "thresholdtorev = 0.000000\n",
      "reflim = 5000\n",
      "mode = a\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "\n",
      "Step 1/2\n",
      "1   \n",
      "\n",
      "Step 2/2\n",
      " 1 / 4 (8 threads)   \n",
      "makedirectionlist (nuc) Version 7.526\n",
      "alg=m, model=DNA200 (2), 1.53 (4.59), 0.37 (1.11), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "directionfile = _direction\n",
      "inputfile = infile\n",
      "subalignment = 0\n",
      "subalignmentoffset = 0\n",
      "4 x 1599 - 1230 d\n",
      "outputhat23=16\n",
      "treein = 0\n",
      "compacttree = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "All-to-all alignment.\n",
      "tbfast-pair (nuc) Version 7.526\n",
      "alg=L, model=DNA200 (2), 2.00 (6.00), -0.10 (-0.30), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "outputhat23=16\n",
      "Loading 'hat3.seed' ... \n",
      "done.\n",
      "Writing hat3 for iterative refinement\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "tbutree = 1, compacttree = 0\n",
      "Constructing a UPGMA tree ... \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment ... \n",
      "STEP     3 /3 (thread    5) \n",
      "done.\n",
      "tbfast (nuc) Version 7.526\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "minimumweight = 0.000010\n",
      "autosubalignment = 0.000000\n",
      "nthread = 8\n",
      "randomseed = 0\n",
      "blosum 62 / kimura 200\n",
      "poffset = 0\n",
      "niter = 16\n",
      "sueff_global = 0.100000\n",
      "nadd = 16\n",
      "Loading 'hat3' ... done.\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "\n",
      "    0 / 4\n",
      "Segment   1/  1    1-1651\n",
      "001-0004-0 (thread    2) identical     001-0001-1 (thread    3) identical     \n",
      "Converged.\n",
      "done\n",
      "dvtditr (nuc) Version 7.526\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " L-INS-i (Probably most accurate, very slow)\n",
      " Iterative refinement method (<16) with LOCAL pairwise alignment information\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 8\n",
      "nadd = 0\n",
      "inputfile = infile\n",
      "thresholdtorev = 0.000000\n",
      "reflim = 5000\n",
      "mode = a\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "\n",
      "Step 1/2\n",
      "1   \n",
      "\n",
      "Step 2/2\n",
      " 1 / 4 (8 threads)   \n",
      "makedirectionlist (nuc) Version 7.526\n",
      "alg=m, model=DNA200 (2), 1.53 (4.59), 0.37 (1.11), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "directionfile = _direction\n",
      "inputfile = infile\n",
      "subalignment = 0\n",
      "subalignmentoffset = 0\n",
      "4 x 984 - 978 d\n",
      "outputhat23=16\n",
      "treein = 0\n",
      "compacttree = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "All-to-all alignment.\n",
      "tbfast-pair (nuc) Version 7.526\n",
      "alg=L, model=DNA200 (2), 2.00 (6.00), -0.10 (-0.30), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "outputhat23=16\n",
      "Loading 'hat3.seed' ... \n",
      "done.\n",
      "Writing hat3 for iterative refinement\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "tbutree = 1, compacttree = 0\n",
      "Constructing a UPGMA tree ... \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment ... \n",
      "STEP     3 /3 (thread    4) \n",
      "done.\n",
      "tbfast (nuc) Version 7.526\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "minimumweight = 0.000010\n",
      "autosubalignment = 0.000000\n",
      "nthread = 8\n",
      "randomseed = 0\n",
      "blosum 62 / kimura 200\n",
      "poffset = 0\n",
      "niter = 16\n",
      "sueff_global = 0.100000\n",
      "nadd = 16\n",
      "Loading 'hat3' ... done.\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "\n",
      "    0 / 4\n",
      "Segment   1/  1    1- 994\n",
      "001-0004-1 (thread    8) identical     \n",
      "Converged.\n",
      "done\n",
      "dvtditr (nuc) Version 7.526\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " L-INS-i (Probably most accurate, very slow)\n",
      " Iterative refinement method (<16) with LOCAL pairwise alignment information\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 8\n",
      "nadd = 0\n",
      "inputfile = infile\n",
      "thresholdtorev = 0.000000\n",
      "reflim = 5000\n",
      "mode = a\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "\n",
      "Step 1/2\n",
      "1   \n",
      "\n",
      "Step 2/2\n",
      " 1 / 4 (8 threads)   \n",
      "makedirectionlist (nuc) Version 7.526\n",
      "alg=m, model=DNA200 (2), 1.53 (4.59), 0.37 (1.11), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "directionfile = _direction\n",
      "inputfile = infile\n",
      "subalignment = 0\n",
      "subalignmentoffset = 0\n",
      "4 x 1008 - 999 d\n",
      "outputhat23=16\n",
      "treein = 0\n",
      "compacttree = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "All-to-all alignment.\n",
      "tbfast-pair (nuc) Version 7.526\n",
      "alg=L, model=DNA200 (2), 2.00 (6.00), -0.10 (-0.30), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "outputhat23=16\n",
      "Loading 'hat3.seed' ... \n",
      "done.\n",
      "Writing hat3 for iterative refinement\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "tbutree = 1, compacttree = 0\n",
      "Constructing a UPGMA tree ... \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment ... \n",
      "STEP     3 /3 (thread    6) \n",
      "done.\n",
      "tbfast (nuc) Version 7.526\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "minimumweight = 0.000010\n",
      "autosubalignment = 0.000000\n",
      "nthread = 8\n",
      "randomseed = 0\n",
      "blosum 62 / kimura 200\n",
      "poffset = 0\n",
      "niter = 16\n",
      "sueff_global = 0.100000\n",
      "nadd = 16\n",
      "Loading 'hat3' ... done.\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "\n",
      "    0 / 4\n",
      "Segment   1/  1    1-1015\n",
      "002-0004-1 (thread    3) identical     001-0002-1 (thread    7) identical     002-0002-1 (thread    1) identical     \n",
      "Converged.\n",
      "done\n",
      "dvtditr (nuc) Version 7.526\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " L-INS-i (Probably most accurate, very slow)\n",
      " Iterative refinement method (<16) with LOCAL pairwise alignment information\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 8\n",
      "nadd = 0\n",
      "inputfile = infile\n",
      "thresholdtorev = 0.000000\n",
      "reflim = 5000\n",
      "mode = a\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "\n",
      "Step 1/2\n",
      "1   \n",
      "\n",
      "Step 2/2\n",
      " 1 / 4 (8 threads)   \n",
      "makedirectionlist (nuc) Version 7.526\n",
      "alg=m, model=DNA200 (2), 1.53 (4.59), 0.37 (1.11), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "directionfile = _direction\n",
      "inputfile = infile\n",
      "subalignment = 0\n",
      "subalignmentoffset = 0\n",
      "4 x 795 - 795 d\n",
      "outputhat23=16\n",
      "treein = 0\n",
      "compacttree = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "All-to-all alignment.\n",
      "tbfast-pair (nuc) Version 7.526\n",
      "alg=L, model=DNA200 (2), 2.00 (6.00), -0.10 (-0.30), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "outputhat23=16\n",
      "Loading 'hat3.seed' ... \n",
      "done.\n",
      "Writing hat3 for iterative refinement\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "tbutree = 1, compacttree = 0\n",
      "Constructing a UPGMA tree ... \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment ... \n",
      "STEP     3 /3 (thread    3) \n",
      "done.\n",
      "tbfast (nuc) Version 7.526\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "minimumweight = 0.000010\n",
      "autosubalignment = 0.000000\n",
      "nthread = 8\n",
      "randomseed = 0\n",
      "blosum 62 / kimura 200\n",
      "poffset = 0\n",
      "niter = 16\n",
      "sueff_global = 0.100000\n",
      "nadd = 16\n",
      "Loading 'hat3' ... done.\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "\n",
      "    0 / 4\n",
      "Segment   1/  1    1- 796\n",
      "001-0004-1 (thread    8) identical     \n",
      "Converged.\n",
      "done\n",
      "dvtditr (nuc) Version 7.526\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " L-INS-i (Probably most accurate, very slow)\n",
      " Iterative refinement method (<16) with LOCAL pairwise alignment information\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 8\n",
      "nadd = 0\n",
      "inputfile = infile\n",
      "thresholdtorev = 0.000000\n",
      "reflim = 5000\n",
      "mode = a\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "\n",
      "Step 1/2\n",
      "1   \n",
      "\n",
      "Step 2/2\n",
      " 1 / 4 (8 threads)   \n",
      "makedirectionlist (nuc) Version 7.526\n",
      "alg=m, model=DNA200 (2), 1.53 (4.59), 0.37 (1.11), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "directionfile = _direction\n",
      "inputfile = infile\n",
      "subalignment = 0\n",
      "subalignmentoffset = 0\n",
      "4 x 810 - 810 d\n",
      "outputhat23=16\n",
      "treein = 0\n",
      "compacttree = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "All-to-all alignment.\n",
      "tbfast-pair (nuc) Version 7.526\n",
      "alg=L, model=DNA200 (2), 2.00 (6.00), -0.10 (-0.30), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "outputhat23=16\n",
      "Loading 'hat3.seed' ... \n",
      "done.\n",
      "Writing hat3 for iterative refinement\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "tbutree = 1, compacttree = 0\n",
      "Constructing a UPGMA tree ... \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment ... \n",
      "STEP     3 /3 (thread    5) \n",
      "done.\n",
      "tbfast (nuc) Version 7.526\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "minimumweight = 0.000010\n",
      "autosubalignment = 0.000000\n",
      "nthread = 8\n",
      "randomseed = 0\n",
      "blosum 62 / kimura 200\n",
      "poffset = 0\n",
      "niter = 16\n",
      "sueff_global = 0.100000\n",
      "nadd = 16\n",
      "Loading 'hat3' ... done.\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "\n",
      "    0 / 4\n",
      "Segment   1/  1    1- 811\n",
      "001-0004-0 (thread    3) identical     \n",
      "Converged.\n",
      "done\n",
      "dvtditr (nuc) Version 7.526\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " L-INS-i (Probably most accurate, very slow)\n",
      " Iterative refinement method (<16) with LOCAL pairwise alignment information\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n",
      "nthread = 8\n",
      "nadd = 0\n",
      "inputfile = infile\n",
      "thresholdtorev = 0.000000\n",
      "reflim = 5000\n",
      "mode = a\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "\n",
      "Step 1/2\n",
      "1   \n",
      "\n",
      "Step 2/2\n",
      " 1 / 4 (8 threads)   \n",
      "makedirectionlist (nuc) Version 7.526\n",
      "alg=m, model=DNA200 (2), 1.53 (4.59), 0.37 (1.11), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "directionfile = _direction\n",
      "inputfile = infile\n",
      "subalignment = 0\n",
      "subalignmentoffset = 0\n",
      "4 x 1209 - 1206 d\n",
      "outputhat23=16\n",
      "treein = 0\n",
      "compacttree = 0\n",
      "stacksize: 8192 kb\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "All-to-all alignment.\n",
      "tbfast-pair (nuc) Version 7.526\n",
      "alg=L, model=DNA200 (2), 2.00 (6.00), -0.10 (-0.30), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "outputhat23=16\n",
      "Loading 'hat3.seed' ... \n",
      "done.\n",
      "Writing hat3 for iterative refinement\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "Gap Penalty = -1.53, +0.00, +0.00\n",
      "tbutree = 1, compacttree = 0\n",
      "Constructing a UPGMA tree ... \n",
      "    0 / 4\n",
      "done.\n",
      "\n",
      "Progressive alignment ... \n",
      "STEP     3 /3 (thread    3) \n",
      "done.\n",
      "tbfast (nuc) Version 7.526\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "minimumweight = 0.000010\n",
      "autosubalignment = 0.000000\n",
      "nthread = 8\n",
      "randomseed = 0\n",
      "blosum 62 / kimura 200\n",
      "poffset = 0\n",
      "niter = 16\n",
      "sueff_global = 0.100000\n",
      "nadd = 16\n",
      "Loading 'hat3' ... done.\n",
      "generating a scoring matrix for nucleotide (dist=200) ... done\n",
      "\n",
      "    0 / 4\n",
      "Segment   1/  1    1-1210\n",
      "002-0004-0 (thread    7) identical     002-0000-1 (thread    2) identical     \n",
      "Converged.\n",
      "done\n",
      "dvtditr (nuc) Version 7.526\n",
      "alg=A, model=DNA200 (2), 1.53 (4.59), -0.00 (-0.00), noshift, amax=0.0\n",
      "8 thread(s)\n",
      "\n",
      "\n",
      "Strategy:\n",
      " L-INS-i (Probably most accurate, very slow)\n",
      " Iterative refinement method (<16) with LOCAL pairwise alignment information\n",
      "\n",
      "If unsure which option to use, try 'mafft --auto input > output'.\n",
      "For more information, see 'mafft --help', 'mafft --man' and the mafft page.\n",
      "\n",
      "The default gap scoring scheme has been changed in version 7.110 (2013 Oct).\n",
      "It tends to insert more gaps into gap-rich regions than previous versions.\n",
      "To disable this change, add the --leavegappyregion option.\n",
      "\n"
     ]
    }
   ],
   "source": [
    "# 使用mafft完成比对\n",
    "! mkdir -p result/phylo_fna_gene\n",
    "! ls result/gene_fna/ | while read file;do mafft --thread 8 --adjustdirection --auto result/gene_fna/${file} > result/phylo_fna_gene/${file%.fna}_aln.fna;done"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "#使用trimal完成修剪，去除gap\n",
    "! ls result/phylo_fna_gene/*_aln.fna | while read file; do trimal -in ${file} -out ${file%_aln.fna}_AlnTrimed.fna -gappyout;done\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "IQ-TREE multicore version 2.3.6 for Linux x86 64-bit built Aug  4 2024\n",
      "Developed by Bui Quang Minh, Nguyen Lam Tung, Olga Chernomor, Heiko Schmidt,\n",
      "Dominik Schrempf, Michael Woodhams, Ly Trong Nhan, Thomas Wong\n",
      "\n",
      "Host:    codespaces-fa7cf2 (AVX2, FMA3, 7 GB RAM)\n",
      "Command: iqtree -s result/phylo_fna_gene/100045at50557_AlnTrimed.fna -nt 2 -m MFP -bb 1000 -alrt 1000 -pre result/phylo_fna_gene/100045at50557_iqtreeout/.tree\n",
      "Seed:    515142 (Using SPRNG - Scalable Parallel Random Number Generator)\n",
      "Time:    Sun Sep  1 10:03:31 2024\n",
      "Kernel:  AVX+FMA - 2 threads (2 CPU cores detected)\n",
      "\n",
      "Reading alignment file result/phylo_fna_gene/100045at50557_AlnTrimed.fna ... Fasta format detected\n",
      "Reading fasta file: done in 0.000112009 secs using 98.21% CPU\n",
      "Alignment most likely contains DNA/RNA sequences\n",
      "Alignment has 4 sequences with 897 columns, 58 distinct patterns\n",
      "15 parsimony-informative, 286 singleton sites, 596 constant sites\n",
      "                        Gap/Ambiguity  Composition  p-value\n",
      "Analyzing sequences: done in 1.3335e-05 secs using 89.99% CPU\n",
      "   1  Culex_pipiens             0.00%    passed     77.58%\n",
      "   2  Culex_pipiens_pallens     0.00%    passed     67.86%\n",
      "   3  Armigeres_subalbatus      0.00%    failed      0.04%\n",
      "   4  Culex_pipiens_molestus    0.00%    passed     23.83%\n",
      "****  TOTAL                     0.00%  1 sequences failed composition chi2 test (p-value<5%; df=3)\n",
      "\n",
      "\n",
      "Create initial parsimony tree by phylogenetic likelihood library (PLL)... 0.000 seconds\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "Perform fast likelihood tree search using GTR+I+G model...\n",
      "Estimate model parameters (epsilon = 5.000)\n",
      "Perform nearest neighbor interchange...\n",
      "Estimate model parameters (epsilon = 1.000)\n",
      "1. Initial log-likelihood: -2394.120\n",
      "Optimal log-likelihood: -2394.072\n",
      "Rate parameters:  A-C: 1.73069  A-G: 4.29870  A-T: 1.77889  C-G: 0.92976  C-T: 9.41889  G-T: 1.00000\n",
      "Base frequencies:  A: 0.305  C: 0.231  G: 0.320  T: 0.144\n",
      "Proportion of invariable sites: 0.332\n",
      "Gamma shape alpha: 1.392\n",
      "Parameters optimization took 1 rounds (0.002 sec)\n",
      "Time for fast ML tree search: 0.008 seconds\n",
      "\n",
      "NOTE: ModelFinder requires 0 MB RAM!\n",
      "ModelFinder will test up to 484 DNA models (sample size: 897 epsilon: 0.100) ...\n",
      " No. Model         -LnL         df  AIC          AICc         BIC\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  1  GTR+F         2402.092     13  4830.185     4830.597     4892.573\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  2  GTR+F+I       2392.957     14  4813.913     4814.390     4881.100\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  3  GTR+F+G4      2394.582     14  4817.164     4817.641     4884.351\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  4  GTR+F+I+G4    2394.116     15  4818.232     4818.776     4890.217\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  5  GTR+F+R2      2394.099     15  4818.197     4818.742     4890.183\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "+R3 reinitialized from +R2 with factor 0.500\n",
      "  6  GTR+F+R3      2394.012     17  4822.024     4822.721     4903.608\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 14  GTR+F+I+R2    2393.678     16  4819.356     4819.975     4896.141\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 15  GTR+F+I+R3    2393.540     18  4823.081     4823.860     4909.464\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 24  SYM+I         2435.448     11  4892.895     4893.194     4945.685\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 25  SYM+G4        2435.249     11  4892.498     4892.796     4945.287\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 26  SYM+I+G4      2435.131     12  4894.261     4894.614     4951.850\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 27  SYM+R2        2435.237     12  4894.475     4894.828     4952.063\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 46  TVM+F+I       2396.193     13  4818.386     4818.798     4880.773\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 47  TVM+F+G4      2399.032     13  4824.064     4824.476     4886.451\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 48  TVM+F+I+G4    2398.702     14  4825.404     4825.880     4892.591\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 49  TVM+F+R2      2396.848     14  4821.695     4822.171     4888.882\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 68  TVMe+I        2435.471     10  4890.943     4891.191     4938.933\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 69  TVMe+G4       2435.251     10  4890.502     4890.751     4938.493\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 70  TVMe+I+G4     2435.163     11  4892.327     4892.625     4945.116\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 71  TVMe+R2       2435.504     11  4893.007     4893.306     4945.797\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 90  TIM3+F+I      2395.197     12  4814.393     4814.746     4871.982\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 91  TIM3+F+G4     2395.638     12  4815.277     4815.630     4872.865\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 92  TIM3+F+I+G4   2395.480     13  4816.959     4817.371     4879.347\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 93  TIM3+F+R2     2395.328     13  4816.656     4817.068     4879.044\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "112  TIM3e+I       2435.945     9   4889.890     4890.093     4933.082\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "113  TIM3e+G4      2435.804     9   4889.608     4889.811     4932.799\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "114  TIM3e+I+G4    2435.886     10  4891.772     4892.020     4939.763\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "115  TIM3e+R2      2436.130     10  4892.260     4892.509     4940.251\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "134  TIM2+F+I      2393.244     12  4810.487     4810.840     4868.076\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "135  TIM2+F+G4     2394.522     12  4813.044     4813.397     4870.633\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "136  TIM2+F+I+G4   2394.199     13  4814.398     4814.810     4876.786\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "137  TIM2+F+R2     2393.732     13  4813.463     4813.876     4875.851\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "156  TIM2e+I       2437.231     9   4892.462     4892.665     4935.654\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "157  TIM2e+G4      2438.270     9   4894.540     4894.743     4937.732\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "158  TIM2e+I+G4    2437.936     10  4895.871     4896.119     4943.862\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "159  TIM2e+R2      2437.492     10  4894.983     4895.232     4942.974\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "178  TIM+F+I       2395.101     12  4814.202     4814.554     4871.790\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "179  TIM+F+G4      2395.739     12  4815.477     4815.830     4873.066\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "180  TIM+F+I+G4    2395.416     13  4816.832     4817.244     4879.220\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "181  TIM+F+R2      2395.207     13  4816.415     4816.827     4878.802\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "200  TIMe+I        2439.072     9   4896.143     4896.346     4939.335\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "201  TIMe+G4       2439.436     9   4896.872     4897.075     4940.063\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "202  TIMe+I+G4     2439.208     10  4898.417     4898.665     4946.407\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "203  TIMe+R2       2439.080     10  4898.160     4898.409     4946.151\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "222  TPM3u+F+I     2400.005     11  4822.010     4822.308     4874.799\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "223  TPM3u+F+G4    2400.676     11  4823.352     4823.650     4876.142\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "224  TPM3u+F+I+G4  2400.386     12  4824.771     4825.124     4882.360\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "225  TPM3u+F+R2    2400.164     12  4824.328     4824.681     4881.917\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "244  TPM3+I        2435.915     8   4887.831     4887.993     4926.223\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "245  TPM3+G4       2435.802     8   4887.604     4887.766     4925.997\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "246  TPM3+I+G4     2435.910     9   4889.821     4890.024     4933.012\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "247  TPM3+R2       2436.179     9   4890.357     4890.560     4933.549\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "266  TPM2u+F+I     2396.438     11  4814.877     4815.175     4867.666\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "267  TPM2u+F+G4    2399.083     11  4820.167     4820.465     4872.956\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "268  TPM2u+F+I+G4  2398.714     12  4821.428     4821.781     4879.017\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "269  TPM2u+F+R2    2396.824     12  4817.648     4818.001     4875.236\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "288  TPM2+I        2437.282     8   4890.564     4890.726     4928.956\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "289  TPM2+G4       2438.391     8   4892.782     4892.945     4931.175\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "290  TPM2+I+G4     2437.944     9   4893.888     4894.091     4937.080\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "291  TPM2+R2       2437.335     9   4892.669     4892.872     4935.861\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "310  K3Pu+F+I      2399.907     11  4821.813     4822.111     4874.603\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "311  K3Pu+F+G4     2400.934     11  4823.868     4824.166     4876.658\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "312  K3Pu+F+I+G4   2400.526     12  4825.052     4825.405     4882.641\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "313  K3Pu+F+R2     2400.023     12  4824.046     4824.399     4881.635\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "332  K3P+I         2439.044     8   4894.087     4894.249     4932.480\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "333  K3P+G4        2439.422     8   4894.844     4895.006     4933.237\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "334  K3P+I+G4      2439.204     9   4896.407     4896.610     4939.599\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "335  K3P+R2        2439.137     9   4896.275     4896.478     4939.466\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "354  TN+F+I        2395.185     11  4812.369     4812.667     4865.159\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "355  TN+F+G4       2395.767     11  4813.533     4813.832     4866.323\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "356  TN+F+I+G4     2395.447     12  4814.893     4815.246     4872.482\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "357  TN+F+R2       2395.251     12  4814.502     4814.855     4872.091\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "376  TNe+I         2439.134     8   4894.268     4894.430     4932.660\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "377  TNe+G4        2439.429     8   4894.858     4895.020     4933.251\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "378  TNe+I+G4      2439.202     9   4896.403     4896.606     4939.595\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "379  TNe+R2        2439.148     9   4896.295     4896.498     4939.487\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "398  HKY+F+I       2400.058     10  4820.116     4820.364     4868.106\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "399  HKY+F+G4      2400.884     10  4821.768     4822.017     4869.759\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "400  HKY+F+I+G4    2400.596     11  4823.193     4823.491     4875.982\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "401  HKY+F+R2      2400.284     11  4822.569     4822.867     4875.358\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "420  K2P+I         2439.084     7   4892.168     4892.294     4925.761\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "421  K2P+G4        2439.397     7   4892.795     4892.921     4926.388\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "422  K2P+I+G4      2439.224     8   4894.448     4894.610     4932.840\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "423  K2P+R2        2439.146     8   4894.293     4894.455     4932.685\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "442  F81+F+I       2434.458     9   4886.917     4887.119     4930.108\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "443  F81+F+G4      2434.255     9   4886.509     4886.712     4929.701\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "444  F81+F+I+G4    2434.431     10  4888.863     4889.111     4936.853\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "445  F81+F+R2      2434.732     10  4889.465     4889.713     4937.455\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "464  JC+I          2480.343     6   4972.685     4972.779     5001.479\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "465  JC+G4         2480.192     6   4972.385     4972.479     5001.179\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "466  JC+I+G4       2480.338     7   4974.677     4974.803     5008.270\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "467  JC+R2         2480.625     7   4975.250     4975.376     5008.844\n",
      "Akaike Information Criterion:           TIM2+F+I\n",
      "Corrected Akaike Information Criterion: TIM2+F+I\n",
      "Bayesian Information Criterion:         TN+F+I\n",
      "Best-fit model: TN+F+I chosen according to BIC\n",
      "\n",
      "All model information printed to result/phylo_fna_gene/100045at50557_iqtreeout/.tree.model.gz\n",
      "CPU time for ModelFinder: 0.591 seconds (0h:0m:0s)\n",
      "Wall-clock time for ModelFinder: 0.524 seconds (0h:0m:0s)\n",
      "Generating 1000 samples for ultrafast bootstrap (seed: 515142)...\n",
      "\n",
      "NOTE: 0 MB RAM (0 GB) is required!\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "Estimate model parameters (epsilon = 0.100)\n",
      "1. Initial log-likelihood: -2434.593\n",
      "2. Current log-likelihood: -2397.513\n",
      "3. Current log-likelihood: -2396.009\n",
      "4. Current log-likelihood: -2395.608\n",
      "5. Current log-likelihood: -2395.403\n",
      "Optimal log-likelihood: -2395.278\n",
      "Rate parameters:  A-C: 1.00000  A-G: 2.89105  A-T: 1.00000  C-G: 1.00000  C-T: 6.53244  G-T: 1.00000\n",
      "Base frequencies:  A: 0.305  C: 0.231  G: 0.320  T: 0.144\n",
      "Proportion of invariable sites: 0.439\n",
      "Parameters optimization took 5 rounds (0.004 sec)\n",
      "Wrote distance file to... \n",
      "Computing ML distances based on estimated model parameters...\n",
      "Calculating distance matrix: done in 3.9794e-05 secs using 163.3% CPU\n",
      "Computing ML distances took 0.001076 sec (of wall-clock time) 0.001094 sec (of CPU time)\n",
      "Setting up auxiliary I and S matrices: done in 0.000180246 secs using 94.32% CPU\n",
      "Computing RapidNJ tree took 0.000554 sec (of wall-clock time) 0.000406 sec (of CPU time)\n",
      "Log-likelihood of RapidNJ tree: -2395.264\n",
      "--------------------------------------------------------------------\n",
      "|             INITIALIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "Generating 99 parsimony trees... 0.010 second\n",
      "Computing log-likelihood of 2 initial trees ... 0.000 seconds\n",
      "Current best score: -2395.264\n",
      "\n",
      "Do NNI search on 3 best initial trees\n",
      "Finish initializing candidate tree set (3)\n",
      "Current best tree score: -2395.264 / CPU time: 0.018\n",
      "Number of iterations: 3\n",
      "--------------------------------------------------------------------\n",
      "|               OPTIMIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2395.263\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2395.263\n",
      "Iteration 10 / LogL: -2395.277 / Time: 0h:0m:0s\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2395.263\n",
      "Iteration 20 / LogL: -2395.264 / Time: 0h:0m:0s\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2395.263\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2395.263\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2395.263\n",
      "Iteration 30 / LogL: -2395.264 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2395.263\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2395.263\n",
      "Iteration 40 / LogL: -2395.277 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2395.263\n",
      "Iteration 50 / LogL: -2395.277 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -2397.245\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2395.263\n",
      "Iteration 60 / LogL: -2395.277 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2395.263\n",
      "Iteration 70 / LogL: -2395.277 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2395.263\n",
      "Iteration 80 / LogL: -2395.264 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2395.263\n",
      "Iteration 90 / LogL: -2395.277 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 100 / LogL: -2395.264 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -2397.245\n",
      "NOTE: Bootstrap correlation coefficient of split occurrence frequencies: 0.816\n",
      "NOTE: UFBoot does not converge, continue at least 100 more iterations\n",
      "Iteration 110 / LogL: -2395.277 / Time: 0h:0m:1s (0h:0m:0s left)\n",
      "Iteration 120 / LogL: -2395.277 / Time: 0h:0m:1s (0h:0m:0s left)\n",
      "Iteration 130 / LogL: -2395.264 / Time: 0h:0m:1s (0h:0m:0s left)\n",
      "Iteration 140 / LogL: -2395.277 / Time: 0h:0m:1s (0h:0m:0s left)\n",
      "Iteration 150 / LogL: -2395.277 / Time: 0h:0m:1s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -2397.245\n",
      "Iteration 160 / LogL: -2395.277 / Time: 0h:0m:1s (0h:0m:0s left)\n",
      "Iteration 170 / LogL: -2395.277 / Time: 0h:0m:1s (0h:0m:0s left)\n",
      "Iteration 180 / LogL: -2395.277 / Time: 0h:0m:1s (0h:0m:0s left)\n",
      "Iteration 190 / LogL: -2395.277 / Time: 0h:0m:1s (0h:0m:0s left)\n",
      "Iteration 200 / LogL: -2395.263 / Time: 0h:0m:1s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -2397.245\n",
      "NOTE: Bootstrap correlation coefficient of split occurrence frequencies: 1.000\n",
      "TREE SEARCH COMPLETED AFTER 200 ITERATIONS / Time: 0h:0m:1s\n",
      "\n",
      "--------------------------------------------------------------------\n",
      "|                    FINALIZING TREE SEARCH                        |\n",
      "--------------------------------------------------------------------\n",
      "Performs final model parameters optimization\n",
      "Estimate model parameters (epsilon = 0.010)\n",
      "1. Initial log-likelihood: -2395.263\n",
      "2. Current log-likelihood: -2395.233\n",
      "3. Current log-likelihood: -2395.203\n",
      "4. Current log-likelihood: -2395.188\n",
      "Optimal log-likelihood: -2395.179\n",
      "Rate parameters:  A-C: 1.00000  A-G: 3.03307  A-T: 1.00000  C-G: 1.00000  C-T: 6.96196  G-T: 1.00000\n",
      "Base frequencies:  A: 0.305  C: 0.231  G: 0.320  T: 0.144\n",
      "Proportion of invariable sites: 0.454\n",
      "Parameters optimization took 4 rounds (0.036 sec)\n",
      "BEST SCORE FOUND : -2395.179\n",
      "\n",
      "Testing tree branches by SH-like aLRT with 1000 replicates...\n",
      "0.002 sec.\n",
      "Creating bootstrap support values...\n",
      "Split supports printed to NEXUS file result/phylo_fna_gene/100045at50557_iqtreeout/.tree.splits.nex\n",
      "Total tree length: 0.716\n",
      "\n",
      "Total number of iterations: 200\n",
      "CPU time used for tree search: 1.463 sec (0h:0m:1s)\n",
      "Wall-clock time used for tree search: 0.843 sec (0h:0m:0s)\n",
      "Total CPU time used: 2.162 sec (0h:0m:2s)\n",
      "Total wall-clock time used: 1.474 sec (0h:0m:1s)\n",
      "\n",
      "Computing bootstrap consensus tree...\n",
      "Reading input file result/phylo_fna_gene/100045at50557_iqtreeout/.tree.splits.nex...\n",
      "4 taxa and 7 splits.\n",
      "Consensus tree written to result/phylo_fna_gene/100045at50557_iqtreeout/.tree.contree\n",
      "Reading input trees file result/phylo_fna_gene/100045at50557_iqtreeout/.tree.contree\n",
      "Log-likelihood of consensus tree: -2395.178\n",
      "\n",
      "Analysis results written to: \n",
      "  IQ-TREE report:                result/phylo_fna_gene/100045at50557_iqtreeout/.tree.iqtree\n",
      "  Maximum-likelihood tree:       result/phylo_fna_gene/100045at50557_iqtreeout/.tree.treefile\n",
      "  Likelihood distances:          result/phylo_fna_gene/100045at50557_iqtreeout/.tree.mldist\n",
      "\n",
      "Ultrafast bootstrap approximation results written to:\n",
      "  Split support values:          result/phylo_fna_gene/100045at50557_iqtreeout/.tree.splits.nex\n",
      "  Consensus tree:                result/phylo_fna_gene/100045at50557_iqtreeout/.tree.contree\n",
      "  Screen log file:               result/phylo_fna_gene/100045at50557_iqtreeout/.tree.log\n",
      "\n",
      "Date and Time: Sun Sep  1 10:03:33 2024\n",
      "IQ-TREE multicore version 2.3.6 for Linux x86 64-bit built Aug  4 2024\n",
      "Developed by Bui Quang Minh, Nguyen Lam Tung, Olga Chernomor, Heiko Schmidt,\n",
      "Dominik Schrempf, Michael Woodhams, Ly Trong Nhan, Thomas Wong\n",
      "\n",
      "Host:    codespaces-fa7cf2 (AVX2, FMA3, 7 GB RAM)\n",
      "Command: iqtree -s result/phylo_fna_gene/10008at50557_AlnTrimed.fna -nt 2 -m MFP -bb 1000 -alrt 1000 -pre result/phylo_fna_gene/10008at50557_iqtreeout/.tree\n",
      "Seed:    14625 (Using SPRNG - Scalable Parallel Random Number Generator)\n",
      "Time:    Sun Sep  1 10:03:33 2024\n",
      "Kernel:  AVX+FMA - 2 threads (2 CPU cores detected)\n",
      "\n",
      "Reading alignment file result/phylo_fna_gene/10008at50557_AlnTrimed.fna ... Fasta format detected\n",
      "Reading fasta file: done in 0.000247271 secs\n",
      "Alignment most likely contains DNA/RNA sequences\n",
      "Alignment has 4 sequences with 3564 columns, 110 distinct patterns\n",
      "46 parsimony-informative, 838 singleton sites, 2680 constant sites\n",
      "                        Gap/Ambiguity  Composition  p-value\n",
      "Analyzing sequences: done in 1.3566e-05 secs using 22.11% CPU\n",
      "   1  Culex_pipiens             0.17%    passed     52.50%\n",
      "   2  Culex_pipiens_pallens     0.08%    passed     78.69%\n",
      "   3  Armigeres_subalbatus      0.67%    failed      0.39%\n",
      "   4  Culex_pipiens_molestus    0.00%    passed     70.65%\n",
      "****  TOTAL                     0.23%  1 sequences failed composition chi2 test (p-value<5%; df=3)\n",
      "\n",
      "\n",
      "Create initial parsimony tree by phylogenetic likelihood library (PLL)... 0.000 seconds\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "Perform fast likelihood tree search using GTR+I+G model...\n",
      "Estimate model parameters (epsilon = 5.000)\n",
      "Perform nearest neighbor interchange...\n",
      "Estimate model parameters (epsilon = 1.000)\n",
      "1. Initial log-likelihood: -8597.386\n",
      "Optimal log-likelihood: -8597.382\n",
      "Rate parameters:  A-C: 0.86491  A-G: 3.10692  A-T: 0.82292  C-G: 1.41086  C-T: 5.01880  G-T: 1.00000\n",
      "Base frequencies:  A: 0.235  C: 0.274  G: 0.299  T: 0.192\n",
      "Proportion of invariable sites: 0.375\n",
      "Gamma shape alpha: 1.250\n",
      "Parameters optimization took 1 rounds (0.002 sec)\n",
      "Time for fast ML tree search: 0.006 seconds\n",
      "\n",
      "NOTE: ModelFinder requires 0 MB RAM!\n",
      "ModelFinder will test up to 484 DNA models (sample size: 3564 epsilon: 0.100) ...\n",
      " No. Model         -LnL         df  AIC          AICc         BIC\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  1  GTR+F         8627.406     13  17280.812    17280.914    17361.134\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  2  GTR+F+I       8597.775     14  17223.550    17223.668    17310.051\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  3  GTR+F+G4      8597.504     14  17223.008    17223.126    17309.509\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  4  GTR+F+I+G4    8597.382     15  17224.764    17224.900    17317.444\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  5  GTR+F+R2      8598.105     15  17226.211    17226.346    17318.890\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  6  GTR+F+R3      8597.568     17  17229.137    17229.309    17334.173\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 14  GTR+F+I+R2    8597.737     16  17227.475    17227.628    17326.333\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 15  GTR+F+I+R3    8597.610     18  17231.220    17231.413    17342.435\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 24  SYM+I         8647.620     11  17317.240    17317.314    17385.205\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 25  SYM+G4        8646.903     11  17315.807    17315.881    17383.772\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 26  SYM+I+G4      8646.996     12  17317.992    17318.080    17392.135\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 27  SYM+R2        8647.440     12  17318.879    17318.967    17393.023\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 46  TVM+F+I       8604.243     13  17234.486    17234.589    17314.809\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 47  TVM+F+G4      8603.929     13  17233.858    17233.961    17314.180\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 48  TVM+F+I+G4    8603.906     14  17235.812    17235.930    17322.313\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 49  TVM+F+R2      8604.467     14  17236.935    17237.053    17323.436\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 68  TVMe+I        8649.791     10  17319.582    17319.644    17381.368\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 69  TVMe+G4       8649.008     10  17318.016    17318.078    17379.803\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 70  TVMe+I+G4     8649.091     11  17320.182    17320.256    17388.147\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 71  TVMe+R2       8649.631     11  17321.262    17321.336    17389.227\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 90  TIM3+F+I      8600.427     12  17224.854    17224.942    17298.998\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 91  TIM3+F+G4     8600.301     12  17224.602    17224.689    17298.745\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 92  TIM3+F+I+G4   8600.297     13  17226.595    17226.697    17306.917\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 93  TIM3+F+R2     8600.680     13  17227.360    17227.463    17307.683\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "112  TIM3e+I       8653.801     9   17325.603    17325.654    17381.211\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "113  TIM3e+G4      8653.308     9   17324.617    17324.667    17380.224\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "114  TIM3e+I+G4    8653.331     10  17326.663    17326.725    17388.449\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "115  TIM3e+R2      8653.651     10  17327.302    17327.364    17389.088\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "134  TIM2+F+I      8598.592     12  17221.184    17221.272    17295.327\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "135  TIM2+F+G4     8598.543     12  17221.085    17221.173    17295.229\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "136  TIM2+F+I+G4   8598.542     13  17223.085    17223.187    17303.407\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "137  TIM2+F+R2     8598.701     13  17223.402    17223.505    17303.724\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "156  TIM2e+I       8653.251     9   17324.502    17324.553    17380.110\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "157  TIM2e+G4      8652.978     9   17323.956    17324.007    17379.564\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "158  TIM2e+I+G4    8652.969     10  17325.939    17326.001    17387.725\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "159  TIM2e+R2      8653.146     10  17326.291    17326.353    17388.078\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "178  TIM+F+I       8600.315     12  17224.631    17224.719    17298.775\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "179  TIM+F+G4      8600.223     12  17224.446    17224.534    17298.590\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "180  TIM+F+I+G4    8600.233     13  17226.467    17226.569    17306.789\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "181  TIM+F+R2      8600.373     13  17226.747    17226.849    17307.069\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "200  TIMe+I        8657.655     9   17333.311    17333.362    17388.919\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "201  TIMe+G4       8657.356     9   17332.713    17332.764    17388.321\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "202  TIMe+I+G4     8657.365     10  17334.730    17334.791    17396.516\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "203  TIMe+R2       8657.547     10  17335.094    17335.156    17396.881\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "222  TPM3u+F+I     8606.438     11  17234.876    17234.950    17302.841\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "223  TPM3u+F+G4    8606.403     11  17234.806    17234.880    17302.771\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "224  TPM3u+F+I+G4  8606.413     12  17236.825    17236.913    17310.969\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "225  TPM3u+F+R2    8606.494     12  17236.988    17237.076    17311.132\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "244  TPM3+I        8655.755     8   17327.510    17327.551    17376.939\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "245  TPM3+G4       8655.287     8   17326.574    17326.614    17376.003\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "246  TPM3+I+G4     8655.290     9   17328.579    17328.630    17384.187\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "247  TPM3+R2       8655.694     9   17329.389    17329.439    17384.996\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "266  TPM2u+F+I     8605.511     11  17233.021    17233.096    17300.986\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "267  TPM2u+F+G4    8605.326     11  17232.651    17232.725    17300.616\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "268  TPM2u+F+I+G4  8605.348     12  17234.696    17234.784    17308.840\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "269  TPM2u+F+R2    8605.491     12  17234.983    17235.071    17309.127\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "288  TPM2+I        8655.906     8   17327.811    17327.852    17377.241\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "289  TPM2+G4       8655.398     8   17326.796    17326.837    17376.225\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "290  TPM2+I+G4     8655.418     9   17328.837    17328.887    17384.445\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "291  TPM2+R2       8655.796     9   17329.591    17329.642    17385.199\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "310  K3Pu+F+I      8606.776     11  17235.551    17235.625    17303.516\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "311  K3Pu+F+G4     8606.716     11  17235.431    17235.506    17303.396\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "312  K3Pu+F+I+G4   8606.728     12  17237.456    17237.544    17311.600\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "313  K3Pu+F+R2     8606.805     12  17237.610    17237.698    17311.754\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "332  K3P+I         8660.011     8   17336.023    17336.063    17385.452\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "333  K3P+G4        8659.569     8   17335.137    17335.178    17384.566\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "334  K3P+I+G4      8659.590     9   17337.180    17337.231    17392.788\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "335  K3P+R2        8659.888     9   17337.776    17337.827    17393.384\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "354  TN+F+I        8600.923     11  17223.846    17223.920    17291.811\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "355  TN+F+G4       8600.968     11  17223.935    17224.010    17291.901\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "356  TN+F+I+G4     8601.019     12  17226.039    17226.127    17300.182\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "357  TN+F+R2       8600.957     12  17225.915    17226.003    17300.058\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "376  TNe+I         8658.552     8   17333.104    17333.144    17382.533\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "377  TNe+G4        8658.478     8   17332.956    17332.997    17382.385\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "378  TNe+I+G4      8658.490     9   17334.980    17335.031    17390.588\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "379  TNe+R2        8658.482     9   17334.964    17335.015    17390.572\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "398  HKY+F+I       8607.351     10  17234.701    17234.763    17296.488\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "399  HKY+F+G4      8607.390     10  17234.780    17234.842    17296.566\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "400  HKY+F+I+G4    8607.442     11  17236.884    17236.959    17304.849\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "401  HKY+F+R2      8607.388     11  17236.776    17236.850    17304.741\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "420  K2P+I         8660.951     7   17335.903    17335.934    17379.153\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "421  K2P+G4        8660.721     7   17335.442    17335.474    17378.693\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "422  K2P+I+G4      8660.751     8   17337.503    17337.543    17386.932\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "423  K2P+R2        8660.865     8   17337.731    17337.771    17387.160\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "442  F81+F+I       8710.569     9   17439.138    17439.188    17494.745\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "443  F81+F+G4      8710.739     9   17439.478    17439.528    17495.086\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "444  F81+F+I+G4    8710.787     10  17441.574    17441.636    17503.360\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "445  F81+F+R2      8710.620     10  17441.240    17441.301    17503.026\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "464  JC+I          8759.896     6   17531.792    17531.816    17568.864\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "465  JC+G4         8760.024     6   17532.049    17532.072    17569.121\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "466  JC+I+G4       8760.057     7   17534.115    17534.146    17577.365\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "467  JC+R2         8760.006     7   17534.012    17534.043    17577.262\n",
      "Akaike Information Criterion:           TIM2+F+G4\n",
      "Corrected Akaike Information Criterion: TIM2+F+G4\n",
      "Bayesian Information Criterion:         TN+F+I\n",
      "Best-fit model: TN+F+I chosen according to BIC\n",
      "\n",
      "All model information printed to result/phylo_fna_gene/10008at50557_iqtreeout/.tree.model.gz\n",
      "CPU time for ModelFinder: 0.417 seconds (0h:0m:0s)\n",
      "Wall-clock time for ModelFinder: 0.312 seconds (0h:0m:0s)\n",
      "Generating 1000 samples for ultrafast bootstrap (seed: 14625)...\n",
      "\n",
      "NOTE: 0 MB RAM (0 GB) is required!\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "Estimate model parameters (epsilon = 0.100)\n",
      "1. Initial log-likelihood: -8710.631\n",
      "2. Current log-likelihood: -8602.339\n",
      "3. Current log-likelihood: -8601.430\n",
      "4. Current log-likelihood: -8601.132\n",
      "5. Current log-likelihood: -8601.006\n",
      "Optimal log-likelihood: -8600.933\n",
      "Rate parameters:  A-C: 1.00000  A-G: 2.97906  A-T: 1.00000  C-G: 1.00000  C-T: 5.05638  G-T: 1.00000\n",
      "Base frequencies:  A: 0.235  C: 0.274  G: 0.299  T: 0.192\n",
      "Proportion of invariable sites: 0.562\n",
      "Parameters optimization took 5 rounds (0.004 sec)\n",
      "Wrote distance file to... \n",
      "Computing ML distances based on estimated model parameters...\n",
      "Calculating distance matrix: done in 4.1497e-05 secs using 226.5% CPU\n",
      "Computing ML distances took 0.000553 sec (of wall-clock time) 0.000658 sec (of CPU time)\n",
      "Setting up auxiliary I and S matrices: done in 0.000163625 secs using 103.3% CPU\n",
      "Computing RapidNJ tree took 0.000504 sec (of wall-clock time) 0.000441 sec (of CPU time)\n",
      "Log-likelihood of RapidNJ tree: -8600.933\n",
      "--------------------------------------------------------------------\n",
      "|             INITIALIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "Generating 99 parsimony trees... 0.015 second\n",
      "Computing log-likelihood of 2 initial trees ... 0.000 seconds\n",
      "Current best score: -8600.933\n",
      "\n",
      "Do NNI search on 3 best initial trees\n",
      "Finish initializing candidate tree set (3)\n",
      "Current best tree score: -8600.933 / CPU time: 0.021\n",
      "Number of iterations: 3\n",
      "--------------------------------------------------------------------\n",
      "|               OPTIMIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "Iteration 10 / LogL: -8601.205 / Time: 0h:0m:0s\n",
      "UPDATE BEST LOG-LIKELIHOOD: -8600.932\n",
      "Iteration 20 / LogL: -8601.209 / Time: 0h:0m:0s\n",
      "UPDATE BEST LOG-LIKELIHOOD: -8600.932\n",
      "Iteration 30 / LogL: -8600.942 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 40 / LogL: -8601.218 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "UPDATE BEST LOG-LIKELIHOOD: -8600.932\n",
      "UPDATE BEST LOG-LIKELIHOOD: -8600.932\n",
      "UPDATE BEST LOG-LIKELIHOOD: -8600.932\n",
      "Iteration 50 / LogL: -8600.932 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -8603.258\n",
      "UPDATE BEST LOG-LIKELIHOOD: -8600.932\n",
      "UPDATE BEST LOG-LIKELIHOOD: -8600.932\n",
      "UPDATE BEST LOG-LIKELIHOOD: -8600.932\n",
      "Iteration 60 / LogL: -8601.218 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "UPDATE BEST LOG-LIKELIHOOD: -8600.932\n",
      "Iteration 70 / LogL: -8601.209 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 80 / LogL: -8601.218 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 90 / LogL: -8601.209 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 100 / LogL: -8601.200 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -8603.258\n",
      "NOTE: Bootstrap correlation coefficient of split occurrence frequencies: 0.999\n",
      "TREE SEARCH COMPLETED AFTER 101 ITERATIONS / Time: 0h:0m:0s\n",
      "\n",
      "--------------------------------------------------------------------\n",
      "|                    FINALIZING TREE SEARCH                        |\n",
      "--------------------------------------------------------------------\n",
      "Performs final model parameters optimization\n",
      "Estimate model parameters (epsilon = 0.010)\n",
      "1. Initial log-likelihood: -8600.932\n",
      "2. Current log-likelihood: -8600.920\n",
      "3. Current log-likelihood: -8600.908\n",
      "Optimal log-likelihood: -8600.900\n",
      "Rate parameters:  A-C: 1.00000  A-G: 3.03015  A-T: 1.00000  C-G: 1.00000  C-T: 5.15466  G-T: 1.00000\n",
      "Base frequencies:  A: 0.235  C: 0.274  G: 0.299  T: 0.192\n",
      "Proportion of invariable sites: 0.567\n",
      "Parameters optimization took 3 rounds (0.002 sec)\n",
      "BEST SCORE FOUND : -8600.900\n",
      "\n",
      "Testing tree branches by SH-like aLRT with 1000 replicates...\n",
      "0.004 sec.\n",
      "Creating bootstrap support values...\n",
      "Split supports printed to NEXUS file result/phylo_fna_gene/10008at50557_iqtreeout/.tree.splits.nex\n",
      "Total tree length: 0.486\n",
      "\n",
      "Total number of iterations: 101\n",
      "CPU time used for tree search: 0.744 sec (0h:0m:0s)\n",
      "Wall-clock time used for tree search: 0.415 sec (0h:0m:0s)\n",
      "Total CPU time used: 1.225 sec (0h:0m:1s)\n",
      "Total wall-clock time used: 0.773 sec (0h:0m:0s)\n",
      "\n",
      "Computing bootstrap consensus tree...\n",
      "Reading input file result/phylo_fna_gene/10008at50557_iqtreeout/.tree.splits.nex...\n",
      "4 taxa and 7 splits.\n",
      "Consensus tree written to result/phylo_fna_gene/10008at50557_iqtreeout/.tree.contree\n",
      "Reading input trees file result/phylo_fna_gene/10008at50557_iqtreeout/.tree.contree\n",
      "Log-likelihood of consensus tree: -8600.900\n",
      "\n",
      "Analysis results written to: \n",
      "  IQ-TREE report:                result/phylo_fna_gene/10008at50557_iqtreeout/.tree.iqtree\n",
      "  Maximum-likelihood tree:       result/phylo_fna_gene/10008at50557_iqtreeout/.tree.treefile\n",
      "  Likelihood distances:          result/phylo_fna_gene/10008at50557_iqtreeout/.tree.mldist\n",
      "\n",
      "Ultrafast bootstrap approximation results written to:\n",
      "  Split support values:          result/phylo_fna_gene/10008at50557_iqtreeout/.tree.splits.nex\n",
      "  Consensus tree:                result/phylo_fna_gene/10008at50557_iqtreeout/.tree.contree\n",
      "  Screen log file:               result/phylo_fna_gene/10008at50557_iqtreeout/.tree.log\n",
      "\n",
      "Date and Time: Sun Sep  1 10:03:33 2024\n",
      "IQ-TREE multicore version 2.3.6 for Linux x86 64-bit built Aug  4 2024\n",
      "Developed by Bui Quang Minh, Nguyen Lam Tung, Olga Chernomor, Heiko Schmidt,\n",
      "Dominik Schrempf, Michael Woodhams, Ly Trong Nhan, Thomas Wong\n",
      "\n",
      "Host:    codespaces-fa7cf2 (AVX2, FMA3, 7 GB RAM)\n",
      "Command: iqtree -s result/phylo_fna_gene/100138at50557_AlnTrimed.fna -nt 2 -m MFP -bb 1000 -alrt 1000 -pre result/phylo_fna_gene/100138at50557_iqtreeout/.tree\n",
      "Seed:    815500 (Using SPRNG - Scalable Parallel Random Number Generator)\n",
      "Time:    Sun Sep  1 10:03:33 2024\n",
      "Kernel:  AVX+FMA - 2 threads (2 CPU cores detected)\n",
      "\n",
      "Reading alignment file result/phylo_fna_gene/100138at50557_AlnTrimed.fna ... Fasta format detected\n",
      "Reading fasta file: done in 0.000102551 secs using 98.49% CPU\n",
      "Alignment most likely contains DNA/RNA sequences\n",
      "Alignment has 4 sequences with 801 columns, 41 distinct patterns\n",
      "8 parsimony-informative, 235 singleton sites, 558 constant sites\n",
      "                        Gap/Ambiguity  Composition  p-value\n",
      "Analyzing sequences: done in 2.1289e-05 secs using 89.25% CPU\n",
      "   1  Culex_pipiens             0.00%    passed     74.82%\n",
      "   2  Culex_pipiens_pallens     0.00%    passed     62.61%\n",
      "   3  Armigeres_subalbatus      5.24%    failed      0.46%\n",
      "   4  Culex_pipiens_molestus    0.00%    passed     75.75%\n",
      "****  TOTAL                     1.31%  1 sequences failed composition chi2 test (p-value<5%; df=3)\n",
      "\n",
      "\n",
      "Create initial parsimony tree by phylogenetic likelihood library (PLL)... 0.000 seconds\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "Perform fast likelihood tree search using GTR+I+G model...\n",
      "Estimate model parameters (epsilon = 5.000)\n",
      "Perform nearest neighbor interchange...\n",
      "Estimate model parameters (epsilon = 1.000)\n",
      "1. Initial log-likelihood: -1975.119\n",
      "Optimal log-likelihood: -1974.922\n",
      "Rate parameters:  A-C: 3.30584  A-G: 4.82555  A-T: 0.96958  C-G: 1.44062  C-T: 6.15364  G-T: 1.00000\n",
      "Base frequencies:  A: 0.234  C: 0.272  G: 0.283  T: 0.211\n",
      "Proportion of invariable sites: 0.351\n",
      "Gamma shape alpha: 1.473\n",
      "Parameters optimization took 1 rounds (0.002 sec)\n",
      "Time for fast ML tree search: 0.006 seconds\n",
      "\n",
      "NOTE: ModelFinder requires 0 MB RAM!\n",
      "ModelFinder will test up to 484 DNA models (sample size: 801 epsilon: 0.100) ...\n",
      " No. Model         -LnL         df  AIC          AICc         BIC\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  1  GTR+F         1978.481     13  3982.961     3983.424     4043.878\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  2  GTR+F+I       1974.923     14  3977.847     3978.381     4043.449\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  3  GTR+F+G4      1974.654     14  3977.308     3977.842     4042.910\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  4  GTR+F+I+G4    1974.724     15  3979.448     3980.060     4049.736\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  5  GTR+F+R2      1975.033     15  3980.066     3980.677     4050.354\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  6  GTR+F+R3      1974.754     17  3983.508     3984.289     4063.167\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 14  GTR+F+I+R2    1974.851     16  3981.702     3982.396     4056.676\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 15  GTR+F+I+R3    1974.727     18  3985.455     3986.329     4069.800\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 23  SYM           1980.692     10  3981.384     3981.662     4028.243\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 24  SYM+I         1977.239     11  3976.477     3976.812     4028.021\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 25  SYM+G4        1976.889     11  3975.779     3976.113     4027.323\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 26  SYM+I+G4      1977.013     12  3978.026     3978.422     4034.257\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 27  SYM+R2        1977.234     12  3978.469     3978.865     4034.699\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 45  TVM+F         1978.811     12  3981.623     3982.019     4037.853\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 46  TVM+F+I       1975.309     13  3976.619     3977.081     4037.535\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 47  TVM+F+G4      1974.900     13  3975.801     3976.263     4036.717\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 48  TVM+F+I+G4    1975.044     14  3978.088     3978.622     4043.690\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 49  TVM+F+R2      1975.223     14  3978.446     3978.980     4044.048\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 67  TVMe          1980.788     9   3979.576     3979.803     4021.749\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 68  TVMe+I        1977.410     10  3974.819     3975.098     4021.678\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 69  TVMe+G4       1976.982     10  3973.964     3974.243     4020.823\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 70  TVMe+I+G4     1977.145     11  3976.289     3976.624     4027.834\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 71  TVMe+R2       1977.271     11  3976.541     3976.876     4028.086\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 89  TIM3+F        1980.518     11  3983.037     3983.371     4034.581\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 90  TIM3+F+I      1976.586     12  3977.173     3977.568     4033.403\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 91  TIM3+F+G4     1976.395     12  3976.790     3977.186     4033.020\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 92  TIM3+F+I+G4   1976.405     13  3978.809     3979.272     4039.726\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 93  TIM3+F+R2     1976.587     13  3979.174     3979.636     4040.090\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "111  TIM3e         1981.984     8   3979.969     3980.150     4017.455\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "112  TIM3e+I       1978.249     9   3974.498     3974.726     4016.671\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "113  TIM3e+G4      1978.017     9   3974.034     3974.262     4016.207\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "114  TIM3e+I+G4    1978.071     10  3976.142     3976.420     4023.000\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "115  TIM3e+R2      1978.193     10  3976.386     3976.665     4023.245\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "133  TIM2+F        1980.858     11  3983.716     3984.050     4035.260\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "134  TIM2+F+I      1976.860     12  3977.719     3978.115     4033.950\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "135  TIM2+F+G4     1976.870     12  3977.740     3978.136     4033.970\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "136  TIM2+F+I+G4   1976.830     13  3979.660     3980.122     4040.576\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "137  TIM2+F+R2     1976.909     13  3979.818     3980.280     4040.734\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "155  TIM2e         1984.454     8   3984.908     3985.090     4022.395\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "156  TIM2e+I       1980.629     9   3979.257     3979.485     4021.430\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "157  TIM2e+G4      1980.598     9   3979.196     3979.424     4021.369\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "158  TIM2e+I+G4    1980.579     10  3981.158     3981.436     4028.016\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "159  TIM2e+R2      1980.644     10  3981.288     3981.567     4028.147\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "177  TIM+F         1982.577     11  3987.154     3987.488     4038.698\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "178  TIM+F+I       1978.051     12  3980.103     3980.499     4036.333\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "179  TIM+F+G4      1977.924     12  3979.848     3980.244     4036.078\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "180  TIM+F+I+G4    1977.928     13  3981.857     3982.319     4042.773\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "181  TIM+F+R2      1978.065     13  3982.131     3982.593     4043.047\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "199  TIMe          1985.519     8   3987.038     3987.220     4024.525\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "200  TIMe+I        1981.309     9   3980.618     3980.846     4022.791\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "201  TIMe+G4       1981.119     9   3980.238     3980.466     4022.411\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "202  TIMe+I+G4     1981.153     10  3982.305     3982.584     4029.164\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "203  TIMe+R2       1981.264     10  3982.528     3982.807     4029.387\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "221  TPM3u+F       1980.835     10  3981.670     3981.949     4028.529\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "222  TPM3u+F+I     1976.897     11  3975.795     3976.129     4027.339\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "223  TPM3u+F+G4    1976.633     11  3975.266     3975.601     4026.811\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "224  TPM3u+F+I+G4  1976.691     12  3977.382     3977.778     4033.613\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "225  TPM3u+F+R2    1976.807     12  3977.615     3978.011     4033.845\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "243  TPM3          1982.082     7   3978.164     3978.305     4010.965\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "244  TPM3+I        1978.377     8   3972.753     3972.935     4010.240\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "245  TPM3+G4       1978.101     8   3972.202     3972.384     4009.689\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "246  TPM3+I+G4     1978.177     9   3974.353     3974.581     4016.526\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "247  TPM3+R2       1978.284     9   3974.569     3974.796     4016.742\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "265  TPM2u+F       1981.166     10  3982.332     3982.610     4029.190\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "266  TPM2u+F+I     1977.314     11  3976.627     3976.962     4028.172\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "267  TPM2u+F+G4    1977.182     11  3976.363     3976.698     4027.908\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "268  TPM2u+F+I+G4  1977.207     12  3978.413     3978.809     4034.644\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "269  TPM2u+F+R2    1977.266     12  3978.532     3978.928     4034.762\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "287  TPM2          1984.552     7   3983.103     3983.245     4015.904\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "288  TPM2+I        1980.827     8   3977.655     3977.837     4015.142\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "289  TPM2+G4       1980.705     8   3977.410     3977.592     4014.897\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "290  TPM2+I+G4     1980.728     9   3979.455     3979.683     4021.628\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "291  TPM2+R2       1980.787     9   3979.574     3979.801     4021.746\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "309  K3Pu+F        1982.875     10  3985.750     3986.029     4032.609\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "310  K3Pu+F+I      1978.500     11  3979.000     3979.335     4030.545\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "311  K3Pu+F+G4     1978.203     11  3978.406     3978.740     4029.950\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "312  K3Pu+F+I+G4   1978.272     12  3980.543     3980.939     4036.773\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "313  K3Pu+F+R2     1978.397     12  3980.794     3981.190     4037.024\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "331  K3P           1985.618     7   3985.236     3985.378     4018.038\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "332  K3P+I         1981.490     8   3978.980     3979.162     4016.467\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "333  K3P+G4        1981.207     8   3978.413     3978.595     4015.900\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "334  K3P+I+G4      1981.270     9   3980.540     3980.768     4022.713\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "335  K3P+R2        1981.385     9   3980.770     3980.997     4022.942\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "353  TN+F          1982.839     10  3985.678     3985.957     4032.537\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "354  TN+F+I        1978.325     11  3978.649     3978.984     4030.194\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "355  TN+F+G4       1978.320     11  3978.640     3978.974     4030.184\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "356  TN+F+I+G4     1978.275     12  3980.549     3980.945     4036.779\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "357  TN+F+R2       1978.351     12  3980.701     3981.097     4036.932\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "375  TNe           1985.707     7   3985.414     3985.555     4018.215\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "376  TNe+I         1981.458     8   3978.916     3979.098     4016.403\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "377  TNe+G4        1981.411     8   3978.822     3979.004     4016.309\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "378  TNe+I+G4      1981.387     9   3980.773     3981.001     4022.946\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "379  TNe+R2        1981.460     9   3980.919     3981.147     4023.092\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "397  HKY+F         1983.134     9   3984.268     3984.496     4026.441\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "398  HKY+F+I       1978.742     10  3977.484     3977.763     4024.343\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "399  HKY+F+G4      1978.588     10  3977.176     3977.455     4024.035\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "400  HKY+F+I+G4    1978.596     11  3979.192     3979.526     4030.736\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "401  HKY+F+R2      1978.688     11  3979.377     3979.711     4030.921\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "419  K2P           1985.808     6   3983.616     3983.721     4011.731\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "420  K2P+I         1981.641     7   3977.283     3977.424     4010.084\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "421  K2P+G4        1981.500     7   3977.000     3977.141     4009.801\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "422  K2P+I+G4      1981.507     8   3979.013     3979.195     4016.500\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "423  K2P+R2        1981.592     8   3979.183     3979.365     4016.670\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "441  F81+F         1998.566     8   4013.131     4013.313     4050.618\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "442  F81+F+I       1992.275     9   4002.550     4002.778     4044.723\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "443  F81+F+G4      1992.239     9   4002.478     4002.706     4044.651\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "444  F81+F+I+G4    1992.258     10  4004.516     4004.795     4051.375\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "445  F81+F+R2      1992.681     10  4005.362     4005.641     4052.221\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "463  JC            2000.994     5   4011.989     4012.064     4035.418\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "464  JC+I          1994.893     6   4001.785     4001.891     4029.900\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "465  JC+G4         1994.829     6   4001.658     4001.763     4029.773\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "466  JC+I+G4       1994.843     7   4003.686     4003.827     4036.487\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "467  JC+R2         1995.209     7   4004.418     4004.559     4037.219\n",
      "Akaike Information Criterion:           TPM3+G4\n",
      "Corrected Akaike Information Criterion: TPM3+G4\n",
      "Bayesian Information Criterion:         TPM3+G4\n",
      "Best-fit model: TPM3+G4 chosen according to BIC\n",
      "\n",
      "All model information printed to result/phylo_fna_gene/100138at50557_iqtreeout/.tree.model.gz\n",
      "CPU time for ModelFinder: 0.255 seconds (0h:0m:0s)\n",
      "Wall-clock time for ModelFinder: 0.198 seconds (0h:0m:0s)\n",
      "Generating 1000 samples for ultrafast bootstrap (seed: 815500)...\n",
      "\n",
      "NOTE: 0 MB RAM (0 GB) is required!\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "Estimate model parameters (epsilon = 0.100)\n",
      "1. Initial log-likelihood: -1995.556\n",
      "2. Current log-likelihood: -1978.398\n",
      "3. Current log-likelihood: -1978.247\n",
      "Optimal log-likelihood: -1978.173\n",
      "Rate parameters:  A-C: 2.57722  A-G: 5.44332  A-T: 1.00000  C-G: 2.57722  C-T: 5.44332  G-T: 1.00000\n",
      "Base frequencies:  A: 0.250  C: 0.250  G: 0.250  T: 0.250\n",
      "Gamma shape alpha: 0.499\n",
      "Parameters optimization took 3 rounds (0.003 sec)\n",
      "Wrote distance file to... \n",
      "Computing ML distances based on estimated model parameters...\n",
      "Calculating distance matrix: done in 8.1281e-05 secs using 182.1% CPU\n",
      "Computing ML distances took 0.000605 sec (of wall-clock time) 0.000575 sec (of CPU time)\n",
      "Setting up auxiliary I and S matrices: done in 0.000184915 secs using 87.07% CPU\n",
      "Computing RapidNJ tree took 0.000731 sec (of wall-clock time) 0.000701 sec (of CPU time)\n",
      "Log-likelihood of RapidNJ tree: -1977.864\n",
      "--------------------------------------------------------------------\n",
      "|             INITIALIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "Generating 98 parsimony trees... 0.010 second\n",
      "Computing log-likelihood of 1 initial trees ... 0.000 seconds\n",
      "Current best score: -1977.864\n",
      "\n",
      "Do NNI search on 3 best initial trees\n",
      "Finish initializing candidate tree set (3)\n",
      "Current best tree score: -1977.864 / CPU time: 0.016\n",
      "Number of iterations: 3\n",
      "--------------------------------------------------------------------\n",
      "|               OPTIMIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "Iteration 10 / LogL: -1978.175 / Time: 0h:0m:0s\n",
      "UPDATE BEST LOG-LIKELIHOOD: -1977.863\n",
      "UPDATE BEST LOG-LIKELIHOOD: -1977.863\n",
      "UPDATE BEST LOG-LIKELIHOOD: -1977.862\n",
      "Iteration 20 / LogL: -1977.862 / Time: 0h:0m:0s\n",
      "UPDATE BEST LOG-LIKELIHOOD: -1977.862\n",
      "Iteration 30 / LogL: -1977.967 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "UPDATE BEST LOG-LIKELIHOOD: -1977.862\n",
      "UPDATE BEST LOG-LIKELIHOOD: -1977.862\n",
      "UPDATE BEST LOG-LIKELIHOOD: -1977.862\n",
      "Iteration 40 / LogL: -1978.111 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "UPDATE BEST LOG-LIKELIHOOD: -1977.862\n",
      "UPDATE BEST LOG-LIKELIHOOD: -1977.862\n",
      "UPDATE BEST LOG-LIKELIHOOD: -1977.862\n",
      "Iteration 50 / LogL: -1977.967 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -1979.237\n",
      "UPDATE BEST LOG-LIKELIHOOD: -1977.862\n",
      "Iteration 60 / LogL: -1978.111 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "UPDATE BEST LOG-LIKELIHOOD: -1977.862\n",
      "UPDATE BEST LOG-LIKELIHOOD: -1977.862\n",
      "Iteration 70 / LogL: -1978.175 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "UPDATE BEST LOG-LIKELIHOOD: -1977.862\n",
      "UPDATE BEST LOG-LIKELIHOOD: -1977.862\n",
      "Iteration 80 / LogL: -1977.862 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "UPDATE BEST LOG-LIKELIHOOD: -1977.862\n",
      "UPDATE BEST LOG-LIKELIHOOD: -1977.862\n",
      "Iteration 90 / LogL: -1978.111 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "UPDATE BEST LOG-LIKELIHOOD: -1977.862\n",
      "UPDATE BEST LOG-LIKELIHOOD: -1977.862\n",
      "Iteration 100 / LogL: -1977.862 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -1979.556\n",
      "NOTE: Bootstrap correlation coefficient of split occurrence frequencies: 0.999\n",
      "TREE SEARCH COMPLETED AFTER 101 ITERATIONS / Time: 0h:0m:0s\n",
      "\n",
      "--------------------------------------------------------------------\n",
      "|                    FINALIZING TREE SEARCH                        |\n",
      "--------------------------------------------------------------------\n",
      "Performs final model parameters optimization\n",
      "Estimate model parameters (epsilon = 0.010)\n",
      "1. Initial log-likelihood: -1977.862\n",
      "2. Current log-likelihood: -1977.847\n",
      "3. Current log-likelihood: -1977.836\n",
      "Optimal log-likelihood: -1977.830\n",
      "Rate parameters:  A-C: 2.56298  A-G: 5.45559  A-T: 1.00000  C-G: 2.56298  C-T: 5.45559  G-T: 1.00000\n",
      "Base frequencies:  A: 0.250  C: 0.250  G: 0.250  T: 0.250\n",
      "Gamma shape alpha: 0.531\n",
      "Parameters optimization took 3 rounds (0.003 sec)\n",
      "BEST SCORE FOUND : -1977.830\n",
      "\n",
      "Testing tree branches by SH-like aLRT with 1000 replicates...\n",
      "0.003 sec.\n",
      "Creating bootstrap support values...\n",
      "Split supports printed to NEXUS file result/phylo_fna_gene/100138at50557_iqtreeout/.tree.splits.nex\n",
      "Total tree length: 0.669\n",
      "\n",
      "Total number of iterations: 101\n",
      "CPU time used for tree search: 0.795 sec (0h:0m:0s)\n",
      "Wall-clock time used for tree search: 0.459 sec (0h:0m:0s)\n",
      "Total CPU time used: 1.120 sec (0h:0m:1s)\n",
      "Total wall-clock time used: 0.712 sec (0h:0m:0s)\n",
      "\n",
      "Computing bootstrap consensus tree...\n",
      "Reading input file result/phylo_fna_gene/100138at50557_iqtreeout/.tree.splits.nex...\n",
      "4 taxa and 7 splits.\n",
      "Consensus tree written to result/phylo_fna_gene/100138at50557_iqtreeout/.tree.contree\n",
      "Reading input trees file result/phylo_fna_gene/100138at50557_iqtreeout/.tree.contree\n",
      "Log-likelihood of consensus tree: -1977.832\n",
      "\n",
      "Analysis results written to: \n",
      "  IQ-TREE report:                result/phylo_fna_gene/100138at50557_iqtreeout/.tree.iqtree\n",
      "  Maximum-likelihood tree:       result/phylo_fna_gene/100138at50557_iqtreeout/.tree.treefile\n",
      "  Likelihood distances:          result/phylo_fna_gene/100138at50557_iqtreeout/.tree.mldist\n",
      "\n",
      "Ultrafast bootstrap approximation results written to:\n",
      "  Split support values:          result/phylo_fna_gene/100138at50557_iqtreeout/.tree.splits.nex\n",
      "  Consensus tree:                result/phylo_fna_gene/100138at50557_iqtreeout/.tree.contree\n",
      "  Screen log file:               result/phylo_fna_gene/100138at50557_iqtreeout/.tree.log\n",
      "\n",
      "Date and Time: Sun Sep  1 10:03:34 2024\n",
      "IQ-TREE multicore version 2.3.6 for Linux x86 64-bit built Aug  4 2024\n",
      "Developed by Bui Quang Minh, Nguyen Lam Tung, Olga Chernomor, Heiko Schmidt,\n",
      "Dominik Schrempf, Michael Woodhams, Ly Trong Nhan, Thomas Wong\n",
      "\n",
      "Host:    codespaces-fa7cf2 (AVX2, FMA3, 7 GB RAM)\n",
      "Command: iqtree -s result/phylo_fna_gene/100236at50557_AlnTrimed.fna -nt 2 -m MFP -bb 1000 -alrt 1000 -pre result/phylo_fna_gene/100236at50557_iqtreeout/.tree\n",
      "Seed:    565821 (Using SPRNG - Scalable Parallel Random Number Generator)\n",
      "Time:    Sun Sep  1 10:03:34 2024\n",
      "Kernel:  AVX+FMA - 2 threads (2 CPU cores detected)\n",
      "\n",
      "Reading alignment file result/phylo_fna_gene/100236at50557_AlnTrimed.fna ... Fasta format detected\n",
      "Reading fasta file: done in 0.000143658 secs\n",
      "Alignment most likely contains DNA/RNA sequences\n",
      "Alignment has 4 sequences with 1194 columns, 57 distinct patterns\n",
      "23 parsimony-informative, 261 singleton sites, 910 constant sites\n",
      "                        Gap/Ambiguity  Composition  p-value\n",
      "Analyzing sequences: done in 1.5329e-05 secs using 91.33% CPU\n",
      "   1  Culex_pipiens             0.00%    passed     71.90%\n",
      "   2  Culex_pipiens_pallens     0.00%    passed     84.78%\n",
      "   3  Armigeres_subalbatus      0.00%    failed      4.83%\n",
      "   4  Culex_pipiens_molestus    0.00%    passed     89.16%\n",
      "****  TOTAL                     0.00%  1 sequences failed composition chi2 test (p-value<5%; df=3)\n",
      "\n",
      "\n",
      "Create initial parsimony tree by phylogenetic likelihood library (PLL)... 0.000 seconds\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "Perform fast likelihood tree search using GTR+I+G model...\n",
      "Estimate model parameters (epsilon = 5.000)\n",
      "Perform nearest neighbor interchange...\n",
      "Estimate model parameters (epsilon = 1.000)\n",
      "1. Initial log-likelihood: -2838.992\n",
      "Optimal log-likelihood: -2838.960\n",
      "Rate parameters:  A-C: 1.22581  A-G: 2.85635  A-T: 0.90319  C-G: 0.89261  C-T: 5.78764  G-T: 1.00000\n",
      "Base frequencies:  A: 0.238  C: 0.249  G: 0.286  T: 0.228\n",
      "Proportion of invariable sites: 0.379\n",
      "Gamma shape alpha: 1.150\n",
      "Parameters optimization took 1 rounds (0.002 sec)\n",
      "Time for fast ML tree search: 0.006 seconds\n",
      "\n",
      "NOTE: ModelFinder requires 0 MB RAM!\n",
      "ModelFinder will test up to 484 DNA models (sample size: 1194 epsilon: 0.100) ...\n",
      " No. Model         -LnL         df  AIC          AICc         BIC\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  1  GTR+F         2849.068     13  5724.136     5724.445     5790.242\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  2  GTR+F+I       2839.320     14  5706.639     5706.996     5777.830\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  3  GTR+F+G4      2839.068     14  5706.136     5706.492     5777.327\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  4  GTR+F+I+G4    2838.968     15  5707.936     5708.343     5784.212\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  5  GTR+F+R2      2839.784     15  5709.569     5709.976     5785.844\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  6  GTR+F+R3      2839.199     17  5712.398     5712.918     5798.844\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 14  GTR+F+I+R2    2839.309     16  5710.618     5711.080     5791.979\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 15  GTR+F+I+R3    2839.161     18  5714.322     5714.904     5805.853\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 24  SYM+I         2843.287     11  5708.575     5708.798     5764.510\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 25  SYM+G4        2843.057     11  5708.114     5708.338     5764.050\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 26  SYM+I+G4      2842.997     12  5709.993     5710.257     5771.014\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 27  SYM+R2        2843.539     12  5711.077     5711.342     5772.098\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 46  TVM+F+I       2843.432     13  5712.863     5713.172     5778.969\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 47  TVM+F+G4      2843.440     13  5712.879     5713.188     5778.985\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 48  TVM+F+I+G4    2843.412     14  5714.824     5715.180     5786.015\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 49  TVM+F+R2      2843.948     14  5715.896     5716.253     5787.087\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 68  TVMe+I        2846.398     10  5712.796     5712.981     5763.646\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 69  TVMe+G4       2846.282     10  5712.564     5712.750     5763.415\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 70  TVMe+I+G4     2846.238     11  5714.476     5714.699     5770.411\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 71  TVMe+R2       2846.555     11  5715.110     5715.334     5771.046\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 90  TIM3+F+I      2839.510     12  5703.019     5703.283     5764.040\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 91  TIM3+F+G4     2839.233     12  5702.466     5702.730     5763.487\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 92  TIM3+F+I+G4   2839.177     13  5704.353     5704.662     5770.459\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 93  TIM3+F+R2     2839.469     13  5704.938     5705.246     5771.044\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "112  TIM3e+I       2843.397     9   5704.794     5704.946     5750.559\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "113  TIM3e+G4      2843.139     9   5704.277     5704.429     5750.043\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "114  TIM3e+I+G4    2843.090     10  5706.180     5706.366     5757.031\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "115  TIM3e+R2      2843.358     10  5706.715     5706.901     5757.566\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "134  TIM2+F+I      2839.447     12  5702.894     5703.158     5763.915\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "135  TIM2+F+G4     2839.156     12  5702.312     5702.576     5763.333\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "136  TIM2+F+I+G4   2839.082     13  5704.165     5704.473     5770.271\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "137  TIM2+F+R2     2839.404     13  5704.808     5705.116     5770.914\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "156  TIM2e+I       2843.558     9   5705.117     5705.269     5750.882\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "157  TIM2e+G4      2843.333     9   5704.667     5704.819     5750.432\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "158  TIM2e+I+G4    2843.275     10  5706.551     5706.737     5757.401\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "159  TIM2e+R2      2843.521     10  5707.042     5707.228     5757.893\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "178  TIM+F+I       2839.427     12  5702.854     5703.118     5763.875\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "179  TIM+F+G4      2839.185     12  5702.371     5702.635     5763.391\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "180  TIM+F+I+G4    2839.120     13  5704.240     5704.548     5770.346\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "181  TIM+F+R2      2839.393     13  5704.786     5705.095     5770.892\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "200  TIMe+I        2843.439     9   5704.877     5705.029     5750.643\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "201  TIMe+G4       2843.234     9   5704.469     5704.621     5750.234\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "202  TIMe+I+G4     2843.177     10  5706.355     5706.541     5757.206\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "203  TIMe+R2       2843.406     10  5706.811     5706.997     5757.662\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "222  TPM3u+F+I     2843.863     11  5709.726     5709.949     5765.661\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "223  TPM3u+F+G4    2843.824     11  5709.648     5709.872     5765.584\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "224  TPM3u+F+I+G4  2843.787     12  5711.574     5711.838     5772.595\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "225  TPM3u+F+R2    2843.976     12  5711.951     5712.215     5772.972\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "244  TPM3+I        2846.514     8   5709.029     5709.150     5749.709\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "245  TPM3+G4       2846.392     8   5708.785     5708.906     5749.465\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "246  TPM3+I+G4     2846.350     9   5710.700     5710.852     5756.466\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "247  TPM3+R2       2846.537     9   5711.073     5711.225     5756.839\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "266  TPM2u+F+I     2843.508     11  5709.017     5709.240     5764.952\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "267  TPM2u+F+G4    2843.511     11  5709.021     5709.245     5764.957\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "268  TPM2u+F+I+G4  2843.501     12  5711.002     5711.266     5772.023\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "269  TPM2u+F+R2    2843.614     12  5711.228     5711.492     5772.249\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "288  TPM2+I        2846.599     8   5709.198     5709.319     5749.878\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "289  TPM2+G4       2846.521     8   5709.041     5709.163     5749.722\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "290  TPM2+I+G4     2846.485     9   5710.970     5711.122     5756.736\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "291  TPM2+R2       2846.613     9   5711.227     5711.379     5756.992\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "310  K3Pu+F+I      2843.768     11  5709.535     5709.758     5765.471\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "311  K3Pu+F+G4     2843.738     11  5709.476     5709.700     5765.412\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "312  K3Pu+F+I+G4   2843.699     12  5711.398     5711.662     5772.419\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "313  K3Pu+F+R2     2843.782     12  5711.563     5711.827     5772.584\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "332  K3P+I         2846.565     8   5709.130     5709.251     5749.810\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "333  K3P+G4        2846.479     8   5708.958     5709.080     5749.639\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "334  K3P+I+G4      2846.435     9   5710.871     5711.023     5756.636\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "335  K3P+R2        2846.553     9   5711.105     5711.257     5756.871\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "354  TN+F+I        2839.592     11  5701.185     5701.408     5757.120\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "355  TN+F+G4       2839.328     11  5700.655     5700.879     5756.591\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "356  TN+F+I+G4     2839.257     12  5702.515     5702.779     5763.535\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "357  TN+F+R2       2839.612     12  5703.225     5703.489     5764.245\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "376  TNe+I         2843.565     8   5703.129     5703.251     5743.810\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "377  TNe+G4        2843.343     8   5702.685     5702.807     5743.366\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "378  TNe+I+G4      2843.291     9   5704.581     5704.733     5750.347\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "379  TNe+R2        2843.556     9   5705.113     5705.265     5750.878\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "398  HKY+F+I       2843.892     10  5707.783     5707.969     5758.634\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "399  HKY+F+G4      2843.874     10  5707.747     5707.933     5758.598\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "400  HKY+F+I+G4    2843.831     11  5709.662     5709.886     5765.598\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "401  HKY+F+R2      2843.935     11  5709.871     5710.094     5765.806\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "420  K2P+I         2846.658     7   5707.316     5707.410     5742.911\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "421  K2P+G4        2846.568     7   5707.137     5707.231     5742.732\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "422  K2P+I+G4      2846.528     8   5709.056     5709.178     5749.737\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "423  K2P+R2        2846.660     8   5709.319     5709.441     5750.000\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "442  F81+F+I       2883.316     9   5784.631     5784.783     5830.397\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "443  F81+F+G4      2883.107     9   5784.214     5784.366     5829.980\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "444  F81+F+I+G4    2883.062     10  5786.124     5786.310     5836.975\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "445  F81+F+R2      2883.286     10  5786.571     5786.757     5837.422\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "464  JC+I          2885.706     6   5783.412     5783.483     5813.922\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "465  JC+G4         2885.524     6   5783.048     5783.119     5813.559\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "466  JC+I+G4       2885.482     7   5784.965     5785.059     5820.560\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "467  JC+R2         2885.683     7   5785.367     5785.461     5820.962\n",
      "Akaike Information Criterion:           TN+F+G4\n",
      "Corrected Akaike Information Criterion: TN+F+G4\n",
      "Bayesian Information Criterion:         K2P+G4\n",
      "Best-fit model: K2P+G4 chosen according to BIC\n",
      "\n",
      "All model information printed to result/phylo_fna_gene/100236at50557_iqtreeout/.tree.model.gz\n",
      "CPU time for ModelFinder: 0.271 seconds (0h:0m:0s)\n",
      "Wall-clock time for ModelFinder: 0.185 seconds (0h:0m:0s)\n",
      "Generating 1000 samples for ultrafast bootstrap (seed: 565821)...\n",
      "\n",
      "NOTE: 0 MB RAM (0 GB) is required!\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "Estimate model parameters (epsilon = 0.100)\n",
      "1. Initial log-likelihood: -2885.646\n",
      "2. Current log-likelihood: -2847.103\n",
      "3. Current log-likelihood: -2846.660\n",
      "Optimal log-likelihood: -2846.599\n",
      "Rate parameters:  A-C: 1.00000  A-G: 4.22531  A-T: 1.00000  C-G: 1.00000  C-T: 4.22531  G-T: 1.00000\n",
      "Base frequencies:  A: 0.250  C: 0.250  G: 0.250  T: 0.250\n",
      "Gamma shape alpha: 0.325\n",
      "Parameters optimization took 3 rounds (0.004 sec)\n",
      "Wrote distance file to... \n",
      "Computing ML distances based on estimated model parameters...\n",
      "Calculating distance matrix: done in 8.1342e-05 secs using 393.4% CPU\n",
      "Computing ML distances took 0.000596 sec (of wall-clock time) 0.000698 sec (of CPU time)\n",
      "Setting up auxiliary I and S matrices: done in 0.000174094 secs using 98.22% CPU\n",
      "Computing RapidNJ tree took 0.000565 sec (of wall-clock time) 0.000467 sec (of CPU time)\n",
      "Log-likelihood of RapidNJ tree: -2846.596\n",
      "--------------------------------------------------------------------\n",
      "|             INITIALIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "Generating 99 parsimony trees... 0.009 second\n",
      "Computing log-likelihood of 2 initial trees ... 0.000 seconds\n",
      "Current best score: -2846.596\n",
      "\n",
      "Do NNI search on 3 best initial trees\n",
      "Finish initializing candidate tree set (3)\n",
      "Current best tree score: -2846.596 / CPU time: 0.017\n",
      "Number of iterations: 3\n",
      "--------------------------------------------------------------------\n",
      "|               OPTIMIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "Iteration 10 / LogL: -2846.931 / Time: 0h:0m:0s\n",
      "Iteration 20 / LogL: -2846.622 / Time: 0h:0m:0s\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2846.596\n",
      "Iteration 30 / LogL: -2846.931 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2846.596\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2846.596\n",
      "Iteration 40 / LogL: -2846.622 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2846.596\n",
      "Iteration 50 / LogL: -2846.622 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -2848.455\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2846.596\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2846.596\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2846.596\n",
      "Iteration 60 / LogL: -2846.622 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 70 / LogL: -2846.914 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 80 / LogL: -2846.926 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 90 / LogL: -2846.622 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 100 / LogL: -2846.598 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -2848.455\n",
      "NOTE: Bootstrap correlation coefficient of split occurrence frequencies: 0.991\n",
      "TREE SEARCH COMPLETED AFTER 101 ITERATIONS / Time: 0h:0m:0s\n",
      "\n",
      "--------------------------------------------------------------------\n",
      "|                    FINALIZING TREE SEARCH                        |\n",
      "--------------------------------------------------------------------\n",
      "Performs final model parameters optimization\n",
      "Estimate model parameters (epsilon = 0.010)\n",
      "1. Initial log-likelihood: -2846.596\n",
      "Optimal log-likelihood: -2846.582\n",
      "Rate parameters:  A-C: 1.00000  A-G: 4.27469  A-T: 1.00000  C-G: 1.00000  C-T: 4.27469  G-T: 1.00000\n",
      "Base frequencies:  A: 0.250  C: 0.250  G: 0.250  T: 0.250\n",
      "Gamma shape alpha: 0.318\n",
      "Parameters optimization took 1 rounds (0.001 sec)\n",
      "BEST SCORE FOUND : -2846.582\n",
      "\n",
      "Testing tree branches by SH-like aLRT with 1000 replicates...\n",
      "0.004 sec.\n",
      "Creating bootstrap support values...\n",
      "Split supports printed to NEXUS file result/phylo_fna_gene/100236at50557_iqtreeout/.tree.splits.nex\n",
      "Total tree length: 0.472\n",
      "\n",
      "Total number of iterations: 101\n",
      "CPU time used for tree search: 0.831 sec (0h:0m:0s)\n",
      "Wall-clock time used for tree search: 0.614 sec (0h:0m:0s)\n",
      "Total CPU time used: 1.178 sec (0h:0m:1s)\n",
      "Total wall-clock time used: 0.854 sec (0h:0m:0s)\n",
      "\n",
      "Computing bootstrap consensus tree...\n",
      "Reading input file result/phylo_fna_gene/100236at50557_iqtreeout/.tree.splits.nex...\n",
      "4 taxa and 7 splits.\n",
      "Consensus tree written to result/phylo_fna_gene/100236at50557_iqtreeout/.tree.contree\n",
      "Reading input trees file result/phylo_fna_gene/100236at50557_iqtreeout/.tree.contree\n",
      "Log-likelihood of consensus tree: -2846.582\n",
      "\n",
      "Analysis results written to: \n",
      "  IQ-TREE report:                result/phylo_fna_gene/100236at50557_iqtreeout/.tree.iqtree\n",
      "  Maximum-likelihood tree:       result/phylo_fna_gene/100236at50557_iqtreeout/.tree.treefile\n",
      "  Likelihood distances:          result/phylo_fna_gene/100236at50557_iqtreeout/.tree.mldist\n",
      "\n",
      "Ultrafast bootstrap approximation results written to:\n",
      "  Split support values:          result/phylo_fna_gene/100236at50557_iqtreeout/.tree.splits.nex\n",
      "  Consensus tree:                result/phylo_fna_gene/100236at50557_iqtreeout/.tree.contree\n",
      "  Screen log file:               result/phylo_fna_gene/100236at50557_iqtreeout/.tree.log\n",
      "\n",
      "Date and Time: Sun Sep  1 10:03:35 2024\n",
      "IQ-TREE multicore version 2.3.6 for Linux x86 64-bit built Aug  4 2024\n",
      "Developed by Bui Quang Minh, Nguyen Lam Tung, Olga Chernomor, Heiko Schmidt,\n",
      "Dominik Schrempf, Michael Woodhams, Ly Trong Nhan, Thomas Wong\n",
      "\n",
      "Host:    codespaces-fa7cf2 (AVX2, FMA3, 7 GB RAM)\n",
      "Command: iqtree -s result/phylo_fna_gene/100341at50557_AlnTrimed.fna -nt 2 -m MFP -bb 1000 -alrt 1000 -pre result/phylo_fna_gene/100341at50557_iqtreeout/.tree\n",
      "Seed:    448558 (Using SPRNG - Scalable Parallel Random Number Generator)\n",
      "Time:    Sun Sep  1 10:03:35 2024\n",
      "Kernel:  AVX+FMA - 2 threads (2 CPU cores detected)\n",
      "\n",
      "Reading alignment file result/phylo_fna_gene/100341at50557_AlnTrimed.fna ... Fasta format detected\n",
      "Reading fasta file: done in 0.000114023 secs using 98.23% CPU\n",
      "Alignment most likely contains DNA/RNA sequences\n",
      "Alignment has 4 sequences with 975 columns, 37 distinct patterns\n",
      "9 parsimony-informative, 259 singleton sites, 707 constant sites\n",
      "                        Gap/Ambiguity  Composition  p-value\n",
      "Analyzing sequences: done in 1.2453e-05 secs\n",
      "   1  Culex_pipiens             0.00%    passed     34.21%\n",
      "   2  Culex_pipiens_pallens     0.00%    passed     44.39%\n",
      "   3  Armigeres_subalbatus      0.00%    failed      0.00%\n",
      "   4  Culex_pipiens_molestus    0.00%    passed     28.09%\n",
      "****  TOTAL                     0.00%  1 sequences failed composition chi2 test (p-value<5%; df=3)\n",
      "\n",
      "\n",
      "Create initial parsimony tree by phylogenetic likelihood library (PLL)... 0.000 seconds\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "Perform fast likelihood tree search using GTR+I+G model...\n",
      "Estimate model parameters (epsilon = 5.000)\n",
      "Perform nearest neighbor interchange...\n",
      "Estimate model parameters (epsilon = 1.000)\n",
      "1. Initial log-likelihood: -2321.641\n",
      "Optimal log-likelihood: -2321.576\n",
      "Rate parameters:  A-C: 2.82781  A-G: 4.26958  A-T: 0.65442  C-G: 2.44663  C-T: 12.40784  G-T: 1.00000\n",
      "Base frequencies:  A: 0.269  C: 0.253  G: 0.296  T: 0.182\n",
      "Proportion of invariable sites: 0.363\n",
      "Gamma shape alpha: 2.213\n",
      "Parameters optimization took 1 rounds (0.003 sec)\n",
      "Time for fast ML tree search: 0.009 seconds\n",
      "\n",
      "NOTE: ModelFinder requires 0 MB RAM!\n",
      "ModelFinder will test up to 484 DNA models (sample size: 975 epsilon: 0.100) ...\n",
      " No. Model         -LnL         df  AIC          AICc         BIC\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  1  GTR+F         2322.589     13  4671.177     4671.556     4734.649\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  2  GTR+F+I       2321.483     14  4670.965     4671.403     4739.319\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  3  GTR+F+G4      2321.136     14  4670.272     4670.709     4738.626\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  4  GTR+F+I+G4    2321.426     15  4672.851     4673.352     4746.088\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  5  GTR+F+R2      2321.352     15  4672.704     4673.204     4745.940\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "+R3 reinitialized from +R2 with factor 0.500\n",
      "  6  GTR+F+R3      2321.441     17  4676.881     4677.521     4759.883\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 14  GTR+F+I+R2    2321.600     16  4675.201     4675.769     4753.320\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 15  GTR+F+I+R3    2321.427     18  4678.854     4679.569     4766.737\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 23  SYM           2333.743     10  4687.486     4687.715     4736.311\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 24  SYM+I         2332.950     11  4687.900     4688.175     4741.607\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 25  SYM+G4        2332.638     11  4687.276     4687.551     4740.983\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 45  TVM+F         2331.971     12  4687.942     4688.266     4746.531\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 46  TVM+F+I       2329.944     13  4685.889     4686.268     4749.360\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 47  TVM+F+G4      2329.973     13  4685.946     4686.325     4749.418\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 67  TVMe          2337.725     9   4693.450     4693.637     4737.392\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 68  TVMe+I        2336.745     10  4693.489     4693.718     4742.314\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 69  TVMe+G4       2336.528     10  4693.057     4693.285     4741.881\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 89  TIM3+F        2322.760     11  4667.520     4667.794     4721.227\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 90  TIM3+F+I      2321.656     12  4667.312     4667.637     4725.902\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 91  TIM3+F+G4     2321.278     12  4666.555     4666.880     4725.145\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "111  TIM3e         2333.933     8   4683.866     4684.016     4722.926\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "112  TIM3e+I       2333.143     9   4684.285     4684.472     4728.227\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "113  TIM3e+G4      2332.797     9   4683.595     4683.781     4727.537\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "133  TIM2+F        2327.326     11  4676.651     4676.925     4730.358\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "134  TIM2+F+I      2325.403     12  4674.805     4675.130     4733.395\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "135  TIM2+F+G4     2325.238     12  4674.477     4674.801     4733.066\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "155  TIM2e         2341.172     8   4698.344     4698.493     4737.403\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "156  TIM2e+I       2339.641     9   4697.283     4697.469     4741.225\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "157  TIM2e+G4      2339.403     9   4696.805     4696.992     4740.747\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "177  TIM+F         2327.234     11  4676.468     4676.743     4730.175\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "178  TIM+F+I       2325.305     12  4674.610     4674.935     4733.200\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "179  TIM+F+G4      2325.039     12  4674.079     4674.403     4732.668\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "199  TIMe          2341.100     8   4698.200     4698.349     4737.260\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "200  TIMe+I        2339.589     9   4697.177     4697.364     4741.119\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "201  TIMe+G4       2339.313     9   4696.626     4696.813     4740.568\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "221  TPM3u+F       2332.144     10  4684.289     4684.517     4733.113\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "222  TPM3u+F+I     2330.057     11  4682.115     4682.389     4735.822\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "223  TPM3u+F+G4    2329.989     11  4681.979     4682.253     4735.685\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "243  TPM3          2337.915     7   4689.830     4689.945     4724.007\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "244  TPM3+I        2336.936     8   4689.873     4690.022     4728.932\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "245  TPM3+G4       2336.706     8   4689.412     4689.561     4728.471\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "265  TPM2u+F       2336.468     10  4692.936     4693.165     4741.761\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "266  TPM2u+F+I     2333.001     11  4688.002     4688.276     4741.709\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "267  TPM2u+F+G4    2333.622     11  4689.245     4689.519     4742.951\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "287  TPM2          2345.142     7   4704.284     4704.399     4738.461\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "288  TPM2+I        2343.188     8   4702.377     4702.526     4741.436\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "289  TPM2+G4       2343.136     8   4702.271     4702.420     4741.331\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "309  K3Pu+F        2336.377     10  4692.754     4692.983     4741.579\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "310  K3Pu+F+I      2333.017     11  4688.034     4688.308     4741.740\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "311  K3Pu+F+G4     2333.412     11  4688.824     4689.098     4742.531\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "331  K3P           2345.068     7   4704.137     4704.253     4738.314\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "332  K3P+I         2343.158     8   4702.316     4702.465     4741.375\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "333  K3P+G4        2343.079     8   4702.157     4702.306     4741.217\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "353  TN+F          2327.326     10  4674.651     4674.879     4723.476\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "354  TN+F+I        2325.259     11  4672.518     4672.792     4726.225\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "355  TN+F+G4       2324.965     11  4671.929     4672.203     4725.636\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "375  TNe           2341.189     7   4696.377     4696.493     4730.554\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "376  TNe+I         2339.613     8   4695.225     4695.375     4734.285\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "377  TNe+G4        2339.340     8   4694.679     4694.828     4733.739\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "397  HKY+F         2336.468     9   4690.936     4691.122     4734.878\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "398  HKY+F+I       2332.968     10  4685.937     4686.165     4734.761\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "399  HKY+F+G4      2333.464     10  4686.928     4687.156     4735.752\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "419  K2P           2345.158     6   4702.316     4702.403     4731.611\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "420  K2P+I         2343.209     7   4700.419     4700.534     4734.596\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "421  K2P+G4        2343.132     7   4700.264     4700.380     4734.441\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "441  F81+F         2361.681     8   4739.363     4739.512     4778.422\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "442  F81+F+I       2361.257     9   4740.515     4740.701     4784.457\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "443  F81+F+G4      2361.302     9   4740.604     4740.790     4784.546\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "463  JC            2372.553     5   4755.106     4755.168     4779.518\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "464  JC+I          2372.029     6   4756.057     4756.144     4785.352\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "465  JC+G4         2372.048     6   4756.096     4756.183     4785.391\n",
      "Akaike Information Criterion:           TIM3+F+G4\n",
      "Corrected Akaike Information Criterion: TIM3+F+G4\n",
      "Bayesian Information Criterion:         TIM3+F\n",
      "Best-fit model: TIM3+F chosen according to BIC\n",
      "\n",
      "All model information printed to result/phylo_fna_gene/100341at50557_iqtreeout/.tree.model.gz\n",
      "CPU time for ModelFinder: 0.219 seconds (0h:0m:0s)\n",
      "Wall-clock time for ModelFinder: 0.152 seconds (0h:0m:0s)\n",
      "Generating 1000 samples for ultrafast bootstrap (seed: 448558)...\n",
      "\n",
      "NOTE: 0 MB RAM (0 GB) is required!\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "Estimate model parameters (epsilon = 0.100)\n",
      "1. Initial log-likelihood: -2365.167\n",
      "2. Current log-likelihood: -2322.916\n",
      "3. Current log-likelihood: -2322.759\n",
      "Optimal log-likelihood: -2322.758\n",
      "Rate parameters:  A-C: 2.12354  A-G: 3.09016  A-T: 1.00000  C-G: 2.12354  C-T: 6.89159  G-T: 1.00000\n",
      "Base frequencies:  A: 0.269  C: 0.253  G: 0.296  T: 0.182\n",
      "Parameters optimization took 3 rounds (0.003 sec)\n",
      "Wrote distance file to... \n",
      "Computing ML distances based on estimated model parameters...\n",
      "Calculating distance matrix: done in 4.85e-05 secs using 164.9% CPU\n",
      "Computing ML distances took 0.001222 sec (of wall-clock time) 0.001241 sec (of CPU time)\n",
      "Setting up auxiliary I and S matrices: done in 0.000175197 secs using 100.5% CPU\n",
      "Computing RapidNJ tree took 0.000575 sec (of wall-clock time) 0.000516 sec (of CPU time)\n",
      "Log-likelihood of RapidNJ tree: -2322.758\n",
      "--------------------------------------------------------------------\n",
      "|             INITIALIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "Generating 99 parsimony trees... 0.011 second\n",
      "Computing log-likelihood of 2 initial trees ... 0.000 seconds\n",
      "Current best score: -2322.758\n",
      "\n",
      "Do NNI search on 3 best initial trees\n",
      "Finish initializing candidate tree set (3)\n",
      "Current best tree score: -2322.758 / CPU time: 0.017\n",
      "Number of iterations: 3\n",
      "--------------------------------------------------------------------\n",
      "|               OPTIMIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "Iteration 10 / LogL: -2322.775 / Time: 0h:0m:0s\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2322.758\n",
      "Iteration 20 / LogL: -2322.761 / Time: 0h:0m:0s\n",
      "Iteration 30 / LogL: -2322.761 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 40 / LogL: -2322.791 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 50 / LogL: -2322.768 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -2327.337\n",
      "Iteration 60 / LogL: -2322.791 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 70 / LogL: -2322.775 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 80 / LogL: -2322.791 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 90 / LogL: -2322.788 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 100 / LogL: -2322.768 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -2327.337\n",
      "NOTE: Bootstrap correlation coefficient of split occurrence frequencies: 1.000\n",
      "TREE SEARCH COMPLETED AFTER 101 ITERATIONS / Time: 0h:0m:0s\n",
      "\n",
      "--------------------------------------------------------------------\n",
      "|                    FINALIZING TREE SEARCH                        |\n",
      "--------------------------------------------------------------------\n",
      "Performs final model parameters optimization\n",
      "Estimate model parameters (epsilon = 0.010)\n",
      "1. Initial log-likelihood: -2322.758\n",
      "Optimal log-likelihood: -2322.758\n",
      "Rate parameters:  A-C: 2.12249  A-G: 3.09018  A-T: 1.00000  C-G: 2.12249  C-T: 6.89085  G-T: 1.00000\n",
      "Base frequencies:  A: 0.269  C: 0.253  G: 0.296  T: 0.182\n",
      "Parameters optimization took 1 rounds (0.001 sec)\n",
      "BEST SCORE FOUND : -2322.758\n",
      "\n",
      "Testing tree branches by SH-like aLRT with 1000 replicates...\n",
      "0.002 sec.\n",
      "Creating bootstrap support values...\n",
      "Split supports printed to NEXUS file result/phylo_fna_gene/100341at50557_iqtreeout/.tree.splits.nex\n",
      "Total tree length: 0.354\n",
      "\n",
      "Total number of iterations: 101\n",
      "CPU time used for tree search: 0.706 sec (0h:0m:0s)\n",
      "Wall-clock time used for tree search: 0.394 sec (0h:0m:0s)\n",
      "Total CPU time used: 0.993 sec (0h:0m:0s)\n",
      "Total wall-clock time used: 0.592 sec (0h:0m:0s)\n",
      "\n",
      "Computing bootstrap consensus tree...\n",
      "Reading input file result/phylo_fna_gene/100341at50557_iqtreeout/.tree.splits.nex...\n",
      "4 taxa and 7 splits.\n",
      "Consensus tree written to result/phylo_fna_gene/100341at50557_iqtreeout/.tree.contree\n",
      "Reading input trees file result/phylo_fna_gene/100341at50557_iqtreeout/.tree.contree\n",
      "Log-likelihood of consensus tree: -2322.758\n",
      "\n",
      "Analysis results written to: \n",
      "  IQ-TREE report:                result/phylo_fna_gene/100341at50557_iqtreeout/.tree.iqtree\n",
      "  Maximum-likelihood tree:       result/phylo_fna_gene/100341at50557_iqtreeout/.tree.treefile\n",
      "  Likelihood distances:          result/phylo_fna_gene/100341at50557_iqtreeout/.tree.mldist\n",
      "\n",
      "Ultrafast bootstrap approximation results written to:\n",
      "  Split support values:          result/phylo_fna_gene/100341at50557_iqtreeout/.tree.splits.nex\n",
      "  Consensus tree:                result/phylo_fna_gene/100341at50557_iqtreeout/.tree.contree\n",
      "  Screen log file:               result/phylo_fna_gene/100341at50557_iqtreeout/.tree.log\n",
      "\n",
      "Date and Time: Sun Sep  1 10:03:36 2024\n",
      "IQ-TREE multicore version 2.3.6 for Linux x86 64-bit built Aug  4 2024\n",
      "Developed by Bui Quang Minh, Nguyen Lam Tung, Olga Chernomor, Heiko Schmidt,\n",
      "Dominik Schrempf, Michael Woodhams, Ly Trong Nhan, Thomas Wong\n",
      "\n",
      "Host:    codespaces-fa7cf2 (AVX2, FMA3, 7 GB RAM)\n",
      "Command: iqtree -s result/phylo_fna_gene/100475at50557_AlnTrimed.fna -nt 2 -m MFP -bb 1000 -alrt 1000 -pre result/phylo_fna_gene/100475at50557_iqtreeout/.tree\n",
      "Seed:    76437 (Using SPRNG - Scalable Parallel Random Number Generator)\n",
      "Time:    Sun Sep  1 10:03:36 2024\n",
      "Kernel:  AVX+FMA - 2 threads (2 CPU cores detected)\n",
      "\n",
      "Reading alignment file result/phylo_fna_gene/100475at50557_AlnTrimed.fna ... Fasta format detected\n",
      "Reading fasta file: done in 0.000109414 secs\n",
      "Alignment most likely contains DNA/RNA sequences\n",
      "Alignment has 4 sequences with 993 columns, 39 distinct patterns\n",
      "8 parsimony-informative, 258 singleton sites, 727 constant sites\n",
      "                        Gap/Ambiguity  Composition  p-value\n",
      "Analyzing sequences: done in 1.3495e-05 secs using 44.46% CPU\n",
      "   1  Culex_pipiens             0.00%    passed     75.16%\n",
      "   2  Culex_pipiens_pallens     0.00%    passed     63.73%\n",
      "   3  Armigeres_subalbatus      0.00%    failed      1.59%\n",
      "   4  Culex_pipiens_molestus    0.00%    passed     83.14%\n",
      "****  TOTAL                     0.00%  1 sequences failed composition chi2 test (p-value<5%; df=3)\n",
      "\n",
      "\n",
      "Create initial parsimony tree by phylogenetic likelihood library (PLL)... 0.000 seconds\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "Perform fast likelihood tree search using GTR+I+G model...\n",
      "Estimate model parameters (epsilon = 5.000)\n",
      "Perform nearest neighbor interchange...\n",
      "Estimate model parameters (epsilon = 1.000)\n",
      "1. Initial log-likelihood: -2348.449\n",
      "Optimal log-likelihood: -2348.149\n",
      "Rate parameters:  A-C: 0.00010  A-G: 4.65708  A-T: 0.64960  C-G: 2.76195  C-T: 5.81149  G-T: 1.00000\n",
      "Warning! Some parameters hit the boundaries\n",
      "Base frequencies:  A: 0.261  C: 0.275  G: 0.296  T: 0.168\n",
      "Proportion of invariable sites: 0.365\n",
      "Gamma shape alpha: 1.094\n",
      "Parameters optimization took 1 rounds (0.002 sec)\n",
      "Time for fast ML tree search: 0.009 seconds\n",
      "\n",
      "NOTE: ModelFinder requires 0 MB RAM!\n",
      "ModelFinder will test up to 484 DNA models (sample size: 993 epsilon: 0.100) ...\n",
      " No. Model         -LnL         df  AIC          AICc         BIC\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  1  GTR+F         2357.481     13  4740.962     4741.333     4804.671\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  2  GTR+F+I       2346.102     14  4720.204     4720.633     4788.814\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  3  GTR+F+G4      2347.880     14  4723.761     4724.190     4792.371\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  4  GTR+F+I+G4    2347.807     15  4725.614     4726.105     4799.125\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  5  GTR+F+R2      2346.423     15  4722.846     4723.337     4796.357\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "+R3 reinitialized from +R2 with factor 0.500\n",
      "+R3 reinitialized from +R2 with factor 0.250\n",
      "+R3 reinitialized from +R2 with factor 0.125\n",
      "  6  GTR+F+R3      2346.443     17  4726.887     4727.515     4810.199\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 14  GTR+F+I+R2    2346.771     16  4725.543     4726.100     4803.954\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 15  GTR+F+I+R3    2346.527     18  4729.054     4729.756     4817.267\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 24  SYM+I         2367.433     11  4756.865     4757.134     4810.773\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 25  SYM+G4        2368.535     11  4759.070     4759.339     4812.978\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 27  SYM+R2        2367.506     12  4759.012     4759.331     4817.821\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 46  TVM+F+I       2347.423     13  4720.846     4721.217     4784.555\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 47  TVM+F+G4      2348.558     13  4723.117     4723.488     4786.826\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 49  TVM+F+R2      2347.480     14  4722.960     4723.389     4791.570\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 68  TVMe+I        2367.387     10  4754.774     4754.998     4803.781\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 69  TVMe+G4       2368.576     10  4757.151     4757.375     4806.159\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 71  TVMe+R2       2367.451     11  4756.901     4757.170     4810.809\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 90  TIM3+F+I      2357.394     12  4738.787     4739.106     4797.596\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 91  TIM3+F+G4     2358.695     12  4741.390     4741.708     4800.199\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 93  TIM3+F+R2     2357.515     13  4741.030     4741.402     4804.740\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "112  TIM3e+I       2379.646     9   4777.291     4777.474     4821.398\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "113  TIM3e+G4      2380.544     9   4779.088     4779.271     4823.195\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "115  TIM3e+R2      2379.748     10  4779.496     4779.720     4828.504\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "134  TIM2+F+I      2347.527     12  4719.053     4719.372     4777.862\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "135  TIM2+F+G4     2349.150     12  4722.299     4722.618     4781.108\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "137  TIM2+F+R2     2347.659     13  4721.317     4721.689     4785.027\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "156  TIM2e+I       2370.538     9   4759.076     4759.259     4803.182\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "157  TIM2e+G4      2371.606     9   4761.212     4761.395     4805.318\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "159  TIM2e+R2      2370.630     10  4761.259     4761.483     4810.267\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "178  TIM+F+I       2353.521     12  4731.043     4731.361     4789.852\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "179  TIM+F+G4      2355.695     12  4735.390     4735.708     4794.199\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "181  TIM+F+R2      2353.762     13  4733.524     4733.896     4797.233\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "200  TIMe+I        2378.950     9   4775.900     4776.083     4820.006\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "201  TIMe+G4       2380.005     9   4778.010     4778.193     4822.116\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "203  TIMe+R2       2378.981     10  4777.962     4778.186     4826.969\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "222  TPM3u+F+I     2358.540     11  4739.081     4739.350     4792.989\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "223  TPM3u+F+G4    2359.456     11  4740.912     4741.181     4794.820\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "225  TPM3u+F+R2    2358.601     12  4741.202     4741.520     4800.011\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "244  TPM3+I        2379.683     8   4775.366     4775.512     4814.572\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "245  TPM3+G4       2380.569     8   4777.137     4777.284     4816.343\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "247  TPM3+R2       2379.748     9   4777.496     4777.679     4821.602\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "266  TPM2u+F+I     2348.865     11  4719.730     4720.000     4773.639\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "267  TPM2u+F+G4    2349.734     11  4721.467     4721.736     4775.375\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "269  TPM2u+F+R2    2348.911     12  4721.823     4722.141     4780.631\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "288  TPM2+I        2370.608     8   4757.217     4757.363     4796.423\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "289  TPM2+G4       2371.612     8   4759.224     4759.370     4798.430\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "291  TPM2+R2       2370.656     9   4759.313     4759.496     4803.419\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "310  K3Pu+F+I      2355.201     11  4732.402     4732.671     4786.310\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "311  K3Pu+F+G4     2356.611     11  4735.222     4735.491     4789.130\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "313  K3Pu+F+R2     2355.305     12  4734.610     4734.928     4793.418\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "332  K3P+I         2378.923     8   4773.847     4773.993     4813.053\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "333  K3P+G4        2380.008     8   4776.017     4776.163     4815.223\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "335  K3P+R2        2379.012     9   4776.024     4776.208     4820.131\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "354  TN+F+I        2357.680     11  4737.361     4737.630     4791.269\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "355  TN+F+G4       2358.982     11  4739.964     4740.233     4793.872\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "357  TN+F+R2       2357.791     12  4739.582     4739.900     4798.391\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "376  TNe+I         2381.826     8   4779.653     4779.799     4818.859\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "377  TNe+G4        2382.701     8   4781.403     4781.549     4820.608\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "379  TNe+R2        2381.895     9   4781.790     4781.973     4825.897\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "398  HKY+F+I       2358.860     10  4737.720     4737.944     4786.727\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "399  HKY+F+G4      2359.725     10  4739.451     4739.675     4788.458\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "401  HKY+F+R2      2358.917     11  4739.833     4740.102     4793.741\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "420  K2P+I         2381.853     7   4777.705     4777.819     4812.011\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "421  K2P+G4        2382.714     7   4779.427     4779.541     4813.732\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "423  K2P+R2        2381.914     8   4779.827     4779.974     4819.033\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "442  F81+F+I       2387.858     9   4793.715     4793.898     4837.822\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "443  F81+F+G4      2387.933     9   4793.866     4794.049     4837.973\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "445  F81+F+R2      2387.887     10  4795.775     4795.999     4844.782\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "464  JC+I          2410.772     6   4833.543     4833.628     4862.948\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "465  JC+G4         2410.839     6   4833.679     4833.764     4863.083\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "467  JC+R2         2410.812     7   4835.623     4835.737     4869.928\n",
      "Akaike Information Criterion:           TIM2+F+I\n",
      "Corrected Akaike Information Criterion: TIM2+F+I\n",
      "Bayesian Information Criterion:         TPM2u+F+I\n",
      "Best-fit model: TPM2u+F+I chosen according to BIC\n",
      "\n",
      "All model information printed to result/phylo_fna_gene/100475at50557_iqtreeout/.tree.model.gz\n",
      "CPU time for ModelFinder: 0.323 seconds (0h:0m:0s)\n",
      "Wall-clock time for ModelFinder: 0.226 seconds (0h:0m:0s)\n",
      "Generating 1000 samples for ultrafast bootstrap (seed: 76437)...\n",
      "\n",
      "NOTE: 0 MB RAM (0 GB) is required!\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "Estimate model parameters (epsilon = 0.100)\n",
      "1. Initial log-likelihood: -2387.888\n",
      "2. Current log-likelihood: -2370.666\n",
      "3. Current log-likelihood: -2351.827\n",
      "4. Current log-likelihood: -2350.175\n",
      "5. Current log-likelihood: -2349.321\n",
      "6. Current log-likelihood: -2348.992\n",
      "7. Current log-likelihood: -2348.887\n",
      "Optimal log-likelihood: -2348.844\n",
      "Rate parameters:  A-C: 0.06567  A-G: 2.43767  A-T: 0.06567  C-G: 1.00000  C-T: 2.43767  G-T: 1.00000\n",
      "Base frequencies:  A: 0.261  C: 0.275  G: 0.296  T: 0.168\n",
      "Proportion of invariable sites: 0.550\n",
      "Parameters optimization took 7 rounds (0.006 sec)\n",
      "Wrote distance file to... \n",
      "Computing ML distances based on estimated model parameters...\n",
      "Calculating distance matrix: done in 4.1688e-05 secs using 203.9% CPU\n",
      "Computing ML distances took 0.000601 sec (of wall-clock time) 0.000619 sec (of CPU time)\n",
      "Setting up auxiliary I and S matrices: done in 0.000175938 secs using 93.78% CPU\n",
      "Computing RapidNJ tree took 0.000584 sec (of wall-clock time) 0.000490 sec (of CPU time)\n",
      "Log-likelihood of RapidNJ tree: -2348.845\n",
      "--------------------------------------------------------------------\n",
      "|             INITIALIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "Generating 99 parsimony trees... 0.011 second\n",
      "Computing log-likelihood of 2 initial trees ... 0.002 seconds\n",
      "Current best score: -2348.844\n",
      "\n",
      "Do NNI search on 3 best initial trees\n",
      "Finish initializing candidate tree set (3)\n",
      "Current best tree score: -2348.844 / CPU time: 0.020\n",
      "Number of iterations: 3\n",
      "--------------------------------------------------------------------\n",
      "|               OPTIMIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2348.844\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2348.844\n",
      "Iteration 10 / LogL: -2348.878 / Time: 0h:0m:0s\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2348.844\n",
      "Iteration 20 / LogL: -2348.876 / Time: 0h:0m:0s\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2348.844\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2348.844\n",
      "Iteration 30 / LogL: -2348.880 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2348.844\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2348.844\n",
      "Iteration 40 / LogL: -2348.880 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2348.844\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2348.844\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2348.844\n",
      "Iteration 50 / LogL: -2348.844 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -2350.074\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2348.844\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2348.844\n",
      "Iteration 60 / LogL: -2348.875 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "UPDATE BEST LOG-LIKELIHOOD: -2348.844\n",
      "Iteration 70 / LogL: -2348.875 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 80 / LogL: -2348.876 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 90 / LogL: -2348.876 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 100 / LogL: -2348.875 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -2350.074\n",
      "NOTE: Bootstrap correlation coefficient of split occurrence frequencies: 0.872\n",
      "NOTE: UFBoot does not converge, continue at least 100 more iterations\n",
      "Iteration 110 / LogL: -2348.844 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 120 / LogL: -2348.876 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 130 / LogL: -2348.858 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 140 / LogL: -2348.875 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 150 / LogL: -2348.880 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -2350.074\n",
      "Iteration 160 / LogL: -2348.880 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 170 / LogL: -2348.880 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 180 / LogL: -2348.878 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 190 / LogL: -2348.858 / Time: 0h:0m:1s (0h:0m:0s left)\n",
      "Iteration 200 / LogL: -2348.878 / Time: 0h:0m:1s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -2350.074\n",
      "NOTE: Bootstrap correlation coefficient of split occurrence frequencies: 0.989\n",
      "NOTE: UFBoot does not converge, continue at least 100 more iterations\n",
      "Iteration 210 / LogL: -2348.875 / Time: 0h:0m:1s (0h:0m:0s left)\n",
      "Iteration 220 / LogL: -2348.876 / Time: 0h:0m:1s (0h:0m:0s left)\n",
      "Iteration 230 / LogL: -2348.844 / Time: 0h:0m:1s (0h:0m:0s left)\n",
      "Iteration 240 / LogL: -2348.880 / Time: 0h:0m:1s (0h:0m:0s left)\n",
      "Iteration 250 / LogL: -2348.858 / Time: 0h:0m:1s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -2350.074\n",
      "Iteration 260 / LogL: -2348.880 / Time: 0h:0m:1s (0h:0m:0s left)\n",
      "Iteration 270 / LogL: -2348.844 / Time: 0h:0m:1s (0h:0m:0s left)\n",
      "Iteration 280 / LogL: -2348.844 / Time: 0h:0m:1s (0h:0m:0s left)\n",
      "Iteration 290 / LogL: -2348.858 / Time: 0h:0m:1s (0h:0m:0s left)\n",
      "Iteration 300 / LogL: -2348.875 / Time: 0h:0m:1s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -2350.074\n",
      "NOTE: Bootstrap correlation coefficient of split occurrence frequencies: 0.587\n",
      "NOTE: UFBoot does not converge, continue at least 100 more iterations\n",
      "Iteration 310 / LogL: -2348.875 / Time: 0h:0m:1s (0h:0m:0s left)\n",
      "Iteration 320 / LogL: -2348.878 / Time: 0h:0m:1s (0h:0m:0s left)\n",
      "Iteration 330 / LogL: -2348.858 / Time: 0h:0m:1s (0h:0m:0s left)\n",
      "Iteration 340 / LogL: -2348.878 / Time: 0h:0m:1s (0h:0m:0s left)\n",
      "Iteration 350 / LogL: -2348.878 / Time: 0h:0m:1s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -2350.074\n",
      "Iteration 360 / LogL: -2348.878 / Time: 0h:0m:1s (0h:0m:0s left)\n",
      "Iteration 370 / LogL: -2348.844 / Time: 0h:0m:1s (0h:0m:0s left)\n",
      "Iteration 380 / LogL: -2348.876 / Time: 0h:0m:1s (0h:0m:0s left)\n",
      "Iteration 390 / LogL: -2348.878 / Time: 0h:0m:1s (0h:0m:0s left)\n",
      "Iteration 400 / LogL: -2348.875 / Time: 0h:0m:1s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -2350.074\n",
      "NOTE: Bootstrap correlation coefficient of split occurrence frequencies: 0.866\n",
      "NOTE: UFBoot does not converge, continue at least 100 more iterations\n",
      "Iteration 410 / LogL: -2348.876 / Time: 0h:0m:2s (0h:0m:0s left)\n",
      "Iteration 420 / LogL: -2348.880 / Time: 0h:0m:2s (0h:0m:0s left)\n",
      "Iteration 430 / LogL: -2348.858 / Time: 0h:0m:2s (0h:0m:0s left)\n",
      "Iteration 440 / LogL: -2348.876 / Time: 0h:0m:2s (0h:0m:0s left)\n",
      "Iteration 450 / LogL: -2348.878 / Time: 0h:0m:2s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -2350.074\n",
      "Iteration 460 / LogL: -2348.876 / Time: 0h:0m:2s (0h:0m:0s left)\n",
      "Iteration 470 / LogL: -2348.880 / Time: 0h:0m:2s (0h:0m:0s left)\n",
      "Iteration 480 / LogL: -2348.880 / Time: 0h:0m:2s (0h:0m:0s left)\n",
      "Iteration 490 / LogL: -2348.878 / Time: 0h:0m:2s (0h:0m:0s left)\n",
      "Iteration 500 / LogL: -2348.878 / Time: 0h:0m:2s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -2350.074\n",
      "NOTE: Bootstrap correlation coefficient of split occurrence frequencies: 0.900\n",
      "NOTE: UFBoot does not converge, continue at least 100 more iterations\n",
      "Iteration 510 / LogL: -2348.844 / Time: 0h:0m:2s (0h:0m:0s left)\n",
      "Iteration 520 / LogL: -2348.858 / Time: 0h:0m:2s (0h:0m:0s left)\n",
      "Iteration 530 / LogL: -2348.876 / Time: 0h:0m:2s (0h:0m:0s left)\n",
      "Iteration 540 / LogL: -2348.878 / Time: 0h:0m:2s (0h:0m:0s left)\n",
      "Iteration 550 / LogL: -2348.876 / Time: 0h:0m:2s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -2350.074\n",
      "Iteration 560 / LogL: -2348.858 / Time: 0h:0m:2s (0h:0m:0s left)\n",
      "Iteration 570 / LogL: -2348.858 / Time: 0h:0m:2s (0h:0m:0s left)\n",
      "Iteration 580 / LogL: -2348.878 / Time: 0h:0m:2s (0h:0m:0s left)\n",
      "Iteration 590 / LogL: -2348.844 / Time: 0h:0m:2s (0h:0m:0s left)\n",
      "Iteration 600 / LogL: -2348.844 / Time: 0h:0m:2s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -2350.074\n",
      "NOTE: Bootstrap correlation coefficient of split occurrence frequencies: 0.844\n",
      "NOTE: UFBoot does not converge, continue at least 100 more iterations\n",
      "Iteration 610 / LogL: -2348.875 / Time: 0h:0m:2s (0h:0m:0s left)\n",
      "Iteration 620 / LogL: -2348.876 / Time: 0h:0m:2s (0h:0m:0s left)\n",
      "Iteration 630 / LogL: -2348.875 / Time: 0h:0m:2s (0h:0m:0s left)\n",
      "Iteration 640 / LogL: -2348.878 / Time: 0h:0m:2s (0h:0m:0s left)\n",
      "Iteration 650 / LogL: -2348.878 / Time: 0h:0m:2s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -2350.074\n",
      "Iteration 660 / LogL: -2348.844 / Time: 0h:0m:2s (0h:0m:0s left)\n",
      "Iteration 670 / LogL: -2348.844 / Time: 0h:0m:3s (0h:0m:0s left)\n",
      "Iteration 680 / LogL: -2348.880 / Time: 0h:0m:3s (0h:0m:0s left)\n",
      "Iteration 690 / LogL: -2348.844 / Time: 0h:0m:3s (0h:0m:0s left)\n",
      "Iteration 700 / LogL: -2348.858 / Time: 0h:0m:3s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -2350.074\n",
      "NOTE: Bootstrap correlation coefficient of split occurrence frequencies: 0.993\n",
      "TREE SEARCH COMPLETED AFTER 700 ITERATIONS / Time: 0h:0m:3s\n",
      "\n",
      "--------------------------------------------------------------------\n",
      "|                    FINALIZING TREE SEARCH                        |\n",
      "--------------------------------------------------------------------\n",
      "Performs final model parameters optimization\n",
      "Estimate model parameters (epsilon = 0.010)\n",
      "1. Initial log-likelihood: -2348.844\n",
      "Optimal log-likelihood: -2348.836\n",
      "Rate parameters:  A-C: 0.06868  A-G: 2.41359  A-T: 0.06868  C-G: 1.00000  C-T: 2.41359  G-T: 1.00000\n",
      "Base frequencies:  A: 0.261  C: 0.275  G: 0.296  T: 0.168\n",
      "Proportion of invariable sites: 0.549\n",
      "Parameters optimization took 1 rounds (0.002 sec)\n",
      "BEST SCORE FOUND : -2348.836\n",
      "\n",
      "Testing tree branches by SH-like aLRT with 1000 replicates...\n",
      "0.002 sec.\n",
      "Creating bootstrap support values...\n",
      "Split supports printed to NEXUS file result/phylo_fna_gene/100475at50557_iqtreeout/.tree.splits.nex\n",
      "Total tree length: 0.613\n",
      "\n",
      "Total number of iterations: 700\n",
      "CPU time used for tree search: 5.036 sec (0h:0m:5s)\n",
      "Wall-clock time used for tree search: 2.878 sec (0h:0m:2s)\n",
      "Total CPU time used: 5.435 sec (0h:0m:5s)\n",
      "Total wall-clock time used: 3.169 sec (0h:0m:3s)\n",
      "\n",
      "Computing bootstrap consensus tree...\n",
      "Reading input file result/phylo_fna_gene/100475at50557_iqtreeout/.tree.splits.nex...\n",
      "4 taxa and 7 splits.\n",
      "Consensus tree written to result/phylo_fna_gene/100475at50557_iqtreeout/.tree.contree\n",
      "Reading input trees file result/phylo_fna_gene/100475at50557_iqtreeout/.tree.contree\n",
      "Log-likelihood of consensus tree: -2348.837\n",
      "\n",
      "Analysis results written to: \n",
      "  IQ-TREE report:                result/phylo_fna_gene/100475at50557_iqtreeout/.tree.iqtree\n",
      "  Maximum-likelihood tree:       result/phylo_fna_gene/100475at50557_iqtreeout/.tree.treefile\n",
      "  Likelihood distances:          result/phylo_fna_gene/100475at50557_iqtreeout/.tree.mldist\n",
      "\n",
      "Ultrafast bootstrap approximation results written to:\n",
      "  Split support values:          result/phylo_fna_gene/100475at50557_iqtreeout/.tree.splits.nex\n",
      "  Consensus tree:                result/phylo_fna_gene/100475at50557_iqtreeout/.tree.contree\n",
      "  Screen log file:               result/phylo_fna_gene/100475at50557_iqtreeout/.tree.log\n",
      "\n",
      "Date and Time: Sun Sep  1 10:03:39 2024\n",
      "IQ-TREE multicore version 2.3.6 for Linux x86 64-bit built Aug  4 2024\n",
      "Developed by Bui Quang Minh, Nguyen Lam Tung, Olga Chernomor, Heiko Schmidt,\n",
      "Dominik Schrempf, Michael Woodhams, Ly Trong Nhan, Thomas Wong\n",
      "\n",
      "Host:    codespaces-fa7cf2 (AVX2, FMA3, 7 GB RAM)\n",
      "Command: iqtree -s result/phylo_fna_gene/100579at50557_AlnTrimed.fna -nt 2 -m MFP -bb 1000 -alrt 1000 -pre result/phylo_fna_gene/100579at50557_iqtreeout/.tree\n",
      "Seed:    281503 (Using SPRNG - Scalable Parallel Random Number Generator)\n",
      "Time:    Sun Sep  1 10:03:39 2024\n",
      "Kernel:  AVX+FMA - 2 threads (2 CPU cores detected)\n",
      "\n",
      "Reading alignment file result/phylo_fna_gene/100579at50557_AlnTrimed.fna ... Fasta format detected\n",
      "Reading fasta file: done in 0.00041184 secs using 28.17% CPU\n",
      "Alignment most likely contains DNA/RNA sequences\n",
      "Alignment has 4 sequences with 795 columns, 25 distinct patterns\n",
      "6 parsimony-informative, 158 singleton sites, 631 constant sites\n",
      "                        Gap/Ambiguity  Composition  p-value\n",
      "Analyzing sequences: done in 1.5118e-05 secs using 185.2% CPU\n",
      "   1  Culex_pipiens             0.00%    passed     97.19%\n",
      "   2  Culex_pipiens_pallens     0.00%    passed     97.12%\n",
      "   3  Armigeres_subalbatus      0.00%    passed     53.07%\n",
      "   4  Culex_pipiens_molestus    0.00%    passed     91.94%\n",
      "****  TOTAL                     0.00%  0 sequences failed composition chi2 test (p-value<5%; df=3)\n",
      "\n",
      "\n",
      "Create initial parsimony tree by phylogenetic likelihood library (PLL)... 0.000 seconds\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "Perform fast likelihood tree search using GTR+I+G model...\n",
      "Estimate model parameters (epsilon = 5.000)\n",
      "Perform nearest neighbor interchange...\n",
      "Estimate model parameters (epsilon = 1.000)\n",
      "1. Initial log-likelihood: -1718.931\n",
      "Optimal log-likelihood: -1718.922\n",
      "Rate parameters:  A-C: 2.06257  A-G: 9.79119  A-T: 0.79299  C-G: 2.60197  C-T: 9.66386  G-T: 1.00000\n",
      "Base frequencies:  A: 0.266  C: 0.242  G: 0.252  T: 0.241\n",
      "Proportion of invariable sites: 0.396\n",
      "Gamma shape alpha: 1.394\n",
      "Parameters optimization took 1 rounds (0.002 sec)\n",
      "Time for fast ML tree search: 0.005 seconds\n",
      "\n",
      "NOTE: ModelFinder requires 0 MB RAM!\n",
      "ModelFinder will test up to 484 DNA models (sample size: 795 epsilon: 0.100) ...\n",
      " No. Model         -LnL         df  AIC          AICc         BIC\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  1  GTR+F         1720.652     13  3467.304     3467.770     3528.122\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  2  GTR+F+I       1719.049     14  3466.097     3466.636     3531.594\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  3  GTR+F+G4      1719.193     14  3466.386     3466.924     3531.883\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  4  GTR+F+I+G4    1718.936     15  3467.872     3468.488     3538.047\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  5  GTR+F+R2      1719.394     15  3468.788     3469.404     3538.963\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "+R3 reinitialized from +R2 with factor 0.500\n",
      "+R3 reinitialized from +R2 with factor 0.250\n",
      "  6  GTR+F+R3      1719.451     17  3472.902     3473.690     3552.434\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 14  GTR+F+I+R2    1718.810     16  3469.620     3470.320     3544.474\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 15  GTR+F+I+R3    1718.888     18  3473.777     3474.658     3557.987\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 23  SYM           1721.658     10  3463.316     3463.597     3510.100\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 24  SYM+I         1719.978     11  3461.956     3462.293     3513.418\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 25  SYM+G4        1720.116     11  3462.233     3462.570     3513.694\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 26  SYM+I+G4      1719.929     12  3463.859     3464.258     3519.999\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 45  TVM+F         1720.701     12  3465.401     3465.800     3521.541\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 46  TVM+F+I       1718.967     13  3463.934     3464.400     3524.752\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 47  TVM+F+G4      1719.080     13  3464.160     3464.626     3524.979\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 48  TVM+F+I+G4    1718.954     14  3465.907     3466.446     3531.404\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 67  TVMe          1721.658     9   3461.316     3461.545     3503.421\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 68  TVMe+I        1720.000     10  3460.000     3460.280     3506.783\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 69  TVMe+G4       1720.069     10  3460.138     3460.419     3506.922\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 70  TVMe+I+G4     1719.993     11  3461.986     3462.323     3513.448\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 89  TIM3+F        1720.956     11  3463.912     3464.250     3515.374\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 90  TIM3+F+I      1719.034     12  3462.067     3462.466     3518.208\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 91  TIM3+F+G4     1719.165     12  3462.330     3462.729     3518.470\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 92  TIM3+F+I+G4   1719.101     13  3464.201     3464.667     3525.020\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "111  TIM3e         1721.911     8   3459.823     3460.006     3497.250\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "112  TIM3e+I       1719.911     9   3457.821     3458.050     3499.926\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "113  TIM3e+G4      1720.071     9   3458.142     3458.372     3500.248\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "114  TIM3e+I+G4    1720.041     10  3460.081     3460.362     3506.865\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "133  TIM2+F        1722.687     11  3467.374     3467.711     3518.836\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "134  TIM2+F+I      1720.827     12  3465.653     3466.052     3521.793\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "135  TIM2+F+G4     1720.957     12  3465.914     3466.313     3522.054\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "136  TIM2+F+I+G4   1720.935     13  3467.871     3468.337     3528.689\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "155  TIM2e         1723.696     8   3463.391     3463.575     3500.818\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "156  TIM2e+I       1721.819     9   3461.637     3461.867     3503.743\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "157  TIM2e+G4      1721.938     9   3461.876     3462.105     3503.981\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "158  TIM2e+I+G4    1721.918     10  3463.835     3464.116     3510.619\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "177  TIM+F         1722.830     11  3467.659     3467.996     3519.121\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "178  TIM+F+I       1720.971     12  3465.941     3466.340     3522.082\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "179  TIM+F+G4      1721.067     12  3466.135     3466.534     3522.275\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "180  TIM+F+I+G4    1721.049     13  3468.098     3468.565     3528.917\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "199  TIMe          1723.794     8   3463.588     3463.771     3501.015\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "200  TIMe+I        1721.912     9   3461.824     3462.053     3503.929\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "201  TIMe+G4       1721.998     9   3461.996     3462.225     3504.101\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "202  TIMe+I+G4     1721.994     10  3463.987     3464.268     3510.771\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "221  TPM3u+F       1721.005     10  3462.010     3462.290     3508.793\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "222  TPM3u+F+I     1719.049     11  3460.098     3460.436     3511.560\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "223  TPM3u+F+G4    1719.073     11  3460.145     3460.482     3511.607\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "224  TPM3u+F+I+G4  1719.065     12  3462.130     3462.528     3518.270\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "243  TPM3          1721.912     7   3457.824     3457.966     3490.572\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "244  TPM3+I        1720.074     8   3456.149     3456.332     3493.576\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "245  TPM3+G4       1720.068     8   3456.136     3456.319     3493.563\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "246  TPM3+I+G4     1720.065     9   3458.131     3458.360     3500.236\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "265  TPM2u+F       1722.733     10  3465.466     3465.747     3512.250\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "266  TPM2u+F+I     1720.864     11  3463.728     3464.065     3515.190\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "267  TPM2u+F+G4    1720.923     11  3463.845     3464.183     3515.307\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "268  TPM2u+F+I+G4  1720.921     12  3465.842     3466.241     3521.982\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "287  TPM2          1723.696     7   3461.391     3461.534     3494.140\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "288  TPM2+I        1721.914     8   3459.827     3460.010     3497.254\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "289  TPM2+G4       1721.963     8   3459.926     3460.109     3497.353\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "290  TPM2+I+G4     1721.963     9   3461.926     3462.155     3504.031\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "309  K3Pu+F        1722.875     10  3465.750     3466.031     3512.534\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "310  K3Pu+F+I      1720.997     11  3463.994     3464.331     3515.455\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "311  K3Pu+F+G4     1721.040     11  3464.080     3464.417     3515.541\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "312  K3Pu+F+I+G4   1721.038     12  3466.077     3466.476     3522.217\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "331  K3P           1723.794     7   3461.588     3461.730     3494.336\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "332  K3P+I         1722.011     8   3460.022     3460.205     3497.449\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "333  K3P+G4        1722.047     8   3460.094     3460.277     3497.521\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "334  K3P+I+G4      1722.047     9   3462.094     3462.323     3504.199\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "353  TN+F          1722.891     10  3465.783     3466.063     3512.566\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "354  TN+F+I        1721.007     11  3464.014     3464.351     3515.475\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "355  TN+F+G4       1721.048     11  3464.096     3464.433     3515.558\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "356  TN+F+I+G4     1721.042     12  3466.085     3466.484     3522.225\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "375  TNe           1723.851     7   3461.702     3461.844     3494.450\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "376  TNe+I         1721.917     8   3459.833     3460.017     3497.260\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "377  TNe+G4        1721.983     8   3459.966     3460.149     3497.393\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "378  TNe+I+G4      1721.986     9   3461.971     3462.201     3504.076\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "397  HKY+F         1722.937     9   3463.875     3464.104     3505.980\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "398  HKY+F+I       1720.987     10  3461.974     3462.255     3508.758\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "399  HKY+F+G4      1721.037     10  3462.074     3462.355     3508.857\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "400  HKY+F+I+G4    1721.043     11  3464.085     3464.422     3515.547\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "419  K2P           1723.852     6   3459.703     3459.810     3487.773\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "420  K2P+I         1722.002     7   3458.005     3458.147     3490.753\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "421  K2P+G4        1722.044     7   3458.088     3458.230     3490.836\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "422  K2P+I+G4      1722.049     8   3460.097     3460.281     3497.524\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "441  F81+F         1759.440     8   3534.881     3535.064     3572.307\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "442  F81+F+I       1758.219     9   3534.438     3534.667     3576.543\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "443  F81+F+G4      1758.226     9   3534.452     3534.681     3576.557\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "444  F81+F+I+G4    1758.230     10  3536.460     3536.741     3583.244\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "463  JC            1760.467     5   3530.934     3531.010     3554.326\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "464  JC+I          1759.287     6   3530.574     3530.681     3558.644\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "465  JC+G4         1759.296     6   3530.593     3530.700     3558.663\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "466  JC+I+G4       1759.303     7   3532.607     3532.749     3565.355\n",
      "Akaike Information Criterion:           TPM3+G4\n",
      "Corrected Akaike Information Criterion: TPM3+G4\n",
      "Bayesian Information Criterion:         K2P\n",
      "Best-fit model: K2P chosen according to BIC\n",
      "\n",
      "All model information printed to result/phylo_fna_gene/100579at50557_iqtreeout/.tree.model.gz\n",
      "CPU time for ModelFinder: 0.195 seconds (0h:0m:0s)\n",
      "Wall-clock time for ModelFinder: 0.158 seconds (0h:0m:0s)\n",
      "Generating 1000 samples for ultrafast bootstrap (seed: 281503)...\n",
      "\n",
      "NOTE: 0 MB RAM (0 GB) is required!\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "Estimate model parameters (epsilon = 0.100)\n",
      "1. Initial log-likelihood: -1767.879\n",
      "2. Current log-likelihood: -1735.653\n",
      "3. Current log-likelihood: -1724.120\n",
      "4. Current log-likelihood: -1723.852\n",
      "Optimal log-likelihood: -1723.851\n",
      "Rate parameters:  A-C: 1.00000  A-G: 4.37731  A-T: 1.00000  C-G: 1.00000  C-T: 4.37731  G-T: 1.00000\n",
      "Base frequencies:  A: 0.250  C: 0.250  G: 0.250  T: 0.250\n",
      "Parameters optimization took 4 rounds (0.002 sec)\n",
      "Wrote distance file to... \n",
      "Computing ML distances based on estimated model parameters...\n",
      "Calculating distance matrix: done in 4.1658e-05 secs using 158.4% CPU\n",
      "Computing ML distances took 0.000525 sec (of wall-clock time) 0.000485 sec (of CPU time)\n",
      "Setting up auxiliary I and S matrices: done in 0.0223441 secs using 98.6% CPU\n",
      "Computing RapidNJ tree took 0.026992 sec (of wall-clock time) 0.026222 sec (of CPU time)\n",
      "Log-likelihood of RapidNJ tree: -1723.851\n",
      "--------------------------------------------------------------------\n",
      "|             INITIALIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "Generating 99 parsimony trees... 0.009 second\n",
      "Computing log-likelihood of 2 initial trees ... 0.000 seconds\n",
      "Current best score: -1723.851\n",
      "\n",
      "Do NNI search on 3 best initial trees\n",
      "Finish initializing candidate tree set (3)\n",
      "Current best tree score: -1723.851 / CPU time: 0.015\n",
      "Number of iterations: 3\n",
      "--------------------------------------------------------------------\n",
      "|               OPTIMIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "Iteration 10 / LogL: -1723.851 / Time: 0h:0m:0s\n",
      "Iteration 20 / LogL: -1723.852 / Time: 0h:0m:0s\n",
      "Iteration 30 / LogL: -1723.860 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 40 / LogL: -1723.852 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 50 / LogL: -1723.852 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -1729.252\n",
      "Iteration 60 / LogL: -1723.859 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 70 / LogL: -1723.860 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 80 / LogL: -1723.852 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 90 / LogL: -1723.852 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 100 / LogL: -1723.852 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -1729.252\n",
      "NOTE: Bootstrap correlation coefficient of split occurrence frequencies: 1.000\n",
      "TREE SEARCH COMPLETED AFTER 101 ITERATIONS / Time: 0h:0m:0s\n",
      "\n",
      "--------------------------------------------------------------------\n",
      "|                    FINALIZING TREE SEARCH                        |\n",
      "--------------------------------------------------------------------\n",
      "Performs final model parameters optimization\n",
      "Estimate model parameters (epsilon = 0.010)\n",
      "1. Initial log-likelihood: -1723.851\n",
      "Optimal log-likelihood: -1723.851\n",
      "Rate parameters:  A-C: 1.00000  A-G: 4.38257  A-T: 1.00000  C-G: 1.00000  C-T: 4.38257  G-T: 1.00000\n",
      "Base frequencies:  A: 0.250  C: 0.250  G: 0.250  T: 0.250\n",
      "Parameters optimization took 1 rounds (0.001 sec)\n",
      "BEST SCORE FOUND : -1723.851\n",
      "\n",
      "Testing tree branches by SH-like aLRT with 1000 replicates...\n",
      "0.002 sec.\n",
      "Creating bootstrap support values...\n",
      "Split supports printed to NEXUS file result/phylo_fna_gene/100579at50557_iqtreeout/.tree.splits.nex\n",
      "Total tree length: 0.249\n",
      "\n",
      "Total number of iterations: 101\n",
      "CPU time used for tree search: 0.723 sec (0h:0m:0s)\n",
      "Wall-clock time used for tree search: 0.403 sec (0h:0m:0s)\n",
      "Total CPU time used: 1.028 sec (0h:0m:1s)\n",
      "Total wall-clock time used: 0.656 sec (0h:0m:0s)\n",
      "\n",
      "Computing bootstrap consensus tree...\n",
      "Reading input file result/phylo_fna_gene/100579at50557_iqtreeout/.tree.splits.nex...\n",
      "4 taxa and 7 splits.\n",
      "Consensus tree written to result/phylo_fna_gene/100579at50557_iqtreeout/.tree.contree\n",
      "Reading input trees file result/phylo_fna_gene/100579at50557_iqtreeout/.tree.contree\n",
      "Log-likelihood of consensus tree: -1723.851\n",
      "\n",
      "Analysis results written to: \n",
      "  IQ-TREE report:                result/phylo_fna_gene/100579at50557_iqtreeout/.tree.iqtree\n",
      "  Maximum-likelihood tree:       result/phylo_fna_gene/100579at50557_iqtreeout/.tree.treefile\n",
      "  Likelihood distances:          result/phylo_fna_gene/100579at50557_iqtreeout/.tree.mldist\n",
      "\n",
      "Ultrafast bootstrap approximation results written to:\n",
      "  Split support values:          result/phylo_fna_gene/100579at50557_iqtreeout/.tree.splits.nex\n",
      "  Consensus tree:                result/phylo_fna_gene/100579at50557_iqtreeout/.tree.contree\n",
      "  Screen log file:               result/phylo_fna_gene/100579at50557_iqtreeout/.tree.log\n",
      "\n",
      "Date and Time: Sun Sep  1 10:03:39 2024\n",
      "IQ-TREE multicore version 2.3.6 for Linux x86 64-bit built Aug  4 2024\n",
      "Developed by Bui Quang Minh, Nguyen Lam Tung, Olga Chernomor, Heiko Schmidt,\n",
      "Dominik Schrempf, Michael Woodhams, Ly Trong Nhan, Thomas Wong\n",
      "\n",
      "Host:    codespaces-fa7cf2 (AVX2, FMA3, 7 GB RAM)\n",
      "Command: iqtree -s result/phylo_fna_gene/100764at50557_AlnTrimed.fna -nt 2 -m MFP -bb 1000 -alrt 1000 -pre result/phylo_fna_gene/100764at50557_iqtreeout/.tree\n",
      "Seed:    971911 (Using SPRNG - Scalable Parallel Random Number Generator)\n",
      "Time:    Sun Sep  1 10:03:39 2024\n",
      "Kernel:  AVX+FMA - 2 threads (2 CPU cores detected)\n",
      "\n",
      "Reading alignment file result/phylo_fna_gene/100764at50557_AlnTrimed.fna ... Fasta format detected\n",
      "Reading fasta file: done in 9.7251e-05 secs\n",
      "Alignment most likely contains DNA/RNA sequences\n",
      "Alignment has 4 sequences with 810 columns, 35 distinct patterns\n",
      "10 parsimony-informative, 117 singleton sites, 683 constant sites\n",
      "                        Gap/Ambiguity  Composition  p-value\n",
      "Analyzing sequences: done in 1.3736e-05 secs using 94.64% CPU\n",
      "   1  Culex_pipiens             0.00%    passed     86.98%\n",
      "   2  Culex_pipiens_pallens     0.00%    passed     91.04%\n",
      "   3  Armigeres_subalbatus      0.00%    passed     11.10%\n",
      "   4  Culex_pipiens_molestus    0.00%    passed     84.87%\n",
      "****  TOTAL                     0.00%  0 sequences failed composition chi2 test (p-value<5%; df=3)\n",
      "\n",
      "\n",
      "Create initial parsimony tree by phylogenetic likelihood library (PLL)... 0.000 seconds\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "Perform fast likelihood tree search using GTR+I+G model...\n",
      "Estimate model parameters (epsilon = 5.000)\n",
      "Perform nearest neighbor interchange...\n",
      "Estimate model parameters (epsilon = 1.000)\n",
      "1. Initial log-likelihood: -1685.715\n",
      "Optimal log-likelihood: -1685.684\n",
      "Rate parameters:  A-C: 0.42690  A-G: 2.47577  A-T: 0.00010  C-G: 1.77576  C-T: 2.36252  G-T: 1.00000\n",
      "Warning! Some parameters hit the boundaries\n",
      "Base frequencies:  A: 0.180  C: 0.285  G: 0.308  T: 0.227\n",
      "Proportion of invariable sites: 0.421\n",
      "Gamma shape alpha: 1.107\n",
      "Parameters optimization took 1 rounds (0.002 sec)\n",
      "Time for fast ML tree search: 0.007 seconds\n",
      "\n",
      "NOTE: ModelFinder requires 0 MB RAM!\n",
      "ModelFinder will test up to 484 DNA models (sample size: 810 epsilon: 0.100) ...\n",
      " No. Model         -LnL         df  AIC          AICc         BIC\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  1  GTR+F         1689.239     13  3404.477     3404.934     3465.539\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  2  GTR+F+I       1685.611     14  3399.221     3399.750     3464.980\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  3  GTR+F+G4      1685.650     14  3399.299     3399.827     3465.058\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  4  GTR+F+I+G4    1685.655     15  3401.309     3401.914     3471.765\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  5  GTR+F+R2      1686.282     15  3402.565     3403.169     3473.020\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  6  GTR+F+R3      1685.978     17  3405.956     3406.729     3485.806\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 14  GTR+F+I+R2    1685.802     16  3403.604     3404.290     3478.756\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 15  GTR+F+I+R3    1685.624     18  3407.248     3408.112     3491.794\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 23  SYM           1703.012     10  3426.024     3426.299     3472.994\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 24  SYM+I         1700.099     11  3422.198     3422.529     3473.865\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 25  SYM+G4        1700.171     11  3422.342     3422.672     3474.009\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 26  SYM+I+G4      1700.186     12  3424.372     3424.763     3480.736\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 27  SYM+R2        1700.406     12  3424.813     3425.204     3481.177\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 45  TVM+F         1689.286     12  3402.573     3402.964     3458.937\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 46  TVM+F+I       1685.667     13  3397.334     3397.791     3458.395\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 47  TVM+F+G4      1685.684     13  3397.367     3397.825     3458.429\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 48  TVM+F+I+G4    1685.675     14  3399.350     3399.878     3465.108\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 49  TVM+F+R2      1686.189     14  3400.378     3400.906     3466.136\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 67  TVMe          1703.019     9   3424.038     3424.263     3466.312\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 68  TVMe+I        1700.112     10  3420.224     3420.499     3467.194\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 69  TVMe+G4       1700.189     10  3420.378     3420.653     3467.348\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 70  TVMe+I+G4     1700.199     11  3422.397     3422.728     3474.064\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 71  TVMe+R2       1700.336     11  3422.671     3423.002     3474.339\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 89  TIM3+F        1697.670     11  3417.340     3417.671     3469.008\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 90  TIM3+F+I      1693.095     12  3410.189     3410.581     3466.554\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 91  TIM3+F+G4     1693.085     12  3410.170     3410.561     3466.534\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 92  TIM3+F+I+G4   1693.108     13  3412.216     3412.674     3473.278\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 93  TIM3+F+R2     1694.054     13  3414.108     3414.566     3475.170\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "111  TIM3e         1715.968     8   3447.935     3448.115     3485.512\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "112  TIM3e+I       1712.404     9   3442.808     3443.033     3485.081\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "113  TIM3e+G4      1712.413     9   3442.826     3443.051     3485.099\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "114  TIM3e+I+G4    1712.419     10  3444.838     3445.113     3491.808\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "115  TIM3e+R2      1712.831     10  3445.663     3445.938     3492.633\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "133  TIM2+F        1692.216     11  3406.432     3406.763     3458.099\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "134  TIM2+F+I      1688.499     12  3400.999     3401.390     3457.363\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "135  TIM2+F+G4     1688.507     12  3401.014     3401.405     3457.378\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "136  TIM2+F+I+G4   1688.516     13  3403.032     3403.489     3464.093\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "137  TIM2+F+R2     1688.898     13  3403.796     3404.253     3464.858\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "155  TIM2e         1707.094     8   3430.188     3430.368     3467.764\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "156  TIM2e+I       1704.209     9   3426.417     3426.642     3468.691\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "157  TIM2e+G4      1704.251     9   3426.502     3426.727     3468.775\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "158  TIM2e+I+G4    1704.270     10  3428.540     3428.816     3475.511\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "159  TIM2e+R2      1704.300     10  3428.600     3428.876     3475.571\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "177  TIM+F         1699.064     11  3420.127     3420.458     3471.795\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "178  TIM+F+I       1694.161     12  3412.322     3412.714     3468.687\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "179  TIM+F+G4      1694.226     12  3412.451     3412.843     3468.816\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "180  TIM+F+I+G4    1694.324     13  3414.648     3415.105     3475.709\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "181  TIM+F+R2      1695.050     13  3416.100     3416.557     3477.162\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "199  TIMe          1718.101     8   3452.201     3452.381     3489.777\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "200  TIMe+I        1714.299     9   3446.597     3446.822     3488.870\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "201  TIMe+G4       1714.361     9   3446.723     3446.948     3488.996\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "202  TIMe+I+G4     1714.400     10  3448.800     3449.075     3495.770\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "203  TIMe+R2       1714.650     10  3449.301     3449.576     3496.271\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "221  TPM3u+F       1697.703     10  3415.406     3415.681     3462.376\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "222  TPM3u+F+I     1693.071     11  3408.142     3408.473     3459.809\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "223  TPM3u+F+G4    1693.051     11  3408.101     3408.432     3459.769\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "224  TPM3u+F+I+G4  1693.075     12  3410.151     3410.542     3466.515\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "225  TPM3u+F+R2    1693.681     12  3411.361     3411.753     3467.726\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "243  TPM3          1715.977     7   3445.953     3446.093     3478.832\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "244  TPM3+I        1712.456     8   3440.911     3441.091     3478.487\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "245  TPM3+G4       1712.467     8   3440.935     3441.114     3478.511\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "246  TPM3+I+G4     1712.488     9   3442.977     3443.202     3485.250\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "247  TPM3+R2       1712.623     9   3443.246     3443.471     3485.519\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "265  TPM2u+F       1692.269     10  3404.537     3404.813     3451.508\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "266  TPM2u+F+I     1688.508     11  3399.016     3399.347     3450.684\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "267  TPM2u+F+G4    1688.514     11  3399.028     3399.359     3450.695\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "268  TPM2u+F+I+G4  1688.527     12  3401.053     3401.445     3457.418\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "269  TPM2u+F+R2    1688.687     12  3401.374     3401.765     3457.738\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "287  TPM2          1707.101     7   3428.201     3428.341     3461.080\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "288  TPM2+I        1704.222     8   3424.444     3424.624     3462.021\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "289  TPM2+G4       1704.265     8   3424.530     3424.710     3462.106\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "290  TPM2+I+G4     1704.289     9   3426.577     3426.802     3468.851\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "291  TPM2+R2       1704.224     9   3426.448     3426.673     3468.721\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "309  K3Pu+F        1699.101     10  3418.203     3418.478     3465.173\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "310  K3Pu+F+I      1694.198     11  3410.396     3410.727     3462.064\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "311  K3Pu+F+G4     1694.222     11  3410.443     3410.774     3462.111\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "312  K3Pu+F+I+G4   1694.344     12  3412.688     3413.079     3469.052\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "313  K3Pu+F+R2     1694.740     12  3413.480     3413.872     3469.845\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "331  K3P           1718.108     7   3450.216     3450.356     3483.095\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "332  K3P+I         1714.338     8   3444.676     3444.856     3482.253\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "333  K3P+G4        1714.402     8   3444.804     3444.984     3482.381\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "334  K3P+I+G4      1714.441     9   3446.882     3447.107     3489.155\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "335  K3P+R2        1714.483     9   3446.967     3447.192     3489.240\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "353  TN+F          1700.321     10  3420.642     3420.917     3467.612\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "354  TN+F+I        1695.447     11  3412.893     3413.224     3464.561\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "355  TN+F+G4       1695.423     11  3412.845     3413.176     3464.513\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "356  TN+F+I+G4     1695.431     12  3414.862     3415.254     3471.227\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "357  TN+F+R2       1695.890     12  3415.781     3416.172     3472.145\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "375  TNe           1719.644     7   3453.287     3453.427     3486.166\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "376  TNe+I         1715.915     8   3447.830     3448.010     3485.407\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "377  TNe+G4        1715.893     8   3447.786     3447.965     3485.362\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "378  TNe+I+G4      1715.868     9   3449.737     3449.962     3492.010\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "379  TNe+R2        1716.016     9   3450.032     3450.257     3492.305\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "397  HKY+F         1700.358     9   3418.716     3418.941     3460.990\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "398  HKY+F+I       1695.446     10  3410.893     3411.168     3457.863\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "399  HKY+F+G4      1695.414     10  3410.829     3411.104     3457.799\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "400  HKY+F+I+G4    1695.403     11  3412.807     3413.138     3464.474\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "401  HKY+F+R2      1695.820     11  3413.639     3413.970     3465.307\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "419  K2P           1719.651     6   3451.302     3451.407     3479.485\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "420  K2P+I         1715.959     7   3445.917     3446.057     3478.797\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "421  K2P+G4        1715.937     7   3445.873     3446.013     3478.753\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "422  K2P+I+G4      1715.928     8   3447.856     3448.036     3485.432\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "423  K2P+R2        1716.009     8   3448.018     3448.198     3485.594\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "441  F81+F         1709.185     8   3434.369     3434.549     3471.946\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "442  F81+F+I       1705.348     9   3428.696     3428.921     3470.970\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "443  F81+F+G4      1705.366     9   3428.733     3428.958     3471.006\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "444  F81+F+I+G4    1705.391     10  3430.783     3431.058     3477.753\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "445  F81+F+R2      1705.414     10  3430.829     3431.104     3477.799\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "463  JC            1727.750     5   3465.501     3465.575     3488.986\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "464  JC+I          1724.588     6   3461.176     3461.281     3489.359\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "465  JC+G4         1724.610     6   3461.220     3461.325     3489.403\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "466  JC+I+G4       1724.633     7   3463.266     3463.406     3496.146\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "467  JC+R2         1724.583     7   3463.165     3463.305     3496.045\n",
      "Akaike Information Criterion:           TVM+F+I\n",
      "Corrected Akaike Information Criterion: TVM+F+I\n",
      "Bayesian Information Criterion:         TPM2u+F+I\n",
      "Best-fit model: TPM2u+F+I chosen according to BIC\n",
      "\n",
      "All model information printed to result/phylo_fna_gene/100764at50557_iqtreeout/.tree.model.gz\n",
      "CPU time for ModelFinder: 0.439 seconds (0h:0m:0s)\n",
      "Wall-clock time for ModelFinder: 0.381 seconds (0h:0m:0s)\n",
      "Generating 1000 samples for ultrafast bootstrap (seed: 971911)...\n",
      "\n",
      "NOTE: 0 MB RAM (0 GB) is required!\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "Estimate model parameters (epsilon = 0.100)\n",
      "1. Initial log-likelihood: -1705.402\n",
      "2. Current log-likelihood: -1688.560\n",
      "Optimal log-likelihood: -1688.514\n",
      "Rate parameters:  A-C: 0.16053  A-G: 1.76944  A-T: 0.16053  C-G: 1.00000  C-T: 1.76944  G-T: 1.00000\n",
      "Base frequencies:  A: 0.180  C: 0.285  G: 0.308  T: 0.227\n",
      "Proportion of invariable sites: 0.639\n",
      "Parameters optimization took 2 rounds (0.004 sec)\n",
      "Wrote distance file to... \n",
      "Computing ML distances based on estimated model parameters...\n",
      "Calculating distance matrix: done in 3.9383e-05 secs using 365.6% CPU\n",
      "Computing ML distances took 0.000878 sec (of wall-clock time) 0.000395 sec (of CPU time)\n",
      "Setting up auxiliary I and S matrices: done in 0.000165609 secs using 93.59% CPU\n",
      "Computing RapidNJ tree took 0.000677 sec (of wall-clock time) 0.000687 sec (of CPU time)\n",
      "Log-likelihood of RapidNJ tree: -1688.514\n",
      "--------------------------------------------------------------------\n",
      "|             INITIALIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "Generating 99 parsimony trees... 0.011 second\n",
      "Computing log-likelihood of 2 initial trees ... 0.005 seconds\n",
      "Current best score: -1688.514\n",
      "\n",
      "Do NNI search on 3 best initial trees\n",
      "Finish initializing candidate tree set (3)\n",
      "Current best tree score: -1688.514 / CPU time: 0.029\n",
      "Number of iterations: 3\n",
      "--------------------------------------------------------------------\n",
      "|               OPTIMIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "UPDATE BEST LOG-LIKELIHOOD: -1688.514\n",
      "Iteration 10 / LogL: -1689.015 / Time: 0h:0m:0s\n",
      "UPDATE BEST LOG-LIKELIHOOD: -1688.514\n",
      "Iteration 20 / LogL: -1688.592 / Time: 0h:0m:0s\n",
      "Iteration 30 / LogL: -1689.015 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 40 / LogL: -1688.592 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 50 / LogL: -1688.592 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -1691.524\n",
      "Iteration 60 / LogL: -1689.015 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 70 / LogL: -1688.592 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 80 / LogL: -1689.015 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 90 / LogL: -1689.015 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 100 / LogL: -1689.015 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -1691.524\n",
      "NOTE: Bootstrap correlation coefficient of split occurrence frequencies: 1.000\n",
      "TREE SEARCH COMPLETED AFTER 101 ITERATIONS / Time: 0h:0m:0s\n",
      "\n",
      "--------------------------------------------------------------------\n",
      "|                    FINALIZING TREE SEARCH                        |\n",
      "--------------------------------------------------------------------\n",
      "Performs final model parameters optimization\n",
      "Estimate model parameters (epsilon = 0.010)\n",
      "1. Initial log-likelihood: -1688.514\n",
      "Optimal log-likelihood: -1688.509\n",
      "Rate parameters:  A-C: 0.15609  A-G: 1.77027  A-T: 0.15609  C-G: 1.00000  C-T: 1.77027  G-T: 1.00000\n",
      "Base frequencies:  A: 0.180  C: 0.285  G: 0.308  T: 0.227\n",
      "Proportion of invariable sites: 0.644\n",
      "Parameters optimization took 1 rounds (0.001 sec)\n",
      "BEST SCORE FOUND : -1688.509\n",
      "\n",
      "Testing tree branches by SH-like aLRT with 1000 replicates...\n",
      "0.003 sec.\n",
      "Creating bootstrap support values...\n",
      "Split supports printed to NEXUS file result/phylo_fna_gene/100764at50557_iqtreeout/.tree.splits.nex\n",
      "Total tree length: 0.238\n",
      "\n",
      "Total number of iterations: 101\n",
      "CPU time used for tree search: 0.750 sec (0h:0m:0s)\n",
      "Wall-clock time used for tree search: 0.539 sec (0h:0m:0s)\n",
      "Total CPU time used: 1.244 sec (0h:0m:1s)\n",
      "Total wall-clock time used: 0.981 sec (0h:0m:0s)\n",
      "\n",
      "Computing bootstrap consensus tree...\n",
      "Reading input file result/phylo_fna_gene/100764at50557_iqtreeout/.tree.splits.nex...\n",
      "4 taxa and 7 splits.\n",
      "Consensus tree written to result/phylo_fna_gene/100764at50557_iqtreeout/.tree.contree\n",
      "Reading input trees file result/phylo_fna_gene/100764at50557_iqtreeout/.tree.contree\n",
      "Log-likelihood of consensus tree: -1688.509\n",
      "\n",
      "Analysis results written to: \n",
      "  IQ-TREE report:                result/phylo_fna_gene/100764at50557_iqtreeout/.tree.iqtree\n",
      "  Maximum-likelihood tree:       result/phylo_fna_gene/100764at50557_iqtreeout/.tree.treefile\n",
      "  Likelihood distances:          result/phylo_fna_gene/100764at50557_iqtreeout/.tree.mldist\n",
      "\n",
      "Ultrafast bootstrap approximation results written to:\n",
      "  Split support values:          result/phylo_fna_gene/100764at50557_iqtreeout/.tree.splits.nex\n",
      "  Consensus tree:                result/phylo_fna_gene/100764at50557_iqtreeout/.tree.contree\n",
      "  Screen log file:               result/phylo_fna_gene/100764at50557_iqtreeout/.tree.log\n",
      "\n",
      "Date and Time: Sun Sep  1 10:03:40 2024\n",
      "IQ-TREE multicore version 2.3.6 for Linux x86 64-bit built Aug  4 2024\n",
      "Developed by Bui Quang Minh, Nguyen Lam Tung, Olga Chernomor, Heiko Schmidt,\n",
      "Dominik Schrempf, Michael Woodhams, Ly Trong Nhan, Thomas Wong\n",
      "\n",
      "Host:    codespaces-fa7cf2 (AVX2, FMA3, 7 GB RAM)\n",
      "Command: iqtree -s result/phylo_fna_gene/100854at50557_AlnTrimed.fna -nt 2 -m MFP -bb 1000 -alrt 1000 -pre result/phylo_fna_gene/100854at50557_iqtreeout/.tree\n",
      "Seed:    984561 (Using SPRNG - Scalable Parallel Random Number Generator)\n",
      "Time:    Sun Sep  1 10:03:40 2024\n",
      "Kernel:  AVX+FMA - 2 threads (2 CPU cores detected)\n",
      "\n",
      "Reading alignment file result/phylo_fna_gene/100854at50557_AlnTrimed.fna ... Fasta format detected\n",
      "Reading fasta file: done in 0.00012872 secs\n",
      "Alignment most likely contains DNA/RNA sequences\n",
      "Alignment has 4 sequences with 1206 columns, 39 distinct patterns\n",
      "14 parsimony-informative, 231 singleton sites, 961 constant sites\n",
      "                        Gap/Ambiguity  Composition  p-value\n",
      "Analyzing sequences: done in 1.4357e-05 secs using 83.58% CPU\n",
      "   1  Culex_pipiens             0.00%    passed     72.10%\n",
      "   2  Culex_pipiens_pallens     0.00%    passed     56.99%\n",
      "   3  Armigeres_subalbatus      0.00%    failed      0.18%\n",
      "   4  Culex_pipiens_molestus    0.00%    passed     60.04%\n",
      "****  TOTAL                     0.00%  1 sequences failed composition chi2 test (p-value<5%; df=3)\n",
      "\n",
      "\n",
      "Create initial parsimony tree by phylogenetic likelihood library (PLL)... 0.000 seconds\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "Perform fast likelihood tree search using GTR+I+G model...\n",
      "Estimate model parameters (epsilon = 5.000)\n",
      "Perform nearest neighbor interchange...\n",
      "Estimate model parameters (epsilon = 1.000)\n",
      "1. Initial log-likelihood: -2620.220\n",
      "Optimal log-likelihood: -2620.219\n",
      "Rate parameters:  A-C: 6.44427  A-G: 21.05155  A-T: 3.92316  C-G: 8.79925  C-T: 46.66209  G-T: 1.00000\n",
      "Base frequencies:  A: 0.308  C: 0.192  G: 0.293  T: 0.208\n",
      "Proportion of invariable sites: 0.397\n",
      "Gamma shape alpha: 1.731\n",
      "Parameters optimization took 1 rounds (0.002 sec)\n",
      "Time for fast ML tree search: 0.007 seconds\n",
      "\n",
      "NOTE: ModelFinder requires 0 MB RAM!\n",
      "ModelFinder will test up to 484 DNA models (sample size: 1206 epsilon: 0.100) ...\n",
      " No. Model         -LnL         df  AIC          AICc         BIC\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  1  GTR+F         2623.514     13  5273.027     5273.332     5339.263\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  2  GTR+F+I       2620.512     14  5269.023     5269.376     5340.354\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  3  GTR+F+G4      2620.231     14  5268.462     5268.814     5339.793\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  4  GTR+F+I+G4    2620.193     15  5270.386     5270.789     5346.812\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  5  GTR+F+R2      2620.781     15  5271.561     5271.965     5347.987\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  6  GTR+F+R3      2620.685     17  5275.369     5275.884     5361.985\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 14  GTR+F+I+R2    2620.503     16  5273.006     5273.463     5354.527\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 15  GTR+F+I+R3    2620.375     18  5276.751     5277.327     5368.462\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 23  SYM           2649.572     10  5319.145     5319.329     5370.095\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 24  SYM+I         2646.612     11  5315.223     5315.444     5371.269\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 25  SYM+G4        2646.247     11  5314.494     5314.715     5370.540\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 26  SYM+I+G4      2646.310     12  5316.621     5316.882     5377.761\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 27  SYM+R2        2646.841     12  5317.683     5317.944     5378.823\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 45  TVM+F         2630.646     12  5285.292     5285.553     5346.433\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 46  TVM+F+I       2626.743     13  5279.485     5279.791     5345.721\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 47  TVM+F+G4      2626.580     13  5279.159     5279.464     5345.395\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 48  TVM+F+I+G4    2626.535     14  5281.070     5281.423     5352.401\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 49  TVM+F+R2      2627.305     14  5282.610     5282.963     5353.941\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 67  TVMe          2650.514     9   5319.027     5319.178     5364.883\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 68  TVMe+I        2647.564     10  5315.128     5315.312     5366.079\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 69  TVMe+G4       2647.250     10  5314.499     5314.683     5365.450\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 70  TVMe+I+G4     2647.324     11  5316.648     5316.869     5372.694\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 71  TVMe+R2       2647.712     11  5317.424     5317.645     5373.470\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 89  TIM3+F        2623.559     11  5269.118     5269.339     5325.164\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 90  TIM3+F+I      2620.843     12  5265.687     5265.948     5326.827\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 91  TIM3+F+G4     2620.607     12  5265.213     5265.475     5326.354\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 92  TIM3+F+I+G4   2620.716     13  5267.432     5267.737     5333.668\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 93  TIM3+F+R2     2620.869     13  5267.738     5268.044     5333.974\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "111  TIM3e         2649.622     8   5315.244     5315.364     5356.005\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "112  TIM3e+I       2646.926     9   5311.853     5312.003     5357.708\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "113  TIM3e+G4      2646.690     9   5311.380     5311.531     5357.236\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "114  TIM3e+I+G4    2646.781     10  5313.562     5313.746     5364.512\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "115  TIM3e+R2      2646.947     10  5313.894     5314.078     5364.844\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "133  TIM2+F        2626.984     11  5275.967     5276.188     5332.013\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "134  TIM2+F+I      2623.969     12  5271.938     5272.200     5333.079\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "135  TIM2+F+G4     2623.848     12  5271.696     5271.957     5332.837\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "136  TIM2+F+I+G4   2623.868     13  5273.736     5274.041     5339.972\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "137  TIM2+F+R2     2624.115     13  5274.231     5274.536     5340.467\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "155  TIM2e         2652.547     8   5321.094     5321.214     5361.855\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "156  TIM2e+I       2649.563     9   5317.126     5317.276     5362.981\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "157  TIM2e+G4      2649.439     9   5316.877     5317.028     5362.733\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "158  TIM2e+I+G4    2649.463     10  5318.926     5319.110     5369.876\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "159  TIM2e+R2      2649.703     10  5319.406     5319.591     5370.357\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "177  TIM+F         2626.990     11  5275.979     5276.200     5332.025\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "178  TIM+F+I       2623.896     12  5271.793     5272.054     5332.933\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "179  TIM+F+G4      2623.769     12  5271.538     5271.799     5332.679\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "180  TIM+F+I+G4    2623.790     13  5273.580     5273.886     5339.816\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "181  TIM+F+R2      2624.033     13  5274.065     5274.370     5340.301\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "199  TIMe          2652.545     8   5321.090     5321.210     5361.850\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "200  TIMe+I        2649.495     9   5316.990     5317.141     5362.846\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "201  TIMe+G4       2649.361     9   5316.722     5316.873     5362.578\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "202  TIMe+I+G4     2649.384     10  5318.769     5318.953     5369.719\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "203  TIMe+R2       2649.615     10  5319.229     5319.413     5370.180\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "221  TPM3u+F       2630.682     10  5281.364     5281.548     5332.314\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "222  TPM3u+F+I     2627.118     11  5276.236     5276.457     5332.282\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "223  TPM3u+F+G4    2626.956     11  5275.913     5276.134     5331.959\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "224  TPM3u+F+I+G4  2626.979     12  5277.958     5278.220     5339.099\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "225  TPM3u+F+R2    2627.300     12  5278.601     5278.862     5339.741\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "243  TPM3          2650.556     7   5315.111     5315.205     5350.777\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "244  TPM3+I        2647.885     8   5311.769     5311.890     5352.530\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "245  TPM3+G4       2647.655     8   5311.310     5311.431     5352.071\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "246  TPM3+I+G4     2647.751     9   5313.502     5313.652     5359.357\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "247  TPM3+R2       2647.834     9   5313.669     5313.819     5359.524\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "265  TPM2u+F       2634.107     10  5288.214     5288.398     5339.164\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "266  TPM2u+F+I     2630.165     11  5282.330     5282.551     5338.376\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "267  TPM2u+F+G4    2630.154     11  5282.308     5282.530     5338.354\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "268  TPM2u+F+I+G4  2630.134     12  5284.268     5284.530     5345.409\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "269  TPM2u+F+R2    2630.514     12  5285.027     5285.289     5346.168\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "287  TPM2          2653.476     7   5320.952     5321.045     5356.617\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "288  TPM2+I        2650.542     8   5317.083     5317.203     5357.844\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "289  TPM2+G4       2650.407     8   5316.813     5316.934     5357.574\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "290  TPM2+I+G4     2650.449     9   5318.898     5319.048     5364.754\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "291  TPM2+R2       2650.582     9   5319.164     5319.315     5365.020\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "309  K3Pu+F        2634.111     10  5288.222     5288.406     5339.172\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "310  K3Pu+F+I      2630.121     11  5282.241     5282.462     5338.287\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "311  K3Pu+F+G4     2630.078     11  5282.157     5282.378     5338.203\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "312  K3Pu+F+I+G4   2630.072     12  5284.143     5284.405     5345.284\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "313  K3Pu+F+R2     2630.428     12  5284.856     5285.117     5345.996\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "331  K3P           2653.475     7   5320.949     5321.042     5356.614\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "332  K3P+I         2650.485     8   5316.971     5317.091     5357.731\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "333  K3P+G4        2650.345     8   5316.691     5316.811     5357.451\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "334  K3P+I+G4      2650.388     9   5318.776     5318.926     5364.631\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "335  K3P+R2        2650.506     9   5319.011     5319.162     5364.867\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "353  TN+F          2626.991     10  5273.983     5274.167     5324.933\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "354  TN+F+I        2623.955     11  5269.909     5270.130     5325.955\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "355  TN+F+G4       2623.853     11  5269.706     5269.927     5325.751\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "356  TN+F+I+G4     2623.878     12  5271.757     5272.018     5332.898\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "357  TN+F+R2       2623.979     12  5271.959     5272.220     5333.100\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "375  TNe           2652.548     7   5319.096     5319.189     5354.761\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "376  TNe+I         2649.580     8   5315.159     5315.280     5355.920\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "377  TNe+G4        2649.464     8   5314.928     5315.048     5355.689\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "378  TNe+I+G4      2649.490     9   5316.981     5317.131     5362.836\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "379  TNe+R2        2649.584     9   5317.167     5317.318     5363.023\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "397  HKY+F         2634.111     9   5286.223     5286.373     5332.079\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "398  HKY+F+I       2630.199     10  5280.398     5280.582     5331.348\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "399  HKY+F+G4      2630.180     10  5280.360     5280.544     5331.311\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "400  HKY+F+I+G4    2630.168     11  5282.335     5282.556     5338.381\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "401  HKY+F+R2      2630.415     11  5282.829     5283.050     5338.875\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "419  K2P           2653.479     6   5318.958     5319.028     5349.529\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "420  K2P+I         2650.576     7   5315.151     5315.245     5350.817\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "421  K2P+G4        2650.450     7   5314.899     5314.993     5350.564\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "422  K2P+I+G4      2650.494     8   5316.989     5317.109     5357.749\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "423  K2P+R2        2650.548     8   5317.095     5317.215     5357.856\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "441  F81+F         2691.277     8   5398.555     5398.675     5439.315\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "442  F81+F+I       2688.757     9   5395.514     5395.665     5441.370\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "443  F81+F+G4      2688.788     9   5395.575     5395.726     5441.431\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "444  F81+F+I+G4    2688.779     10  5397.558     5397.742     5448.509\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "445  F81+F+R2      2688.811     10  5397.621     5397.805     5448.572\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "463  JC            2717.204     5   5444.407     5444.457     5469.883\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "464  JC+I          2714.727     6   5441.454     5441.524     5472.024\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "465  JC+G4         2714.756     6   5441.511     5441.581     5472.082\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "466  JC+I+G4       2714.748     7   5443.496     5443.589     5479.161\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "467  JC+R2         2714.772     7   5443.544     5443.637     5479.209\n",
      "Akaike Information Criterion:           TIM3+F+G4\n",
      "Corrected Akaike Information Criterion: TIM3+F+G4\n",
      "Bayesian Information Criterion:         TN+F\n",
      "Best-fit model: TN+F chosen according to BIC\n",
      "\n",
      "All model information printed to result/phylo_fna_gene/100854at50557_iqtreeout/.tree.model.gz\n",
      "CPU time for ModelFinder: 0.322 seconds (0h:0m:0s)\n",
      "Wall-clock time for ModelFinder: 0.224 seconds (0h:0m:0s)\n",
      "Generating 1000 samples for ultrafast bootstrap (seed: 984561)...\n",
      "\n",
      "NOTE: 0 MB RAM (0 GB) is required!\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "Estimate model parameters (epsilon = 0.100)\n",
      "1. Initial log-likelihood: -2694.247\n",
      "2. Current log-likelihood: -2627.130\n",
      "3. Current log-likelihood: -2626.992\n",
      "Optimal log-likelihood: -2626.991\n",
      "Rate parameters:  A-C: 1.00000  A-G: 3.50547  A-T: 1.00000  C-G: 1.00000  C-T: 6.61380  G-T: 1.00000\n",
      "Base frequencies:  A: 0.308  C: 0.192  G: 0.293  T: 0.208\n",
      "Parameters optimization took 3 rounds (0.005 sec)\n",
      "Wrote distance file to... \n",
      "Computing ML distances based on estimated model parameters...\n",
      "Calculating distance matrix: done in 4.0005e-05 secs using 220% CPU\n",
      "Computing ML distances took 0.000535 sec (of wall-clock time) 0.000504 sec (of CPU time)\n",
      "Setting up auxiliary I and S matrices: done in 0.000183793 secs\n",
      "Computing RapidNJ tree took 0.000700 sec (of wall-clock time) 0.000010 sec (of CPU time)\n",
      "Log-likelihood of RapidNJ tree: -2626.991\n",
      "--------------------------------------------------------------------\n",
      "|             INITIALIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "Generating 99 parsimony trees... 0.010 second\n",
      "Computing log-likelihood of 2 initial trees ... 0.000 seconds\n",
      "Current best score: -2626.991\n",
      "\n",
      "Do NNI search on 3 best initial trees\n",
      "Finish initializing candidate tree set (3)\n",
      "Current best tree score: -2626.991 / CPU time: 0.016\n",
      "Number of iterations: 3\n",
      "--------------------------------------------------------------------\n",
      "|               OPTIMIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "Iteration 10 / LogL: -2627.018 / Time: 0h:0m:0s\n",
      "Iteration 20 / LogL: -2627.018 / Time: 0h:0m:0s\n",
      "Iteration 30 / LogL: -2627.018 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 40 / LogL: -2627.045 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 50 / LogL: -2627.045 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -2642.773\n",
      "Iteration 60 / LogL: -2627.047 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 70 / LogL: -2627.017 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 80 / LogL: -2626.991 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 90 / LogL: -2627.047 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 100 / LogL: -2627.018 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -2642.820\n",
      "NOTE: Bootstrap correlation coefficient of split occurrence frequencies: 1.000\n",
      "TREE SEARCH COMPLETED AFTER 101 ITERATIONS / Time: 0h:0m:0s\n",
      "\n",
      "--------------------------------------------------------------------\n",
      "|                    FINALIZING TREE SEARCH                        |\n",
      "--------------------------------------------------------------------\n",
      "Performs final model parameters optimization\n",
      "Estimate model parameters (epsilon = 0.010)\n",
      "1. Initial log-likelihood: -2626.991\n",
      "Optimal log-likelihood: -2626.991\n",
      "Rate parameters:  A-C: 1.00000  A-G: 3.50944  A-T: 1.00000  C-G: 1.00000  C-T: 6.62322  G-T: 1.00000\n",
      "Base frequencies:  A: 0.308  C: 0.192  G: 0.293  T: 0.208\n",
      "Parameters optimization took 1 rounds (0.001 sec)\n",
      "BEST SCORE FOUND : -2626.991\n",
      "\n",
      "Testing tree branches by SH-like aLRT with 1000 replicates...\n",
      "0.003 sec.\n",
      "Creating bootstrap support values...\n",
      "Split supports printed to NEXUS file result/phylo_fna_gene/100854at50557_iqtreeout/.tree.splits.nex\n",
      "Total tree length: 0.247\n",
      "\n",
      "Total number of iterations: 101\n",
      "CPU time used for tree search: 0.736 sec (0h:0m:0s)\n",
      "Wall-clock time used for tree search: 0.446 sec (0h:0m:0s)\n",
      "Total CPU time used: 1.121 sec (0h:0m:1s)\n",
      "Total wall-clock time used: 0.728 sec (0h:0m:0s)\n",
      "\n",
      "Computing bootstrap consensus tree...\n",
      "Reading input file result/phylo_fna_gene/100854at50557_iqtreeout/.tree.splits.nex...\n",
      "4 taxa and 6 splits.\n",
      "Consensus tree written to result/phylo_fna_gene/100854at50557_iqtreeout/.tree.contree\n",
      "Reading input trees file result/phylo_fna_gene/100854at50557_iqtreeout/.tree.contree\n",
      "Log-likelihood of consensus tree: -2626.991\n",
      "\n",
      "Analysis results written to: \n",
      "  IQ-TREE report:                result/phylo_fna_gene/100854at50557_iqtreeout/.tree.iqtree\n",
      "  Maximum-likelihood tree:       result/phylo_fna_gene/100854at50557_iqtreeout/.tree.treefile\n",
      "  Likelihood distances:          result/phylo_fna_gene/100854at50557_iqtreeout/.tree.mldist\n",
      "\n",
      "Ultrafast bootstrap approximation results written to:\n",
      "  Split support values:          result/phylo_fna_gene/100854at50557_iqtreeout/.tree.splits.nex\n",
      "  Consensus tree:                result/phylo_fna_gene/100854at50557_iqtreeout/.tree.contree\n",
      "  Screen log file:               result/phylo_fna_gene/100854at50557_iqtreeout/.tree.log\n",
      "\n",
      "Date and Time: Sun Sep  1 10:03:41 2024\n"
     ]
    }
   ],
   "source": [
    "#使用iqtree完成，如果数据量过大，可以使用fasttree快速构建一个树文件，暂时代替\n",
    "! ls result/phylo_fna_gene/*_AlnTrimed.fna | while read file;do mkdir -p ${file%_AlnTrimed.fna}_iqtreeout;done\n",
    "! ls result/phylo_fna_gene/*_AlnTrimed.fna | while read file; do iqtree -s $file -nt 2 -m MFP -bb 1000 -alrt 1000 -pre ${file%_AlnTrimed.fna}_iqtreeout/$(basename ${i%_AlnTrimed.fna}.tree); done\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# 2、利用修剪后的基因比对序列建物种树"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "#先将Trimal后的序列提取出来\n",
    "! mkdir -p result/Aln_fna\n",
    "! cp -r result/phylo_fna_gene/*_AlnTrimed.fna result/Aln_fna/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "# 按照物种划分序列，得到对应物种的全序列，注意，缺失的基因用gap替换\n",
    "! python3 src/species.py result/Aln_fna/ result/species_fna/ fna"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "IQ-TREE multicore version 2.3.6 for Linux x86 64-bit built Aug  4 2024\n",
      "Developed by Bui Quang Minh, Nguyen Lam Tung, Olga Chernomor, Heiko Schmidt,\n",
      "Dominik Schrempf, Michael Woodhams, Ly Trong Nhan, Thomas Wong\n",
      "\n",
      "Host:    codespaces-fa7cf2 (AVX2, FMA3, 7 GB RAM)\n",
      "Command: iqtree -s result/species.fna -nt 2 -m MFP -bb 1000 -alrt 1000 -pre result/species_iqtreeout/species.tree\n",
      "Seed:    662907 (Using SPRNG - Scalable Parallel Random Number Generator)\n",
      "Time:    Sun Sep  1 10:05:55 2024\n",
      "Kernel:  AVX+FMA - 2 threads (2 CPU cores detected)\n",
      "\n",
      "Reading alignment file result/species.fna ... Fasta format detected\n",
      "Reading fasta file: done in 0.000935354 secs using 29.51% CPU\n",
      "Alignment most likely contains DNA/RNA sequences\n",
      "Alignment has 4 sequences with 11235 columns, 153 distinct patterns\n",
      "139 parsimony-informative, 2643 singleton sites, 8453 constant sites\n",
      "                        Gap/Ambiguity  Composition  p-value\n",
      "Analyzing sequences: done in 2.5527e-05 secs using 31.34% CPU\n",
      "   1  Armigeres_subalbatus      0.59%    failed      0.00%\n",
      "   2  Culex_pipiens             0.05%    failed      1.94%\n",
      "   3  Culex_pipiens_molestus    0.00%    failed      2.11%\n",
      "   4  Culex_pipiens_pallens     0.03%    failed      4.29%\n",
      "****  TOTAL                     0.17%  4 sequences failed composition chi2 test (p-value<5%; df=3)\n",
      "\n",
      "\n",
      "Create initial parsimony tree by phylogenetic likelihood library (PLL)... 0.000 seconds\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "Perform fast likelihood tree search using GTR+I+G model...\n",
      "Estimate model parameters (epsilon = 5.000)\n",
      "Perform nearest neighbor interchange...\n",
      "Estimate model parameters (epsilon = 1.000)\n",
      "1. Initial log-likelihood: -26742.277\n",
      "Optimal log-likelihood: -26742.260\n",
      "Rate parameters:  A-C: 1.34058  A-G: 4.13292  A-T: 0.87265  C-G: 1.77323  C-T: 6.57432  G-T: 1.00000\n",
      "Base frequencies:  A: 0.252  C: 0.256  G: 0.294  T: 0.198\n",
      "Proportion of invariable sites: 0.377\n",
      "Gamma shape alpha: 1.323\n",
      "Parameters optimization took 1 rounds (0.001 sec)\n",
      "Time for fast ML tree search: 0.008 seconds\n",
      "\n",
      "NOTE: ModelFinder requires 0 MB RAM!\n",
      "ModelFinder will test up to 484 DNA models (sample size: 11235 epsilon: 0.100) ...\n",
      " No. Model         -LnL         df  AIC          AICc         BIC\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  1  GTR+F         26818.794    13  53663.589    53663.621    53758.837\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  2  GTR+F+I       26742.237    14  53512.474    53512.512    53615.049\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  3  GTR+F+G4      26742.679    14  53513.359    53513.396    53615.934\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  4  GTR+F+I+G4    26742.264    15  53514.527    53514.570    53624.429\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  5  GTR+F+R2      26742.634    15  53515.268    53515.311    53625.170\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "  6  GTR+F+R3      26742.546    17  53519.093    53519.147    53643.648\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 14  GTR+F+I+R2    26742.496    16  53516.992    53517.041    53634.221\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 15  GTR+F+I+R3    26742.350    18  53520.700    53520.761    53652.583\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 24  SYM+I         26839.536    11  53701.072    53701.096    53781.667\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 25  SYM+G4        26839.187    11  53700.374    53700.397    53780.968\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 26  SYM+I+G4      26839.124    12  53702.248    53702.276    53790.170\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 46  TVM+F+I       26760.192    13  53546.384    53546.416    53641.632\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 47  TVM+F+G4      26761.231    13  53548.462    53548.494    53643.710\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 48  TVM+F+I+G4    26761.106    14  53550.212    53550.249    53652.787\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 68  TVMe+I        26843.669    10  53707.338    53707.358    53780.606\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 69  TVMe+G4       26843.315    10  53706.630    53706.650    53779.898\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 70  TVMe+I+G4     26843.214    11  53708.428    53708.452    53789.023\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 90  TIM3+F+I      26745.272    12  53514.544    53514.571    53602.465\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 91  TIM3+F+G4     26745.526    12  53515.052    53515.080    53602.973\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      " 92  TIM3+F+I+G4   26745.375    13  53516.751    53516.783    53611.999\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "112  TIM3e+I       26846.244    9   53710.488    53710.504    53776.429\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "113  TIM3e+G4      26845.871    9   53709.742    53709.758    53775.683\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "114  TIM3e+I+G4    26845.777    10  53711.555    53711.574    53784.823\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "134  TIM2+F+I      26752.308    12  53528.615    53528.643    53616.537\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "135  TIM2+F+G4     26753.926    12  53531.851    53531.879    53619.772\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "136  TIM2+F+I+G4   26753.764    13  53533.528    53533.561    53628.777\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "156  TIM2e+I       26864.269    9   53746.538    53746.554    53812.479\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "157  TIM2e+G4      26865.356    9   53748.712    53748.728    53814.653\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "158  TIM2e+I+G4    26865.210    10  53750.420    53750.440    53823.688\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "178  TIM+F+I       26754.032    12  53532.064    53532.091    53619.985\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "179  TIM+F+G4      26755.565    12  53535.130    53535.158    53623.051\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "180  TIM+F+I+G4    26755.568    13  53537.135    53537.168    53632.383\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "200  TIMe+I        26869.279    9   53756.558    53756.574    53822.499\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "201  TIMe+G4       26870.376    9   53758.753    53758.769    53824.694\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "202  TIMe+I+G4     26870.323    10  53760.646    53760.665    53833.914\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "222  TPM3u+F+I     26762.305    11  53546.609    53546.633    53627.204\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "223  TPM3u+F+G4    26763.614    11  53549.228    53549.252    53629.823\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "224  TPM3u+F+I+G4  26763.513    12  53551.027    53551.054    53638.948\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "244  TPM3+I        26850.038    8   53716.075    53716.088    53774.690\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "245  TPM3+G4       26849.741    8   53715.483    53715.496    53774.097\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "246  TPM3+I+G4     26849.679    9   53717.358    53717.374    53783.299\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "266  TPM2u+F+I     26771.538    11  53565.075    53565.099    53645.670\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "267  TPM2u+F+G4    26773.386    11  53568.772    53568.795    53649.366\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "268  TPM2u+F+I+G4  26773.469    12  53570.938    53570.966    53658.859\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "288  TPM2+I        26869.256    8   53754.512    53754.525    53813.127\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "289  TPM2+G4       26869.915    8   53755.831    53755.843    53814.445\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "290  TPM2+I+G4     26869.844    9   53757.688    53757.704    53823.629\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "310  K3Pu+F+I      26772.429    11  53566.857    53566.881    53647.452\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "311  K3Pu+F+G4     26774.551    11  53571.101    53571.125    53651.696\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "312  K3Pu+F+I+G4   26774.665    12  53573.329    53573.357    53661.251\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "332  K3P+I         26873.836    8   53763.671    53763.684    53822.286\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "333  K3P+G4        26874.636    8   53765.271    53765.284    53823.886\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "334  K3P+I+G4      26874.637    9   53767.274    53767.290    53833.216\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "354  TN+F+I        26754.881    11  53531.761    53531.785    53612.356\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "355  TN+F+G4       26756.361    11  53534.722    53534.746    53615.317\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "356  TN+F+I+G4     26756.314    12  53536.629    53536.657    53624.550\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "376  TNe+I         26870.069    8   53756.138    53756.150    53814.752\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "377  TNe+G4        26871.270    8   53758.540    53758.553    53817.155\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "378  TNe+I+G4      26871.214    9   53760.428    53760.444    53826.369\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "398  HKY+F+I       26773.244    10  53566.488    53566.507    53639.756\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "399  HKY+F+G4      26775.320    10  53570.640    53570.659    53643.908\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "400  HKY+F+I+G4    26775.432    11  53572.864    53572.888    53653.459\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "420  K2P+I         26874.633    7   53763.266    53763.276    53814.554\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "421  K2P+G4        26875.536    7   53765.072    53765.082    53816.359\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "422  K2P+I+G4      26875.544    8   53767.087    53767.100    53825.701\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "442  F81+F+I       27125.605    9   54269.209    54269.225    54335.151\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "443  F81+F+G4      27125.647    9   54269.294    54269.310    54335.235\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "444  F81+F+I+G4    27125.673    10  54271.345    54271.365    54344.613\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "464  JC+I          27227.636    6   54467.272    54467.279    54511.232\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "465  JC+G4         27227.683    6   54467.366    54467.373    54511.326\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "466  JC+I+G4       27227.705    7   54469.411    54469.421    54520.698\n",
      "Akaike Information Criterion:           GTR+F+I\n",
      "Corrected Akaike Information Criterion: GTR+F+I\n",
      "Bayesian Information Criterion:         TIM3+F+I\n",
      "Best-fit model: TIM3+F+I chosen according to BIC\n",
      "\n",
      "All model information printed to result/species_iqtreeout/species.tree.model.gz\n",
      "CPU time for ModelFinder: 0.399 seconds (0h:0m:0s)\n",
      "Wall-clock time for ModelFinder: 0.264 seconds (0h:0m:0s)\n",
      "Generating 1000 samples for ultrafast bootstrap (seed: 662907)...\n",
      "\n",
      "NOTE: 0 MB RAM (0 GB) is required!\n",
      "WARNING: Number of threads seems too high for short alignments. Use -T AUTO to determine best number of threads.\n",
      "Estimate model parameters (epsilon = 0.100)\n",
      "1. Initial log-likelihood: -27126.778\n",
      "2. Current log-likelihood: -26745.865\n",
      "3. Current log-likelihood: -26745.418\n",
      "Optimal log-likelihood: -26745.310\n",
      "Rate parameters:  A-C: 1.61858  A-G: 4.27554  A-T: 1.00000  C-G: 1.61858  C-T: 6.80303  G-T: 1.00000\n",
      "Base frequencies:  A: 0.252  C: 0.256  G: 0.294  T: 0.198\n",
      "Proportion of invariable sites: 0.537\n",
      "Parameters optimization took 3 rounds (0.004 sec)\n",
      "Wrote distance file to... \n",
      "Computing ML distances based on estimated model parameters...\n",
      "Calculating distance matrix: done in 3.9985e-05 secs using 182.6% CPU\n",
      "Computing ML distances took 0.000293 sec (of wall-clock time) 0.000283 sec (of CPU time)\n",
      "Setting up auxiliary I and S matrices: done in 0.000175047 secs using 105.1% CPU\n",
      "Computing RapidNJ tree took 0.000532 sec (of wall-clock time) 0.000560 sec (of CPU time)\n",
      "Log-likelihood of RapidNJ tree: -26745.310\n",
      "--------------------------------------------------------------------\n",
      "|             INITIALIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "Generating 99 parsimony trees... 0.022 second\n",
      "Computing log-likelihood of 2 initial trees ... 0.000 seconds\n",
      "Current best score: -26745.310\n",
      "\n",
      "Do NNI search on 3 best initial trees\n",
      "Finish initializing candidate tree set (3)\n",
      "Current best tree score: -26745.310 / CPU time: 0.030\n",
      "Number of iterations: 3\n",
      "--------------------------------------------------------------------\n",
      "|               OPTIMIZING CANDIDATE TREE SET                      |\n",
      "--------------------------------------------------------------------\n",
      "Iteration 10 / LogL: -26745.906 / Time: 0h:0m:0s\n",
      "Iteration 20 / LogL: -26745.889 / Time: 0h:0m:0s\n",
      "UPDATE BEST LOG-LIKELIHOOD: -26745.309\n",
      "Iteration 30 / LogL: -26745.527 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "UPDATE BEST LOG-LIKELIHOOD: -26745.309\n",
      "UPDATE BEST LOG-LIKELIHOOD: -26745.309\n",
      "Iteration 40 / LogL: -26745.889 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "UPDATE BEST LOG-LIKELIHOOD: -26745.309\n",
      "Iteration 50 / LogL: -26745.889 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -26752.520\n",
      "Iteration 60 / LogL: -26745.887 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 70 / LogL: -26745.527 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 80 / LogL: -26745.887 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 90 / LogL: -26745.887 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Iteration 100 / LogL: -26745.887 / Time: 0h:0m:0s (0h:0m:0s left)\n",
      "Log-likelihood cutoff on original alignment: -26752.520\n",
      "NOTE: Bootstrap correlation coefficient of split occurrence frequencies: 1.000\n",
      "TREE SEARCH COMPLETED AFTER 101 ITERATIONS / Time: 0h:0m:0s\n",
      "\n",
      "--------------------------------------------------------------------\n",
      "|                    FINALIZING TREE SEARCH                        |\n",
      "--------------------------------------------------------------------\n",
      "Performs final model parameters optimization\n",
      "Estimate model parameters (epsilon = 0.010)\n",
      "1. Initial log-likelihood: -26745.309\n",
      "2. Current log-likelihood: -26745.295\n",
      "3. Current log-likelihood: -26745.281\n",
      "4. Current log-likelihood: -26745.269\n",
      "Optimal log-likelihood: -26745.261\n",
      "Rate parameters:  A-C: 1.60311  A-G: 4.26534  A-T: 1.00000  C-G: 1.60311  C-T: 6.85780  G-T: 1.00000\n",
      "Base frequencies:  A: 0.252  C: 0.256  G: 0.294  T: 0.198\n",
      "Proportion of invariable sites: 0.541\n",
      "Parameters optimization took 4 rounds (0.002 sec)\n",
      "BEST SCORE FOUND : -26745.261\n",
      "\n",
      "Testing tree branches by SH-like aLRT with 1000 replicates...\n",
      "0.008 sec.\n",
      "Creating bootstrap support values...\n",
      "Split supports printed to NEXUS file result/species_iqtreeout/species.tree.splits.nex\n",
      "Total tree length: 0.457\n",
      "\n",
      "Total number of iterations: 101\n",
      "CPU time used for tree search: 0.799 sec (0h:0m:0s)\n",
      "Wall-clock time used for tree search: 0.502 sec (0h:0m:0s)\n",
      "Total CPU time used: 1.288 sec (0h:0m:1s)\n",
      "Total wall-clock time used: 0.823 sec (0h:0m:0s)\n",
      "\n",
      "Computing bootstrap consensus tree...\n",
      "Reading input file result/species_iqtreeout/species.tree.splits.nex...\n",
      "4 taxa and 7 splits.\n",
      "Consensus tree written to result/species_iqtreeout/species.tree.contree\n",
      "Reading input trees file result/species_iqtreeout/species.tree.contree\n",
      "Log-likelihood of consensus tree: -26745.261\n",
      "\n",
      "Analysis results written to: \n",
      "  IQ-TREE report:                result/species_iqtreeout/species.tree.iqtree\n",
      "  Maximum-likelihood tree:       result/species_iqtreeout/species.tree.treefile\n",
      "  Likelihood distances:          result/species_iqtreeout/species.tree.mldist\n",
      "\n",
      "Ultrafast bootstrap approximation results written to:\n",
      "  Split support values:          result/species_iqtreeout/species.tree.splits.nex\n",
      "  Consensus tree:                result/species_iqtreeout/species.tree.contree\n",
      "  Screen log file:               result/species_iqtreeout/species.tree.log\n",
      "\n",
      "Date and Time: Sun Sep  1 10:05:56 2024\n"
     ]
    }
   ],
   "source": [
    "#  由于本身序列就是比对修剪过后的，可以直接用来进行建树\n",
    "! cat result/species_fna/* > result/species.fna\n",
    "! mkdir -p result/species_iqtreeout\n",
    "! iqtree -s result/species.fna -nt 2 -m MFP -bb 1000 -alrt 1000 -pre result/species_iqtreeout/species.tree"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "base",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.12.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
